{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "e6c09049",
   "metadata": {},
   "source": [
    "<a target=\"_blank\" href=\"https://colab.research.google.com/github/visze/foundations_in_medicine/blob/master/biomedical_databases/tutorial.ipynb\">\n",
    "  <img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/>\n",
    "</a>"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6b79e1c3",
   "metadata": {},
   "source": [
    "# üß¨ Tutorial Biomedical Databases"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "38b2d16c",
   "metadata": {},
   "source": [
    "## üß¨ Background\n",
    "\n",
    "Several genes are linked to hereditary cancer, including **BRCA1**, **BRCA2**, and **TP53**. These genes play a crucial role in the body's ability to repair DNA damage and regulate cell growth. üî¨ Mutations in these genes can lead to an increased risk of developing certain types of cancer, particularly breast and ovarian cancer."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b0f179c5",
   "metadata": {},
   "source": [
    "### ‚ùìQuestion\n",
    "\n",
    "\n",
    "> **üí° Look up the gene TERT in various databases. What is its function, and what diseases are associated with mutations in this gene?**\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4e1a7293",
   "metadata": {},
   "source": [
    "Besides coding mutations, regulatory mutations can also impact gene function. Gene coding regions constitute only around 2% of the human genome, while regulatory regions make up a much larger portion (up to 15-20% depending on the source). Their main purpose is to regulate the expression of genes. üß´ Every cell has the same sequence of DNA, but their purpose, shape, etc., differ. We can define two different aspects of regulation:\n",
    "\n",
    "1. **üó∫Ô∏è Spatial regulation:**\n",
    "   - Organs\n",
    "   - Cell types\n",
    "   - Single cells (e.g., different immune cells)\n",
    "2. **‚è±Ô∏è Temporal regulation:**\n",
    "   - Cell cycle phase\n",
    "   - Differentiation phase\n",
    "   - Developmental stage\n",
    "   - Induced transcription (environmental stimuli)\n",
    "\n",
    "The regulation of transcription is controlled by the binding of proteins or RNA to regulatory nucleic acid sequences. These sequences are often classified into **promoters** and **enhancers**.\n",
    "\n",
    "**üö¶ Promoters** are typically located directly upstream of the gene they regulate and serve as the binding site for RNA polymerase and other transcription factors necessary for initiating transcription. **‚ö° Enhancers**, on the other hand, can be located far away from the gene they regulate (upstream, downstream, or even within introns) and function to increase the transcriptional activity of a gene by providing additional binding sites for transcription factors."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bd38c67f",
   "metadata": {},
   "source": [
    "![Regulation](resources/picture/regulation.png)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "95791425",
   "metadata": {},
   "source": [
    "Mutations in regulatory regions can affect gene expression levels, leading to diseases such as cancer. ü¶† TERT promoter mutations have been identified in various cancers, including melanoma, glioblastoma, and bladder cancer. These mutations can lead to increased TERT expression, which may contribute to the uncontrolled growth of cancer cells. üìà"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3197b0f2",
   "metadata": {},
   "source": [
    "## üîç Variants in the TERT promoter region"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f77daaee",
   "metadata": {},
   "source": [
    "Now we will look up different databases to find information about variants in the TERT promoter region. üìö We'll try to solve this question \"programmatically\"‚Äîthis means using code to query the databases instead of using their web interfaces (when it makes sense). üíª"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f1e8dc48",
   "metadata": {},
   "source": [
    "### ‚ùìQuestion\n",
    "\n",
    "> **üí° Find out the genomic coordinates (GRCh38) of the TERT promoter region.**\n",
    "\n",
    "üìå Helpful resources:\n",
    "- [UCSC Genome Browser](https://genome.ucsc.edu/) üåê\n",
    "- [ENCODE SCREEN database](https://screen.wenglab.org). SCREEN is a database derived from ENCODE data. They annotate candidate cis-regulatory elements (cCREs) in the human and mouse genomes based on a large number of functional genomics experiments. üß™"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3177b4cd",
   "metadata": {},
   "source": [
    "### üåç Population variants from gnomAD"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c83c211f",
   "metadata": {},
   "source": [
    "GnomAD uses [GraphQL](https://graphql.org/) to provide programmatic access to their database. üîå GraphQL is an alternative to REST APIs. Instead of multiple endpoints, there is only one endpoint. The user defines the structure of the response data in the request. ‚ú®"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2aa1afd4",
   "metadata": {},
   "source": [
    "First we need to install the GQL GraphQL client library for Python. üì¶"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "c9fa688d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Requirement already satisfied: gql in /home/mschubach/miniforge3/envs/jupyter/lib/python3.12/site-packages (4.0.0)\n",
      "Requirement already satisfied: graphql-core<3.3,>=3.2 in /home/mschubach/miniforge3/envs/jupyter/lib/python3.12/site-packages (from gql) (3.2.7)\n",
      "Requirement already satisfied: yarl<2.0,>=1.6 in /home/mschubach/miniforge3/envs/jupyter/lib/python3.12/site-packages (from gql) (1.22.0)\n",
      "Requirement already satisfied: backoff<3.0,>=1.11.1 in /home/mschubach/miniforge3/envs/jupyter/lib/python3.12/site-packages (from gql) (2.2.1)\n",
      "Requirement already satisfied: anyio<5,>=3.0 in /home/mschubach/miniforge3/envs/jupyter/lib/python3.12/site-packages (from gql) (4.4.0)\n",
      "Requirement already satisfied: idna>=2.8 in /home/mschubach/miniforge3/envs/jupyter/lib/python3.12/site-packages (from anyio<5,>=3.0->gql) (3.7)\n",
      "Requirement already satisfied: sniffio>=1.1 in /home/mschubach/miniforge3/envs/jupyter/lib/python3.12/site-packages (from anyio<5,>=3.0->gql) (1.3.1)\n",
      "Requirement already satisfied: multidict>=4.0 in /home/mschubach/miniforge3/envs/jupyter/lib/python3.12/site-packages (from yarl<2.0,>=1.6->gql) (6.0.5)\n",
      "Requirement already satisfied: propcache>=0.2.1 in /home/mschubach/miniforge3/envs/jupyter/lib/python3.12/site-packages (from yarl<2.0,>=1.6->gql) (0.3.1)\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "pip install gql"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2b0b7b1d",
   "metadata": {},
   "source": [
    "Starting with GraphQL is not that easy. ü§î GnomAD has a (very short) documentation here: https://gnomad.broadinstitute.org/data#api.\n",
    "\n",
    "However, it only shows you the `gene` query. But there is more! üéØ You have `region`, `transcript`, `variant`, and many more searches. You can find them out by clicking on this link: https://gnomad.broadinstitute.org/api?"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "03a002ee",
   "metadata": {},
   "source": [
    "### ‚ùì TODO\n",
    "\n",
    "> **üí° Insert your region in the query by replacing `<chr_number>`, `<start_position>`, `<end_position>`**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "20e3ca97",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'region': {'variants': [{'variant_id': '5-1294944-T-C', 'chrom': '5', 'pos': 1294944, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 0, 'an': 1403334}}, {'variant_id': '5-1294945-G-A', 'chrom': '5', 'pos': 1294945, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 22, 'an': 1401036}}, {'variant_id': '5-1294945-G-T', 'chrom': '5', 'pos': 1294945, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1401144}}, {'variant_id': '5-1294946-C-CGCA', 'chrom': '5', 'pos': 1294946, 'ref': 'C', 'alt': 'CGCA', 'joint': {'ac': 1, 'an': 1400604}}, {'variant_id': '5-1294946-C-T', 'chrom': '5', 'pos': 1294946, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1400602}}, {'variant_id': '5-1294946-CGCAGCAGGGAGCGCACGGCTCG-C', 'chrom': '5', 'pos': 1294946, 'ref': 'CGCAGCAGGGAGCGCACGGCTCG', 'alt': 'C', 'joint': {'ac': 1, 'an': 1400604}}, {'variant_id': '5-1294946-C-A', 'chrom': '5', 'pos': 1294946, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 1, 'an': 1400602}}, {'variant_id': '5-1294947-G-A', 'chrom': '5', 'pos': 1294947, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1393228}}, {'variant_id': '5-1294947-G-C', 'chrom': '5', 'pos': 1294947, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1393230}}, {'variant_id': '5-1294947-G-T', 'chrom': '5', 'pos': 1294947, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1393230}}, {'variant_id': '5-1294948-C-A', 'chrom': '5', 'pos': 1294948, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 1, 'an': 1393494}}, {'variant_id': '5-1294948-C-T', 'chrom': '5', 'pos': 1294948, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1393494}}, {'variant_id': '5-1294949-A-G', 'chrom': '5', 'pos': 1294949, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1393516}}, {'variant_id': '5-1294950-G-A', 'chrom': '5', 'pos': 1294950, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1390046}}, {'variant_id': '5-1294950-G-T', 'chrom': '5', 'pos': 1294950, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1390046}}, {'variant_id': '5-1294951-C-A', 'chrom': '5', 'pos': 1294951, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 2, 'an': 1389898}}, {'variant_id': '5-1294951-CAGGG-C', 'chrom': '5', 'pos': 1294951, 'ref': 'CAGGG', 'alt': 'C', 'joint': {'ac': 1, 'an': 1389900}}, {'variant_id': '5-1294951-C-T', 'chrom': '5', 'pos': 1294951, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 2, 'an': 1389900}}, {'variant_id': '5-1294952-A-G', 'chrom': '5', 'pos': 1294952, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1390276}}, {'variant_id': '5-1294952-A-AGGGAGCGCACGGCTCGGCAGCG', 'chrom': '5', 'pos': 1294952, 'ref': 'A', 'alt': 'AGGGAGCGCACGGCTCGGCAGCG', 'joint': {'ac': 1, 'an': 1390276}}, {'variant_id': '5-1294952-A-C', 'chrom': '5', 'pos': 1294952, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 0, 'an': 1390276}}, {'variant_id': '5-1294952-AGGGAGCGCACGGCTCGGCAGCG-A', 'chrom': '5', 'pos': 1294952, 'ref': 'AGGGAGCGCACGGCTCGGCAGCG', 'alt': 'A', 'joint': {'ac': 0, 'an': 1390266}}, {'variant_id': '5-1294953-G-A', 'chrom': '5', 'pos': 1294953, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 1381780}}, {'variant_id': '5-1294953-G-T', 'chrom': '5', 'pos': 1294953, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1381780}}, {'variant_id': '5-1294954-G-A', 'chrom': '5', 'pos': 1294954, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1381336}}, {'variant_id': '5-1294954-G-C', 'chrom': '5', 'pos': 1294954, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1381334}}, {'variant_id': '5-1294954-G-T', 'chrom': '5', 'pos': 1294954, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 2, 'an': 1381336}}, {'variant_id': '5-1294955-G-A', 'chrom': '5', 'pos': 1294955, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1381738}}, {'variant_id': '5-1294955-G-T', 'chrom': '5', 'pos': 1294955, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1381738}}, {'variant_id': '5-1294956-A-G', 'chrom': '5', 'pos': 1294956, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1376688}}, {'variant_id': '5-1294957-G-GCTCCC', 'chrom': '5', 'pos': 1294957, 'ref': 'G', 'alt': 'GCTCCC', 'joint': {'ac': 1, 'an': 1376536}}, {'variant_id': '5-1294957-G-T', 'chrom': '5', 'pos': 1294957, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1376536}}, {'variant_id': '5-1294957-G-A', 'chrom': '5', 'pos': 1294957, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 1376536}}, {'variant_id': '5-1294958-C-A', 'chrom': '5', 'pos': 1294958, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1371810}}, {'variant_id': '5-1294958-C-T', 'chrom': '5', 'pos': 1294958, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 2, 'an': 1371812}}, {'variant_id': '5-1294958-C-G', 'chrom': '5', 'pos': 1294958, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 1371812}}, {'variant_id': '5-1294959-G-C', 'chrom': '5', 'pos': 1294959, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1367474}}, {'variant_id': '5-1294959-G-A', 'chrom': '5', 'pos': 1294959, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1367472}}, {'variant_id': '5-1294959-G-T', 'chrom': '5', 'pos': 1294959, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1367474}}, {'variant_id': '5-1294960-C-A', 'chrom': '5', 'pos': 1294960, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 1, 'an': 1367802}}, {'variant_id': '5-1294960-C-T', 'chrom': '5', 'pos': 1294960, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 1367802}}, {'variant_id': '5-1294961-A-G', 'chrom': '5', 'pos': 1294961, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1367812}}, {'variant_id': '5-1294962-C-A', 'chrom': '5', 'pos': 1294962, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1366950}}, {'variant_id': '5-1294962-C-T', 'chrom': '5', 'pos': 1294962, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1366948}}, {'variant_id': '5-1294962-C-G', 'chrom': '5', 'pos': 1294962, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 1, 'an': 1366950}}, {'variant_id': '5-1294963-G-T', 'chrom': '5', 'pos': 1294963, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 17, 'an': 1350022}}, {'variant_id': '5-1294963-G-C', 'chrom': '5', 'pos': 1294963, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 11, 'an': 1349914}}, {'variant_id': '5-1294963-G-A', 'chrom': '5', 'pos': 1294963, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 1349914}}, {'variant_id': '5-1294964-G-T', 'chrom': '5', 'pos': 1294964, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1349032}}, {'variant_id': '5-1294964-G-A', 'chrom': '5', 'pos': 1294964, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1349032}}, {'variant_id': '5-1294965-C-A', 'chrom': '5', 'pos': 1294965, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1348900}}, {'variant_id': '5-1294965-C-G', 'chrom': '5', 'pos': 1294965, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 1348900}}, {'variant_id': '5-1294965-C-T', 'chrom': '5', 'pos': 1294965, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1348900}}, {'variant_id': '5-1294966-T-C', 'chrom': '5', 'pos': 1294966, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 0, 'an': 1345134}}, {'variant_id': '5-1294967-C-G', 'chrom': '5', 'pos': 1294967, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 1, 'an': 1344580}}, {'variant_id': '5-1294967-C-A', 'chrom': '5', 'pos': 1294967, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1344578}}, {'variant_id': '5-1294967-C-T', 'chrom': '5', 'pos': 1294967, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 2, 'an': 1344580}}, {'variant_id': '5-1294968-G-C', 'chrom': '5', 'pos': 1294968, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 1338470}}, {'variant_id': '5-1294968-G-T', 'chrom': '5', 'pos': 1294968, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1338468}}, {'variant_id': '5-1294968-G-A', 'chrom': '5', 'pos': 1294968, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1338470}}, {'variant_id': '5-1294969-G-T', 'chrom': '5', 'pos': 1294969, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1334352}}, {'variant_id': '5-1294969-G-A', 'chrom': '5', 'pos': 1294969, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 1334244}}, {'variant_id': '5-1294970-C-T', 'chrom': '5', 'pos': 1294970, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1334346}}, {'variant_id': '5-1294970-C-A', 'chrom': '5', 'pos': 1294970, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1334344}}, {'variant_id': '5-1294971-A-G', 'chrom': '5', 'pos': 1294971, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1333628}}, {'variant_id': '5-1294972-G-T', 'chrom': '5', 'pos': 1294972, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1332168}}, {'variant_id': '5-1294972-G-A', 'chrom': '5', 'pos': 1294972, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 3, 'an': 1332168}}, {'variant_id': '5-1294972-GC-G', 'chrom': '5', 'pos': 1294972, 'ref': 'GC', 'alt': 'G', 'joint': {'ac': 0, 'an': 1332166}}, {'variant_id': '5-1294973-C-T', 'chrom': '5', 'pos': 1294973, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 1330560}}, {'variant_id': '5-1294973-C-G', 'chrom': '5', 'pos': 1294973, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 1330560}}, {'variant_id': '5-1294973-C-A', 'chrom': '5', 'pos': 1294973, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 1, 'an': 1330560}}, {'variant_id': '5-1294973-CG-C', 'chrom': '5', 'pos': 1294973, 'ref': 'CG', 'alt': 'C', 'joint': {'ac': 1, 'an': 1330550}}, {'variant_id': '5-1294973-C-CG', 'chrom': '5', 'pos': 1294973, 'ref': 'C', 'alt': 'CG', 'joint': {'ac': 0, 'an': 1330560}}, {'variant_id': '5-1294974-G-A', 'chrom': '5', 'pos': 1294974, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1331550}}, {'variant_id': '5-1294974-G-C', 'chrom': '5', 'pos': 1294974, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1331550}}, {'variant_id': '5-1294974-G-T', 'chrom': '5', 'pos': 1294974, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1331550}}, {'variant_id': '5-1294975-G-T', 'chrom': '5', 'pos': 1294975, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1327410}}, {'variant_id': '5-1294975-G-C', 'chrom': '5', 'pos': 1294975, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1327410}}, {'variant_id': '5-1294975-G-A', 'chrom': '5', 'pos': 1294975, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1327410}}, {'variant_id': '5-1294976-G-T', 'chrom': '5', 'pos': 1294976, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1327602}}, {'variant_id': '5-1294976-G-A', 'chrom': '5', 'pos': 1294976, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1327602}}, {'variant_id': '5-1294977-G-T', 'chrom': '5', 'pos': 1294977, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1327644}}, {'variant_id': '5-1294977-G-A', 'chrom': '5', 'pos': 1294977, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1327644}}, {'variant_id': '5-1294978-A-G', 'chrom': '5', 'pos': 1294978, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 37, 'an': 1324526}}, {'variant_id': '5-1294979-G-A', 'chrom': '5', 'pos': 1294979, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 1325896}}, {'variant_id': '5-1294979-G-T', 'chrom': '5', 'pos': 1294979, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1325896}}, {'variant_id': '5-1294980-C-T', 'chrom': '5', 'pos': 1294980, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 1321432}}, {'variant_id': '5-1294980-C-G', 'chrom': '5', 'pos': 1294980, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 2, 'an': 1321432}}, {'variant_id': '5-1294980-C-A', 'chrom': '5', 'pos': 1294980, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1321430}}, {'variant_id': '5-1294981-G-A', 'chrom': '5', 'pos': 1294981, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1322484}}, {'variant_id': '5-1294981-G-T', 'chrom': '5', 'pos': 1294981, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1322484}}, {'variant_id': '5-1294982-C-T', 'chrom': '5', 'pos': 1294982, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1320348}}, {'variant_id': '5-1294982-C-G', 'chrom': '5', 'pos': 1294982, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 1320350}}, {'variant_id': '5-1294982-C-A', 'chrom': '5', 'pos': 1294982, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1320350}}, {'variant_id': '5-1294983-G-T', 'chrom': '5', 'pos': 1294983, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1321418}}, {'variant_id': '5-1294983-G-C', 'chrom': '5', 'pos': 1294983, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 1321418}}, {'variant_id': '5-1294983-GC-G', 'chrom': '5', 'pos': 1294983, 'ref': 'GC', 'alt': 'G', 'joint': {'ac': 0, 'an': 1321418}}, {'variant_id': '5-1294983-G-A', 'chrom': '5', 'pos': 1294983, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1321418}}, {'variant_id': '5-1294984-C-A', 'chrom': '5', 'pos': 1294984, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1320278}}, {'variant_id': '5-1294984-C-T', 'chrom': '5', 'pos': 1294984, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 1320278}}, {'variant_id': '5-1294985-G-C', 'chrom': '5', 'pos': 1294985, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 2, 'an': 1320526}}, {'variant_id': '5-1294985-G-T', 'chrom': '5', 'pos': 1294985, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 1320526}}, {'variant_id': '5-1294985-G-A', 'chrom': '5', 'pos': 1294985, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 1320526}}, {'variant_id': '5-1294986-G-T', 'chrom': '5', 'pos': 1294986, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1320402}}, {'variant_id': '5-1294986-G-A', 'chrom': '5', 'pos': 1294986, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1320402}}, {'variant_id': '5-1294987-C-T', 'chrom': '5', 'pos': 1294987, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1320238}}, {'variant_id': '5-1294987-C-A', 'chrom': '5', 'pos': 1294987, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1320236}}, {'variant_id': '5-1294988-A-G', 'chrom': '5', 'pos': 1294988, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1318792}}, {'variant_id': '5-1294989-T-C', 'chrom': '5', 'pos': 1294989, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 0, 'an': 1317682}}, {'variant_id': '5-1294990-C-T', 'chrom': '5', 'pos': 1294990, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 1317348}}, {'variant_id': '5-1294990-C-G', 'chrom': '5', 'pos': 1294990, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 1, 'an': 1317240}}, {'variant_id': '5-1294990-C-A', 'chrom': '5', 'pos': 1294990, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1317348}}, {'variant_id': '5-1294991-G-A', 'chrom': '5', 'pos': 1294991, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 3, 'an': 1317992}}, {'variant_id': '5-1294991-G-T', 'chrom': '5', 'pos': 1294991, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1317992}}, {'variant_id': '5-1294992-C-T', 'chrom': '5', 'pos': 1294992, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 2, 'an': 1312640}}, {'variant_id': '5-1294992-CG-C', 'chrom': '5', 'pos': 1294992, 'ref': 'CG', 'alt': 'C', 'joint': {'ac': 1, 'an': 1312626}}, {'variant_id': '5-1294992-C-G', 'chrom': '5', 'pos': 1294992, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 1312640}}, {'variant_id': '5-1294992-C-CG', 'chrom': '5', 'pos': 1294992, 'ref': 'C', 'alt': 'CG', 'joint': {'ac': 0, 'an': 1312634}}, {'variant_id': '5-1294992-C-A', 'chrom': '5', 'pos': 1294992, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 1, 'an': 1312640}}, {'variant_id': '5-1294993-G-A', 'chrom': '5', 'pos': 1294993, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1314968}}, {'variant_id': '5-1294993-G-C', 'chrom': '5', 'pos': 1294993, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 8, 'an': 1314860}}, {'variant_id': '5-1294993-G-T', 'chrom': '5', 'pos': 1294993, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 5, 'an': 1314860}}, {'variant_id': '5-1294994-G-C', 'chrom': '5', 'pos': 1294994, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 2, 'an': 1315502}}, {'variant_id': '5-1294994-G-A', 'chrom': '5', 'pos': 1294994, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1315502}}, {'variant_id': '5-1294994-G-T', 'chrom': '5', 'pos': 1294994, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 1315610}}, {'variant_id': '5-1294995-G-T', 'chrom': '5', 'pos': 1294995, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 13, 'an': 1315568}}, {'variant_id': '5-1294995-G-A', 'chrom': '5', 'pos': 1294995, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1315676}}, {'variant_id': '5-1294996-G-T', 'chrom': '5', 'pos': 1294996, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1315474}}, {'variant_id': '5-1294996-G-A', 'chrom': '5', 'pos': 1294996, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1315474}}, {'variant_id': '5-1294996-G-C', 'chrom': '5', 'pos': 1294996, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 3, 'an': 1315366}}, {'variant_id': '5-1294997-G-A', 'chrom': '5', 'pos': 1294997, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 5, 'an': 1315984}}, {'variant_id': '5-1294997-G-T', 'chrom': '5', 'pos': 1294997, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 6, 'an': 1315984}}, {'variant_id': '5-1294998-TG-T', 'chrom': '5', 'pos': 1294998, 'ref': 'TG', 'alt': 'T', 'joint': {'ac': 1, 'an': 1308190}}, {'variant_id': '5-1294998-T-G', 'chrom': '5', 'pos': 1294998, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 1, 'an': 1308190}}, {'variant_id': '5-1294998-TGGCCGG-T', 'chrom': '5', 'pos': 1294998, 'ref': 'TGGCCGG', 'alt': 'T', 'joint': {'ac': 0, 'an': 1308178}}, {'variant_id': '5-1294998-T-C', 'chrom': '5', 'pos': 1294998, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 38, 'an': 1308190}}, {'variant_id': '5-1294999-G-A', 'chrom': '5', 'pos': 1294999, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1314458}}, {'variant_id': '5-1294999-G-T', 'chrom': '5', 'pos': 1294999, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1314458}}, {'variant_id': '5-1295000-G-A', 'chrom': '5', 'pos': 1295000, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1312596}}, {'variant_id': '5-1295000-G-T', 'chrom': '5', 'pos': 1295000, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 4, 'an': 1312486}}, {'variant_id': '5-1295001-C-A', 'chrom': '5', 'pos': 1295001, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1312594}}, {'variant_id': '5-1295001-C-G', 'chrom': '5', 'pos': 1295001, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 1312594}}, {'variant_id': '5-1295001-C-T', 'chrom': '5', 'pos': 1295001, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 3, 'an': 1312592}}, {'variant_id': '5-1295002-C-T', 'chrom': '5', 'pos': 1295002, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 1308990}}, {'variant_id': '5-1295002-CG-C', 'chrom': '5', 'pos': 1295002, 'ref': 'CG', 'alt': 'C', 'joint': {'ac': 2, 'an': 1308972}}, {'variant_id': '5-1295002-C-A', 'chrom': '5', 'pos': 1295002, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1308990}}, {'variant_id': '5-1295002-C-G', 'chrom': '5', 'pos': 1295002, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 8, 'an': 1308990}}, {'variant_id': '5-1295003-GGGGCCA-G', 'chrom': '5', 'pos': 1295003, 'ref': 'GGGGCCA', 'alt': 'G', 'joint': {'ac': 0, 'an': 1309062}}, {'variant_id': '5-1295003-G-T', 'chrom': '5', 'pos': 1295003, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 4, 'an': 1309066}}, {'variant_id': '5-1295003-G-C', 'chrom': '5', 'pos': 1295003, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1309066}}, {'variant_id': '5-1295003-G-A', 'chrom': '5', 'pos': 1295003, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1309066}}, {'variant_id': '5-1295004-G-T', 'chrom': '5', 'pos': 1295004, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1307504}}, {'variant_id': '5-1295004-G-A', 'chrom': '5', 'pos': 1295004, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 4, 'an': 1307504}}, {'variant_id': '5-1295005-G-T', 'chrom': '5', 'pos': 1295005, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1307256}}, {'variant_id': '5-1295005-G-A', 'chrom': '5', 'pos': 1295005, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1307256}}, {'variant_id': '5-1295006-GC-G', 'chrom': '5', 'pos': 1295006, 'ref': 'GC', 'alt': 'G', 'joint': {'ac': 0, 'an': 1305778}}, {'variant_id': '5-1295006-G-T', 'chrom': '5', 'pos': 1295006, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 1305674}}, {'variant_id': '5-1295006-G-C', 'chrom': '5', 'pos': 1295006, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 1305782}}, {'variant_id': '5-1295006-G-A', 'chrom': '5', 'pos': 1295006, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1305674}}, {'variant_id': '5-1295007-C-A', 'chrom': '5', 'pos': 1295007, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1298048}}, {'variant_id': '5-1295007-C-T', 'chrom': '5', 'pos': 1295007, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 1298048}}, {'variant_id': '5-1295008-C-A', 'chrom': '5', 'pos': 1295008, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1298582}}, {'variant_id': '5-1295008-C-T', 'chrom': '5', 'pos': 1295008, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1298584}}, {'variant_id': '5-1295009-AG-A', 'chrom': '5', 'pos': 1295009, 'ref': 'AG', 'alt': 'A', 'joint': {'ac': 1, 'an': 1296918}}, {'variant_id': '5-1295009-A-G', 'chrom': '5', 'pos': 1295009, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1296920}}, {'variant_id': '5-1295010-G-T', 'chrom': '5', 'pos': 1295010, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1299374}}, {'variant_id': '5-1295010-G-A', 'chrom': '5', 'pos': 1295010, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 3, 'an': 1299376}}, {'variant_id': '5-1295011-G-T', 'chrom': '5', 'pos': 1295011, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1295098}}, {'variant_id': '5-1295011-G-A', 'chrom': '5', 'pos': 1295011, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1295100}}, {'variant_id': '5-1295012-G-C', 'chrom': '5', 'pos': 1295012, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1293538}}, {'variant_id': '5-1295012-G-T', 'chrom': '5', 'pos': 1295012, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1293538}}, {'variant_id': '5-1295012-G-A', 'chrom': '5', 'pos': 1295012, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1293428}}, {'variant_id': '5-1295013-C-T', 'chrom': '5', 'pos': 1295013, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1292690}}, {'variant_id': '5-1295013-CT-C', 'chrom': '5', 'pos': 1295013, 'ref': 'CT', 'alt': 'C', 'joint': {'ac': 0, 'an': 1292688}}, {'variant_id': '5-1295013-C-A', 'chrom': '5', 'pos': 1295013, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 1, 'an': 1292690}}, {'variant_id': '5-1295014-T-C', 'chrom': '5', 'pos': 1295014, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 0, 'an': 1289108}}, {'variant_id': '5-1295015-T-A', 'chrom': '5', 'pos': 1295015, 'ref': 'T', 'alt': 'A', 'joint': {'ac': 0, 'an': 1286882}}, {'variant_id': '5-1295015-T-G', 'chrom': '5', 'pos': 1295015, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 58, 'an': 1286778}}, {'variant_id': '5-1295015-T-C', 'chrom': '5', 'pos': 1295015, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 0, 'an': 1286880}}, {'variant_id': '5-1295015-TC-T', 'chrom': '5', 'pos': 1295015, 'ref': 'TC', 'alt': 'T', 'joint': {'ac': 1, 'an': 1286874}}, {'variant_id': '5-1295016-C-T', 'chrom': '5', 'pos': 1295016, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1285978}}, {'variant_id': '5-1295016-C-A', 'chrom': '5', 'pos': 1295016, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1285978}}, {'variant_id': '5-1295017-C-A', 'chrom': '5', 'pos': 1295017, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1286486}}, {'variant_id': '5-1295017-C-G', 'chrom': '5', 'pos': 1295017, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 1286486}}, {'variant_id': '5-1295017-C-T', 'chrom': '5', 'pos': 1295017, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1286486}}, {'variant_id': '5-1295018-C-A', 'chrom': '5', 'pos': 1295018, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 15, 'an': 1285216}}, {'variant_id': '5-1295018-C-T', 'chrom': '5', 'pos': 1295018, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1285214}}, {'variant_id': '5-1295019-A-T', 'chrom': '5', 'pos': 1295019, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 1, 'an': 1279382}}, {'variant_id': '5-1295019-A-G', 'chrom': '5', 'pos': 1295019, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 1, 'an': 1279382}}, {'variant_id': '5-1295020-C-A', 'chrom': '5', 'pos': 1295020, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1278918}}, {'variant_id': '5-1295020-C-T', 'chrom': '5', 'pos': 1295020, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 2, 'an': 1278918}}, {'variant_id': '5-1295020-C-G', 'chrom': '5', 'pos': 1295020, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 13, 'an': 1278802}}, {'variant_id': '5-1295021-G-A', 'chrom': '5', 'pos': 1295021, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1272576}}, {'variant_id': '5-1295021-G-T', 'chrom': '5', 'pos': 1295021, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 1272574}}, {'variant_id': '5-1295021-G-C', 'chrom': '5', 'pos': 1295021, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 65, 'an': 1272468}}, {'variant_id': '5-1295022-T-C', 'chrom': '5', 'pos': 1295022, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 1, 'an': 1264782}}, {'variant_id': '5-1295023-G-T', 'chrom': '5', 'pos': 1295023, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1271464}}, {'variant_id': '5-1295023-G-A', 'chrom': '5', 'pos': 1295023, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1271466}}, {'variant_id': '5-1295024-C-A', 'chrom': '5', 'pos': 1295024, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 1, 'an': 1239158}}, {'variant_id': '5-1295024-C-T', 'chrom': '5', 'pos': 1295024, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1239158}}, {'variant_id': '5-1295025-G-A', 'chrom': '5', 'pos': 1295025, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 1239186}}, {'variant_id': '5-1295025-G-C', 'chrom': '5', 'pos': 1295025, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1239292}}, {'variant_id': '5-1295025-G-T', 'chrom': '5', 'pos': 1295025, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1239290}}, {'variant_id': '5-1295026-C-T', 'chrom': '5', 'pos': 1295026, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 12, 'an': 1243182}}, {'variant_id': '5-1295026-C-A', 'chrom': '5', 'pos': 1295026, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1243182}}, {'variant_id': '5-1295026-C-G', 'chrom': '5', 'pos': 1295026, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 6, 'an': 1243182}}, {'variant_id': '5-1295027-A-T', 'chrom': '5', 'pos': 1295027, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 1, 'an': 1238644}}, {'variant_id': '5-1295027-A-C', 'chrom': '5', 'pos': 1295027, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 8, 'an': 1238542}}, {'variant_id': '5-1295027-A-G', 'chrom': '5', 'pos': 1295027, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1238650}}, {'variant_id': '5-1295028-G-C', 'chrom': '5', 'pos': 1295028, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1243334}}, {'variant_id': '5-1295028-G-A', 'chrom': '5', 'pos': 1295028, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1243332}}, {'variant_id': '5-1295028-G-T', 'chrom': '5', 'pos': 1295028, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1243334}}, {'variant_id': '5-1295029-C-A', 'chrom': '5', 'pos': 1295029, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1245858}}, {'variant_id': '5-1295029-C-T', 'chrom': '5', 'pos': 1295029, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1245858}}, {'variant_id': '5-1295029-C-G', 'chrom': '5', 'pos': 1295029, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 3, 'an': 1245858}}, {'variant_id': '5-1295029-CAG-C', 'chrom': '5', 'pos': 1295029, 'ref': 'CAG', 'alt': 'C', 'joint': {'ac': 0, 'an': 1245852}}, {'variant_id': '5-1295030-A-C', 'chrom': '5', 'pos': 1295030, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 1, 'an': 1240644}}, {'variant_id': '5-1295030-A-G', 'chrom': '5', 'pos': 1295030, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1240644}}, {'variant_id': '5-1295030-A-T', 'chrom': '5', 'pos': 1295030, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 0, 'an': 1240644}}, {'variant_id': '5-1295030-A-AG', 'chrom': '5', 'pos': 1295030, 'ref': 'A', 'alt': 'AG', 'joint': {'ac': 11, 'an': 1240644}}, {'variant_id': '5-1295031-G-A', 'chrom': '5', 'pos': 1295031, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 1240464}}, {'variant_id': '5-1295031-G-T', 'chrom': '5', 'pos': 1295031, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1240464}}, {'variant_id': '5-1295032-G-T', 'chrom': '5', 'pos': 1295032, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1231830}}, {'variant_id': '5-1295032-G-A', 'chrom': '5', 'pos': 1295032, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1231830}}, {'variant_id': '5-1295032-G-C', 'chrom': '5', 'pos': 1295032, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1231830}}, {'variant_id': '5-1295032-G-GTCCTGCTGCGC', 'chrom': '5', 'pos': 1295032, 'ref': 'G', 'alt': 'GTCCTGCTGCGC', 'joint': {'ac': 0, 'an': 1231824}}, {'variant_id': '5-1295033-A-C', 'chrom': '5', 'pos': 1295033, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 1, 'an': 1215048}}, {'variant_id': '5-1295033-A-G', 'chrom': '5', 'pos': 1295033, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 1, 'an': 1215048}}, {'variant_id': '5-1295034-C-G', 'chrom': '5', 'pos': 1295034, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 1224994}}, {'variant_id': '5-1295034-C-T', 'chrom': '5', 'pos': 1295034, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 8, 'an': 1224994}}, {'variant_id': '5-1295034-C-A', 'chrom': '5', 'pos': 1295034, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 6, 'an': 1224982}}, {'variant_id': '5-1295035-G-GAATTCCTACGTAC', 'chrom': '5', 'pos': 1295035, 'ref': 'G', 'alt': 'GAATTCCTACGTAC', 'joint': {'ac': 1, 'an': 1225222}}, {'variant_id': '5-1295035-G-A', 'chrom': '5', 'pos': 1295035, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 1225216}}, {'variant_id': '5-1295035-G-T', 'chrom': '5', 'pos': 1295035, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1225216}}, {'variant_id': '5-1295035-G-C', 'chrom': '5', 'pos': 1295035, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 1225222}}, {'variant_id': '5-1295035-G-GAATT', 'chrom': '5', 'pos': 1295035, 'ref': 'G', 'alt': 'GAATT', 'joint': {'ac': 1, 'an': 1225222}}, {'variant_id': '5-1295036-C-T', 'chrom': '5', 'pos': 1295036, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1221844}}, {'variant_id': '5-1295036-C-A', 'chrom': '5', 'pos': 1295036, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1221844}}, {'variant_id': '5-1295036-C-CCTACGTACCACCACACTGGGATCCGGCCGGCGCCT', 'chrom': '5', 'pos': 1295036, 'ref': 'C', 'alt': 'CCTACGTACCACCACACTGGGATCCGGCCGGCGCCT', 'joint': {'ac': 1, 'an': 1221844}}, {'variant_id': '5-1295037-A-ACCACACTGGGATCCG', 'chrom': '5', 'pos': 1295037, 'ref': 'A', 'alt': 'ACCACACTGGGATCCG', 'joint': {'ac': 0, 'an': 1211982}}, {'variant_id': '5-1295037-A-ACCACACTGGGATCCGGCC', 'chrom': '5', 'pos': 1295037, 'ref': 'A', 'alt': 'ACCACACTGGGATCCGGCC', 'joint': {'ac': 1, 'an': 1211982}}, {'variant_id': '5-1295037-A-T', 'chrom': '5', 'pos': 1295037, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 1, 'an': 1211982}}, {'variant_id': '5-1295037-A-G', 'chrom': '5', 'pos': 1295037, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1211982}}, {'variant_id': '5-1295038-G-A', 'chrom': '5', 'pos': 1295038, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1221212}}, {'variant_id': '5-1295038-G-T', 'chrom': '5', 'pos': 1295038, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1221212}}, {'variant_id': '5-1295039-C-A', 'chrom': '5', 'pos': 1295039, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1212418}}, {'variant_id': '5-1295039-C-G', 'chrom': '5', 'pos': 1295039, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 1212418}}, {'variant_id': '5-1295039-C-T', 'chrom': '5', 'pos': 1295039, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1212418}}, {'variant_id': '5-1295040-G-A', 'chrom': '5', 'pos': 1295040, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1211818}}, {'variant_id': '5-1295040-G-T', 'chrom': '5', 'pos': 1295040, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1211814}}, {'variant_id': '5-1295041-C-T', 'chrom': '5', 'pos': 1295041, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 1201352}}, {'variant_id': '5-1295041-C-A', 'chrom': '5', 'pos': 1295041, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 2, 'an': 1201352}}, {'variant_id': '5-1295042-T-A', 'chrom': '5', 'pos': 1295042, 'ref': 'T', 'alt': 'A', 'joint': {'ac': 2, 'an': 1134910}}, {'variant_id': '5-1295042-T-C', 'chrom': '5', 'pos': 1295042, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 12, 'an': 1134910}}, {'variant_id': '5-1295043-G-C', 'chrom': '5', 'pos': 1295043, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1193066}}, {'variant_id': '5-1295043-G-A', 'chrom': '5', 'pos': 1295043, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1193066}}, {'variant_id': '5-1295043-G-T', 'chrom': '5', 'pos': 1295043, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1193064}}, {'variant_id': '5-1295044-C-T', 'chrom': '5', 'pos': 1295044, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 2, 'an': 1193540}}, {'variant_id': '5-1295044-C-A', 'chrom': '5', 'pos': 1295044, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 1, 'an': 1193648}}, {'variant_id': '5-1295045-C-T', 'chrom': '5', 'pos': 1295045, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1181738}}, {'variant_id': '5-1295045-C-A', 'chrom': '5', 'pos': 1295045, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1181738}}, {'variant_id': '5-1295046-T-G', 'chrom': '5', 'pos': 1295046, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 43, 'an': 1165730}}, {'variant_id': '5-1295046-T-C', 'chrom': '5', 'pos': 1295046, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 0, 'an': 1165848}}, {'variant_id': '5-1295046-T-A', 'chrom': '5', 'pos': 1295046, 'ref': 'T', 'alt': 'A', 'joint': {'ac': 0, 'an': 1165846}}, {'variant_id': '5-1295047-G-A', 'chrom': '5', 'pos': 1295047, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 1152190}}, {'variant_id': '5-1295047-G-T', 'chrom': '5', 'pos': 1295047, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1152190}}, {'variant_id': '5-1295048-A-G', 'chrom': '5', 'pos': 1295048, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1177338}}, {'variant_id': '5-1295049-A-G', 'chrom': '5', 'pos': 1295049, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1171224}}, {'variant_id': '5-1295050-A-G', 'chrom': '5', 'pos': 1295050, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 1103920}}, {'variant_id': '5-1295050-A-C', 'chrom': '5', 'pos': 1295050, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 5, 'an': 1103920}}, {'variant_id': '5-1295051-C-T', 'chrom': '5', 'pos': 1295051, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 6, 'an': 1170916}}, {'variant_id': '5-1295051-C-A', 'chrom': '5', 'pos': 1295051, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1170916}}, {'variant_id': '5-1295052-T-C', 'chrom': '5', 'pos': 1295052, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 2, 'an': 994092}}, {'variant_id': '5-1295052-T-G', 'chrom': '5', 'pos': 1295052, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 0, 'an': 994192}}, {'variant_id': '5-1295052-T-A', 'chrom': '5', 'pos': 1295052, 'ref': 'T', 'alt': 'A', 'joint': {'ac': 0, 'an': 994188}}, {'variant_id': '5-1295053-C-T', 'chrom': '5', 'pos': 1295053, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 4, 'an': 1158426}}, {'variant_id': '5-1295053-C-G', 'chrom': '5', 'pos': 1295053, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 2, 'an': 1158424}}, {'variant_id': '5-1295053-C-A', 'chrom': '5', 'pos': 1295053, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 5, 'an': 1158426}}, {'variant_id': '5-1295054-G-A', 'chrom': '5', 'pos': 1295054, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 3, 'an': 1148198}}, {'variant_id': '5-1295054-G-T', 'chrom': '5', 'pos': 1295054, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1148198}}, {'variant_id': '5-1295054-G-C', 'chrom': '5', 'pos': 1295054, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 1148200}}, {'variant_id': '5-1295055-C-G', 'chrom': '5', 'pos': 1295055, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 1, 'an': 1143596}}, {'variant_id': '5-1295055-C-T', 'chrom': '5', 'pos': 1295055, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 1143596}}, {'variant_id': '5-1295055-C-A', 'chrom': '5', 'pos': 1295055, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 1, 'an': 1143596}}, {'variant_id': '5-1295056-G-T', 'chrom': '5', 'pos': 1295056, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1132486}}, {'variant_id': '5-1295056-G-A', 'chrom': '5', 'pos': 1295056, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1132490}}, {'variant_id': '5-1295056-G-C', 'chrom': '5', 'pos': 1295056, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1132490}}, {'variant_id': '5-1295057-C-G', 'chrom': '5', 'pos': 1295057, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 1145144}}, {'variant_id': '5-1295057-C-A', 'chrom': '5', 'pos': 1295057, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1145144}}, {'variant_id': '5-1295058-C-A', 'chrom': '5', 'pos': 1295058, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1105718}}, {'variant_id': '5-1295058-C-T', 'chrom': '5', 'pos': 1295058, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 1105718}}, {'variant_id': '5-1295059-G-C', 'chrom': '5', 'pos': 1295059, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 2, 'an': 1124764}}, {'variant_id': '5-1295059-G-A', 'chrom': '5', 'pos': 1295059, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1124762}}, {'variant_id': '5-1295059-G-T', 'chrom': '5', 'pos': 1295059, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 4, 'an': 1124870}}, {'variant_id': '5-1295060-C-A', 'chrom': '5', 'pos': 1295060, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 1, 'an': 1049018}}, {'variant_id': '5-1295060-C-G', 'chrom': '5', 'pos': 1295060, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 1049018}}, {'variant_id': '5-1295060-C-T', 'chrom': '5', 'pos': 1295060, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 4, 'an': 1049018}}, {'variant_id': '5-1295061-G-A', 'chrom': '5', 'pos': 1295061, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 5, 'an': 1114106}}, {'variant_id': '5-1295061-G-T', 'chrom': '5', 'pos': 1295061, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1114216}}, {'variant_id': '5-1295062-A-T', 'chrom': '5', 'pos': 1295062, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 4, 'an': 817170}}, {'variant_id': '5-1295063-G-A', 'chrom': '5', 'pos': 1295063, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 30, 'an': 1116540}}, {'variant_id': '5-1295063-G-T', 'chrom': '5', 'pos': 1295063, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1116646}}, {'variant_id': '5-1295064-G-T', 'chrom': '5', 'pos': 1295064, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1076776}}, {'variant_id': '5-1295064-G-A', 'chrom': '5', 'pos': 1295064, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1076778}}, {'variant_id': '5-1295064-G-C', 'chrom': '5', 'pos': 1295064, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1076778}}, {'variant_id': '5-1295065-A-G', 'chrom': '5', 'pos': 1295065, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 823868}}, {'variant_id': '5-1295065-A-T', 'chrom': '5', 'pos': 1295065, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 2, 'an': 823868}}, {'variant_id': '5-1295066-G-A', 'chrom': '5', 'pos': 1295066, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1085342}}, {'variant_id': '5-1295066-G-T', 'chrom': '5', 'pos': 1295066, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1085340}}, {'variant_id': '5-1295067-AGGGCGGGGCCGCGGAAAGGAAGGGGAGGGGCTGGGAGGGCCCGGAGG-A', 'chrom': '5', 'pos': 1295067, 'ref': 'AGGGCGGGGCCGCGGAAAGGAAGGGGAGGGGCTGGGAGGGCCCGGAGG', 'alt': 'A', 'joint': {'ac': 0, 'an': 575382}}, {'variant_id': '5-1295067-AGGGCGGGGCCGCGGAAAGGAAGGGGAGGGGCTGGGAGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACG-A', 'chrom': '5', 'pos': 1295067, 'ref': 'AGGGCGGGGCCGCGGAAAGGAAGGGGAGGGGCTGGGAGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACG', 'alt': 'A', 'joint': {'ac': 11, 'an': 575368}}, {'variant_id': '5-1295067-A-G', 'chrom': '5', 'pos': 1295067, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 1, 'an': 575386}}, {'variant_id': '5-1295067-AG-A', 'chrom': '5', 'pos': 1295067, 'ref': 'AG', 'alt': 'A', 'joint': {'ac': 0, 'an': 575386}}, {'variant_id': '5-1295067-A-C', 'chrom': '5', 'pos': 1295067, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 1, 'an': 575386}}, {'variant_id': '5-1295067-A-T', 'chrom': '5', 'pos': 1295067, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 5, 'an': 575384}}, {'variant_id': '5-1295068-G-T', 'chrom': '5', 'pos': 1295068, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1084134}}, {'variant_id': '5-1295068-G-A', 'chrom': '5', 'pos': 1295068, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 20, 'an': 1084028}}, {'variant_id': '5-1295069-G-T', 'chrom': '5', 'pos': 1295069, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1089136}}, {'variant_id': '5-1295069-G-A', 'chrom': '5', 'pos': 1295069, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1089136}}, {'variant_id': '5-1295070-G-A', 'chrom': '5', 'pos': 1295070, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 1077718}}, {'variant_id': '5-1295070-GCGGGGCCGCGGAAAGGAAGGGGAGGGGCTGGGAGGGCCCGGAGGGGGCTGGGC-G', 'chrom': '5', 'pos': 1295070, 'ref': 'GCGGGGCCGCGGAAAGGAAGGGGAGGGGCTGGGAGGGCCCGGAGGGGGCTGGGC', 'alt': 'G', 'joint': {'ac': 0, 'an': 1077718}}, {'variant_id': '5-1295070-G-T', 'chrom': '5', 'pos': 1295070, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1077716}}, {'variant_id': '5-1295071-C-G', 'chrom': '5', 'pos': 1295071, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 5, 'an': 329814}}, {'variant_id': '5-1295071-C-A', 'chrom': '5', 'pos': 1295071, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 329820}}, {'variant_id': '5-1295071-C-T', 'chrom': '5', 'pos': 1295071, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 329766}}, {'variant_id': '5-1295072-G-GGGGCCGCGGAAAGGAAGGGGAGGGGCTGGGAGGGCCCGGAGGGGGCT', 'chrom': '5', 'pos': 1295072, 'ref': 'G', 'alt': 'GGGGCCGCGGAAAGGAAGGGGAGGGGCTGGGAGGGCCCGGAGGGGGCT', 'joint': {'ac': 1, 'an': 1067516}}, {'variant_id': '5-1295072-G-T', 'chrom': '5', 'pos': 1295072, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1067516}}, {'variant_id': '5-1295072-G-A', 'chrom': '5', 'pos': 1295072, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1067516}}, {'variant_id': '5-1295073-G-A', 'chrom': '5', 'pos': 1295073, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1069398}}, {'variant_id': '5-1295073-G-T', 'chrom': '5', 'pos': 1295073, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1069506}}, {'variant_id': '5-1295074-G-T', 'chrom': '5', 'pos': 1295074, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 1066896}}, {'variant_id': '5-1295074-G-A', 'chrom': '5', 'pos': 1295074, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 1066896}}, {'variant_id': '5-1295075-G-A', 'chrom': '5', 'pos': 1295075, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 12, 'an': 973884}}, {'variant_id': '5-1295075-G-GCCGCGGAAAGGAAGGGGAGGGGCTGGGAGGGC', 'chrom': '5', 'pos': 1295075, 'ref': 'G', 'alt': 'GCCGCGGAAAGGAAGGGGAGGGGCTGGGAGGGC', 'joint': {'ac': 1, 'an': 973884}}, {'variant_id': '5-1295075-G-T', 'chrom': '5', 'pos': 1295075, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 973880}}, {'variant_id': '5-1295075-G-C', 'chrom': '5', 'pos': 1295075, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 973884}}, {'variant_id': '5-1295076-C-A', 'chrom': '5', 'pos': 1295076, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 1043112}}, {'variant_id': '5-1295076-C-T', 'chrom': '5', 'pos': 1295076, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 1043110}}, {'variant_id': '5-1295077-C-G', 'chrom': '5', 'pos': 1295077, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 773772}}, {'variant_id': '5-1295077-C-T', 'chrom': '5', 'pos': 1295077, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 773772}}, {'variant_id': '5-1295077-C-A', 'chrom': '5', 'pos': 1295077, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 773772}}, {'variant_id': '5-1295078-G-A', 'chrom': '5', 'pos': 1295078, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 1016936}}, {'variant_id': '5-1295078-G-T', 'chrom': '5', 'pos': 1295078, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 1016936}}, {'variant_id': '5-1295078-G-C', 'chrom': '5', 'pos': 1295078, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 1016934}}, {'variant_id': '5-1295079-C-A', 'chrom': '5', 'pos': 1295079, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 468318}}, {'variant_id': '5-1295079-C-T', 'chrom': '5', 'pos': 1295079, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 468318}}, {'variant_id': '5-1295079-C-G', 'chrom': '5', 'pos': 1295079, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 1, 'an': 468318}}, {'variant_id': '5-1295080-G-T', 'chrom': '5', 'pos': 1295080, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 979986}}, {'variant_id': '5-1295080-G-A', 'chrom': '5', 'pos': 1295080, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 979986}}, {'variant_id': '5-1295081-G-C', 'chrom': '5', 'pos': 1295081, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 492356}}, {'variant_id': '5-1295081-GAAAGGAA-G', 'chrom': '5', 'pos': 1295081, 'ref': 'GAAAGGAA', 'alt': 'G', 'joint': {'ac': 0, 'an': 492352}}, {'variant_id': '5-1295081-G-A', 'chrom': '5', 'pos': 1295081, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 492356}}, {'variant_id': '5-1295081-GA-G', 'chrom': '5', 'pos': 1295081, 'ref': 'GA', 'alt': 'G', 'joint': {'ac': 0, 'an': 492356}}, {'variant_id': '5-1295081-G-T', 'chrom': '5', 'pos': 1295081, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 5, 'an': 492352}}, {'variant_id': '5-1295082-A-G', 'chrom': '5', 'pos': 1295082, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 1, 'an': 986386}}, {'variant_id': '5-1295083-A-C', 'chrom': '5', 'pos': 1295083, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 0, 'an': 913458}}, {'variant_id': '5-1295083-A-G', 'chrom': '5', 'pos': 1295083, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 913462}}, {'variant_id': '5-1295084-A-C', 'chrom': '5', 'pos': 1295084, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 2, 'an': 433696}}, {'variant_id': '5-1295084-AGGAAG-A', 'chrom': '5', 'pos': 1295084, 'ref': 'AGGAAG', 'alt': 'A', 'joint': {'ac': 0, 'an': 433702}}, {'variant_id': '5-1295084-AGGAAGGGGAGGGGCTGGGAGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGCGGGGTCCGCGCGGAGGAGGCGGAGCT-A', 'chrom': '5', 'pos': 1295084, 'ref': 'AGGAAGGGGAGGGGCTGGGAGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGCGGGGTCCGCGCGGAGGAGGCGGAGCT', 'alt': 'A', 'joint': {'ac': 0, 'an': 433702}}, {'variant_id': '5-1295084-A-G', 'chrom': '5', 'pos': 1295084, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 433702}}, {'variant_id': '5-1295084-A-T', 'chrom': '5', 'pos': 1295084, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 5, 'an': 433684}}, {'variant_id': '5-1295085-G-T', 'chrom': '5', 'pos': 1295085, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 939294}}, {'variant_id': '5-1295085-G-A', 'chrom': '5', 'pos': 1295085, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 939294}}, {'variant_id': '5-1295086-G-T', 'chrom': '5', 'pos': 1295086, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 627078}}, {'variant_id': '5-1295086-G-A', 'chrom': '5', 'pos': 1295086, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 627082}}, {'variant_id': '5-1295086-G-C', 'chrom': '5', 'pos': 1295086, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 627080}}, {'variant_id': '5-1295086-GA-G', 'chrom': '5', 'pos': 1295086, 'ref': 'GA', 'alt': 'G', 'joint': {'ac': 0, 'an': 627080}}, {'variant_id': '5-1295087-A-G', 'chrom': '5', 'pos': 1295087, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 786032}}, {'variant_id': '5-1295087-A-C', 'chrom': '5', 'pos': 1295087, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 0, 'an': 786032}}, {'variant_id': '5-1295088-A-G', 'chrom': '5', 'pos': 1295088, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 169360}}, {'variant_id': '5-1295088-A-AGGGGAGGGGC', 'chrom': '5', 'pos': 1295088, 'ref': 'A', 'alt': 'AGGGGAGGGGC', 'joint': {'ac': 5, 'an': 169298}}, {'variant_id': '5-1295088-A-T', 'chrom': '5', 'pos': 1295088, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 52, 'an': 169322}}, {'variant_id': '5-1295088-AGGGGAGGGGCTGGGAGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTC-A', 'chrom': '5', 'pos': 1295088, 'ref': 'AGGGGAGGGGCTGGGAGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTC', 'alt': 'A', 'joint': {'ac': 0, 'an': 169358}}, {'variant_id': '5-1295088-A-C', 'chrom': '5', 'pos': 1295088, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 6, 'an': 169350}}, {'variant_id': '5-1295089-GGGGAGGGGCT-G', 'chrom': '5', 'pos': 1295089, 'ref': 'GGGGAGGGGCT', 'alt': 'G', 'joint': {'ac': 0, 'an': 913008}}, {'variant_id': '5-1295089-G-GGGGAGGGGCT', 'chrom': '5', 'pos': 1295089, 'ref': 'G', 'alt': 'GGGGAGGGGCT', 'joint': {'ac': 8, 'an': 913012}}, {'variant_id': '5-1295089-G-C', 'chrom': '5', 'pos': 1295089, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 913008}}, {'variant_id': '5-1295089-G-A', 'chrom': '5', 'pos': 1295089, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 913008}}, {'variant_id': '5-1295089-G-T', 'chrom': '5', 'pos': 1295089, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 913008}}, {'variant_id': '5-1295090-GGGAGGGGCTGGGAGGGCCC-G', 'chrom': '5', 'pos': 1295090, 'ref': 'GGGAGGGGCTGGGAGGGCCC', 'alt': 'G', 'joint': {'ac': 0, 'an': 822660}}, {'variant_id': '5-1295090-G-T', 'chrom': '5', 'pos': 1295090, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 24, 'an': 822558}}, {'variant_id': '5-1295090-G-A', 'chrom': '5', 'pos': 1295090, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 822662}}, {'variant_id': '5-1295091-G-T', 'chrom': '5', 'pos': 1295091, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 823762}}, {'variant_id': '5-1295091-G-A', 'chrom': '5', 'pos': 1295091, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 823762}}, {'variant_id': '5-1295092-G-T', 'chrom': '5', 'pos': 1295092, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 762344}}, {'variant_id': '5-1295092-G-A', 'chrom': '5', 'pos': 1295092, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 762344}}, {'variant_id': '5-1295092-GA-G', 'chrom': '5', 'pos': 1295092, 'ref': 'GA', 'alt': 'G', 'joint': {'ac': 0, 'an': 762344}}, {'variant_id': '5-1295092-GAGGGGCTGGGAGGGCCCGGA-G', 'chrom': '5', 'pos': 1295092, 'ref': 'GAGGGGCTGGGAGGGCCCGGA', 'alt': 'G', 'joint': {'ac': 0, 'an': 762342}}, {'variant_id': '5-1295093-AGGGGCTGGGAGGGCCCGGAG-A', 'chrom': '5', 'pos': 1295093, 'ref': 'AGGGGCTGGGAGGGCCCGGAG', 'alt': 'A', 'joint': {'ac': 1, 'an': 95758}}, {'variant_id': '5-1295093-A-C', 'chrom': '5', 'pos': 1295093, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 1, 'an': 95820}}, {'variant_id': '5-1295093-A-AT', 'chrom': '5', 'pos': 1295093, 'ref': 'A', 'alt': 'AT', 'joint': {'ac': 0, 'an': 95820}}, {'variant_id': '5-1295093-A-G', 'chrom': '5', 'pos': 1295093, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 6, 'an': 95782}}, {'variant_id': '5-1295093-A-T', 'chrom': '5', 'pos': 1295093, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 154, 'an': 95796}}, {'variant_id': '5-1295094-G-T', 'chrom': '5', 'pos': 1295094, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 812092}}, {'variant_id': '5-1295094-G-A', 'chrom': '5', 'pos': 1295094, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 812092}}, {'variant_id': '5-1295095-G-T', 'chrom': '5', 'pos': 1295095, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 808530}}, {'variant_id': '5-1295095-G-A', 'chrom': '5', 'pos': 1295095, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 808530}}, {'variant_id': '5-1295096-G-A', 'chrom': '5', 'pos': 1295096, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 807672}}, {'variant_id': '5-1295096-G-T', 'chrom': '5', 'pos': 1295096, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 807672}}, {'variant_id': '5-1295096-GGCTGGGAGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGCGGGGTCCGCGCGGAGGAGGCGGA-G', 'chrom': '5', 'pos': 1295096, 'ref': 'GGCTGGGAGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGCGGGGTCCGCGCGGAGGAGGCGGA', 'alt': 'G', 'joint': {'ac': 1, 'an': 807672}}, {'variant_id': '5-1295097-G-A', 'chrom': '5', 'pos': 1295097, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 794202}}, {'variant_id': '5-1295097-GCT-G', 'chrom': '5', 'pos': 1295097, 'ref': 'GCT', 'alt': 'G', 'joint': {'ac': 0, 'an': 794202}}, {'variant_id': '5-1295097-GCTGGGA-G', 'chrom': '5', 'pos': 1295097, 'ref': 'GCTGGGA', 'alt': 'G', 'joint': {'ac': 0, 'an': 794202}}, {'variant_id': '5-1295097-G-T', 'chrom': '5', 'pos': 1295097, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 794202}}, {'variant_id': '5-1295098-C-T', 'chrom': '5', 'pos': 1295098, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 218452}}, {'variant_id': '5-1295098-C-G', 'chrom': '5', 'pos': 1295098, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 2, 'an': 218448}}, {'variant_id': '5-1295098-C-A', 'chrom': '5', 'pos': 1295098, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 218452}}, {'variant_id': '5-1295099-TGGGAGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGCG-T', 'chrom': '5', 'pos': 1295099, 'ref': 'TGGGAGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGCG', 'alt': 'T', 'joint': {'ac': 0, 'an': 71746}}, {'variant_id': '5-1295099-T-G', 'chrom': '5', 'pos': 1295099, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 36, 'an': 71676}}, {'variant_id': '5-1295099-TGGGAGGGCCCGGA-T', 'chrom': '5', 'pos': 1295099, 'ref': 'TGGGAGGGCCCGGA', 'alt': 'T', 'joint': {'ac': 0, 'an': 71746}}, {'variant_id': '5-1295099-T-C', 'chrom': '5', 'pos': 1295099, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 0, 'an': 71746}}, {'variant_id': '5-1295100-G-A', 'chrom': '5', 'pos': 1295100, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 778588}}, {'variant_id': '5-1295100-G-T', 'chrom': '5', 'pos': 1295100, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 3, 'an': 778588}}, {'variant_id': '5-1295101-G-C', 'chrom': '5', 'pos': 1295101, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 738684}}, {'variant_id': '5-1295101-G-T', 'chrom': '5', 'pos': 1295101, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 738684}}, {'variant_id': '5-1295101-G-A', 'chrom': '5', 'pos': 1295101, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 31, 'an': 738684}}, {'variant_id': '5-1295102-GA-G', 'chrom': '5', 'pos': 1295102, 'ref': 'GA', 'alt': 'G', 'joint': {'ac': 0, 'an': 647344}}, {'variant_id': '5-1295102-G-T', 'chrom': '5', 'pos': 1295102, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 647342}}, {'variant_id': '5-1295102-G-A', 'chrom': '5', 'pos': 1295102, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 647346}}, {'variant_id': '5-1295102-GAGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGCGGGGTCCGCGCGGAGGAGGCGGAGCTGGA-G', 'chrom': '5', 'pos': 1295102, 'ref': 'GAGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGCGGGGTCCGCGCGGAGGAGGCGGAGCTGGA', 'alt': 'G', 'joint': {'ac': 0, 'an': 647346}}, {'variant_id': '5-1295103-A-G', 'chrom': '5', 'pos': 1295103, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 2, 'an': 96238}}, {'variant_id': '5-1295103-AGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGC-A', 'chrom': '5', 'pos': 1295103, 'ref': 'AGGGCCCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGC', 'alt': 'A', 'joint': {'ac': 0, 'an': 96280}}, {'variant_id': '5-1295103-A-T', 'chrom': '5', 'pos': 1295103, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 41, 'an': 96266}}, {'variant_id': '5-1295103-A-C', 'chrom': '5', 'pos': 1295103, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 1, 'an': 96278}}, {'variant_id': '5-1295104-G-A', 'chrom': '5', 'pos': 1295104, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 722756}}, {'variant_id': '5-1295104-G-T', 'chrom': '5', 'pos': 1295104, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 722756}}, {'variant_id': '5-1295105-G-T', 'chrom': '5', 'pos': 1295105, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 706164}}, {'variant_id': '5-1295105-G-A', 'chrom': '5', 'pos': 1295105, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 706164}}, {'variant_id': '5-1295106-G-T', 'chrom': '5', 'pos': 1295106, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 6, 'an': 259604}}, {'variant_id': '5-1295106-G-C', 'chrom': '5', 'pos': 1295106, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 259730}}, {'variant_id': '5-1295106-GCCCGGAGGGGGCTGGGCCGGGGACCC-G', 'chrom': '5', 'pos': 1295106, 'ref': 'GCCCGGAGGGGGCTGGGCCGGGGACCC', 'alt': 'G', 'joint': {'ac': 0, 'an': 259728}}, {'variant_id': '5-1295106-G-A', 'chrom': '5', 'pos': 1295106, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 259732}}, {'variant_id': '5-1295107-C-T', 'chrom': '5', 'pos': 1295107, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 18, 'an': 681454}}, {'variant_id': '5-1295107-CCCGG-C', 'chrom': '5', 'pos': 1295107, 'ref': 'CCCGG', 'alt': 'C', 'joint': {'ac': 0, 'an': 681558}}, {'variant_id': '5-1295107-C-CT', 'chrom': '5', 'pos': 1295107, 'ref': 'C', 'alt': 'CT', 'joint': {'ac': 0, 'an': 681560}}, {'variant_id': '5-1295107-C-A', 'chrom': '5', 'pos': 1295107, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 5, 'an': 681454}}, {'variant_id': '5-1295108-C-T', 'chrom': '5', 'pos': 1295108, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 651956}}, {'variant_id': '5-1295108-CCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGCGGGGTCCGCG-C', 'chrom': '5', 'pos': 1295108, 'ref': 'CCGGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGCGGGGTCCGCG', 'alt': 'C', 'joint': {'ac': 0, 'an': 651954}}, {'variant_id': '5-1295108-C-G', 'chrom': '5', 'pos': 1295108, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 1, 'an': 651956}}, {'variant_id': '5-1295109-C-G', 'chrom': '5', 'pos': 1295109, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 5, 'an': 146554}}, {'variant_id': '5-1295109-C-T', 'chrom': '5', 'pos': 1295109, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 146558}}, {'variant_id': '5-1295110-G-C', 'chrom': '5', 'pos': 1295110, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 2, 'an': 671906}}, {'variant_id': '5-1295110-G-A', 'chrom': '5', 'pos': 1295110, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 672014}}, {'variant_id': '5-1295110-GGAGGGGGCTGGGCCGGGGACCC-G', 'chrom': '5', 'pos': 1295110, 'ref': 'GGAGGGGGCTGGGCCGGGGACCC', 'alt': 'G', 'joint': {'ac': 0, 'an': 672012}}, {'variant_id': '5-1295110-GGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGCGGGGTCCGCGCGGAGGAGGCGGAGCTGGAAGGT-G', 'chrom': '5', 'pos': 1295110, 'ref': 'GGAGGGGGCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGCGGGGTCCGCGCGGAGGAGGCGGAGCTGGAAGGT', 'alt': 'G', 'joint': {'ac': 1, 'an': 672014}}, {'variant_id': '5-1295111-G-T', 'chrom': '5', 'pos': 1295111, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 640764}}, {'variant_id': '5-1295111-GA-G', 'chrom': '5', 'pos': 1295111, 'ref': 'GA', 'alt': 'G', 'joint': {'ac': 0, 'an': 640748}}, {'variant_id': '5-1295112-AG-A', 'chrom': '5', 'pos': 1295112, 'ref': 'AG', 'alt': 'A', 'joint': {'ac': 0, 'an': 98898}}, {'variant_id': '5-1295112-A-C', 'chrom': '5', 'pos': 1295112, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 0, 'an': 98898}}, {'variant_id': '5-1295112-A-T', 'chrom': '5', 'pos': 1295112, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 2, 'an': 98898}}, {'variant_id': '5-1295112-A-G', 'chrom': '5', 'pos': 1295112, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 2, 'an': 98846}}, {'variant_id': '5-1295112-A-AG', 'chrom': '5', 'pos': 1295112, 'ref': 'A', 'alt': 'AG', 'joint': {'ac': 1, 'an': 98898}}, {'variant_id': '5-1295113-G-A', 'chrom': '5', 'pos': 1295113, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 62, 'an': 655850}}, {'variant_id': '5-1295113-G-T', 'chrom': '5', 'pos': 1295113, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 4, 'an': 656050}}, {'variant_id': '5-1295113-G-C', 'chrom': '5', 'pos': 1295113, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 6, 'an': 655944}}, {'variant_id': '5-1295114-G-C', 'chrom': '5', 'pos': 1295114, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 656866}}, {'variant_id': '5-1295114-GGGGC-G', 'chrom': '5', 'pos': 1295114, 'ref': 'GGGGC', 'alt': 'G', 'joint': {'ac': 0, 'an': 656858}}, {'variant_id': '5-1295114-G-A', 'chrom': '5', 'pos': 1295114, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 656866}}, {'variant_id': '5-1295114-G-T', 'chrom': '5', 'pos': 1295114, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 8, 'an': 656758}}, {'variant_id': '5-1295114-GGGGCT-G', 'chrom': '5', 'pos': 1295114, 'ref': 'GGGGCT', 'alt': 'G', 'joint': {'ac': 0, 'an': 656866}}, {'variant_id': '5-1295115-G-A', 'chrom': '5', 'pos': 1295115, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 632368}}, {'variant_id': '5-1295115-G-T', 'chrom': '5', 'pos': 1295115, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 632474}}, {'variant_id': '5-1295116-G-T', 'chrom': '5', 'pos': 1295116, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 633286}}, {'variant_id': '5-1295116-G-A', 'chrom': '5', 'pos': 1295116, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 633286}}, {'variant_id': '5-1295117-G-T', 'chrom': '5', 'pos': 1295117, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 624700}}, {'variant_id': '5-1295117-GCTGGGCCGGGGACCC-G', 'chrom': '5', 'pos': 1295117, 'ref': 'GCTGGGCCGGGGACCC', 'alt': 'G', 'joint': {'ac': 0, 'an': 624698}}, {'variant_id': '5-1295117-G-C', 'chrom': '5', 'pos': 1295117, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 624700}}, {'variant_id': '5-1295117-GCT-G', 'chrom': '5', 'pos': 1295117, 'ref': 'GCT', 'alt': 'G', 'joint': {'ac': 0, 'an': 624688}}, {'variant_id': '5-1295117-GC-G', 'chrom': '5', 'pos': 1295117, 'ref': 'GC', 'alt': 'G', 'joint': {'ac': 0, 'an': 624700}}, {'variant_id': '5-1295117-GCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGC-G', 'chrom': '5', 'pos': 1295117, 'ref': 'GCTGGGCCGGGGACCCGGGAGGGGTCGGGACGGGGC', 'alt': 'G', 'joint': {'ac': 0, 'an': 624700}}, {'variant_id': '5-1295117-GCTGGGCC-G', 'chrom': '5', 'pos': 1295117, 'ref': 'GCTGGGCC', 'alt': 'G', 'joint': {'ac': 1, 'an': 624676}}, {'variant_id': '5-1295117-G-A', 'chrom': '5', 'pos': 1295117, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 624700}}, {'variant_id': '5-1295118-C-G', 'chrom': '5', 'pos': 1295118, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 218722}}, {'variant_id': '5-1295118-CT-C', 'chrom': '5', 'pos': 1295118, 'ref': 'CT', 'alt': 'C', 'joint': {'ac': 0, 'an': 218730}}, {'variant_id': '5-1295118-C-T', 'chrom': '5', 'pos': 1295118, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 218734}}, {'variant_id': '5-1295118-C-A', 'chrom': '5', 'pos': 1295118, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 218734}}, {'variant_id': '5-1295119-T-G', 'chrom': '5', 'pos': 1295119, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 1, 'an': 104308}}, {'variant_id': '5-1295119-T-C', 'chrom': '5', 'pos': 1295119, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 0, 'an': 104380}}, {'variant_id': '5-1295120-GGGCC-G', 'chrom': '5', 'pos': 1295120, 'ref': 'GGGCC', 'alt': 'G', 'joint': {'ac': 0, 'an': 613482}}, {'variant_id': '5-1295120-G-C', 'chrom': '5', 'pos': 1295120, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 613482}}, {'variant_id': '5-1295120-GGGCCGGGGACCC-G', 'chrom': '5', 'pos': 1295120, 'ref': 'GGGCCGGGGACCC', 'alt': 'G', 'joint': {'ac': 0, 'an': 613476}}, {'variant_id': '5-1295120-GGGCCGGGGACCCGGGAGGGGTCGGGACGGGGC-G', 'chrom': '5', 'pos': 1295120, 'ref': 'GGGCCGGGGACCCGGGAGGGGTCGGGACGGGGC', 'alt': 'G', 'joint': {'ac': 0, 'an': 613480}}, {'variant_id': '5-1295120-G-A', 'chrom': '5', 'pos': 1295120, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 613482}}, {'variant_id': '5-1295121-GGCC-G', 'chrom': '5', 'pos': 1295121, 'ref': 'GGCC', 'alt': 'G', 'joint': {'ac': 0, 'an': 611892}}, {'variant_id': '5-1295121-G-T', 'chrom': '5', 'pos': 1295121, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 611892}}, {'variant_id': '5-1295121-G-A', 'chrom': '5', 'pos': 1295121, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 5, 'an': 611784}}, {'variant_id': '5-1295122-GCC-G', 'chrom': '5', 'pos': 1295122, 'ref': 'GCC', 'alt': 'G', 'joint': {'ac': 0, 'an': 567050}}, {'variant_id': '5-1295122-G-T', 'chrom': '5', 'pos': 1295122, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 567086}}, {'variant_id': '5-1295122-GC-G', 'chrom': '5', 'pos': 1295122, 'ref': 'GC', 'alt': 'G', 'joint': {'ac': 1, 'an': 567086}}, {'variant_id': '5-1295122-G-A', 'chrom': '5', 'pos': 1295122, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 567086}}, {'variant_id': '5-1295123-C-A', 'chrom': '5', 'pos': 1295123, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 543384}}, {'variant_id': '5-1295123-C-G', 'chrom': '5', 'pos': 1295123, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 543380}}, {'variant_id': '5-1295123-C-T', 'chrom': '5', 'pos': 1295123, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 543384}}, {'variant_id': '5-1295123-CCGGGG-C', 'chrom': '5', 'pos': 1295123, 'ref': 'CCGGGG', 'alt': 'C', 'joint': {'ac': 0, 'an': 543384}}, {'variant_id': '5-1295124-C-A', 'chrom': '5', 'pos': 1295124, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 111564}}, {'variant_id': '5-1295124-C-T', 'chrom': '5', 'pos': 1295124, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 111564}}, {'variant_id': '5-1295124-C-G', 'chrom': '5', 'pos': 1295124, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 14, 'an': 111488}}, {'variant_id': '5-1295124-CG-C', 'chrom': '5', 'pos': 1295124, 'ref': 'CG', 'alt': 'C', 'joint': {'ac': 0, 'an': 111564}}, {'variant_id': '5-1295125-G-C', 'chrom': '5', 'pos': 1295125, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 581944}}, {'variant_id': '5-1295125-G-A', 'chrom': '5', 'pos': 1295125, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 581836}}, {'variant_id': '5-1295125-G-T', 'chrom': '5', 'pos': 1295125, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 581944}}, {'variant_id': '5-1295126-G-T', 'chrom': '5', 'pos': 1295126, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 585802}}, {'variant_id': '5-1295126-G-A', 'chrom': '5', 'pos': 1295126, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 585910}}, {'variant_id': '5-1295126-GGGACC-G', 'chrom': '5', 'pos': 1295126, 'ref': 'GGGACC', 'alt': 'G', 'joint': {'ac': 0, 'an': 585910}}, {'variant_id': '5-1295127-G-T', 'chrom': '5', 'pos': 1295127, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 559822}}, {'variant_id': '5-1295127-G-A', 'chrom': '5', 'pos': 1295127, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 559822}}, {'variant_id': '5-1295127-GGACCCGGGAGGGGTCGGGACGGGGC-G', 'chrom': '5', 'pos': 1295127, 'ref': 'GGACCCGGGAGGGGTCGGGACGGGGC', 'alt': 'G', 'joint': {'ac': 0, 'an': 559820}}, {'variant_id': '5-1295127-GGACCC-G', 'chrom': '5', 'pos': 1295127, 'ref': 'GGACCC', 'alt': 'G', 'joint': {'ac': 0, 'an': 559802}}, {'variant_id': '5-1295128-GACCCGGGA-G', 'chrom': '5', 'pos': 1295128, 'ref': 'GACCCGGGA', 'alt': 'G', 'joint': {'ac': 0, 'an': 386056}}, {'variant_id': '5-1295128-G-A', 'chrom': '5', 'pos': 1295128, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 386064}}, {'variant_id': '5-1295128-G-T', 'chrom': '5', 'pos': 1295128, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 386060}}, {'variant_id': '5-1295128-G-C', 'chrom': '5', 'pos': 1295128, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 386060}}, {'variant_id': '5-1295128-GACCC-G', 'chrom': '5', 'pos': 1295128, 'ref': 'GACCC', 'alt': 'G', 'joint': {'ac': 0, 'an': 385860}}, {'variant_id': '5-1295128-GACC-G', 'chrom': '5', 'pos': 1295128, 'ref': 'GACC', 'alt': 'G', 'joint': {'ac': 0, 'an': 386044}}, {'variant_id': '5-1295129-ACCCGGGAGGGGTCGGGACGGG-A', 'chrom': '5', 'pos': 1295129, 'ref': 'ACCCGGGAGGGGTCGGGACGGG', 'alt': 'A', 'joint': {'ac': 0, 'an': 228226}}, {'variant_id': '5-1295129-AC-A', 'chrom': '5', 'pos': 1295129, 'ref': 'AC', 'alt': 'A', 'joint': {'ac': 1, 'an': 228142}}, {'variant_id': '5-1295129-A-C', 'chrom': '5', 'pos': 1295129, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 0, 'an': 228226}}, {'variant_id': '5-1295129-A-G', 'chrom': '5', 'pos': 1295129, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 228222}}, {'variant_id': '5-1295130-C-T', 'chrom': '5', 'pos': 1295130, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 9, 'an': 533860}}, {'variant_id': '5-1295130-C-G', 'chrom': '5', 'pos': 1295130, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 533968}}, {'variant_id': '5-1295130-C-A', 'chrom': '5', 'pos': 1295130, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 1, 'an': 533968}}, {'variant_id': '5-1295131-C-A', 'chrom': '5', 'pos': 1295131, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 510974}}, {'variant_id': '5-1295131-C-T', 'chrom': '5', 'pos': 1295131, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 46, 'an': 510974}}, {'variant_id': '5-1295132-C-T', 'chrom': '5', 'pos': 1295132, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 105190}}, {'variant_id': '5-1295132-C-CG', 'chrom': '5', 'pos': 1295132, 'ref': 'C', 'alt': 'CG', 'joint': {'ac': 0, 'an': 105188}}, {'variant_id': '5-1295132-CG-C', 'chrom': '5', 'pos': 1295132, 'ref': 'CG', 'alt': 'C', 'joint': {'ac': 2, 'an': 105140}}, {'variant_id': '5-1295132-C-G', 'chrom': '5', 'pos': 1295132, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 8, 'an': 105082}}, {'variant_id': '5-1295132-C-A', 'chrom': '5', 'pos': 1295132, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 105190}}, {'variant_id': '5-1295133-G-A', 'chrom': '5', 'pos': 1295133, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 513858}}, {'variant_id': '5-1295133-GGGAGGGGTCGGGAC-G', 'chrom': '5', 'pos': 1295133, 'ref': 'GGGAGGGGTCGGGAC', 'alt': 'G', 'joint': {'ac': 0, 'an': 513858}}, {'variant_id': '5-1295133-G-C', 'chrom': '5', 'pos': 1295133, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 513858}}, {'variant_id': '5-1295134-G-T', 'chrom': '5', 'pos': 1295134, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 520758}}, {'variant_id': '5-1295134-G-GGGT', 'chrom': '5', 'pos': 1295134, 'ref': 'G', 'alt': 'GGGT', 'joint': {'ac': 0, 'an': 520756}}, {'variant_id': '5-1295134-G-A', 'chrom': '5', 'pos': 1295134, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 520758}}, {'variant_id': '5-1295134-G-C', 'chrom': '5', 'pos': 1295134, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 2, 'an': 520758}}, {'variant_id': '5-1295134-GGAGGGGTCGGGACGGGGC-G', 'chrom': '5', 'pos': 1295134, 'ref': 'GGAGGGGTCGGGACGGGGC', 'alt': 'G', 'joint': {'ac': 0, 'an': 520748}}, {'variant_id': '5-1295135-G-C', 'chrom': '5', 'pos': 1295135, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 484394}}, {'variant_id': '5-1295135-G-T', 'chrom': '5', 'pos': 1295135, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 484394}}, {'variant_id': '5-1295135-GA-G', 'chrom': '5', 'pos': 1295135, 'ref': 'GA', 'alt': 'G', 'joint': {'ac': 0, 'an': 484376}}, {'variant_id': '5-1295135-G-A', 'chrom': '5', 'pos': 1295135, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 13, 'an': 484372}}, {'variant_id': '5-1295136-A-G', 'chrom': '5', 'pos': 1295136, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 108874}}, {'variant_id': '5-1295136-AG-A', 'chrom': '5', 'pos': 1295136, 'ref': 'AG', 'alt': 'A', 'joint': {'ac': 0, 'an': 108890}}, {'variant_id': '5-1295136-A-C', 'chrom': '5', 'pos': 1295136, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 0, 'an': 108890}}, {'variant_id': '5-1295136-A-T', 'chrom': '5', 'pos': 1295136, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 51, 'an': 108788}}, {'variant_id': '5-1295137-G-T', 'chrom': '5', 'pos': 1295137, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 508666}}, {'variant_id': '5-1295137-G-A', 'chrom': '5', 'pos': 1295137, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 508666}}, {'variant_id': '5-1295137-GGGGTCGGGAC-G', 'chrom': '5', 'pos': 1295137, 'ref': 'GGGGTCGGGAC', 'alt': 'G', 'joint': {'ac': 0, 'an': 508664}}, {'variant_id': '5-1295138-G-T', 'chrom': '5', 'pos': 1295138, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 491204}}, {'variant_id': '5-1295138-G-A', 'chrom': '5', 'pos': 1295138, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 491204}}, {'variant_id': '5-1295139-G-A', 'chrom': '5', 'pos': 1295139, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 491496}}, {'variant_id': '5-1295139-G-T', 'chrom': '5', 'pos': 1295139, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 491496}}, {'variant_id': '5-1295139-GGTCGGGACGGGGC-G', 'chrom': '5', 'pos': 1295139, 'ref': 'GGTCGGGACGGGGC', 'alt': 'G', 'joint': {'ac': 0, 'an': 491484}}, {'variant_id': '5-1295139-G-C', 'chrom': '5', 'pos': 1295139, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 491496}}, {'variant_id': '5-1295140-G-T', 'chrom': '5', 'pos': 1295140, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 472034}}, {'variant_id': '5-1295140-GTC-G', 'chrom': '5', 'pos': 1295140, 'ref': 'GTC', 'alt': 'G', 'joint': {'ac': 1, 'an': 471846}}, {'variant_id': '5-1295140-G-A', 'chrom': '5', 'pos': 1295140, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 471930}}, {'variant_id': '5-1295140-GTCGGGAC-G', 'chrom': '5', 'pos': 1295140, 'ref': 'GTCGGGAC', 'alt': 'G', 'joint': {'ac': 0, 'an': 472032}}, {'variant_id': '5-1295140-GT-G', 'chrom': '5', 'pos': 1295140, 'ref': 'GT', 'alt': 'G', 'joint': {'ac': 0, 'an': 472030}}, {'variant_id': '5-1295141-T-A', 'chrom': '5', 'pos': 1295141, 'ref': 'T', 'alt': 'A', 'joint': {'ac': 0, 'an': 297558}}, {'variant_id': '5-1295141-T-C', 'chrom': '5', 'pos': 1295141, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 0, 'an': 297558}}, {'variant_id': '5-1295141-T-G', 'chrom': '5', 'pos': 1295141, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 0, 'an': 297546}}, {'variant_id': '5-1295142-C-A', 'chrom': '5', 'pos': 1295142, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 131866}}, {'variant_id': '5-1295142-CG-C', 'chrom': '5', 'pos': 1295142, 'ref': 'CG', 'alt': 'C', 'joint': {'ac': 0, 'an': 131866}}, {'variant_id': '5-1295142-C-G', 'chrom': '5', 'pos': 1295142, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 5, 'an': 131618}}, {'variant_id': '5-1295142-C-T', 'chrom': '5', 'pos': 1295142, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 2, 'an': 131796}}, {'variant_id': '5-1295143-G-C', 'chrom': '5', 'pos': 1295143, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 476260}}, {'variant_id': '5-1295143-G-A', 'chrom': '5', 'pos': 1295143, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 476368}}, {'variant_id': '5-1295143-G-T', 'chrom': '5', 'pos': 1295143, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 476366}}, {'variant_id': '5-1295144-G-A', 'chrom': '5', 'pos': 1295144, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 476706}}, {'variant_id': '5-1295144-GGACGGGGC-G', 'chrom': '5', 'pos': 1295144, 'ref': 'GGACGGGGC', 'alt': 'G', 'joint': {'ac': 0, 'an': 476544}}, {'variant_id': '5-1295144-G-T', 'chrom': '5', 'pos': 1295144, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 476706}}, {'variant_id': '5-1295145-GAC-G', 'chrom': '5', 'pos': 1295145, 'ref': 'GAC', 'alt': 'G', 'joint': {'ac': 0, 'an': 445278}}, {'variant_id': '5-1295145-G-T', 'chrom': '5', 'pos': 1295145, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 445288}}, {'variant_id': '5-1295145-G-C', 'chrom': '5', 'pos': 1295145, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 445288}}, {'variant_id': '5-1295145-G-A', 'chrom': '5', 'pos': 1295145, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 445288}}, {'variant_id': '5-1295146-A-G', 'chrom': '5', 'pos': 1295146, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 426268}}, {'variant_id': '5-1295146-A-C', 'chrom': '5', 'pos': 1295146, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 0, 'an': 426266}}, {'variant_id': '5-1295147-C-G', 'chrom': '5', 'pos': 1295147, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 4, 'an': 170040}}, {'variant_id': '5-1295147-C-A', 'chrom': '5', 'pos': 1295147, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 170326}}, {'variant_id': '5-1295147-C-CG', 'chrom': '5', 'pos': 1295147, 'ref': 'C', 'alt': 'CG', 'joint': {'ac': 0, 'an': 170324}}, {'variant_id': '5-1295147-C-CGGGGCGGGGGACCGGGGTGGGGTGGGGAG', 'chrom': '5', 'pos': 1295147, 'ref': 'C', 'alt': 'CGGGGCGGGGGACCGGGGTGGGGTGGGGAG', 'joint': {'ac': 0, 'an': 170326}}, {'variant_id': '5-1295147-C-T', 'chrom': '5', 'pos': 1295147, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 170326}}, {'variant_id': '5-1295148-G-T', 'chrom': '5', 'pos': 1295148, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 458580}}, {'variant_id': '5-1295148-G-A', 'chrom': '5', 'pos': 1295148, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 458580}}, {'variant_id': '5-1295149-G-A', 'chrom': '5', 'pos': 1295149, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 458298}}, {'variant_id': '5-1295149-G-T', 'chrom': '5', 'pos': 1295149, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 458298}}, {'variant_id': '5-1295149-GGGCGGGGTCCGCGCGGAGGA-G', 'chrom': '5', 'pos': 1295149, 'ref': 'GGGCGGGGTCCGCGCGGAGGA', 'alt': 'G', 'joint': {'ac': 0, 'an': 458298}}, {'variant_id': '5-1295150-G-A', 'chrom': '5', 'pos': 1295150, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 457460}}, {'variant_id': '5-1295150-G-T', 'chrom': '5', 'pos': 1295150, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 457460}}, {'variant_id': '5-1295150-GGC-G', 'chrom': '5', 'pos': 1295150, 'ref': 'GGC', 'alt': 'G', 'joint': {'ac': 0, 'an': 457456}}, {'variant_id': '5-1295150-G-C', 'chrom': '5', 'pos': 1295150, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 457460}}, {'variant_id': '5-1295151-GC-G', 'chrom': '5', 'pos': 1295151, 'ref': 'GC', 'alt': 'G', 'joint': {'ac': 0, 'an': 456142}}, {'variant_id': '5-1295151-G-A', 'chrom': '5', 'pos': 1295151, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 456144}}, {'variant_id': '5-1295152-C-T', 'chrom': '5', 'pos': 1295152, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 2, 'an': 308186}}, {'variant_id': '5-1295152-C-CGGGGTCCCGGGGTGGGGGGGGGCGGGGGG', 'chrom': '5', 'pos': 1295152, 'ref': 'C', 'alt': 'CGGGGTCCCGGGGTGGGGGGGGGCGGGGGG', 'joint': {'ac': 0, 'an': 308184}}, {'variant_id': '5-1295152-CG-C', 'chrom': '5', 'pos': 1295152, 'ref': 'CG', 'alt': 'C', 'joint': {'ac': 0, 'an': 308186}}, {'variant_id': '5-1295152-C-CGGGGCCCCGGGGGGGGGGGGGGCGGGGGG', 'chrom': '5', 'pos': 1295152, 'ref': 'C', 'alt': 'CGGGGCCCCGGGGGGGGGGGGGGCGGGGGG', 'joint': {'ac': 0, 'an': 308184}}, {'variant_id': '5-1295152-C-CGGGGGG', 'chrom': '5', 'pos': 1295152, 'ref': 'C', 'alt': 'CGGGGGG', 'joint': {'ac': 0, 'an': 308182}}, {'variant_id': '5-1295152-C-A', 'chrom': '5', 'pos': 1295152, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 308186}}, {'variant_id': '5-1295152-C-G', 'chrom': '5', 'pos': 1295152, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 1, 'an': 307990}}, {'variant_id': '5-1295152-C-CGGGGTACCGGGGTGGGGGGGGTAGGGGGG', 'chrom': '5', 'pos': 1295152, 'ref': 'C', 'alt': 'CGGGGTACCGGGGTGGGGGGGGTAGGGGGG', 'joint': {'ac': 0, 'an': 308186}}, {'variant_id': '5-1295152-C-CGGGGGCCCGGGGGGGGGGGGGGAGGGGGG', 'chrom': '5', 'pos': 1295152, 'ref': 'C', 'alt': 'CGGGGGCCCGGGGGGGGGGGGGGAGGGGGG', 'joint': {'ac': 0, 'an': 308184}}, {'variant_id': '5-1295153-GGGGTC-G', 'chrom': '5', 'pos': 1295153, 'ref': 'GGGGTC', 'alt': 'G', 'joint': {'ac': 0, 'an': 442400}}, {'variant_id': '5-1295153-G-T', 'chrom': '5', 'pos': 1295153, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 442400}}, {'variant_id': '5-1295153-G-A', 'chrom': '5', 'pos': 1295153, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 442400}}, {'variant_id': '5-1295154-G-A', 'chrom': '5', 'pos': 1295154, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 441172}}, {'variant_id': '5-1295154-G-T', 'chrom': '5', 'pos': 1295154, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 441172}}, {'variant_id': '5-1295155-G-T', 'chrom': '5', 'pos': 1295155, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 441326}}, {'variant_id': '5-1295155-G-A', 'chrom': '5', 'pos': 1295155, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 441326}}, {'variant_id': '5-1295156-G-C', 'chrom': '5', 'pos': 1295156, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 440980}}, {'variant_id': '5-1295156-G-T', 'chrom': '5', 'pos': 1295156, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 440980}}, {'variant_id': '5-1295156-G-A', 'chrom': '5', 'pos': 1295156, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 440980}}, {'variant_id': '5-1295157-T-C', 'chrom': '5', 'pos': 1295157, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 0, 'an': 436322}}, {'variant_id': '5-1295157-T-G', 'chrom': '5', 'pos': 1295157, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 1, 'an': 436214}}, {'variant_id': '5-1295159-C-G', 'chrom': '5', 'pos': 1295159, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 1, 'an': 438076}}, {'variant_id': '5-1295159-C-T', 'chrom': '5', 'pos': 1295159, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 438076}}, {'variant_id': '5-1295159-C-A', 'chrom': '5', 'pos': 1295159, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 438076}}, {'variant_id': '5-1295160-G-T', 'chrom': '5', 'pos': 1295160, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 437922}}, {'variant_id': '5-1295160-G-A', 'chrom': '5', 'pos': 1295160, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 437922}}, {'variant_id': '5-1295161-C-T', 'chrom': '5', 'pos': 1295161, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 437106}}, {'variant_id': '5-1295161-C-G', 'chrom': '5', 'pos': 1295161, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 437212}}, {'variant_id': '5-1295162-G-A', 'chrom': '5', 'pos': 1295162, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 6, 'an': 434268}}, {'variant_id': '5-1295162-G-T', 'chrom': '5', 'pos': 1295162, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 434268}}, {'variant_id': '5-1295163-C-G', 'chrom': '5', 'pos': 1295163, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 433444}}, {'variant_id': '5-1295163-C-T', 'chrom': '5', 'pos': 1295163, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 433444}}, {'variant_id': '5-1295164-G-T', 'chrom': '5', 'pos': 1295164, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 434906}}, {'variant_id': '5-1295164-G-A', 'chrom': '5', 'pos': 1295164, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 434906}}, {'variant_id': '5-1295165-G-C', 'chrom': '5', 'pos': 1295165, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 435384}}, {'variant_id': '5-1295166-A-G', 'chrom': '5', 'pos': 1295166, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 435362}}, {'variant_id': '5-1295166-A-T', 'chrom': '5', 'pos': 1295166, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 0, 'an': 435362}}, {'variant_id': '5-1295167-G-A', 'chrom': '5', 'pos': 1295167, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 435958}}, {'variant_id': '5-1295167-G-T', 'chrom': '5', 'pos': 1295167, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 436066}}, {'variant_id': '5-1295168-G-A', 'chrom': '5', 'pos': 1295168, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 434888}}, {'variant_id': '5-1295168-G-T', 'chrom': '5', 'pos': 1295168, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 434888}}, {'variant_id': '5-1295169-A-T', 'chrom': '5', 'pos': 1295169, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 0, 'an': 433732}}, {'variant_id': '5-1295169-A-G', 'chrom': '5', 'pos': 1295169, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 433732}}, {'variant_id': '5-1295170-G-T', 'chrom': '5', 'pos': 1295170, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 433348}}, {'variant_id': '5-1295171-G-A', 'chrom': '5', 'pos': 1295171, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 433568}}, {'variant_id': '5-1295172-C-CCCAGTGCCT', 'chrom': '5', 'pos': 1295172, 'ref': 'C', 'alt': 'CCCAGTGCCT', 'joint': {'ac': 0, 'an': 432346}}, {'variant_id': '5-1295172-C-A', 'chrom': '5', 'pos': 1295172, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 432346}}, {'variant_id': '5-1295172-C-T', 'chrom': '5', 'pos': 1295172, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 432346}}, {'variant_id': '5-1295173-G-A', 'chrom': '5', 'pos': 1295173, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 429372}}, {'variant_id': '5-1295173-G-T', 'chrom': '5', 'pos': 1295173, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 429372}}, {'variant_id': '5-1295174-G-T', 'chrom': '5', 'pos': 1295174, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 429778}}, {'variant_id': '5-1295175-A-G', 'chrom': '5', 'pos': 1295175, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 429396}}, {'variant_id': '5-1295175-A-T', 'chrom': '5', 'pos': 1295175, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 0, 'an': 429396}}, {'variant_id': '5-1295176-G-GTGCGTGCC', 'chrom': '5', 'pos': 1295176, 'ref': 'G', 'alt': 'GTGCGTGCC', 'joint': {'ac': 0, 'an': 429074}}, {'variant_id': '5-1295176-G-T', 'chrom': '5', 'pos': 1295176, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 429074}}, {'variant_id': '5-1295176-G-A', 'chrom': '5', 'pos': 1295176, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 429074}}, {'variant_id': '5-1295177-C-T', 'chrom': '5', 'pos': 1295177, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 429026}}, {'variant_id': '5-1295177-C-A', 'chrom': '5', 'pos': 1295177, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 1, 'an': 428918}}, {'variant_id': '5-1295177-C-G', 'chrom': '5', 'pos': 1295177, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 1, 'an': 429026}}, {'variant_id': '5-1295178-T-G', 'chrom': '5', 'pos': 1295178, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 0, 'an': 427500}}, {'variant_id': '5-1295178-T-C', 'chrom': '5', 'pos': 1295178, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 0, 'an': 427500}}, {'variant_id': '5-1295179-G-A', 'chrom': '5', 'pos': 1295179, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 425450}}, {'variant_id': '5-1295179-G-T', 'chrom': '5', 'pos': 1295179, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 425450}}, {'variant_id': '5-1295180-G-T', 'chrom': '5', 'pos': 1295180, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 425674}}, {'variant_id': '5-1295181-A-G', 'chrom': '5', 'pos': 1295181, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 425236}}, {'variant_id': '5-1295181-A-AAGGTGAAGGGGCAGGAC', 'chrom': '5', 'pos': 1295181, 'ref': 'A', 'alt': 'AAGGTGAAGGGGCAGGAC', 'joint': {'ac': 1, 'an': 425128}}, {'variant_id': '5-1295182-A-G', 'chrom': '5', 'pos': 1295182, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 77, 'an': 424486}}, {'variant_id': '5-1295182-A-T', 'chrom': '5', 'pos': 1295182, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 0, 'an': 424486}}, {'variant_id': '5-1295183-G-T', 'chrom': '5', 'pos': 1295183, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 424690}}, {'variant_id': '5-1295183-G-A', 'chrom': '5', 'pos': 1295183, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 424690}}, {'variant_id': '5-1295184-G-A', 'chrom': '5', 'pos': 1295184, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 423502}}, {'variant_id': '5-1295184-G-T', 'chrom': '5', 'pos': 1295184, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 423502}}, {'variant_id': '5-1295185-T-G', 'chrom': '5', 'pos': 1295185, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 1, 'an': 420942}}, {'variant_id': '5-1295185-T-C', 'chrom': '5', 'pos': 1295185, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 1, 'an': 420836}}, {'variant_id': '5-1295185-T-A', 'chrom': '5', 'pos': 1295185, 'ref': 'T', 'alt': 'A', 'joint': {'ac': 0, 'an': 420942}}, {'variant_id': '5-1295186-G-A', 'chrom': '5', 'pos': 1295186, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 420936}}, {'variant_id': '5-1295186-G-C', 'chrom': '5', 'pos': 1295186, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 420936}}, {'variant_id': '5-1295186-G-T', 'chrom': '5', 'pos': 1295186, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 420936}}, {'variant_id': '5-1295187-A-C', 'chrom': '5', 'pos': 1295187, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 0, 'an': 420260}}, {'variant_id': '5-1295187-A-G', 'chrom': '5', 'pos': 1295187, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 1, 'an': 420260}}, {'variant_id': '5-1295188-A-G', 'chrom': '5', 'pos': 1295188, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 420026}}, {'variant_id': '5-1295188-A-T', 'chrom': '5', 'pos': 1295188, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 0, 'an': 420026}}, {'variant_id': '5-1295189-G-C', 'chrom': '5', 'pos': 1295189, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 412780}}, {'variant_id': '5-1295189-G-T', 'chrom': '5', 'pos': 1295189, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 412888}}, {'variant_id': '5-1295189-G-A', 'chrom': '5', 'pos': 1295189, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 412888}}, {'variant_id': '5-1295190-G-T', 'chrom': '5', 'pos': 1295190, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 413188}}, {'variant_id': '5-1295190-G-A', 'chrom': '5', 'pos': 1295190, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 413188}}, {'variant_id': '5-1295190-G-C', 'chrom': '5', 'pos': 1295190, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 25, 'an': 413082}}, {'variant_id': '5-1295191-G-T', 'chrom': '5', 'pos': 1295191, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 8, 'an': 413078}}, {'variant_id': '5-1295192-G-T', 'chrom': '5', 'pos': 1295192, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 413330}}, {'variant_id': '5-1295192-G-A', 'chrom': '5', 'pos': 1295192, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 413330}}, {'variant_id': '5-1295193-C-G', 'chrom': '5', 'pos': 1295193, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 412642}}, {'variant_id': '5-1295193-C-A', 'chrom': '5', 'pos': 1295193, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 412642}}, {'variant_id': '5-1295193-C-T', 'chrom': '5', 'pos': 1295193, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 11, 'an': 412534}}, {'variant_id': '5-1295194-A-G', 'chrom': '5', 'pos': 1295194, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 411746}}, {'variant_id': '5-1295196-G-T', 'chrom': '5', 'pos': 1295196, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 411328}}, {'variant_id': '5-1295196-G-A', 'chrom': '5', 'pos': 1295196, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 411328}}, {'variant_id': '5-1295197-A-ACGGGTGCC', 'chrom': '5', 'pos': 1295197, 'ref': 'A', 'alt': 'ACGGGTGCC', 'joint': {'ac': 2, 'an': 410224}}, {'variant_id': '5-1295197-A-G', 'chrom': '5', 'pos': 1295197, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 410224}}, {'variant_id': '5-1295197-A-C', 'chrom': '5', 'pos': 1295197, 'ref': 'A', 'alt': 'C', 'joint': {'ac': 0, 'an': 410224}}, {'variant_id': '5-1295198-C-A', 'chrom': '5', 'pos': 1295198, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 407688}}, {'variant_id': '5-1295198-CG-C', 'chrom': '5', 'pos': 1295198, 'ref': 'CG', 'alt': 'C', 'joint': {'ac': 1, 'an': 407688}}, {'variant_id': '5-1295198-C-G', 'chrom': '5', 'pos': 1295198, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 1, 'an': 407688}}, {'variant_id': '5-1295198-C-T', 'chrom': '5', 'pos': 1295198, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 3, 'an': 407582}}, {'variant_id': '5-1295199-G-C', 'chrom': '5', 'pos': 1295199, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 2, 'an': 408144}}, {'variant_id': '5-1295199-G-A', 'chrom': '5', 'pos': 1295199, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 408144}}, {'variant_id': '5-1295199-G-T', 'chrom': '5', 'pos': 1295199, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 3, 'an': 408144}}, {'variant_id': '5-1295200-G-A', 'chrom': '5', 'pos': 1295200, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 408158}}, {'variant_id': '5-1295201-G-A', 'chrom': '5', 'pos': 1295201, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 2, 'an': 408140}}, {'variant_id': '5-1295201-G-T', 'chrom': '5', 'pos': 1295201, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 34, 'an': 408140}}, {'variant_id': '5-1295201-G-C', 'chrom': '5', 'pos': 1295201, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 3, 'an': 408140}}, {'variant_id': '5-1295202-T-G', 'chrom': '5', 'pos': 1295202, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 0, 'an': 406058}}, {'variant_id': '5-1295203-G-GCCCGGGTC', 'chrom': '5', 'pos': 1295203, 'ref': 'G', 'alt': 'GCCCGGGTC', 'joint': {'ac': 1, 'an': 407132}}, {'variant_id': '5-1295203-GC-G', 'chrom': '5', 'pos': 1295203, 'ref': 'GC', 'alt': 'G', 'joint': {'ac': 0, 'an': 407132}}, {'variant_id': '5-1295203-G-T', 'chrom': '5', 'pos': 1295203, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 407024}}, {'variant_id': '5-1295203-G-C', 'chrom': '5', 'pos': 1295203, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 0, 'an': 407132}}, {'variant_id': '5-1295203-G-A', 'chrom': '5', 'pos': 1295203, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 407132}}, {'variant_id': '5-1295204-C-A', 'chrom': '5', 'pos': 1295204, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 406424}}, {'variant_id': '5-1295204-C-G', 'chrom': '5', 'pos': 1295204, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 406424}}, {'variant_id': '5-1295204-C-T', 'chrom': '5', 'pos': 1295204, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 2, 'an': 406316}}, {'variant_id': '5-1295205-C-T', 'chrom': '5', 'pos': 1295205, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 406526}}, {'variant_id': '5-1295205-C-G', 'chrom': '5', 'pos': 1295205, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 0, 'an': 406526}}, {'variant_id': '5-1295206-C-G', 'chrom': '5', 'pos': 1295206, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 2, 'an': 404446}}, {'variant_id': '5-1295206-C-A', 'chrom': '5', 'pos': 1295206, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 404446}}, {'variant_id': '5-1295206-C-T', 'chrom': '5', 'pos': 1295206, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 37, 'an': 404340}}, {'variant_id': '5-1295207-G-T', 'chrom': '5', 'pos': 1295207, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 405818}}, {'variant_id': '5-1295207-G-A', 'chrom': '5', 'pos': 1295207, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 405710}}, {'variant_id': '5-1295207-G-C', 'chrom': '5', 'pos': 1295207, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1635, 'an': 405818}}, {'variant_id': '5-1295208-G-C', 'chrom': '5', 'pos': 1295208, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 405658}}, {'variant_id': '5-1295208-G-A', 'chrom': '5', 'pos': 1295208, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 405658}}, {'variant_id': '5-1295209-G-T', 'chrom': '5', 'pos': 1295209, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 0, 'an': 404350}}, {'variant_id': '5-1295209-G-A', 'chrom': '5', 'pos': 1295209, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 0, 'an': 404350}}, {'variant_id': '5-1295209-G-C', 'chrom': '5', 'pos': 1295209, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 4, 'an': 404350}}, {'variant_id': '5-1295210-TC-T', 'chrom': '5', 'pos': 1295210, 'ref': 'TC', 'alt': 'T', 'joint': {'ac': 0, 'an': 402150}}, {'variant_id': '5-1295210-T-C', 'chrom': '5', 'pos': 1295210, 'ref': 'T', 'alt': 'C', 'joint': {'ac': 0, 'an': 402150}}, {'variant_id': '5-1295210-T-G', 'chrom': '5', 'pos': 1295210, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 1, 'an': 402150}}, {'variant_id': '5-1295210-T-TCCCCGCGCTGCACCAGCCGCCAGCCCTGGGG', 'chrom': '5', 'pos': 1295210, 'ref': 'T', 'alt': 'TCCCCGCGCTGCACCAGCCGCCAGCCCTGGGG', 'joint': {'ac': 0, 'an': 402150}}, {'variant_id': '5-1295211-C-T', 'chrom': '5', 'pos': 1295211, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 11, 'an': 404234}}, {'variant_id': '5-1295211-C-G', 'chrom': '5', 'pos': 1295211, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 2, 'an': 404342}}, {'variant_id': '5-1295211-C-A', 'chrom': '5', 'pos': 1295211, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 404342}}, {'variant_id': '5-1295212-C-T', 'chrom': '5', 'pos': 1295212, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 0, 'an': 404050}}, {'variant_id': '5-1295212-C-G', 'chrom': '5', 'pos': 1295212, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 2, 'an': 403942}}, {'variant_id': '5-1295213-C-T', 'chrom': '5', 'pos': 1295213, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 404098}}, {'variant_id': '5-1295214-C-A', 'chrom': '5', 'pos': 1295214, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 404098}}, {'variant_id': '5-1295215-A-G', 'chrom': '5', 'pos': 1295215, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 0, 'an': 403400}}, {'variant_id': '5-1295216-G-T', 'chrom': '5', 'pos': 1295216, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 3, 'an': 403862}}, {'variant_id': '5-1295217-T-A', 'chrom': '5', 'pos': 1295217, 'ref': 'T', 'alt': 'A', 'joint': {'ac': 0, 'an': 401448}}, {'variant_id': '5-1295217-T-G', 'chrom': '5', 'pos': 1295217, 'ref': 'T', 'alt': 'G', 'joint': {'ac': 0, 'an': 401448}}, {'variant_id': '5-1295217-TC-T', 'chrom': '5', 'pos': 1295217, 'ref': 'TC', 'alt': 'T', 'joint': {'ac': 0, 'an': 401448}}, {'variant_id': '5-1295218-C-T', 'chrom': '5', 'pos': 1295218, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 403532}}, {'variant_id': '5-1295218-C-A', 'chrom': '5', 'pos': 1295218, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 403532}}, {'variant_id': '5-1295219-C-A', 'chrom': '5', 'pos': 1295219, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 0, 'an': 403586}}, {'variant_id': '5-1295219-C-T', 'chrom': '5', 'pos': 1295219, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 3, 'an': 403478}}, {'variant_id': '5-1295225-C-T', 'chrom': '5', 'pos': 1295225, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 58, 'an': 152198}}, {'variant_id': '5-1295226-C-T', 'chrom': '5', 'pos': 1295226, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 152060}}, {'variant_id': '5-1295227-A-T', 'chrom': '5', 'pos': 1295227, 'ref': 'A', 'alt': 'T', 'joint': {'ac': 8, 'an': 151934}}, {'variant_id': '5-1295233-G-A', 'chrom': '5', 'pos': 1295233, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 1, 'an': 152098}}, {'variant_id': '5-1295234-A-G', 'chrom': '5', 'pos': 1295234, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 38593, 'an': 151924}}, {'variant_id': '5-1295244-C-T', 'chrom': '5', 'pos': 1295244, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 151840}}, {'variant_id': '5-1295249-C-T', 'chrom': '5', 'pos': 1295249, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 2, 'an': 151948}}, {'variant_id': '5-1295257-C-T', 'chrom': '5', 'pos': 1295257, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 152236}}, {'variant_id': '5-1295258-C-T', 'chrom': '5', 'pos': 1295258, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 3909, 'an': 152190}}, {'variant_id': '5-1295260-G-C', 'chrom': '5', 'pos': 1295260, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 152210}}, {'variant_id': '5-1295262-G-A', 'chrom': '5', 'pos': 1295262, 'ref': 'G', 'alt': 'A', 'joint': {'ac': 4, 'an': 152042}}, {'variant_id': '5-1295269-C-T', 'chrom': '5', 'pos': 1295269, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 3, 'an': 151842}}, {'variant_id': '5-1295275-G-T', 'chrom': '5', 'pos': 1295275, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 2, 'an': 152148}}, {'variant_id': '5-1295278-C-T', 'chrom': '5', 'pos': 1295278, 'ref': 'C', 'alt': 'T', 'joint': {'ac': 1, 'an': 151712}}, {'variant_id': '5-1295281-C-A', 'chrom': '5', 'pos': 1295281, 'ref': 'C', 'alt': 'A', 'joint': {'ac': 22, 'an': 152166}}, {'variant_id': '5-1295284-G-T', 'chrom': '5', 'pos': 1295284, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 152156}}, {'variant_id': '5-1295284-G-C', 'chrom': '5', 'pos': 1295284, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 21, 'an': 152264}}, {'variant_id': '5-1295285-G-T', 'chrom': '5', 'pos': 1295285, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 1, 'an': 151990}}, {'variant_id': '5-1295288-C-G', 'chrom': '5', 'pos': 1295288, 'ref': 'C', 'alt': 'G', 'joint': {'ac': 5, 'an': 152244}}, {'variant_id': '5-1295290-G-C', 'chrom': '5', 'pos': 1295290, 'ref': 'G', 'alt': 'C', 'joint': {'ac': 1, 'an': 152218}}, {'variant_id': '5-1295293-A-G', 'chrom': '5', 'pos': 1295293, 'ref': 'A', 'alt': 'G', 'joint': {'ac': 5, 'an': 151864}}, {'variant_id': '5-1295294-G-T', 'chrom': '5', 'pos': 1295294, 'ref': 'G', 'alt': 'T', 'joint': {'ac': 5, 'an': 152160}}]}}\n"
     ]
    }
   ],
   "source": [
    "from gql import gql, Client\n",
    "from gql.transport.aiohttp import AIOHTTPTransport\n",
    "\n",
    "transport = AIOHTTPTransport(url=\"https://gnomad.broadinstitute.org/api\")\n",
    "client = Client(transport=transport, fetch_schema_from_transport=True)\n",
    "\n",
    "query = gql(\n",
    "    \"\"\"\n",
    "      \n",
    "    query RegionQuery {\n",
    "\n",
    "      region(chrom: \"5\", start: 1294944, stop: 1295294, reference_genome: GRCh38) {\n",
    "        variants(dataset: gnomad_r4) {\n",
    "          variant_id\n",
    "          chrom\n",
    "          pos\n",
    "          ref\n",
    "          alt\n",
    "          joint {\n",
    "            ac\n",
    "            an\n",
    "          }\n",
    "        }\n",
    "      }\n",
    "    }\n",
    "    \"\"\"\n",
    ")\n",
    "result = await client.execute_async(query)\n",
    "print(result)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "20e34bb3",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "chrom",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pos",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "ref",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "alt",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ac",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "an",
         "rawType": "int64",
         "type": "integer"
        }
       ],
       "ref": "0033b56a-8a7c-43e1-b008-cd99d4a1bccf",
       "rows": [
        [
         "0",
         "5-1294944-T-C",
         "5",
         "1294944",
         "T",
         "C",
         "0",
         "1403334"
        ],
        [
         "1",
         "5-1294945-G-A",
         "5",
         "1294945",
         "G",
         "A",
         "22",
         "1401036"
        ],
        [
         "2",
         "5-1294945-G-T",
         "5",
         "1294945",
         "G",
         "T",
         "0",
         "1401144"
        ],
        [
         "3",
         "5-1294946-C-CGCA",
         "5",
         "1294946",
         "C",
         "CGCA",
         "1",
         "1400604"
        ],
        [
         "4",
         "5-1294946-C-T",
         "5",
         "1294946",
         "C",
         "T",
         "0",
         "1400602"
        ],
        [
         "5",
         "5-1294946-CGCAGCAGGGAGCGCACGGCTCG-C",
         "5",
         "1294946",
         "CGCAGCAGGGAGCGCACGGCTCG",
         "C",
         "1",
         "1400604"
        ],
        [
         "6",
         "5-1294946-C-A",
         "5",
         "1294946",
         "C",
         "A",
         "1",
         "1400602"
        ],
        [
         "7",
         "5-1294947-G-A",
         "5",
         "1294947",
         "G",
         "A",
         "2",
         "1393228"
        ],
        [
         "8",
         "5-1294947-G-C",
         "5",
         "1294947",
         "G",
         "C",
         "0",
         "1393230"
        ],
        [
         "9",
         "5-1294947-G-T",
         "5",
         "1294947",
         "G",
         "T",
         "0",
         "1393230"
        ],
        [
         "10",
         "5-1294948-C-A",
         "5",
         "1294948",
         "C",
         "A",
         "1",
         "1393494"
        ],
        [
         "11",
         "5-1294948-C-T",
         "5",
         "1294948",
         "C",
         "T",
         "0",
         "1393494"
        ],
        [
         "12",
         "5-1294949-A-G",
         "5",
         "1294949",
         "A",
         "G",
         "0",
         "1393516"
        ],
        [
         "13",
         "5-1294950-G-A",
         "5",
         "1294950",
         "G",
         "A",
         "0",
         "1390046"
        ],
        [
         "14",
         "5-1294950-G-T",
         "5",
         "1294950",
         "G",
         "T",
         "0",
         "1390046"
        ],
        [
         "15",
         "5-1294951-C-A",
         "5",
         "1294951",
         "C",
         "A",
         "2",
         "1389898"
        ],
        [
         "16",
         "5-1294951-CAGGG-C",
         "5",
         "1294951",
         "CAGGG",
         "C",
         "1",
         "1389900"
        ],
        [
         "17",
         "5-1294951-C-T",
         "5",
         "1294951",
         "C",
         "T",
         "2",
         "1389900"
        ],
        [
         "18",
         "5-1294952-A-G",
         "5",
         "1294952",
         "A",
         "G",
         "0",
         "1390276"
        ],
        [
         "19",
         "5-1294952-A-AGGGAGCGCACGGCTCGGCAGCG",
         "5",
         "1294952",
         "A",
         "AGGGAGCGCACGGCTCGGCAGCG",
         "1",
         "1390276"
        ],
        [
         "20",
         "5-1294952-A-C",
         "5",
         "1294952",
         "A",
         "C",
         "0",
         "1390276"
        ],
        [
         "21",
         "5-1294952-AGGGAGCGCACGGCTCGGCAGCG-A",
         "5",
         "1294952",
         "AGGGAGCGCACGGCTCGGCAGCG",
         "A",
         "0",
         "1390266"
        ],
        [
         "22",
         "5-1294953-G-A",
         "5",
         "1294953",
         "G",
         "A",
         "1",
         "1381780"
        ],
        [
         "23",
         "5-1294953-G-T",
         "5",
         "1294953",
         "G",
         "T",
         "0",
         "1381780"
        ],
        [
         "24",
         "5-1294954-G-A",
         "5",
         "1294954",
         "G",
         "A",
         "2",
         "1381336"
        ],
        [
         "25",
         "5-1294954-G-C",
         "5",
         "1294954",
         "G",
         "C",
         "0",
         "1381334"
        ],
        [
         "26",
         "5-1294954-G-T",
         "5",
         "1294954",
         "G",
         "T",
         "2",
         "1381336"
        ],
        [
         "27",
         "5-1294955-G-A",
         "5",
         "1294955",
         "G",
         "A",
         "0",
         "1381738"
        ],
        [
         "28",
         "5-1294955-G-T",
         "5",
         "1294955",
         "G",
         "T",
         "0",
         "1381738"
        ],
        [
         "29",
         "5-1294956-A-G",
         "5",
         "1294956",
         "A",
         "G",
         "0",
         "1376688"
        ],
        [
         "30",
         "5-1294957-G-GCTCCC",
         "5",
         "1294957",
         "G",
         "GCTCCC",
         "1",
         "1376536"
        ],
        [
         "31",
         "5-1294957-G-T",
         "5",
         "1294957",
         "G",
         "T",
         "0",
         "1376536"
        ],
        [
         "32",
         "5-1294957-G-A",
         "5",
         "1294957",
         "G",
         "A",
         "1",
         "1376536"
        ],
        [
         "33",
         "5-1294958-C-A",
         "5",
         "1294958",
         "C",
         "A",
         "0",
         "1371810"
        ],
        [
         "34",
         "5-1294958-C-T",
         "5",
         "1294958",
         "C",
         "T",
         "2",
         "1371812"
        ],
        [
         "35",
         "5-1294958-C-G",
         "5",
         "1294958",
         "C",
         "G",
         "0",
         "1371812"
        ],
        [
         "36",
         "5-1294959-G-C",
         "5",
         "1294959",
         "G",
         "C",
         "0",
         "1367474"
        ],
        [
         "37",
         "5-1294959-G-A",
         "5",
         "1294959",
         "G",
         "A",
         "0",
         "1367472"
        ],
        [
         "38",
         "5-1294959-G-T",
         "5",
         "1294959",
         "G",
         "T",
         "0",
         "1367474"
        ],
        [
         "39",
         "5-1294960-C-A",
         "5",
         "1294960",
         "C",
         "A",
         "1",
         "1367802"
        ],
        [
         "40",
         "5-1294960-C-T",
         "5",
         "1294960",
         "C",
         "T",
         "1",
         "1367802"
        ],
        [
         "41",
         "5-1294961-A-G",
         "5",
         "1294961",
         "A",
         "G",
         "0",
         "1367812"
        ],
        [
         "42",
         "5-1294962-C-A",
         "5",
         "1294962",
         "C",
         "A",
         "0",
         "1366950"
        ],
        [
         "43",
         "5-1294962-C-T",
         "5",
         "1294962",
         "C",
         "T",
         "0",
         "1366948"
        ],
        [
         "44",
         "5-1294962-C-G",
         "5",
         "1294962",
         "C",
         "G",
         "1",
         "1366950"
        ],
        [
         "45",
         "5-1294963-G-T",
         "5",
         "1294963",
         "G",
         "T",
         "17",
         "1350022"
        ],
        [
         "46",
         "5-1294963-G-C",
         "5",
         "1294963",
         "G",
         "C",
         "11",
         "1349914"
        ],
        [
         "47",
         "5-1294963-G-A",
         "5",
         "1294963",
         "G",
         "A",
         "1",
         "1349914"
        ],
        [
         "48",
         "5-1294964-G-T",
         "5",
         "1294964",
         "G",
         "T",
         "0",
         "1349032"
        ],
        [
         "49",
         "5-1294964-G-A",
         "5",
         "1294964",
         "G",
         "A",
         "0",
         "1349032"
        ]
       ],
       "shape": {
        "columns": 7,
        "rows": 769
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>chrom</th>\n",
       "      <th>pos</th>\n",
       "      <th>ref</th>\n",
       "      <th>alt</th>\n",
       "      <th>ac</th>\n",
       "      <th>an</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>5-1294944-T-C</td>\n",
       "      <td>5</td>\n",
       "      <td>1294944</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>0</td>\n",
       "      <td>1403334</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5-1294945-G-A</td>\n",
       "      <td>5</td>\n",
       "      <td>1294945</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>22</td>\n",
       "      <td>1401036</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>5-1294945-G-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1294945</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>0</td>\n",
       "      <td>1401144</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5-1294946-C-CGCA</td>\n",
       "      <td>5</td>\n",
       "      <td>1294946</td>\n",
       "      <td>C</td>\n",
       "      <td>CGCA</td>\n",
       "      <td>1</td>\n",
       "      <td>1400604</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5-1294946-C-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1294946</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>0</td>\n",
       "      <td>1400602</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>764</th>\n",
       "      <td>5-1295285-G-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1295285</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>1</td>\n",
       "      <td>151990</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>765</th>\n",
       "      <td>5-1295288-C-G</td>\n",
       "      <td>5</td>\n",
       "      <td>1295288</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>5</td>\n",
       "      <td>152244</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>766</th>\n",
       "      <td>5-1295290-G-C</td>\n",
       "      <td>5</td>\n",
       "      <td>1295290</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>1</td>\n",
       "      <td>152218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>767</th>\n",
       "      <td>5-1295293-A-G</td>\n",
       "      <td>5</td>\n",
       "      <td>1295293</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>5</td>\n",
       "      <td>151864</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>768</th>\n",
       "      <td>5-1295294-G-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1295294</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>5</td>\n",
       "      <td>152160</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>769 rows √ó 7 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "           variant_id chrom      pos ref   alt  ac       an\n",
       "0       5-1294944-T-C     5  1294944   T     C   0  1403334\n",
       "1       5-1294945-G-A     5  1294945   G     A  22  1401036\n",
       "2       5-1294945-G-T     5  1294945   G     T   0  1401144\n",
       "3    5-1294946-C-CGCA     5  1294946   C  CGCA   1  1400604\n",
       "4       5-1294946-C-T     5  1294946   C     T   0  1400602\n",
       "..                ...   ...      ...  ..   ...  ..      ...\n",
       "764     5-1295285-G-T     5  1295285   G     T   1   151990\n",
       "765     5-1295288-C-G     5  1295288   C     G   5   152244\n",
       "766     5-1295290-G-C     5  1295290   G     C   1   152218\n",
       "767     5-1295293-A-G     5  1295293   A     G   5   151864\n",
       "768     5-1295294-G-T     5  1295294   G     T   5   152160\n",
       "\n",
       "[769 rows x 7 columns]"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "import pandas as pd\n",
    "\n",
    "# Extract variants data from the result dictionary\n",
    "variants_data = result['region']['variants']\n",
    "\n",
    "# Create a pandas DataFrame\n",
    "df_gnomad_variants = pd.DataFrame(variants_data)\n",
    "\n",
    "# Expand the 'joint' column into separate 'ac' and 'an' columns\n",
    "df_gnomad_variants['ac'] = df_gnomad_variants['joint'].apply(lambda x: x['ac'])\n",
    "df_gnomad_variants['an'] = df_gnomad_variants['joint'].apply(lambda x: x['an'])\n",
    "\n",
    "# Drop the original 'joint' column\n",
    "df_gnomad_variants = df_gnomad_variants.drop(columns=['joint'])\n",
    "\n",
    "# Display the DataFrame\n",
    "df_gnomad_variants"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b0d8c7a3",
   "metadata": {},
   "source": [
    "### ‚ùì Question\n",
    "\n",
    "> **üí° How many variants did you find?**\n",
    ">\n",
    "> **üí° What is the most common variant?**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "a2b33675",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "chrom",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pos",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "ref",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "alt",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ac",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "an",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "af",
         "rawType": "float64",
         "type": "float"
        }
       ],
       "ref": "a8d817b2-09cd-43b9-b7bc-b33553ec0bbb",
       "rows": [
        [
         "751",
         "5-1295234-A-G",
         "5",
         "1295234",
         "A",
         "G",
         "38593",
         "151924",
         "0.2540283299544509"
        ]
       ],
       "shape": {
        "columns": 8,
        "rows": 1
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>chrom</th>\n",
       "      <th>pos</th>\n",
       "      <th>ref</th>\n",
       "      <th>alt</th>\n",
       "      <th>ac</th>\n",
       "      <th>an</th>\n",
       "      <th>af</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>751</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>5</td>\n",
       "      <td>1295234</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>38593</td>\n",
       "      <td>151924</td>\n",
       "      <td>0.254028</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "        variant_id chrom      pos ref alt     ac      an        af\n",
       "751  5-1295234-A-G     5  1295234   A   G  38593  151924  0.254028"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Calculate allele frequency\n",
    "df_gnomad_variants[\"af\"] = df_gnomad_variants[\"ac\"] / df_gnomad_variants[\"an\"]\n",
    "\n",
    "# Find the most common variant\n",
    "df_gnomad_variants.sort_values(by=\"af\", ascending=False).head(1)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ab10b785",
   "metadata": {},
   "source": [
    "### üè• Clinically annotated variants from ClinVar\n",
    "\n",
    "ClinVar usually provides downloadable files for bulk access to their data. üì• Usually, people use these files. ClinVar has an API for submission of new data and also for retrieval. However, ClinVar data is also available through the gnomAD API (maybe not the most recent one). But we already used gnomAD successfully, so we'll continue here with gnomAD to avoid learning any new API. üéì\n",
    "\n",
    "ClinVar data is available through the `clinvar_variant` field in the gnomAD API. You can query by variant ID or search within a region. üîé"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "e17ee32e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "ClinVar Region Query Result:\n",
      "{'region': {'clinvar_variants': [{'variant_id': '5-1294944-T-C', 'chrom': '5', 'pos': 1294944, 'ref': 'T', 'alt': 'C', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294945-G-A', 'chrom': '5', 'pos': 1294945, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294946-C-T', 'chrom': '5', 'pos': 1294946, 'ref': 'C', 'alt': 'T', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294950-G-A', 'chrom': '5', 'pos': 1294950, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294950-G-T', 'chrom': '5', 'pos': 1294950, 'ref': 'G', 'alt': 'T', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294951-C-G', 'chrom': '5', 'pos': 1294951, 'ref': 'C', 'alt': 'G', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294951-C-T', 'chrom': '5', 'pos': 1294951, 'ref': 'C', 'alt': 'T', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294951-C-A', 'chrom': '5', 'pos': 1294951, 'ref': 'C', 'alt': 'A', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294952-AGGGAGCGCACGGCTCGGCAGCG-A', 'chrom': '5', 'pos': 1294952, 'ref': 'AGGGAGCGCACGGCTCGGCAGCG', 'alt': 'A', 'clinical_significance': 'Pathogenic'}, {'variant_id': '5-1294953-G-A', 'chrom': '5', 'pos': 1294953, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294953-G-C', 'chrom': '5', 'pos': 1294953, 'ref': 'G', 'alt': 'C', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294954-G-A', 'chrom': '5', 'pos': 1294954, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294954-G-T', 'chrom': '5', 'pos': 1294954, 'ref': 'G', 'alt': 'T', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294956-A-G', 'chrom': '5', 'pos': 1294956, 'ref': 'A', 'alt': 'G', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294957-G-A', 'chrom': '5', 'pos': 1294957, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294957-G-T', 'chrom': '5', 'pos': 1294957, 'ref': 'G', 'alt': 'T', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294958-C-G', 'chrom': '5', 'pos': 1294958, 'ref': 'C', 'alt': 'G', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294963-G-A', 'chrom': '5', 'pos': 1294963, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294963-G-C', 'chrom': '5', 'pos': 1294963, 'ref': 'G', 'alt': 'C', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294963-G-T', 'chrom': '5', 'pos': 1294963, 'ref': 'G', 'alt': 'T', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294964-G-A', 'chrom': '5', 'pos': 1294964, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294966-T-C', 'chrom': '5', 'pos': 1294966, 'ref': 'T', 'alt': 'C', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294967-C-T', 'chrom': '5', 'pos': 1294967, 'ref': 'C', 'alt': 'T', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294967-C-A', 'chrom': '5', 'pos': 1294967, 'ref': 'C', 'alt': 'A', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294968-G-C', 'chrom': '5', 'pos': 1294968, 'ref': 'G', 'alt': 'C', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294968-G-T', 'chrom': '5', 'pos': 1294968, 'ref': 'G', 'alt': 'T', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294969-G-A', 'chrom': '5', 'pos': 1294969, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294970-C-T', 'chrom': '5', 'pos': 1294970, 'ref': 'C', 'alt': 'T', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294974-G-C', 'chrom': '5', 'pos': 1294974, 'ref': 'G', 'alt': 'C', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294977-G-T', 'chrom': '5', 'pos': 1294977, 'ref': 'G', 'alt': 'T', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294978-A-G', 'chrom': '5', 'pos': 1294978, 'ref': 'A', 'alt': 'G', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294978-A-AGC', 'chrom': '5', 'pos': 1294978, 'ref': 'A', 'alt': 'AGC', 'clinical_significance': 'Pathogenic'}, {'variant_id': '5-1294980-C-G', 'chrom': '5', 'pos': 1294980, 'ref': 'C', 'alt': 'G', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294981-G-C', 'chrom': '5', 'pos': 1294981, 'ref': 'G', 'alt': 'C', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294981-G-T', 'chrom': '5', 'pos': 1294981, 'ref': 'G', 'alt': 'T', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294981-G-A', 'chrom': '5', 'pos': 1294981, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294982-C-G', 'chrom': '5', 'pos': 1294982, 'ref': 'C', 'alt': 'G', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294984-C-T', 'chrom': '5', 'pos': 1294984, 'ref': 'C', 'alt': 'T', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294984-C-A', 'chrom': '5', 'pos': 1294984, 'ref': 'C', 'alt': 'A', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294984-C-G', 'chrom': '5', 'pos': 1294984, 'ref': 'C', 'alt': 'G', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294985-G-A', 'chrom': '5', 'pos': 1294985, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294985-G-C', 'chrom': '5', 'pos': 1294985, 'ref': 'G', 'alt': 'C', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294986-G-T', 'chrom': '5', 'pos': 1294986, 'ref': 'G', 'alt': 'T', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294986-G-C', 'chrom': '5', 'pos': 1294986, 'ref': 'G', 'alt': 'C', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294991-G-A', 'chrom': '5', 'pos': 1294991, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294992-C-T', 'chrom': '5', 'pos': 1294992, 'ref': 'C', 'alt': 'T', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294992-C-CG', 'chrom': '5', 'pos': 1294992, 'ref': 'C', 'alt': 'CG', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294994-G-C', 'chrom': '5', 'pos': 1294994, 'ref': 'G', 'alt': 'C', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1294994-G-T', 'chrom': '5', 'pos': 1294994, 'ref': 'G', 'alt': 'T', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1294995-G-T', 'chrom': '5', 'pos': 1294995, 'ref': 'G', 'alt': 'T', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1295020-C-G', 'chrom': '5', 'pos': 1295020, 'ref': 'C', 'alt': 'G', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295034-C-A', 'chrom': '5', 'pos': 1295034, 'ref': 'C', 'alt': 'A', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295045-CT-C', 'chrom': '5', 'pos': 1295045, 'ref': 'CT', 'alt': 'C', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295046-T-G', 'chrom': '5', 'pos': 1295046, 'ref': 'T', 'alt': 'G', 'clinical_significance': 'Conflicting classifications of pathogenicity'}, {'variant_id': '5-1295046-T-C', 'chrom': '5', 'pos': 1295046, 'ref': 'T', 'alt': 'C', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295046-T-A', 'chrom': '5', 'pos': 1295046, 'ref': 'T', 'alt': 'A', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295060-C-G', 'chrom': '5', 'pos': 1295060, 'ref': 'C', 'alt': 'G', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295071-CG-C', 'chrom': '5', 'pos': 1295071, 'ref': 'CG', 'alt': 'C', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295075-G-A', 'chrom': '5', 'pos': 1295075, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295078-G-A', 'chrom': '5', 'pos': 1295078, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295112-A-T', 'chrom': '5', 'pos': 1295112, 'ref': 'A', 'alt': 'T', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295113-G-A', 'chrom': '5', 'pos': 1295113, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Likely pathogenic'}, {'variant_id': '5-1295128-G-A', 'chrom': '5', 'pos': 1295128, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295135-G-A', 'chrom': '5', 'pos': 1295135, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Pathogenic'}, {'variant_id': '5-1295136-A-G', 'chrom': '5', 'pos': 1295136, 'ref': 'A', 'alt': 'G', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295154-G-C', 'chrom': '5', 'pos': 1295154, 'ref': 'G', 'alt': 'C', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1295162-G-A', 'chrom': '5', 'pos': 1295162, 'ref': 'G', 'alt': 'A', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295163-C-G', 'chrom': '5', 'pos': 1295163, 'ref': 'C', 'alt': 'G', 'clinical_significance': 'Likely benign'}, {'variant_id': '5-1295181-A-G', 'chrom': '5', 'pos': 1295181, 'ref': 'A', 'alt': 'G', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295182-A-G', 'chrom': '5', 'pos': 1295182, 'ref': 'A', 'alt': 'G', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295207-G-C', 'chrom': '5', 'pos': 1295207, 'ref': 'G', 'alt': 'C', 'clinical_significance': 'Benign/Likely benign'}, {'variant_id': '5-1295211-C-T', 'chrom': '5', 'pos': 1295211, 'ref': 'C', 'alt': 'T', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295225-C-T', 'chrom': '5', 'pos': 1295225, 'ref': 'C', 'alt': 'T', 'clinical_significance': 'Uncertain significance'}, {'variant_id': '5-1295234-A-G', 'chrom': '5', 'pos': 1295234, 'ref': 'A', 'alt': 'G', 'clinical_significance': 'Benign'}, {'variant_id': '5-1295258-C-T', 'chrom': '5', 'pos': 1295258, 'ref': 'C', 'alt': 'T', 'clinical_significance': 'Benign'}]}}\n"
     ]
    }
   ],
   "source": [
    "# Query ClinVar variants within a region\n",
    "clinvar_region_query = gql(\n",
    "    \"\"\"\n",
    "    query ClinVarRegionQuery {\n",
    "      region(chrom: \"5\", start: 1294944, stop: 1295294, reference_genome: GRCh38) {\n",
    "        clinvar_variants {\n",
    "          variant_id\n",
    "          chrom\n",
    "          pos\n",
    "          ref\n",
    "          alt\n",
    "          clinical_significance\n",
    "        }\n",
    "      }\n",
    "    }\n",
    "    \"\"\"\n",
    ")\n",
    "\n",
    "clinvar_region_result = await client.execute_async(clinvar_region_query)\n",
    "print(\"\\nClinVar Region Query Result:\")\n",
    "print(clinvar_region_result)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "9748049b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "chrom",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pos",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "ref",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "alt",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "clinical_significance",
         "rawType": "object",
         "type": "string"
        }
       ],
       "ref": "808b3c12-606b-4487-9b7b-68c5e6f5eb33",
       "rows": [
        [
         "0",
         "5-1294944-T-C",
         "5",
         "1294944",
         "T",
         "C",
         "Uncertain significance"
        ],
        [
         "1",
         "5-1294945-G-A",
         "5",
         "1294945",
         "G",
         "A",
         "Likely benign"
        ],
        [
         "2",
         "5-1294946-C-T",
         "5",
         "1294946",
         "C",
         "T",
         "Uncertain significance"
        ],
        [
         "3",
         "5-1294950-G-A",
         "5",
         "1294950",
         "G",
         "A",
         "Likely benign"
        ],
        [
         "4",
         "5-1294950-G-T",
         "5",
         "1294950",
         "G",
         "T",
         "Uncertain significance"
        ],
        [
         "5",
         "5-1294951-C-G",
         "5",
         "1294951",
         "C",
         "G",
         "Likely benign"
        ],
        [
         "6",
         "5-1294951-C-T",
         "5",
         "1294951",
         "C",
         "T",
         "Likely benign"
        ],
        [
         "7",
         "5-1294951-C-A",
         "5",
         "1294951",
         "C",
         "A",
         "Likely benign"
        ],
        [
         "8",
         "5-1294952-AGGGAGCGCACGGCTCGGCAGCG-A",
         "5",
         "1294952",
         "AGGGAGCGCACGGCTCGGCAGCG",
         "A",
         "Pathogenic"
        ],
        [
         "9",
         "5-1294953-G-A",
         "5",
         "1294953",
         "G",
         "A",
         "Likely benign"
        ],
        [
         "10",
         "5-1294953-G-C",
         "5",
         "1294953",
         "G",
         "C",
         "Uncertain significance"
        ],
        [
         "11",
         "5-1294954-G-A",
         "5",
         "1294954",
         "G",
         "A",
         "Likely benign"
        ],
        [
         "12",
         "5-1294954-G-T",
         "5",
         "1294954",
         "G",
         "T",
         "Likely benign"
        ],
        [
         "13",
         "5-1294956-A-G",
         "5",
         "1294956",
         "A",
         "G",
         "Uncertain significance"
        ],
        [
         "14",
         "5-1294957-G-A",
         "5",
         "1294957",
         "G",
         "A",
         "Likely benign"
        ],
        [
         "15",
         "5-1294957-G-T",
         "5",
         "1294957",
         "G",
         "T",
         "Likely benign"
        ],
        [
         "16",
         "5-1294958-C-G",
         "5",
         "1294958",
         "C",
         "G",
         "Uncertain significance"
        ],
        [
         "17",
         "5-1294963-G-A",
         "5",
         "1294963",
         "G",
         "A",
         "Likely benign"
        ],
        [
         "18",
         "5-1294963-G-C",
         "5",
         "1294963",
         "G",
         "C",
         "Likely benign"
        ],
        [
         "19",
         "5-1294963-G-T",
         "5",
         "1294963",
         "G",
         "T",
         "Likely benign"
        ],
        [
         "20",
         "5-1294964-G-A",
         "5",
         "1294964",
         "G",
         "A",
         "Uncertain significance"
        ],
        [
         "21",
         "5-1294966-T-C",
         "5",
         "1294966",
         "T",
         "C",
         "Likely benign"
        ],
        [
         "22",
         "5-1294967-C-T",
         "5",
         "1294967",
         "C",
         "T",
         "Uncertain significance"
        ],
        [
         "23",
         "5-1294967-C-A",
         "5",
         "1294967",
         "C",
         "A",
         "Uncertain significance"
        ],
        [
         "24",
         "5-1294968-G-C",
         "5",
         "1294968",
         "G",
         "C",
         "Uncertain significance"
        ],
        [
         "25",
         "5-1294968-G-T",
         "5",
         "1294968",
         "G",
         "T",
         "Likely benign"
        ],
        [
         "26",
         "5-1294969-G-A",
         "5",
         "1294969",
         "G",
         "A",
         "Likely benign"
        ],
        [
         "27",
         "5-1294970-C-T",
         "5",
         "1294970",
         "C",
         "T",
         "Uncertain significance"
        ],
        [
         "28",
         "5-1294974-G-C",
         "5",
         "1294974",
         "G",
         "C",
         "Uncertain significance"
        ],
        [
         "29",
         "5-1294977-G-T",
         "5",
         "1294977",
         "G",
         "T",
         "Uncertain significance"
        ],
        [
         "30",
         "5-1294978-A-G",
         "5",
         "1294978",
         "A",
         "G",
         "Likely benign"
        ],
        [
         "31",
         "5-1294978-A-AGC",
         "5",
         "1294978",
         "A",
         "AGC",
         "Pathogenic"
        ],
        [
         "32",
         "5-1294980-C-G",
         "5",
         "1294980",
         "C",
         "G",
         "Uncertain significance"
        ],
        [
         "33",
         "5-1294981-G-C",
         "5",
         "1294981",
         "G",
         "C",
         "Likely benign"
        ],
        [
         "34",
         "5-1294981-G-T",
         "5",
         "1294981",
         "G",
         "T",
         "Likely benign"
        ],
        [
         "35",
         "5-1294981-G-A",
         "5",
         "1294981",
         "G",
         "A",
         "Likely benign"
        ],
        [
         "36",
         "5-1294982-C-G",
         "5",
         "1294982",
         "C",
         "G",
         "Uncertain significance"
        ],
        [
         "37",
         "5-1294984-C-T",
         "5",
         "1294984",
         "C",
         "T",
         "Likely benign"
        ],
        [
         "38",
         "5-1294984-C-A",
         "5",
         "1294984",
         "C",
         "A",
         "Likely benign"
        ],
        [
         "39",
         "5-1294984-C-G",
         "5",
         "1294984",
         "C",
         "G",
         "Likely benign"
        ],
        [
         "40",
         "5-1294985-G-A",
         "5",
         "1294985",
         "G",
         "A",
         "Uncertain significance"
        ],
        [
         "41",
         "5-1294985-G-C",
         "5",
         "1294985",
         "G",
         "C",
         "Uncertain significance"
        ],
        [
         "42",
         "5-1294986-G-T",
         "5",
         "1294986",
         "G",
         "T",
         "Uncertain significance"
        ],
        [
         "43",
         "5-1294986-G-C",
         "5",
         "1294986",
         "G",
         "C",
         "Uncertain significance"
        ],
        [
         "44",
         "5-1294991-G-A",
         "5",
         "1294991",
         "G",
         "A",
         "Uncertain significance"
        ],
        [
         "45",
         "5-1294992-C-T",
         "5",
         "1294992",
         "C",
         "T",
         "Likely benign"
        ],
        [
         "46",
         "5-1294992-C-CG",
         "5",
         "1294992",
         "C",
         "CG",
         "Likely benign"
        ],
        [
         "47",
         "5-1294994-G-C",
         "5",
         "1294994",
         "G",
         "C",
         "Uncertain significance"
        ],
        [
         "48",
         "5-1294994-G-T",
         "5",
         "1294994",
         "G",
         "T",
         "Likely benign"
        ],
        [
         "49",
         "5-1294995-G-T",
         "5",
         "1294995",
         "G",
         "T",
         "Likely benign"
        ]
       ],
       "shape": {
        "columns": 6,
        "rows": 75
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>chrom</th>\n",
       "      <th>pos</th>\n",
       "      <th>ref</th>\n",
       "      <th>alt</th>\n",
       "      <th>clinical_significance</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>5-1294944-T-C</td>\n",
       "      <td>5</td>\n",
       "      <td>1294944</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5-1294945-G-A</td>\n",
       "      <td>5</td>\n",
       "      <td>1294945</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>Likely benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>5-1294946-C-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1294946</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5-1294950-G-A</td>\n",
       "      <td>5</td>\n",
       "      <td>1294950</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>Likely benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5-1294950-G-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1294950</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>5-1295207-G-C</td>\n",
       "      <td>5</td>\n",
       "      <td>1295207</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>Benign/Likely benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>5-1295211-C-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1295211</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>5-1295225-C-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1295225</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>5</td>\n",
       "      <td>1295234</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>Benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>5-1295258-C-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1295258</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>Benign</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>75 rows √ó 6 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "       variant_id chrom      pos ref alt   clinical_significance\n",
       "0   5-1294944-T-C     5  1294944   T   C  Uncertain significance\n",
       "1   5-1294945-G-A     5  1294945   G   A           Likely benign\n",
       "2   5-1294946-C-T     5  1294946   C   T  Uncertain significance\n",
       "3   5-1294950-G-A     5  1294950   G   A           Likely benign\n",
       "4   5-1294950-G-T     5  1294950   G   T  Uncertain significance\n",
       "..            ...   ...      ...  ..  ..                     ...\n",
       "70  5-1295207-G-C     5  1295207   G   C    Benign/Likely benign\n",
       "71  5-1295211-C-T     5  1295211   C   T  Uncertain significance\n",
       "72  5-1295225-C-T     5  1295225   C   T  Uncertain significance\n",
       "73  5-1295234-A-G     5  1295234   A   G                  Benign\n",
       "74  5-1295258-C-T     5  1295258   C   T                  Benign\n",
       "\n",
       "[75 rows x 6 columns]"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "\n",
    "# Extract variants data from the result dictionary\n",
    "clinvar_variants_data = clinvar_region_result['region']['clinvar_variants']\n",
    "\n",
    "# Create a pandas DataFrame\n",
    "df_clinvar_variants = pd.DataFrame(clinvar_variants_data)\n",
    "\n",
    "# Display the DataFrame\n",
    "df_clinvar_variants"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9fcd7dcb",
   "metadata": {},
   "source": [
    "### ‚ùì Question\n",
    "\n",
    "> **üí° How many ClinVar variants did you find?**\n",
    ">\n",
    "> **üí° Are all of them pathogenic? What is their clinical significance?**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "bd1fea12",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "clinical_significance",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "count",
         "rawType": "int64",
         "type": "integer"
        }
       ],
       "ref": "5fdcbee6-fd17-4c9f-8f4c-90abbb31482a",
       "rows": [
        [
         "Uncertain significance",
         "38"
        ],
        [
         "Likely benign",
         "29"
        ],
        [
         "Pathogenic",
         "3"
        ],
        [
         "Benign",
         "2"
        ],
        [
         "Conflicting classifications of pathogenicity",
         "1"
        ],
        [
         "Likely pathogenic",
         "1"
        ],
        [
         "Benign/Likely benign",
         "1"
        ]
       ],
       "shape": {
        "columns": 1,
        "rows": 7
       }
      },
      "text/plain": [
       "clinical_significance\n",
       "Uncertain significance                          38\n",
       "Likely benign                                   29\n",
       "Pathogenic                                       3\n",
       "Benign                                           2\n",
       "Conflicting classifications of pathogenicity     1\n",
       "Likely pathogenic                                1\n",
       "Benign/Likely benign                             1\n",
       "Name: count, dtype: int64"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_clinvar_variants['clinical_significance'].value_counts()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1c213d81",
   "metadata": {},
   "source": [
    "### üîó Annotate population variants with ClinVar data"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3db2ab14",
   "metadata": {},
   "source": [
    "Merge gnomAD variants with ClinVar variants to find clinical significance of variants in the region. üß©"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "dd4b6245",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "chrom",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pos",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "ref",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "alt",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ac",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "an",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "af",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "clinical_significance",
         "rawType": "object",
         "type": "unknown"
        }
       ],
       "ref": "345a9a9b-801c-42f9-a291-cb8d352dee5c",
       "rows": [
        [
         "0",
         "5-1294944-T-C",
         "5",
         "1294944",
         "T",
         "C",
         "0",
         "1403334",
         "0.0",
         "Uncertain significance"
        ],
        [
         "1",
         "5-1294945-G-A",
         "5",
         "1294945",
         "G",
         "A",
         "22",
         "1401036",
         "1.5702665741636904e-05",
         "Likely benign"
        ],
        [
         "2",
         "5-1294945-G-T",
         "5",
         "1294945",
         "G",
         "T",
         "0",
         "1401144",
         "0.0",
         null
        ],
        [
         "3",
         "5-1294946-C-CGCA",
         "5",
         "1294946",
         "C",
         "CGCA",
         "1",
         "1400604",
         "7.139776839135116e-07",
         null
        ],
        [
         "4",
         "5-1294946-C-T",
         "5",
         "1294946",
         "C",
         "T",
         "0",
         "1400602",
         "0.0",
         "Uncertain significance"
        ],
        [
         "5",
         "5-1294946-CGCAGCAGGGAGCGCACGGCTCG-C",
         "5",
         "1294946",
         "CGCAGCAGGGAGCGCACGGCTCG",
         "C",
         "1",
         "1400604",
         "7.139776839135116e-07",
         null
        ],
        [
         "6",
         "5-1294946-C-A",
         "5",
         "1294946",
         "C",
         "A",
         "1",
         "1400602",
         "7.139787034432337e-07",
         null
        ],
        [
         "7",
         "5-1294947-G-A",
         "5",
         "1294947",
         "G",
         "A",
         "2",
         "1393228",
         "1.4355152207678858e-06",
         null
        ],
        [
         "8",
         "5-1294947-G-C",
         "5",
         "1294947",
         "G",
         "C",
         "0",
         "1393230",
         "0.0",
         null
        ],
        [
         "9",
         "5-1294947-G-T",
         "5",
         "1294947",
         "G",
         "T",
         "0",
         "1393230",
         "0.0",
         null
        ],
        [
         "10",
         "5-1294948-C-A",
         "5",
         "1294948",
         "C",
         "A",
         "1",
         "1393494",
         "7.176205997298876e-07",
         null
        ],
        [
         "11",
         "5-1294948-C-T",
         "5",
         "1294948",
         "C",
         "T",
         "0",
         "1393494",
         "0.0",
         null
        ],
        [
         "12",
         "5-1294949-A-G",
         "5",
         "1294949",
         "A",
         "G",
         "0",
         "1393516",
         "0.0",
         null
        ],
        [
         "13",
         "5-1294950-G-A",
         "5",
         "1294950",
         "G",
         "A",
         "0",
         "1390046",
         "0.0",
         "Likely benign"
        ],
        [
         "14",
         "5-1294950-G-T",
         "5",
         "1294950",
         "G",
         "T",
         "0",
         "1390046",
         "0.0",
         "Uncertain significance"
        ],
        [
         "15",
         "5-1294951-C-A",
         "5",
         "1294951",
         "C",
         "A",
         "2",
         "1389898",
         "1.438954513208883e-06",
         "Likely benign"
        ],
        [
         "16",
         "5-1294951-CAGGG-C",
         "5",
         "1294951",
         "CAGGG",
         "C",
         "1",
         "1389900",
         "7.194762213108856e-07",
         null
        ],
        [
         "17",
         "5-1294951-C-T",
         "5",
         "1294951",
         "C",
         "T",
         "2",
         "1389900",
         "1.4389524426217713e-06",
         "Likely benign"
        ],
        [
         "18",
         "5-1294952-A-G",
         "5",
         "1294952",
         "A",
         "G",
         "0",
         "1390276",
         "0.0",
         null
        ],
        [
         "19",
         "5-1294952-A-AGGGAGCGCACGGCTCGGCAGCG",
         "5",
         "1294952",
         "A",
         "AGGGAGCGCACGGCTCGGCAGCG",
         "1",
         "1390276",
         "7.192816390414565e-07",
         null
        ],
        [
         "20",
         "5-1294952-A-C",
         "5",
         "1294952",
         "A",
         "C",
         "0",
         "1390276",
         "0.0",
         null
        ],
        [
         "21",
         "5-1294952-AGGGAGCGCACGGCTCGGCAGCG-A",
         "5",
         "1294952",
         "AGGGAGCGCACGGCTCGGCAGCG",
         "A",
         "0",
         "1390266",
         "0.0",
         "Pathogenic"
        ],
        [
         "22",
         "5-1294953-G-A",
         "5",
         "1294953",
         "G",
         "A",
         "1",
         "1381780",
         "7.237042076162631e-07",
         "Likely benign"
        ],
        [
         "23",
         "5-1294953-G-T",
         "5",
         "1294953",
         "G",
         "T",
         "0",
         "1381780",
         "0.0",
         null
        ],
        [
         "24",
         "5-1294954-G-A",
         "5",
         "1294954",
         "G",
         "A",
         "2",
         "1381336",
         "1.4478736527535661e-06",
         "Likely benign"
        ],
        [
         "25",
         "5-1294954-G-C",
         "5",
         "1294954",
         "G",
         "C",
         "0",
         "1381334",
         "0.0",
         null
        ],
        [
         "26",
         "5-1294954-G-T",
         "5",
         "1294954",
         "G",
         "T",
         "2",
         "1381336",
         "1.4478736527535661e-06",
         "Likely benign"
        ],
        [
         "27",
         "5-1294955-G-A",
         "5",
         "1294955",
         "G",
         "A",
         "0",
         "1381738",
         "0.0",
         null
        ],
        [
         "28",
         "5-1294955-G-T",
         "5",
         "1294955",
         "G",
         "T",
         "0",
         "1381738",
         "0.0",
         null
        ],
        [
         "29",
         "5-1294956-A-G",
         "5",
         "1294956",
         "A",
         "G",
         "0",
         "1376688",
         "0.0",
         "Uncertain significance"
        ],
        [
         "30",
         "5-1294957-G-GCTCCC",
         "5",
         "1294957",
         "G",
         "GCTCCC",
         "1",
         "1376536",
         "7.264612040658581e-07",
         null
        ],
        [
         "31",
         "5-1294957-G-T",
         "5",
         "1294957",
         "G",
         "T",
         "0",
         "1376536",
         "0.0",
         "Likely benign"
        ],
        [
         "32",
         "5-1294957-G-A",
         "5",
         "1294957",
         "G",
         "A",
         "1",
         "1376536",
         "7.264612040658581e-07",
         "Likely benign"
        ],
        [
         "33",
         "5-1294958-C-A",
         "5",
         "1294958",
         "C",
         "A",
         "0",
         "1371810",
         "0.0",
         null
        ],
        [
         "34",
         "5-1294958-C-T",
         "5",
         "1294958",
         "C",
         "T",
         "2",
         "1371812",
         "1.457925721600336e-06",
         null
        ],
        [
         "35",
         "5-1294958-C-G",
         "5",
         "1294958",
         "C",
         "G",
         "0",
         "1371812",
         "0.0",
         "Uncertain significance"
        ],
        [
         "36",
         "5-1294959-G-C",
         "5",
         "1294959",
         "G",
         "C",
         "0",
         "1367474",
         "0.0",
         null
        ],
        [
         "37",
         "5-1294959-G-A",
         "5",
         "1294959",
         "G",
         "A",
         "0",
         "1367472",
         "0.0",
         null
        ],
        [
         "38",
         "5-1294959-G-T",
         "5",
         "1294959",
         "G",
         "T",
         "0",
         "1367474",
         "0.0",
         null
        ],
        [
         "39",
         "5-1294960-C-A",
         "5",
         "1294960",
         "C",
         "A",
         "1",
         "1367802",
         "7.310999691475813e-07",
         null
        ],
        [
         "40",
         "5-1294960-C-T",
         "5",
         "1294960",
         "C",
         "T",
         "1",
         "1367802",
         "7.310999691475813e-07",
         null
        ],
        [
         "41",
         "5-1294961-A-G",
         "5",
         "1294961",
         "A",
         "G",
         "0",
         "1367812",
         "0.0",
         null
        ],
        [
         "42",
         "5-1294962-C-A",
         "5",
         "1294962",
         "C",
         "A",
         "0",
         "1366950",
         "0.0",
         null
        ],
        [
         "43",
         "5-1294962-C-T",
         "5",
         "1294962",
         "C",
         "T",
         "0",
         "1366948",
         "0.0",
         null
        ],
        [
         "44",
         "5-1294962-C-G",
         "5",
         "1294962",
         "C",
         "G",
         "1",
         "1366950",
         "7.315556530963093e-07",
         null
        ],
        [
         "45",
         "5-1294963-G-T",
         "5",
         "1294963",
         "G",
         "T",
         "17",
         "1350022",
         "1.2592387383316716e-05",
         "Likely benign"
        ],
        [
         "46",
         "5-1294963-G-C",
         "5",
         "1294963",
         "G",
         "C",
         "11",
         "1349914",
         "8.148667248432123e-06",
         "Likely benign"
        ],
        [
         "47",
         "5-1294963-G-A",
         "5",
         "1294963",
         "G",
         "A",
         "1",
         "1349914",
         "7.407879316756475e-07",
         "Likely benign"
        ],
        [
         "48",
         "5-1294964-G-T",
         "5",
         "1294964",
         "G",
         "T",
         "0",
         "1349032",
         "0.0",
         null
        ],
        [
         "49",
         "5-1294964-G-A",
         "5",
         "1294964",
         "G",
         "A",
         "0",
         "1349032",
         "0.0",
         "Uncertain significance"
        ]
       ],
       "shape": {
        "columns": 9,
        "rows": 769
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>chrom</th>\n",
       "      <th>pos</th>\n",
       "      <th>ref</th>\n",
       "      <th>alt</th>\n",
       "      <th>ac</th>\n",
       "      <th>an</th>\n",
       "      <th>af</th>\n",
       "      <th>clinical_significance</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>5-1294944-T-C</td>\n",
       "      <td>5</td>\n",
       "      <td>1294944</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>0</td>\n",
       "      <td>1403334</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5-1294945-G-A</td>\n",
       "      <td>5</td>\n",
       "      <td>1294945</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>22</td>\n",
       "      <td>1401036</td>\n",
       "      <td>1.570267e-05</td>\n",
       "      <td>Likely benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>5-1294945-G-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1294945</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>0</td>\n",
       "      <td>1401144</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5-1294946-C-CGCA</td>\n",
       "      <td>5</td>\n",
       "      <td>1294946</td>\n",
       "      <td>C</td>\n",
       "      <td>CGCA</td>\n",
       "      <td>1</td>\n",
       "      <td>1400604</td>\n",
       "      <td>7.139777e-07</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5-1294946-C-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1294946</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>0</td>\n",
       "      <td>1400602</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>764</th>\n",
       "      <td>5-1295285-G-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1295285</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>1</td>\n",
       "      <td>151990</td>\n",
       "      <td>6.579380e-06</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>765</th>\n",
       "      <td>5-1295288-C-G</td>\n",
       "      <td>5</td>\n",
       "      <td>1295288</td>\n",
       "      <td>C</td>\n",
       "      <td>G</td>\n",
       "      <td>5</td>\n",
       "      <td>152244</td>\n",
       "      <td>3.284202e-05</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>766</th>\n",
       "      <td>5-1295290-G-C</td>\n",
       "      <td>5</td>\n",
       "      <td>1295290</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>1</td>\n",
       "      <td>152218</td>\n",
       "      <td>6.569525e-06</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>767</th>\n",
       "      <td>5-1295293-A-G</td>\n",
       "      <td>5</td>\n",
       "      <td>1295293</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>5</td>\n",
       "      <td>151864</td>\n",
       "      <td>3.292420e-05</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>768</th>\n",
       "      <td>5-1295294-G-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1295294</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>5</td>\n",
       "      <td>152160</td>\n",
       "      <td>3.286015e-05</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>769 rows √ó 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "           variant_id chrom      pos ref   alt  ac       an            af  \\\n",
       "0       5-1294944-T-C     5  1294944   T     C   0  1403334  0.000000e+00   \n",
       "1       5-1294945-G-A     5  1294945   G     A  22  1401036  1.570267e-05   \n",
       "2       5-1294945-G-T     5  1294945   G     T   0  1401144  0.000000e+00   \n",
       "3    5-1294946-C-CGCA     5  1294946   C  CGCA   1  1400604  7.139777e-07   \n",
       "4       5-1294946-C-T     5  1294946   C     T   0  1400602  0.000000e+00   \n",
       "..                ...   ...      ...  ..   ...  ..      ...           ...   \n",
       "764     5-1295285-G-T     5  1295285   G     T   1   151990  6.579380e-06   \n",
       "765     5-1295288-C-G     5  1295288   C     G   5   152244  3.284202e-05   \n",
       "766     5-1295290-G-C     5  1295290   G     C   1   152218  6.569525e-06   \n",
       "767     5-1295293-A-G     5  1295293   A     G   5   151864  3.292420e-05   \n",
       "768     5-1295294-G-T     5  1295294   G     T   5   152160  3.286015e-05   \n",
       "\n",
       "      clinical_significance  \n",
       "0    Uncertain significance  \n",
       "1             Likely benign  \n",
       "2                       NaN  \n",
       "3                       NaN  \n",
       "4    Uncertain significance  \n",
       "..                      ...  \n",
       "764                     NaN  \n",
       "765                     NaN  \n",
       "766                     NaN  \n",
       "767                     NaN  \n",
       "768                     NaN  \n",
       "\n",
       "[769 rows x 9 columns]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Merge gnomAD variants with ClinVar variants\n",
    "# Create a common key for merging based on variant_id, chromosome, position, ref, and alt\n",
    "df_merged_variants = df_gnomad_variants.merge(\n",
    "    df_clinvar_variants,\n",
    "    left_on=['variant_id', 'chrom', 'pos', 'ref', 'alt'],\n",
    "    right_on=['variant_id', 'chrom', 'pos', 'ref', 'alt'],\n",
    "    how='left'\n",
    ")\n",
    "\n",
    "# Display the merged DataFrame\n",
    "df_merged_variants"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d35eaf53",
   "metadata": {},
   "source": [
    "### ‚ùì Question\n",
    "\n",
    "> **üí° What clinical significance do the top 5 variants with the highest allele frequency have?**\n",
    ">\n",
    "> **üí° What is the allele frequency or allele count of pathogenic variants?**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "469e6f2f",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "af",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "clinical_significance",
         "rawType": "object",
         "type": "string"
        }
       ],
       "ref": "c3e3cf49-b9f8-4a5c-92a3-147f3e83de4f",
       "rows": [
        [
         "751",
         "5-1295234-A-G",
         "0.2540283299544509",
         "Benign"
        ],
        [
         "755",
         "5-1295258-C-T",
         "0.025684999014389906",
         "Benign"
        ],
        [
         "721",
         "5-1295207-G-C",
         "0.004028899654524935",
         "Benign/Likely benign"
        ],
        [
         "747",
         "5-1295225-C-T",
         "0.00038108253722125124",
         "Uncertain significance"
        ],
        [
         "660",
         "5-1295182-A-G",
         "0.0001813958528667612",
         "Uncertain significance"
        ]
       ],
       "shape": {
        "columns": 3,
        "rows": 5
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>af</th>\n",
       "      <th>clinical_significance</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>751</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>0.254028</td>\n",
       "      <td>Benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>755</th>\n",
       "      <td>5-1295258-C-T</td>\n",
       "      <td>0.025685</td>\n",
       "      <td>Benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>721</th>\n",
       "      <td>5-1295207-G-C</td>\n",
       "      <td>0.004029</td>\n",
       "      <td>Benign/Likely benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>747</th>\n",
       "      <td>5-1295225-C-T</td>\n",
       "      <td>0.000381</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>660</th>\n",
       "      <td>5-1295182-A-G</td>\n",
       "      <td>0.000181</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "        variant_id        af   clinical_significance\n",
       "751  5-1295234-A-G  0.254028                  Benign\n",
       "755  5-1295258-C-T  0.025685                  Benign\n",
       "721  5-1295207-G-C  0.004029    Benign/Likely benign\n",
       "747  5-1295225-C-T  0.000381  Uncertain significance\n",
       "660  5-1295182-A-G  0.000181  Uncertain significance"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_merged_variants[df_merged_variants['clinical_significance'].notna()].sort_values(by=\"af\", ascending=False).head(5)[['variant_id', 'af', 'clinical_significance']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "390cb22e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "af",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "ac",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "clinical_significance",
         "rawType": "object",
         "type": "string"
        }
       ],
       "ref": "d631104f-b85e-4f5f-84d0-3449b0678200",
       "rows": [
        [
         "21",
         "5-1294952-AGGGAGCGCACGGCTCGGCAGCG-A",
         "0.0",
         "0",
         "Pathogenic"
        ],
        [
         "540",
         "5-1295135-G-A",
         "2.683887590529593e-05",
         "13",
         "Pathogenic"
        ]
       ],
       "shape": {
        "columns": 4,
        "rows": 2
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>af</th>\n",
       "      <th>ac</th>\n",
       "      <th>clinical_significance</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>5-1294952-AGGGAGCGCACGGCTCGGCAGCG-A</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>0</td>\n",
       "      <td>Pathogenic</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>540</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>0.000027</td>\n",
       "      <td>13</td>\n",
       "      <td>Pathogenic</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                              variant_id        af  ac clinical_significance\n",
       "21   5-1294952-AGGGAGCGCACGGCTCGGCAGCG-A  0.000000   0            Pathogenic\n",
       "540                        5-1295135-G-A  0.000027  13            Pathogenic"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_merged_variants[df_merged_variants['clinical_significance'] == 'Pathogenic'][['variant_id', 'af', 'ac', 'clinical_significance']]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9fa0b972",
   "metadata": {},
   "source": [
    "## üéØ Linking Motifs to Variant Effects"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cfbdeea6",
   "metadata": {},
   "source": [
    "We know that regulation is controlled by the binding of proteins or RNA to regulatory nucleic acid sequences. üß¨ The proteins are called **transcription factors (TFs)**. TFs bind to specific DNA sequences, often referred to as **motifs**. üîë Mutations in these motifs can disrupt the binding of TFs, but also create new binding sites, leading to changes in gene expression. The direction of the expression change depends on whether the binding TF is an activator or repressor of transcription. ‚öñÔ∏è\n",
    "\n",
    "Keep in mind that binding of TFs is highly tissue-specific. üé≠ Not every TF is present in every cell type. Therefore, mutations in motifs may have different effects depending on the cellular context. Also, we will find many more motifs than actual binding events. Just because a motif is present does not mean that the TF actually binds there in a given cell type. ‚ö†Ô∏è"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b4e2228e",
   "metadata": {},
   "source": [
    "We will now analyze our variants from our previous analysis in the context of motifs. üî¨ We will use all variants that have an entry in ClinVar and will link them to creation/disruption of motifs. Therefore, we use a tool called [FIMO](https://meme-suite.org/meme/tools/fimo) üõ†Ô∏è and a motif database called [HOCOMOCO](https://hocomoco13.autosome.org/). üìä"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ff0e6af2",
   "metadata": {},
   "source": [
    "### üöÄ Getting Started\n",
    "\n",
    "We will now run FIMO from the MEME suite in a container. üê≥ It can be Docker, Apptainer, or in Colab we will use uDocker. Therefore, we will need some additional software and data files. üìÇ"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9627fc5c",
   "metadata": {},
   "source": [
    "**NOTE:** You might want to collapse the code cells of the \"Getting Started\" section (by clicking the \"arrow down\" button left of the \"Getting Started\" section) and click on the appearing button (‚ñ∂Ô∏è) to run all the 9 cells at once because they take some time (approx. 3-5 minutes).\n",
    "\n",
    "- We will use the `IPython` package to beautify the notebook by removing the output of some cells. More information about it can be found [here](https://ipython.org/install.html)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "80ed7ef3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Apptainer is installed! We will use Apptainer to run FIMO\n"
     ]
    }
   ],
   "source": [
    "from IPython.display import clear_output\n",
    "\n",
    "tutorial_container = \"memesuite/memesuite:latest\"\n",
    "docker = False\n",
    "apptainer = False\n",
    "\n",
    "docker = !docker --help &>/dev/null; if [ $? -eq 0 ]; then echo 1; else echo 0; fi\n",
    "docker = bool(int(docker[0]))\n",
    "apptainer = !apptainer --help &>/dev/null; if [ $? -eq 0 ]; then echo 1; else echo 0; fi\n",
    "apptainer = bool(int(apptainer[0]))\n",
    "\n",
    "if docker:\n",
    "    print(\"Docker is installed! We will use Docker to run FIMO\")\n",
    "elif apptainer:\n",
    "    print(\"Apptainer is installed! We will use Apptainer to run FIMO\")\n",
    "else:\n",
    "    print(\"Neither Docker nor Apptainer is installed. We assume you run the tutorial on Colab and we will install uDocker for Colab\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "a85213df",
   "metadata": {},
   "outputs": [],
   "source": [
    "def udocker_init():\n",
    "    import os\n",
    "    if not os.path.exists(\"/home/user\"):\n",
    "        !pip install udocker > /dev/null\n",
    "        !udocker --allow-root install > /dev/null\n",
    "        !useradd -m user > /dev/null\n",
    "    print(f'Docker-in-Colab 1.1.0\\n')\n",
    "    print(f'Usage:     udocker(\"--help\")')\n",
    "    print(f'Examples:  https://github.com/indigo-dc/udocker?tab=readme-ov-file#examples')\n",
    "\n",
    "    def execute(command: str):\n",
    "        user_prompt = \"\\033[1;32muser@pc\\033[0m\"\n",
    "        print(f\"{user_prompt}$ udocker {command}\")\n",
    "        !su - user -c \"udocker $command\"\n",
    "\n",
    "    return execute\n",
    "\n",
    "if not docker and not apptainer:\n",
    "    udocker = udocker_init()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "293227b7",
   "metadata": {},
   "outputs": [],
   "source": [
    "def runContainer(command):\n",
    "    user_prompt = \"\\033[1;32muser@pc\\033[0m\"\n",
    "    if docker:\n",
    "        print(f\"{user_prompt}$ docker run -v=${{PWD}}/resources:/data/run/resources:ro -v=${{PWD}}/results:/data/run/results:rw -u $(id -u):$(id -g) --workdir /data/run {tutorial_container} {command}\")\n",
    "        !docker run -v=${{PWD}}/resources:/data/run/resources:ro -v=${{PWD}}/results:/data/run/results:rw -u $(id -u):$(id -g) --workdir /data/run {tutorial_container} {command}\n",
    "    elif apptainer:\n",
    "        print(f\"{user_prompt}$ apptainer run -B=${{PWD}}/resources:/data/run/resources -B=${{PWD}}/results:/data/run/results --cwd /data/run memesuite.sif {command}\")\n",
    "        !apptainer run -B=${{PWD}}/resources:/data/run/resources -B=${{PWD}}/results:/data/run/results --cwd /data/run memesuite.sif {command}\n",
    "    else:\n",
    "        !chmod o+rw results\n",
    "        udocker(f\"run -v=${{PWD}}:/data/run --workdir /data/run memesuite {command}\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "66748e31",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "apptainer pull memesuite.sif docker://memesuite/memesuite:latest\n",
      "\u001b[31mFATAL:  \u001b[0m Image file already exists: \"memesuite.sif\" - will not overwrite\n"
     ]
    }
   ],
   "source": [
    "if docker:\n",
    "    print(f\"docker pull {tutorial_container}\")\n",
    "    !docker pull \"{tutorial_container}\"\n",
    "    print(f\"docker create --name=memesuite {tutorial_container}\")\n",
    "    !docker create --name=memesuite \"{tutorial_container}\" > /dev/null\n",
    "elif apptainer:\n",
    "    print(f\"apptainer pull memesuite.sif docker://{tutorial_container}\")\n",
    "    !apptainer pull memesuite.sif docker://{tutorial_container} > /dev/null\n",
    "else:\n",
    "    udocker(f\"pull {tutorial_container}\")\n",
    "    udocker(f\"create --name=memesuite {tutorial_container}\")\n",
    "    clear_output()\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "36fb6b06",
   "metadata": {
    "vscode": {
     "languageId": "shellscript"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "File 'H13CORE_meme_format.meme.gz' already exists. Skipping...\n",
      "File 'gene_matrix_specificity_by_percentage.tsv.gz' already exists. Skipping...\n",
      "File 'hocomoco_v13/H13CORE-CLUSTERED_annotation.jsonl' already exists. Skipping...\n",
      "File 'resources/hocomoco_v13/H13CORE_meme_format.meme' already exists. Skipping...\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "\n",
    "mkdir -p results\n",
    "mkdir -p resources\n",
    "mkdir -p resources/hocomoco_v13\n",
    "\n",
    "if [ ! -f \"resources/hocomoco_v13/H13CORE_meme_format.meme.gz\" ]; then\n",
    "    wget -q https://raw.githubusercontent.com/kircherlab/ISMB-2025_IGVF-MPRA-Tutorial/main/06_variant_effects_and_motifs/resources/hocomoco_v13/H13CORE_meme_format.meme.gz -O resources/hocomoco_v13/H13CORE_meme_format.meme.gz\n",
    "else\n",
    "    echo \"File 'H13CORE_meme_format.meme.gz' already exists. Skipping...\"\n",
    "fi\n",
    "\n",
    "if [ ! -f resources/hocomoco_v13/H13CORE_meme_format.meme ]; then\n",
    "  gzip -dc resources/hocomoco_v13/H13CORE_meme_format.meme.gz > resources/hocomoco_v13/H13CORE_meme_format.meme\n",
    "else\n",
    "    echo \"File 'resources/hocomoco_v13/H13CORE_meme_format.meme' already exists. Skipping...\"\n",
    "fi\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9a73c761",
   "metadata": {},
   "source": [
    "Now we install the required Python libraries. üì¶‚ú®"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "aad55c77",
   "metadata": {
    "vscode": {
     "languageId": "shellscript"
    }
   },
   "outputs": [],
   "source": [
    "# Install dependencies\n",
    "!pip install biopython seaborn pandas numpy matplotlib\n",
    "\n",
    "# clear cell output\n",
    "clear_output()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c61d0b99",
   "metadata": {},
   "source": [
    "---\n",
    "\n",
    "### üîç Motif Identification from Sequence\n",
    "\n",
    "Let's explore how to identify transcription factor binding motifs in DNA sequences. üß¨‚ú®"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ad408e73",
   "metadata": {},
   "source": [
    "#### üóÑÔ∏è TFBS Databases: JASPAR & HOCOMOCO\n",
    "\n",
    "Transcription factors (TFs) regulate gene expression by binding to specific DNA motifs. üîó Identifying these motifs is crucial for understanding gene regulation, cell differentiation, and disease. üéØ\n",
    "\n",
    "**Key Databases:**\n",
    "- [JASPAR](https://jaspar.elixir.no/) üìö\n",
    "- [HOCOMOCO](https://hocomoco13.autosome.org/) üìä\n",
    "\n",
    "Both provide position weight matrices (PWMs) for TFBSs.  \n",
    "> **‚ö†Ô∏è Note:**  \n",
    "> Genome-wide PWM scanning can yield many false positives due to short motif lengths and genome size."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "62e2a081",
   "metadata": {},
   "source": [
    "Here, we'll focus on **HOCOMOCO v13 (H13)** in .meme format. üìÑ\n",
    "\n",
    "- **üèÜ HOCOMOCO v13:**  \n",
    "  - Curated for human and mouse TFBSs üê≠\n",
    "  - ~1,200 motifs for 1,000 TFs\n",
    "  - Includes ChIP-seq and computational predictions üß™\n",
    "  - Quality control metrics (e.g., supporting sequences, conservation) ‚úÖ\n",
    "  - [Download HOCOMOCO v13](https://hocomoco13.autosome.org/downloads_v13) üì•"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4002a421",
   "metadata": {},
   "source": [
    "![HOCOMOCOv13 Website Screenshot](resources/picture/hocomocov13_website.png) üñºÔ∏è"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "57ab8771",
   "metadata": {},
   "source": [
    "#### üõ†Ô∏è TFBS Identification Tools\n",
    "\n",
    "Several tools predict TFBSs using PWMs: üîß\n",
    "\n",
    "- **FIMO** (Find Individual Motif Occurrences, MEME suite) üéØ\n",
    "- **HOMER** (Hypergeometric Optimization of Motif EnRichment) üìä\n",
    "- ...and more! ‚ú®\n",
    "\n",
    "We'll use **FIMO** for this tutorial. üöÄ"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dda84bb8",
   "metadata": {},
   "source": [
    "**üìã FIMO Usage:**\n",
    "\n",
    "- Input: TFBS database (.meme format) + FASTA sequences üìÑ\n",
    "- Output: Table of motif matches (`motif_id`, sequence name, start/end, matched sequence, etc.) üìä\n",
    "\n",
    "We'll use the `.tsv` output for downstream analysis. üî¨"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e4e15040",
   "metadata": {},
   "source": [
    "So we first need to prepare our input files for FIMO. üìù We want to use all our variants that have an entry in ClinVar. Also, we only look (for simplicity) at single nucleotide variants (SNVs). üß¨"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "c1a44a19",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "chrom",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pos",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "ref",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "alt",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ac",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "an",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "af",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "clinical_significance",
         "rawType": "object",
         "type": "string"
        }
       ],
       "ref": "6dd1eae6-298d-463f-a938-30f45502c770",
       "rows": [
        [
         "0",
         "5-1294944-T-C",
         "5",
         "1294944",
         "T",
         "C",
         "0",
         "1403334",
         "0.0",
         "Uncertain significance"
        ],
        [
         "1",
         "5-1294945-G-A",
         "5",
         "1294945",
         "G",
         "A",
         "22",
         "1401036",
         "1.5702665741636904e-05",
         "Likely benign"
        ],
        [
         "4",
         "5-1294946-C-T",
         "5",
         "1294946",
         "C",
         "T",
         "0",
         "1400602",
         "0.0",
         "Uncertain significance"
        ],
        [
         "13",
         "5-1294950-G-A",
         "5",
         "1294950",
         "G",
         "A",
         "0",
         "1390046",
         "0.0",
         "Likely benign"
        ],
        [
         "14",
         "5-1294950-G-T",
         "5",
         "1294950",
         "G",
         "T",
         "0",
         "1390046",
         "0.0",
         "Uncertain significance"
        ],
        [
         "15",
         "5-1294951-C-A",
         "5",
         "1294951",
         "C",
         "A",
         "2",
         "1389898",
         "1.438954513208883e-06",
         "Likely benign"
        ],
        [
         "17",
         "5-1294951-C-T",
         "5",
         "1294951",
         "C",
         "T",
         "2",
         "1389900",
         "1.4389524426217713e-06",
         "Likely benign"
        ],
        [
         "22",
         "5-1294953-G-A",
         "5",
         "1294953",
         "G",
         "A",
         "1",
         "1381780",
         "7.237042076162631e-07",
         "Likely benign"
        ],
        [
         "24",
         "5-1294954-G-A",
         "5",
         "1294954",
         "G",
         "A",
         "2",
         "1381336",
         "1.4478736527535661e-06",
         "Likely benign"
        ],
        [
         "26",
         "5-1294954-G-T",
         "5",
         "1294954",
         "G",
         "T",
         "2",
         "1381336",
         "1.4478736527535661e-06",
         "Likely benign"
        ],
        [
         "29",
         "5-1294956-A-G",
         "5",
         "1294956",
         "A",
         "G",
         "0",
         "1376688",
         "0.0",
         "Uncertain significance"
        ],
        [
         "31",
         "5-1294957-G-T",
         "5",
         "1294957",
         "G",
         "T",
         "0",
         "1376536",
         "0.0",
         "Likely benign"
        ],
        [
         "32",
         "5-1294957-G-A",
         "5",
         "1294957",
         "G",
         "A",
         "1",
         "1376536",
         "7.264612040658581e-07",
         "Likely benign"
        ],
        [
         "35",
         "5-1294958-C-G",
         "5",
         "1294958",
         "C",
         "G",
         "0",
         "1371812",
         "0.0",
         "Uncertain significance"
        ],
        [
         "45",
         "5-1294963-G-T",
         "5",
         "1294963",
         "G",
         "T",
         "17",
         "1350022",
         "1.2592387383316716e-05",
         "Likely benign"
        ],
        [
         "46",
         "5-1294963-G-C",
         "5",
         "1294963",
         "G",
         "C",
         "11",
         "1349914",
         "8.148667248432123e-06",
         "Likely benign"
        ],
        [
         "47",
         "5-1294963-G-A",
         "5",
         "1294963",
         "G",
         "A",
         "1",
         "1349914",
         "7.407879316756475e-07",
         "Likely benign"
        ],
        [
         "49",
         "5-1294964-G-A",
         "5",
         "1294964",
         "G",
         "A",
         "0",
         "1349032",
         "0.0",
         "Uncertain significance"
        ],
        [
         "53",
         "5-1294966-T-C",
         "5",
         "1294966",
         "T",
         "C",
         "0",
         "1345134",
         "0.0",
         "Likely benign"
        ],
        [
         "55",
         "5-1294967-C-A",
         "5",
         "1294967",
         "C",
         "A",
         "0",
         "1344578",
         "0.0",
         "Uncertain significance"
        ],
        [
         "56",
         "5-1294967-C-T",
         "5",
         "1294967",
         "C",
         "T",
         "2",
         "1344580",
         "1.4874533311517351e-06",
         "Uncertain significance"
        ],
        [
         "57",
         "5-1294968-G-C",
         "5",
         "1294968",
         "G",
         "C",
         "1",
         "1338470",
         "7.471217135983623e-07",
         "Uncertain significance"
        ],
        [
         "58",
         "5-1294968-G-T",
         "5",
         "1294968",
         "G",
         "T",
         "0",
         "1338468",
         "0.0",
         "Likely benign"
        ],
        [
         "61",
         "5-1294969-G-A",
         "5",
         "1294969",
         "G",
         "A",
         "1",
         "1334244",
         "7.494880996279541e-07",
         "Likely benign"
        ],
        [
         "62",
         "5-1294970-C-T",
         "5",
         "1294970",
         "C",
         "T",
         "0",
         "1334346",
         "0.0",
         "Uncertain significance"
        ],
        [
         "74",
         "5-1294974-G-C",
         "5",
         "1294974",
         "G",
         "C",
         "0",
         "1331550",
         "0.0",
         "Uncertain significance"
        ],
        [
         "81",
         "5-1294977-G-T",
         "5",
         "1294977",
         "G",
         "T",
         "0",
         "1327644",
         "0.0",
         "Uncertain significance"
        ],
        [
         "83",
         "5-1294978-A-G",
         "5",
         "1294978",
         "A",
         "G",
         "37",
         "1324526",
         "2.793452148164702e-05",
         "Likely benign"
        ],
        [
         "87",
         "5-1294980-C-G",
         "5",
         "1294980",
         "C",
         "G",
         "2",
         "1321432",
         "1.5135095865697214e-06",
         "Uncertain significance"
        ],
        [
         "89",
         "5-1294981-G-A",
         "5",
         "1294981",
         "G",
         "A",
         "0",
         "1322484",
         "0.0",
         "Likely benign"
        ],
        [
         "90",
         "5-1294981-G-T",
         "5",
         "1294981",
         "G",
         "T",
         "0",
         "1322484",
         "0.0",
         "Likely benign"
        ],
        [
         "92",
         "5-1294982-C-G",
         "5",
         "1294982",
         "C",
         "G",
         "0",
         "1320350",
         "0.0",
         "Uncertain significance"
        ],
        [
         "98",
         "5-1294984-C-A",
         "5",
         "1294984",
         "C",
         "A",
         "0",
         "1320278",
         "0.0",
         "Likely benign"
        ],
        [
         "99",
         "5-1294984-C-T",
         "5",
         "1294984",
         "C",
         "T",
         "1",
         "1320278",
         "7.574162411249751e-07",
         "Likely benign"
        ],
        [
         "100",
         "5-1294985-G-C",
         "5",
         "1294985",
         "G",
         "C",
         "2",
         "1320526",
         "1.514547990724908e-06",
         "Uncertain significance"
        ],
        [
         "102",
         "5-1294985-G-A",
         "5",
         "1294985",
         "G",
         "A",
         "1",
         "1320526",
         "7.57273995362454e-07",
         "Uncertain significance"
        ],
        [
         "103",
         "5-1294986-G-T",
         "5",
         "1294986",
         "G",
         "T",
         "0",
         "1320402",
         "0.0",
         "Uncertain significance"
        ],
        [
         "112",
         "5-1294991-G-A",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance"
        ],
        [
         "114",
         "5-1294992-C-T",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign"
        ],
        [
         "122",
         "5-1294994-G-C",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance"
        ],
        [
         "124",
         "5-1294994-G-T",
         "5",
         "1294994",
         "G",
         "T",
         "1",
         "1315610",
         "7.601036781416985e-07",
         "Likely benign"
        ],
        [
         "125",
         "5-1294995-G-T",
         "5",
         "1294995",
         "G",
         "T",
         "13",
         "1315568",
         "9.881663281563553e-06",
         "Likely benign"
        ],
        [
         "191",
         "5-1295020-C-G",
         "5",
         "1295020",
         "C",
         "G",
         "13",
         "1278802",
         "1.0165764520230653e-05",
         "Uncertain significance"
        ],
        [
         "230",
         "5-1295034-C-A",
         "5",
         "1295034",
         "C",
         "A",
         "6",
         "1224982",
         "4.898031154743498e-06",
         "Uncertain significance"
        ],
        [
         "261",
         "5-1295046-T-G",
         "5",
         "1295046",
         "T",
         "G",
         "43",
         "1165730",
         "3.688675765400221e-05",
         "Conflicting classifications of pathogenicity"
        ],
        [
         "262",
         "5-1295046-T-C",
         "5",
         "1295046",
         "T",
         "C",
         "0",
         "1165848",
         "0.0",
         "Uncertain significance"
        ],
        [
         "263",
         "5-1295046-T-A",
         "5",
         "1295046",
         "T",
         "A",
         "0",
         "1165846",
         "0.0",
         "Uncertain significance"
        ],
        [
         "295",
         "5-1295060-C-G",
         "5",
         "1295060",
         "C",
         "G",
         "0",
         "1049018",
         "0.0",
         "Uncertain significance"
        ],
        [
         "332",
         "5-1295075-G-A",
         "5",
         "1295075",
         "G",
         "A",
         "12",
         "973884",
         "1.2321796024988602e-05",
         "Uncertain significance"
        ],
        [
         "341",
         "5-1295078-G-A",
         "5",
         "1295078",
         "G",
         "A",
         "2",
         "1016936",
         "1.966692102551193e-06",
         "Uncertain significance"
        ]
       ],
       "shape": {
        "columns": 9,
        "rows": 64
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>chrom</th>\n",
       "      <th>pos</th>\n",
       "      <th>ref</th>\n",
       "      <th>alt</th>\n",
       "      <th>ac</th>\n",
       "      <th>an</th>\n",
       "      <th>af</th>\n",
       "      <th>clinical_significance</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>5-1294944-T-C</td>\n",
       "      <td>5</td>\n",
       "      <td>1294944</td>\n",
       "      <td>T</td>\n",
       "      <td>C</td>\n",
       "      <td>0</td>\n",
       "      <td>1403334</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5-1294945-G-A</td>\n",
       "      <td>5</td>\n",
       "      <td>1294945</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>22</td>\n",
       "      <td>1401036</td>\n",
       "      <td>0.000016</td>\n",
       "      <td>Likely benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5-1294946-C-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1294946</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>0</td>\n",
       "      <td>1400602</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>5-1294950-G-A</td>\n",
       "      <td>5</td>\n",
       "      <td>1294950</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>0</td>\n",
       "      <td>1390046</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>Likely benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>5-1294950-G-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1294950</td>\n",
       "      <td>G</td>\n",
       "      <td>T</td>\n",
       "      <td>0</td>\n",
       "      <td>1390046</td>\n",
       "      <td>0.000000</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>721</th>\n",
       "      <td>5-1295207-G-C</td>\n",
       "      <td>5</td>\n",
       "      <td>1295207</td>\n",
       "      <td>G</td>\n",
       "      <td>C</td>\n",
       "      <td>1635</td>\n",
       "      <td>405818</td>\n",
       "      <td>0.004029</td>\n",
       "      <td>Benign/Likely benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>731</th>\n",
       "      <td>5-1295211-C-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1295211</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>11</td>\n",
       "      <td>404234</td>\n",
       "      <td>0.000027</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>747</th>\n",
       "      <td>5-1295225-C-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1295225</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>58</td>\n",
       "      <td>152198</td>\n",
       "      <td>0.000381</td>\n",
       "      <td>Uncertain significance</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>751</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>5</td>\n",
       "      <td>1295234</td>\n",
       "      <td>A</td>\n",
       "      <td>G</td>\n",
       "      <td>38593</td>\n",
       "      <td>151924</td>\n",
       "      <td>0.254028</td>\n",
       "      <td>Benign</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>755</th>\n",
       "      <td>5-1295258-C-T</td>\n",
       "      <td>5</td>\n",
       "      <td>1295258</td>\n",
       "      <td>C</td>\n",
       "      <td>T</td>\n",
       "      <td>3909</td>\n",
       "      <td>152190</td>\n",
       "      <td>0.025685</td>\n",
       "      <td>Benign</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>64 rows √ó 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        variant_id chrom      pos ref alt     ac       an        af  \\\n",
       "0    5-1294944-T-C     5  1294944   T   C      0  1403334  0.000000   \n",
       "1    5-1294945-G-A     5  1294945   G   A     22  1401036  0.000016   \n",
       "4    5-1294946-C-T     5  1294946   C   T      0  1400602  0.000000   \n",
       "13   5-1294950-G-A     5  1294950   G   A      0  1390046  0.000000   \n",
       "14   5-1294950-G-T     5  1294950   G   T      0  1390046  0.000000   \n",
       "..             ...   ...      ...  ..  ..    ...      ...       ...   \n",
       "721  5-1295207-G-C     5  1295207   G   C   1635   405818  0.004029   \n",
       "731  5-1295211-C-T     5  1295211   C   T     11   404234  0.000027   \n",
       "747  5-1295225-C-T     5  1295225   C   T     58   152198  0.000381   \n",
       "751  5-1295234-A-G     5  1295234   A   G  38593   151924  0.254028   \n",
       "755  5-1295258-C-T     5  1295258   C   T   3909   152190  0.025685   \n",
       "\n",
       "      clinical_significance  \n",
       "0    Uncertain significance  \n",
       "1             Likely benign  \n",
       "4    Uncertain significance  \n",
       "13            Likely benign  \n",
       "14   Uncertain significance  \n",
       "..                      ...  \n",
       "721    Benign/Likely benign  \n",
       "731  Uncertain significance  \n",
       "747  Uncertain significance  \n",
       "751                  Benign  \n",
       "755                  Benign  \n",
       "\n",
       "[64 rows x 9 columns]"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df_clinvar_snvs = df_merged_variants[df_merged_variants['clinical_significance'].notna()]\n",
    "df_clinvar_snvs = df_clinvar_snvs[df_clinvar_snvs['alt'].str.len() == 1]\n",
    "df_clinvar_snvs = df_clinvar_snvs[df_clinvar_snvs['ref'].str.len() == 1]\n",
    "df_clinvar_snvs"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f3e11fb1",
   "metadata": {},
   "source": [
    "Now we need the FASTA sequence around the variant positions to run FIMO. üß¨ This can be done in various ways:\n",
    "1. Copy and paste the sequence from a genome browser (the non-bioinformatics way) üìã\n",
    "2. Use a tool like `bedtools` to extract the sequences programmatically. This requires the human reference genome to be available locally üóÑÔ∏è\n",
    "3. Use a Python library like Biopython or `pyfaidx` to fetch the sequences from a local file üêç\n",
    "4. Use an online service like the UCSC Table Browser or Ensembl REST API to fetch sequences üåê\n",
    "\n",
    "Because we already started with APIs, we will use the UCSC REST API to fetch the sequences. üöÄ"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "58de218b",
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "\n",
    "# Create a FASTA file with sequences for variants in df_clinvar_snvs\n",
    "extend_around_variant = 10\n",
    "fasta_lines = []\n",
    "\n",
    "for _, row in df_clinvar_snvs.iterrows():\n",
    "    variant_id = row['variant_id']\n",
    "    chrom = row['chrom'].replace('chr', '')\n",
    "    pos = row['pos']\n",
    "    ref = row['ref']\n",
    "    alt = row['alt']\n",
    "    \n",
    "    # Get sequence context (20bp window around variant)\n",
    "    seq_params = {\n",
    "        \"genome\": \"hg38\",\n",
    "        \"chrom\": f\"chr{chrom}\",\n",
    "        \"start\": pos - extend_around_variant - 1,\n",
    "        \"end\": pos + extend_around_variant\n",
    "    }\n",
    "    seq_r = requests.get(\"https://api.genome.ucsc.edu/getData/sequence\", params=seq_params, timeout=30)\n",
    "    seq_r.raise_for_status()\n",
    "    ref_seq = seq_r.json()[\"dna\"]\n",
    "\n",
    "    # Create alt sequence by substituting the variant\n",
    "    alt_seq = ref_seq[:extend_around_variant] + alt + ref_seq[extend_around_variant + len(ref):]\n",
    "    \n",
    "    # Add ref_seq and alt_seq to the DataFrame row\n",
    "    df_clinvar_snvs.at[row.name, 'variant_pos'] = extend_around_variant + 1\n",
    "    df_clinvar_snvs.at[row.name, 'ref_sequence'] = ref_seq\n",
    "    df_clinvar_snvs.at[row.name, 'alt_sequence'] = alt_seq\n",
    "    \n",
    "    fasta_lines.append(f\">REF_{variant_id}\\n{ref_seq}\")\n",
    "    fasta_lines.append(f\">ALT_{variant_id}\\n{alt_seq}\")\n",
    "\n",
    "    df_clinvar_snvs.at[row.name, 'REF'] = f\"REF_{variant_id}\"\n",
    "    df_clinvar_snvs.at[row.name, 'ALT'] = f\"ALT_{variant_id}\"\n",
    "\n",
    "# Write to FASTA file\n",
    "with open(\"resources/clinvar_variants_fimo_input.fa\", \"w\") as f:\n",
    "    f.write(\"\\n\".join(fasta_lines))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "f2a902c5",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ref_sequence",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "alt_sequence",
         "rawType": "object",
         "type": "string"
        }
       ],
       "ref": "41b34746-39d8-4425-9ba8-c8a22feb6846",
       "rows": [
        [
         "0",
         "5-1294944-T-C",
         "CGGTAGTGGCTGCGCAGCAGG",
         "CGGTAGTGGCCGCGCAGCAGG"
        ],
        [
         "1",
         "5-1294945-G-A",
         "GGTAGTGGCTGCGCAGCAGGG",
         "GGTAGTGGCTACGCAGCAGGG"
        ],
        [
         "4",
         "5-1294946-C-T",
         "GTAGTGGCTGCGCAGCAGGGA",
         "GTAGTGGCTGTGCAGCAGGGA"
        ],
        [
         "13",
         "5-1294950-G-A",
         "TGGCTGCGCAGCAGGGAGCGC",
         "TGGCTGCGCAACAGGGAGCGC"
        ],
        [
         "14",
         "5-1294950-G-T",
         "TGGCTGCGCAGCAGGGAGCGC",
         "TGGCTGCGCATCAGGGAGCGC"
        ],
        [
         "15",
         "5-1294951-C-A",
         "GGCTGCGCAGCAGGGAGCGCA",
         "GGCTGCGCAGAAGGGAGCGCA"
        ],
        [
         "17",
         "5-1294951-C-T",
         "GGCTGCGCAGCAGGGAGCGCA",
         "GGCTGCGCAGTAGGGAGCGCA"
        ],
        [
         "22",
         "5-1294953-G-A",
         "CTGCGCAGCAGGGAGCGCACG",
         "CTGCGCAGCAAGGAGCGCACG"
        ],
        [
         "24",
         "5-1294954-G-A",
         "TGCGCAGCAGGGAGCGCACGG",
         "TGCGCAGCAGAGAGCGCACGG"
        ],
        [
         "26",
         "5-1294954-G-T",
         "TGCGCAGCAGGGAGCGCACGG",
         "TGCGCAGCAGTGAGCGCACGG"
        ],
        [
         "29",
         "5-1294956-A-G",
         "CGCAGCAGGGAGCGCACGGCT",
         "CGCAGCAGGGGGCGCACGGCT"
        ],
        [
         "31",
         "5-1294957-G-T",
         "GCAGCAGGGAGCGCACGGCTC",
         "GCAGCAGGGATCGCACGGCTC"
        ],
        [
         "32",
         "5-1294957-G-A",
         "GCAGCAGGGAGCGCACGGCTC",
         "GCAGCAGGGAACGCACGGCTC"
        ],
        [
         "35",
         "5-1294958-C-G",
         "CAGCAGGGAGCGCACGGCTCG",
         "CAGCAGGGAGGGCACGGCTCG"
        ],
        [
         "45",
         "5-1294963-G-T",
         "GGGAGCGCACGGCTCGGCAGC",
         "GGGAGCGCACTGCTCGGCAGC"
        ],
        [
         "46",
         "5-1294963-G-C",
         "GGGAGCGCACGGCTCGGCAGC",
         "GGGAGCGCACCGCTCGGCAGC"
        ],
        [
         "47",
         "5-1294963-G-A",
         "GGGAGCGCACGGCTCGGCAGC",
         "GGGAGCGCACAGCTCGGCAGC"
        ],
        [
         "49",
         "5-1294964-G-A",
         "GGAGCGCACGGCTCGGCAGCG",
         "GGAGCGCACGACTCGGCAGCG"
        ],
        [
         "53",
         "5-1294966-T-C",
         "AGCGCACGGCTCGGCAGCGGG",
         "AGCGCACGGCCCGGCAGCGGG"
        ],
        [
         "55",
         "5-1294967-C-A",
         "GCGCACGGCTCGGCAGCGGGG",
         "GCGCACGGCTAGGCAGCGGGG"
        ],
        [
         "56",
         "5-1294967-C-T",
         "GCGCACGGCTCGGCAGCGGGG",
         "GCGCACGGCTTGGCAGCGGGG"
        ],
        [
         "57",
         "5-1294968-G-C",
         "CGCACGGCTCGGCAGCGGGGA",
         "CGCACGGCTCCGCAGCGGGGA"
        ],
        [
         "58",
         "5-1294968-G-T",
         "CGCACGGCTCGGCAGCGGGGA",
         "CGCACGGCTCTGCAGCGGGGA"
        ],
        [
         "61",
         "5-1294969-G-A",
         "GCACGGCTCGGCAGCGGGGAG",
         "GCACGGCTCGACAGCGGGGAG"
        ],
        [
         "62",
         "5-1294970-C-T",
         "CACGGCTCGGCAGCGGGGAGC",
         "CACGGCTCGGTAGCGGGGAGC"
        ],
        [
         "74",
         "5-1294974-G-C",
         "GCTCGGCAGCGGGGAGCGCGC",
         "GCTCGGCAGCCGGGAGCGCGC"
        ],
        [
         "81",
         "5-1294977-G-T",
         "CGGCAGCGGGGAGCGCGCGGC",
         "CGGCAGCGGGTAGCGCGCGGC"
        ],
        [
         "83",
         "5-1294978-A-G",
         "GGCAGCGGGGAGCGCGCGGCA",
         "GGCAGCGGGGGGCGCGCGGCA"
        ],
        [
         "87",
         "5-1294980-C-G",
         "CAGCGGGGAGCGCGCGGCATC",
         "CAGCGGGGAGGGCGCGGCATC"
        ],
        [
         "89",
         "5-1294981-G-A",
         "AGCGGGGAGCGCGCGGCATCG",
         "AGCGGGGAGCACGCGGCATCG"
        ],
        [
         "90",
         "5-1294981-G-T",
         "AGCGGGGAGCGCGCGGCATCG",
         "AGCGGGGAGCTCGCGGCATCG"
        ],
        [
         "92",
         "5-1294982-C-G",
         "GCGGGGAGCGCGCGGCATCGC",
         "GCGGGGAGCGGGCGGCATCGC"
        ],
        [
         "98",
         "5-1294984-C-A",
         "GGGGAGCGCGCGGCATCGCGG",
         "GGGGAGCGCGAGGCATCGCGG"
        ],
        [
         "99",
         "5-1294984-C-T",
         "GGGGAGCGCGCGGCATCGCGG",
         "GGGGAGCGCGTGGCATCGCGG"
        ],
        [
         "100",
         "5-1294985-G-C",
         "GGGAGCGCGCGGCATCGCGGG",
         "GGGAGCGCGCCGCATCGCGGG"
        ],
        [
         "102",
         "5-1294985-G-A",
         "GGGAGCGCGCGGCATCGCGGG",
         "GGGAGCGCGCAGCATCGCGGG"
        ],
        [
         "103",
         "5-1294986-G-T",
         "GGAGCGCGCGGCATCGCGGGG",
         "GGAGCGCGCGTCATCGCGGGG"
        ],
        [
         "112",
         "5-1294991-G-A",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC"
        ],
        [
         "114",
         "5-1294992-C-T",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC"
        ],
        [
         "122",
         "5-1294994-G-C",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG"
        ],
        [
         "124",
         "5-1294994-G-T",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGTGGGTGGCCGG"
        ],
        [
         "125",
         "5-1294995-G-T",
         "GGCATCGCGGGGGTGGCCGGG",
         "GGCATCGCGGTGGTGGCCGGG"
        ],
        [
         "191",
         "5-1295020-C-G",
         "GGGCTTCCCACGTGCGCAGCA",
         "GGGCTTCCCAGGTGCGCAGCA"
        ],
        [
         "230",
         "5-1295034-C-A",
         "CGCAGCAGGACGCAGCGCTGC",
         "CGCAGCAGGAAGCAGCGCTGC"
        ],
        [
         "261",
         "5-1295046-T-G",
         "CAGCGCTGCCTGAAACTCGCG",
         "CAGCGCTGCCGGAAACTCGCG"
        ],
        [
         "262",
         "5-1295046-T-C",
         "CAGCGCTGCCTGAAACTCGCG",
         "CAGCGCTGCCCGAAACTCGCG"
        ],
        [
         "263",
         "5-1295046-T-A",
         "CAGCGCTGCCTGAAACTCGCG",
         "CAGCGCTGCCAGAAACTCGCG"
        ],
        [
         "295",
         "5-1295060-C-G",
         "ACTCGCGCCGCGAGGAGAGGG",
         "ACTCGCGCCGGGAGGAGAGGG"
        ],
        [
         "332",
         "5-1295075-G-A",
         "AGAGGGCGGGGCCGCGGAAAG",
         "AGAGGGCGGGACCGCGGAAAG"
        ],
        [
         "341",
         "5-1295078-G-A",
         "GGGCGGGGCCGCGGAAAGGAA",
         "GGGCGGGGCCACGGAAAGGAA"
        ]
       ],
       "shape": {
        "columns": 3,
        "rows": 64
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>ref_sequence</th>\n",
       "      <th>alt_sequence</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>5-1294944-T-C</td>\n",
       "      <td>CGGTAGTGGCTGCGCAGCAGG</td>\n",
       "      <td>CGGTAGTGGCCGCGCAGCAGG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5-1294945-G-A</td>\n",
       "      <td>GGTAGTGGCTGCGCAGCAGGG</td>\n",
       "      <td>GGTAGTGGCTACGCAGCAGGG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5-1294946-C-T</td>\n",
       "      <td>GTAGTGGCTGCGCAGCAGGGA</td>\n",
       "      <td>GTAGTGGCTGTGCAGCAGGGA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>5-1294950-G-A</td>\n",
       "      <td>TGGCTGCGCAGCAGGGAGCGC</td>\n",
       "      <td>TGGCTGCGCAACAGGGAGCGC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>5-1294950-G-T</td>\n",
       "      <td>TGGCTGCGCAGCAGGGAGCGC</td>\n",
       "      <td>TGGCTGCGCATCAGGGAGCGC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>721</th>\n",
       "      <td>5-1295207-G-C</td>\n",
       "      <td>ACGGGTGCCCGGGTCCCCAGT</td>\n",
       "      <td>ACGGGTGCCCCGGTCCCCAGT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>731</th>\n",
       "      <td>5-1295211-C-T</td>\n",
       "      <td>GTGCCCGGGTCCCCAGTCCCT</td>\n",
       "      <td>GTGCCCGGGTTCCCAGTCCCT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>747</th>\n",
       "      <td>5-1295225-C-T</td>\n",
       "      <td>AGTCCCTCCGCCACGTGGGAA</td>\n",
       "      <td>AGTCCCTCCGTCACGTGGGAA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>751</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>GCCACGTGGGAAGCGCGGTCC</td>\n",
       "      <td>GCCACGTGGGGAGCGCGGTCC</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>755</th>\n",
       "      <td>5-1295258-C-T</td>\n",
       "      <td>GCGTCTGTGCCCGCGAATCCA</td>\n",
       "      <td>GCGTCTGTGCTCGCGAATCCA</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>64 rows √ó 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        variant_id           ref_sequence           alt_sequence\n",
       "0    5-1294944-T-C  CGGTAGTGGCTGCGCAGCAGG  CGGTAGTGGCCGCGCAGCAGG\n",
       "1    5-1294945-G-A  GGTAGTGGCTGCGCAGCAGGG  GGTAGTGGCTACGCAGCAGGG\n",
       "4    5-1294946-C-T  GTAGTGGCTGCGCAGCAGGGA  GTAGTGGCTGTGCAGCAGGGA\n",
       "13   5-1294950-G-A  TGGCTGCGCAGCAGGGAGCGC  TGGCTGCGCAACAGGGAGCGC\n",
       "14   5-1294950-G-T  TGGCTGCGCAGCAGGGAGCGC  TGGCTGCGCATCAGGGAGCGC\n",
       "..             ...                    ...                    ...\n",
       "721  5-1295207-G-C  ACGGGTGCCCGGGTCCCCAGT  ACGGGTGCCCCGGTCCCCAGT\n",
       "731  5-1295211-C-T  GTGCCCGGGTCCCCAGTCCCT  GTGCCCGGGTTCCCAGTCCCT\n",
       "747  5-1295225-C-T  AGTCCCTCCGCCACGTGGGAA  AGTCCCTCCGTCACGTGGGAA\n",
       "751  5-1295234-A-G  GCCACGTGGGAAGCGCGGTCC  GCCACGTGGGGAGCGCGGTCC\n",
       "755  5-1295258-C-T  GCGTCTGTGCCCGCGAATCCA  GCGTCTGTGCTCGCGAATCCA\n",
       "\n",
       "[64 rows x 3 columns]"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Display the DataFrame with sequences\n",
    "df_clinvar_snvs[['variant_id', 'ref_sequence', 'alt_sequence']]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "53ba96e0",
   "metadata": {},
   "source": [
    "Our input file for FIMO looks like this: üìÑ‚ú®"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "240aeb71",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ">REF_5-1294944-T-C\n",
      "CGGTAGTGGCTGCGCAGCAGG\n",
      ">ALT_5-1294944-T-C\n",
      "CGGTAGTGGCCGCGCAGCAGG\n",
      ">REF_5-1294945-G-A\n",
      "GGTAGTGGCTGCGCAGCAGGG\n",
      ">ALT_5-1294945-G-A\n",
      "GGTAGTGGCTACGCAGCAGGG\n",
      ">REF_5-1294946-C-T\n",
      "GTAGTGGCTGCGCAGCAGGGA\n"
     ]
    }
   ],
   "source": [
    "%%bash\n",
    "cat resources/clinvar_variants_fimo_input.fa | head"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "de6fb13b",
   "metadata": {},
   "source": [
    "Let's run FIMO on the recently created input file: üöÄüíª"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "31d7b426",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\u001b[1;32muser@pc\u001b[0m$ apptainer run -B=${PWD}/resources:/data/run/resources -B=${PWD}/results:/data/run/results --cwd /data/run memesuite.sif fimo --o results/finding_tfbs_in_variants_H13 resources/hocomoco_v13/H13CORE_meme_format.meme resources/clinvar_variants_fimo_input.fa\n",
      "Using motif +AHCTF1.H13CORE.0.B.B of width 7.\n",
      "Using motif -AHCTF1.H13CORE.0.B.B of width 7.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +AHR.H13CORE.0.P.B of width 10.\n",
      "Using motif -AHR.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +AHRR.H13CORE.0.P.C of width 11.\n",
      "Using motif -AHRR.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ALX1.H13CORE.0.SM.B of width 20.\n",
      "Using motif -ALX1.H13CORE.0.SM.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ALX3.H13CORE.0.SM.B of width 12.\n",
      "Using motif -ALX3.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ALX3.H13CORE.1.S.B of width 20.\n",
      "Using motif -ALX3.H13CORE.1.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ALX4.H13CORE.0.S.B of width 20.\n",
      "Using motif -ALX4.H13CORE.0.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ALX4.H13CORE.1.SM.B of width 13.\n",
      "Using motif -ALX4.H13CORE.1.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ANDR.H13CORE.0.P.B of width 17.\n",
      "Using motif -ANDR.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ANDR.H13CORE.1.S.C of width 8.\n",
      "Using motif -ANDR.H13CORE.1.S.C of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ANDR.H13CORE.2.P.B of width 19.\n",
      "Using motif -ANDR.H13CORE.2.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +AP2A.H13CORE.0.PSM.A of width 15.\n",
      "Using motif -AP2A.H13CORE.0.PSM.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +AP2B.H13CORE.0.SM.B of width 18.\n",
      "Using motif -AP2B.H13CORE.0.SM.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +AP2C.H13CORE.0.PSM.A of width 15.\n",
      "Using motif -AP2C.H13CORE.0.PSM.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +AP2E.H13CORE.0.SM.B of width 16.\n",
      "Using motif -AP2E.H13CORE.0.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ARGFX.H13CORE.0.SM.B of width 15.\n",
      "Using motif -ARGFX.H13CORE.0.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ARI1A.H13CORE.0.P.C of width 11.\n",
      "Using motif -ARI1A.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ARNT.H13CORE.0.P.B of width 9.\n",
      "Using motif -ARNT.H13CORE.0.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ARNT.H13CORE.1.P.B of width 19.\n",
      "Using motif -ARNT.H13CORE.1.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ARNT2.H13CORE.0.P.B of width 13.\n",
      "Using motif -ARNT2.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.695312\n",
      "#   Estimated pi_0=0.709034\n",
      "Using motif +ARNT2.H13CORE.1.P.B of width 11.\n",
      "Using motif -ARNT2.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.724153\n",
      "#   Estimated pi_0=0.725397\n",
      "Using motif +ARNT2.H13CORE.2.S.C of width 12.\n",
      "Using motif -ARNT2.H13CORE.2.S.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.771234\n",
      "#   Estimated pi_0=0.771234\n",
      "Using motif +ARX.H13CORE.0.S.B of width 18.\n",
      "Using motif -ARX.H13CORE.0.S.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ARX.H13CORE.1.SM.B of width 12.\n",
      "Using motif -ARX.H13CORE.1.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ASCL1.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -ASCL1.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ASCL2.H13CORE.0.PSM.A of width 9.\n",
      "Using motif -ASCL2.H13CORE.0.PSM.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF1.H13CORE.0.P.B of width 10.\n",
      "Using motif -ATF1.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF1.H13CORE.1.P.B of width 19.\n",
      "Using motif -ATF1.H13CORE.1.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF2.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -ATF2.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF2.H13CORE.1.P.B of width 11.\n",
      "Using motif -ATF2.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF3.H13CORE.0.P.B of width 10.\n",
      "Using motif -ATF3.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF3.H13CORE.1.P.B of width 10.\n",
      "Using motif -ATF3.H13CORE.1.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF3.H13CORE.2.P.B of width 12.\n",
      "Using motif -ATF3.H13CORE.2.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF3.H13CORE.3.SM.B of width 15.\n",
      "Using motif -ATF3.H13CORE.3.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF4.H13CORE.0.P.B of width 13.\n",
      "Using motif -ATF4.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF4.H13CORE.1.S.B of width 12.\n",
      "Using motif -ATF4.H13CORE.1.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF4.H13CORE.2.SM.B of width 17.\n",
      "Using motif -ATF4.H13CORE.2.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF6A.H13CORE.0.SM.B of width 15.\n",
      "Using motif -ATF6A.H13CORE.0.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF6B.H13CORE.0.SM.B of width 14.\n",
      "Using motif -ATF6B.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATF7.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -ATF7.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATMIN.H13CORE.0.P.B of width 21.\n",
      "Using motif -ATMIN.H13CORE.0.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.273438\n",
      "#   Estimated pi_0=0.273438\n",
      "Using motif +ATOH1.H13CORE.0.P.B of width 10.\n",
      "Using motif -ATOH1.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATOH1.H13CORE.1.SM.B of width 14.\n",
      "Using motif -ATOH1.H13CORE.1.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ATOH8.H13CORE.0.P.C of width 11.\n",
      "Using motif -ATOH8.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BACH1.H13CORE.0.P.B of width 12.\n",
      "Using motif -BACH1.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BACH2.H13CORE.0.P.B of width 11.\n",
      "Using motif -BACH2.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BACH2.H13CORE.1.SM.B of width 17.\n",
      "Using motif -BACH2.H13CORE.1.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BARH1.H13CORE.0.SM.B of width 12.\n",
      "Using motif -BARH1.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BARH1.H13CORE.1.S.B of width 18.\n",
      "Using motif -BARH1.H13CORE.1.S.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BARH2.H13CORE.0.SM.B of width 12.\n",
      "Using motif -BARH2.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BARH2.H13CORE.1.S.B of width 20.\n",
      "Using motif -BARH2.H13CORE.1.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BARH2.H13CORE.2.S.C of width 14.\n",
      "Using motif -BARH2.H13CORE.2.S.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BARX1.H13CORE.0.P.C of width 11.\n",
      "Using motif -BARX1.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BARX1.H13CORE.1.S.B of width 16.\n",
      "Using motif -BARX1.H13CORE.1.S.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BARX2.H13CORE.0.SM.B of width 13.\n",
      "Using motif -BARX2.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BATF.H13CORE.0.P.B of width 19.\n",
      "Using motif -BATF.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BATF.H13CORE.1.P.B of width 11.\n",
      "Using motif -BATF.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BATF2.H13CORE.0.PSG.A of width 13.\n",
      "Using motif -BATF2.H13CORE.0.PSG.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BATF3.H13CORE.0.P.B of width 18.\n",
      "Using motif -BATF3.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BATF3.H13CORE.1.P.B of width 16.\n",
      "Using motif -BATF3.H13CORE.1.P.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BATF3.H13CORE.2.SM.B of width 14.\n",
      "Using motif -BATF3.H13CORE.2.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BCL6.H13CORE.0.PSM.A of width 16.\n",
      "Using motif -BCL6.H13CORE.0.PSM.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BCL6B.H13CORE.0.PSM.A of width 15.\n",
      "Using motif -BCL6B.H13CORE.0.PSM.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BHA15.H13CORE.0.P.B of width 12.\n",
      "Using motif -BHA15.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BHA15.H13CORE.1.SM.B of width 14.\n",
      "Using motif -BHA15.H13CORE.1.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BHE22.H13CORE.0.SM.B of width 11.\n",
      "Using motif -BHE22.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BHE23.H13CORE.0.SM.B of width 11.\n",
      "Using motif -BHE23.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BHE40.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -BHE40.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.7342\n",
      "#   Estimated pi_0=0.735237\n",
      "Using motif +BHE40.H13CORE.1.S.C of width 22.\n",
      "Using motif -BHE40.H13CORE.1.S.C of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BHE41.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -BHE41.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.690625\n",
      "#   Estimated pi_0=0.690625\n",
      "Using motif +BHLHA9.H13CORE.0.P.B of width 10.\n",
      "Using motif -BHLHA9.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +BMAL1.H13CORE.0.PSM.A of width 8.\n",
      "Using motif -BMAL1.H13CORE.0.PSM.A of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3584 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.684092\n",
      "#   Estimated pi_0=0.684458\n",
      "Using motif +BSH.H13CORE.0.SM.B of width 12.\n",
      "Using motif -BSH.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CAMTA1.H13CORE.0.PSGI.A of width 13.\n",
      "Using motif -CAMTA1.H13CORE.0.PSGI.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.687569\n",
      "#   Estimated pi_0=0.693618\n",
      "Using motif +CAMTA2.H13CORE.0.PSGI.A of width 15.\n",
      "Using motif -CAMTA2.H13CORE.0.PSGI.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CASZ1.H13CORE.0.SGI.A of width 26.\n",
      "Using motif -CASZ1.H13CORE.0.SGI.A of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CDX1.H13CORE.0.M.B of width 11.\n",
      "Using motif -CDX1.H13CORE.0.M.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CDX1.H13CORE.1.S.B of width 13.\n",
      "Using motif -CDX1.H13CORE.1.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CDX2.H13CORE.0.PS.A of width 13.\n",
      "Using motif -CDX2.H13CORE.0.PS.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CDX2.H13CORE.1.M.C of width 10.\n",
      "Using motif -CDX2.H13CORE.1.M.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CDX4.H13CORE.0.S.C of width 14.\n",
      "Using motif -CDX4.H13CORE.0.S.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CDX4.H13CORE.1.M.C of width 13.\n",
      "Using motif -CDX4.H13CORE.1.M.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CEBPA.H13CORE.0.P.B of width 12.\n",
      "Using motif -CEBPA.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CEBPB.H13CORE.0.P.B of width 11.\n",
      "Using motif -CEBPB.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CEBPB.H13CORE.1.SM.B of width 13.\n",
      "Using motif -CEBPB.H13CORE.1.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 1.00177\n",
      "#   Estimated pi_0=1\n",
      "Using motif +CEBPD.H13CORE.0.P.B of width 11.\n",
      "Using motif -CEBPD.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CEBPD.H13CORE.1.SM.B of width 13.\n",
      "Using motif -CEBPD.H13CORE.1.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.996737\n",
      "#   Estimated pi_0=0.996737\n",
      "Using motif +CEBPE.H13CORE.0.P.B of width 11.\n",
      "Using motif -CEBPE.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CEBPE.H13CORE.1.SM.B of width 13.\n",
      "Using motif -CEBPE.H13CORE.1.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CEBPG.H13CORE.0.P.B of width 12.\n",
      "Using motif -CEBPG.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CEBPG.H13CORE.1.SM.B of width 12.\n",
      "Using motif -CEBPG.H13CORE.1.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 1.00024\n",
      "#   Estimated pi_0=1\n",
      "Using motif +CEBPG.H13CORE.2.P.B of width 12.\n",
      "Using motif -CEBPG.H13CORE.2.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CENPB.H13CORE.0.S.B of width 18.\n",
      "Using motif -CENPB.H13CORE.0.S.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CGGBP1.H13CORE.0.PSGIB.A of width 11.\n",
      "Using motif -CGGBP1.H13CORE.0.PSGIB.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.588778\n",
      "#   Estimated pi_0=0.588778\n",
      "Using motif +CLOCK.H13CORE.0.M.C of width 11.\n",
      "Using motif -CLOCK.H13CORE.0.M.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CLOCK.H13CORE.1.PS.A of width 12.\n",
      "Using motif -CLOCK.H13CORE.1.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.758057\n",
      "#   Estimated pi_0=0.758057\n",
      "Using motif +COE1.H13CORE.0.P.B of width 14.\n",
      "Using motif -COE1.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +COE2.H13CORE.0.P.B of width 14.\n",
      "Using motif -COE2.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +COE3.H13CORE.0.P.B of width 13.\n",
      "Using motif -COE3.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +COT1.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -COT1.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +COT1.H13CORE.1.PSM.A of width 16.\n",
      "Using motif -COT1.H13CORE.1.PSM.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +COT2.H13CORE.0.P.B of width 14.\n",
      "Using motif -COT2.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +COT2.H13CORE.1.P.B of width 12.\n",
      "Using motif -COT2.H13CORE.1.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CPEB1.H13CORE.0.S.B of width 8.\n",
      "Using motif -CPEB1.H13CORE.0.S.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CPEB1.H13CORE.1.S.B of width 18.\n",
      "Using motif -CPEB1.H13CORE.1.S.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CPXCR1.H13CORE.0.P.C of width 15.\n",
      "Using motif -CPXCR1.H13CORE.0.P.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.405543\n",
      "#   Estimated pi_0=0.407366\n",
      "Using motif +CR3L1.H13CORE.0.PSM.A of width 14.\n",
      "Using motif -CR3L1.H13CORE.0.PSM.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.8071\n",
      "#   Estimated pi_0=0.816191\n",
      "Using motif +CR3L2.H13CORE.0.PS.A of width 11.\n",
      "Using motif -CR3L2.H13CORE.0.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.524148\n",
      "#   Estimated pi_0=0.524148\n",
      "Using motif +CR3L4.H13CORE.0.M.C of width 13.\n",
      "Using motif -CR3L4.H13CORE.0.M.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.712493\n",
      "#   Estimated pi_0=0.712493\n",
      "Using motif +CR3L4.H13CORE.1.S.C of width 11.\n",
      "Using motif -CR3L4.H13CORE.1.S.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.757348\n",
      "#   Estimated pi_0=0.757348\n",
      "Using motif +CREB1.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -CREB1.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CREB3.H13CORE.0.SM.B of width 14.\n",
      "Using motif -CREB3.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CREB3L3.H13CORE.0.PSGIB.A of width 14.\n",
      "Using motif -CREB3L3.H13CORE.0.PSGIB.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.757052\n",
      "#   Estimated pi_0=0.757708\n",
      "Using motif +CREB5.H13CORE.0.P.B of width 11.\n",
      "Using motif -CREB5.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CREB5.H13CORE.1.SM.B of width 12.\n",
      "Using motif -CREB5.H13CORE.1.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CREM.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -CREM.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CREM.H13CORE.1.P.C of width 18.\n",
      "Using motif -CREM.H13CORE.1.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CRX.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -CRX.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CTCF.H13CORE.0.P.B of width 20.\n",
      "Using motif -CTCF.H13CORE.0.P.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 512 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.459559\n",
      "#   Estimated pi_0=0.46131\n",
      "Using motif +CTCFL.H13CORE.0.P.B of width 16.\n",
      "Using motif -CTCFL.H13CORE.0.P.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 23 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.126302\n",
      "#   Estimated pi_0=0.126302\n",
      "Using motif +CUX1.H13CORE.0.PM.A of width 13.\n",
      "Using motif -CUX1.H13CORE.0.PM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CUX1.H13CORE.1.S.B of width 14.\n",
      "Using motif -CUX1.H13CORE.1.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CUX1.H13CORE.2.P.C of width 19.\n",
      "Using motif -CUX1.H13CORE.2.P.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CUX2.H13CORE.0.P.C of width 10.\n",
      "Using motif -CUX2.H13CORE.0.P.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CUX2.H13CORE.1.SM.B of width 13.\n",
      "Using motif -CUX2.H13CORE.1.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CUX2.H13CORE.2.S.C of width 13.\n",
      "Using motif -CUX2.H13CORE.2.S.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +CXXC4.H13CORE.0.PSGIB.A of width 9.\n",
      "Using motif -CXXC4.H13CORE.0.PSGIB.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DACH1.H13CORE.0.S.B of width 17.\n",
      "Using motif -DACH1.H13CORE.0.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DACH2.H13CORE.0.S.B of width 12.\n",
      "Using motif -DACH2.H13CORE.0.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DBP.H13CORE.0.SM.B of width 14.\n",
      "Using motif -DBP.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DDIT3.H13CORE.0.P.B of width 11.\n",
      "Using motif -DDIT3.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DLX1.H13CORE.0.S.B of width 17.\n",
      "Using motif -DLX1.H13CORE.0.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DLX1.H13CORE.1.PSM.A of width 11.\n",
      "Using motif -DLX1.H13CORE.1.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DLX2.H13CORE.0.PSM.A of width 13.\n",
      "Using motif -DLX2.H13CORE.0.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DLX2.H13CORE.1.S.B of width 20.\n",
      "Using motif -DLX2.H13CORE.1.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DLX3.H13CORE.0.SM.B of width 13.\n",
      "Using motif -DLX3.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DLX3.H13CORE.1.S.B of width 18.\n",
      "Using motif -DLX3.H13CORE.1.S.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DLX4.H13CORE.0.SM.B of width 13.\n",
      "Using motif -DLX4.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DLX4.H13CORE.1.S.C of width 18.\n",
      "Using motif -DLX4.H13CORE.1.S.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DLX5.H13CORE.0.PSM.A of width 13.\n",
      "Using motif -DLX5.H13CORE.0.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DLX6.H13CORE.0.SM.B of width 13.\n",
      "Using motif -DLX6.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DMBX1.H13CORE.0.SM.B of width 9.\n",
      "Using motif -DMBX1.H13CORE.0.SM.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DMRT1.H13CORE.0.PS.A of width 16.\n",
      "Using motif -DMRT1.H13CORE.0.PS.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DMRT1.H13CORE.1.M.C of width 13.\n",
      "Using motif -DMRT1.H13CORE.1.M.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DMRT3.H13CORE.0.S.C of width 16.\n",
      "Using motif -DMRT3.H13CORE.0.S.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DMRTA.H13CORE.0.SM.B of width 16.\n",
      "Using motif -DMRTA.H13CORE.0.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DMRTB.H13CORE.0.P.C of width 16.\n",
      "Using motif -DMRTB.H13CORE.0.P.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DMRTD.H13CORE.0.SM.B of width 14.\n",
      "Using motif -DMRTD.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DMTA2.H13CORE.0.SM.B of width 16.\n",
      "Using motif -DMTA2.H13CORE.0.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DMTF1.H13CORE.0.PSGIB.A of width 14.\n",
      "Using motif -DMTF1.H13CORE.0.PSGIB.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.556641\n",
      "#   Estimated pi_0=0.556641\n",
      "Using motif +DNTTIP1.H13CORE.0.PSG.A of width 11.\n",
      "Using motif -DNTTIP1.H13CORE.0.PSG.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 12 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.682815\n",
      "#   Estimated pi_0=0.684512\n",
      "Using motif +DPF1.H13CORE.0.SM.B of width 14.\n",
      "Using motif -DPF1.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DPRX.H13CORE.0.SM.B of width 12.\n",
      "Using motif -DPRX.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DRGX.H13CORE.0.S.B of width 20.\n",
      "Using motif -DRGX.H13CORE.0.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DRGX.H13CORE.1.SM.B of width 15.\n",
      "Using motif -DRGX.H13CORE.1.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DUX4.H13CORE.0.P.B of width 12.\n",
      "Using motif -DUX4.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +DUXA.H13CORE.0.SM.B of width 14.\n",
      "Using motif -DUXA.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +E2F1.H13CORE.0.P.B of width 13.\n",
      "Using motif -E2F1.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.364583\n",
      "#   Estimated pi_0=0.364583\n",
      "Using motif +E2F1.H13CORE.1.S.B of width 14.\n",
      "Using motif -E2F1.H13CORE.1.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +E2F2.H13CORE.0.S.B of width 16.\n",
      "Using motif -E2F2.H13CORE.0.S.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +E2F3.H13CORE.0.P.B of width 11.\n",
      "Using motif -E2F3.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.453125\n",
      "#   Estimated pi_0=0.461296\n",
      "Using motif +E2F3.H13CORE.1.SM.B of width 15.\n",
      "Using motif -E2F3.H13CORE.1.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +E2F4.H13CORE.0.P.B of width 15.\n",
      "Using motif -E2F4.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.387277\n",
      "#   Estimated pi_0=0.387277\n",
      "Using motif +E2F4.H13CORE.1.P.B of width 13.\n",
      "Using motif -E2F4.H13CORE.1.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.288194\n",
      "#   Estimated pi_0=0.288194\n",
      "Using motif +E2F4.H13CORE.2.S.B of width 16.\n",
      "Using motif -E2F4.H13CORE.2.S.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +E2F6.H13CORE.0.P.B of width 11.\n",
      "Using motif -E2F6.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.416193\n",
      "#   Estimated pi_0=0.416193\n",
      "Using motif +E2F7.H13CORE.0.S.C of width 16.\n",
      "Using motif -E2F7.H13CORE.0.S.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +E2F7.H13CORE.1.P.B of width 12.\n",
      "Using motif -E2F7.H13CORE.1.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +E2F8.H13CORE.0.P.C of width 11.\n",
      "Using motif -E2F8.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +E2F8.H13CORE.1.SM.B of width 13.\n",
      "Using motif -E2F8.H13CORE.1.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +E4F1.H13CORE.0.P.B of width 13.\n",
      "Using motif -E4F1.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.786876\n",
      "#   Estimated pi_0=0.790174\n",
      "Using motif +EGR1.H13CORE.0.PS.A of width 16.\n",
      "Using motif -EGR1.H13CORE.0.PS.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.71239\n",
      "#   Estimated pi_0=0.714532\n",
      "Using motif +EGR2.H13CORE.0.PSM.A of width 14.\n",
      "Using motif -EGR2.H13CORE.0.PSM.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.715554\n",
      "#   Estimated pi_0=0.720688\n",
      "Using motif +EGR3.H13CORE.0.PSM.A of width 13.\n",
      "Using motif -EGR3.H13CORE.0.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.709541\n",
      "#   Estimated pi_0=0.712057\n",
      "Using motif +EGR4.H13CORE.0.S.B of width 10.\n",
      "Using motif -EGR4.H13CORE.0.S.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.87255\n",
      "#   Estimated pi_0=0.87255\n",
      "Using motif +EHF.H13CORE.0.P.B of width 15.\n",
      "Using motif -EHF.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.709424\n",
      "#   Estimated pi_0=0.709424\n",
      "Using motif +ELF1.H13CORE.0.PSM.A of width 13.\n",
      "Using motif -ELF1.H13CORE.0.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.731078\n",
      "#   Estimated pi_0=0.731078\n",
      "Using motif +ELF2.H13CORE.0.PS.A of width 12.\n",
      "Using motif -ELF2.H13CORE.0.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.77753\n",
      "#   Estimated pi_0=0.782585\n",
      "Using motif +ELF2.H13CORE.1.M.B of width 13.\n",
      "Using motif -ELF2.H13CORE.1.M.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.875289\n",
      "#   Estimated pi_0=0.881883\n",
      "Using motif +ELF3.H13CORE.0.S.B of width 13.\n",
      "Using motif -ELF3.H13CORE.0.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.926184\n",
      "#   Estimated pi_0=0.940493\n",
      "Using motif +ELF3.H13CORE.1.PM.A of width 13.\n",
      "Using motif -ELF3.H13CORE.1.PM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.958748\n",
      "#   Estimated pi_0=0.968938\n",
      "Using motif +ELF4.H13CORE.0.PS.A of width 11.\n",
      "Using motif -ELF4.H13CORE.0.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.702857\n",
      "#   Estimated pi_0=0.709564\n",
      "Using motif +ELF4.H13CORE.1.M.B of width 15.\n",
      "Using motif -ELF4.H13CORE.1.M.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.951943\n",
      "#   Estimated pi_0=0.960709\n",
      "Using motif +ELF5.H13CORE.0.PSM.A of width 16.\n",
      "Using motif -ELF5.H13CORE.0.PSM.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.896345\n",
      "#   Estimated pi_0=0.904756\n",
      "Using motif +ELF5.H13CORE.1.S.B of width 14.\n",
      "Using motif -ELF5.H13CORE.1.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.903799\n",
      "#   Estimated pi_0=0.905273\n",
      "Using motif +ELK1.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -ELK1.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.850163\n",
      "#   Estimated pi_0=0.860995\n",
      "Using motif +ELK3.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -ELK3.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.921474\n",
      "#   Estimated pi_0=0.921474\n",
      "Using motif +ELK4.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -ELK4.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.81568\n",
      "#   Estimated pi_0=0.820313\n",
      "Using motif +EMX1.H13CORE.0.SM.B of width 13.\n",
      "Using motif -EMX1.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +EMX2.H13CORE.0.S.B of width 13.\n",
      "Using motif -EMX2.H13CORE.0.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +EOMES.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -EOMES.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +EPAS1.H13CORE.0.P.B of width 11.\n",
      "Using motif -EPAS1.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ERF.H13CORE.0.PS.A of width 11.\n",
      "Using motif -ERF.H13CORE.0.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.944062\n",
      "#   Estimated pi_0=0.944062\n",
      "Using motif +ERG.H13CORE.0.P.B of width 12.\n",
      "Using motif -ERG.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.979368\n",
      "#   Estimated pi_0=0.985243\n",
      "Using motif +ERG.H13CORE.1.SM.B of width 12.\n",
      "Using motif -ERG.H13CORE.1.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.960478\n",
      "#   Estimated pi_0=0.967742\n",
      "Using motif +ERR1.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -ERR1.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ERR2.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -ERR2.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ERR3.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -ERR3.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ESR1.H13CORE.0.P.B of width 15.\n",
      "Using motif -ESR1.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ESR1.H13CORE.1.P.B of width 9.\n",
      "Using motif -ESR1.H13CORE.1.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ESR2.H13CORE.0.P.B of width 15.\n",
      "Using motif -ESR2.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ESR2.H13CORE.1.P.B of width 7.\n",
      "Using motif -ESR2.H13CORE.1.P.B of width 7.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ESX1.H13CORE.0.SM.B of width 11.\n",
      "Using motif -ESX1.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ESX1.H13CORE.1.S.B of width 17.\n",
      "Using motif -ESX1.H13CORE.1.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ETS1.H13CORE.0.S.B of width 12.\n",
      "Using motif -ETS1.H13CORE.0.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ETS1.H13CORE.1.P.B of width 13.\n",
      "Using motif -ETS1.H13CORE.1.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.926125\n",
      "#   Estimated pi_0=0.926959\n",
      "Using motif +ETS2.H13CORE.0.S.C of width 19.\n",
      "Using motif -ETS2.H13CORE.0.S.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ETS2.H13CORE.1.P.B of width 12.\n",
      "Using motif -ETS2.H13CORE.1.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.641687\n",
      "#   Estimated pi_0=0.641687\n",
      "Using motif +ETV1.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -ETV1.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ETV1.H13CORE.1.PM.A of width 13.\n",
      "Using motif -ETV1.H13CORE.1.PM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.940972\n",
      "#   Estimated pi_0=0.951107\n",
      "Using motif +ETV2.H13CORE.0.S.B of width 11.\n",
      "Using motif -ETV2.H13CORE.0.S.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.998601\n",
      "#   Estimated pi_0=0.998601\n",
      "Using motif +ETV2.H13CORE.1.PM.A of width 13.\n",
      "Using motif -ETV2.H13CORE.1.PM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.915016\n",
      "#   Estimated pi_0=0.915016\n",
      "Using motif +ETV3.H13CORE.0.SM.B of width 12.\n",
      "Using motif -ETV3.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.980314\n",
      "#   Estimated pi_0=0.987865\n",
      "Using motif +ETV4.H13CORE.0.P.B of width 10.\n",
      "Using motif -ETV4.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.622396\n",
      "#   Estimated pi_0=0.622396\n",
      "Using motif +ETV4.H13CORE.1.SM.B of width 13.\n",
      "Using motif -ETV4.H13CORE.1.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ETV5.H13CORE.0.PS.A of width 11.\n",
      "Using motif -ETV5.H13CORE.0.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ETV5.H13CORE.1.PM.A of width 13.\n",
      "Using motif -ETV5.H13CORE.1.PM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.91059\n",
      "#   Estimated pi_0=0.922168\n",
      "Using motif +ETV6.H13CORE.0.PS.A of width 10.\n",
      "Using motif -ETV6.H13CORE.0.PS.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.905121\n",
      "#   Estimated pi_0=0.905121\n",
      "Using motif +ETV6.H13CORE.1.P.B of width 12.\n",
      "Using motif -ETV6.H13CORE.1.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.809375\n",
      "#   Estimated pi_0=0.814951\n",
      "Using motif +ETV7.H13CORE.0.SM.B of width 19.\n",
      "Using motif -ETV7.H13CORE.0.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 768 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.96305\n",
      "#   Estimated pi_0=0.967262\n",
      "Using motif +ETV7.H13CORE.1.P.C of width 7.\n",
      "Using motif -ETV7.H13CORE.1.P.C of width 7.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3840 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.870814\n",
      "#   Estimated pi_0=0.874607\n",
      "Using motif +EVX1.H13CORE.0.SM.B of width 12.\n",
      "Using motif -EVX1.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +EVX2.H13CORE.0.SM.B of width 13.\n",
      "Using motif -EVX2.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +EVX2.H13CORE.1.S.B of width 17.\n",
      "Using motif -EVX2.H13CORE.1.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FAM200B.H13CORE.0.PSGIB.A of width 14.\n",
      "Using motif -FAM200B.H13CORE.0.PSGIB.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.445616\n",
      "#   Estimated pi_0=0.445616\n",
      "Using motif +FBXL19.H13CORE.0.PSGIB.A of width 7.\n",
      "Using motif -FBXL19.H13CORE.0.PSGIB.A of width 7.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FER3L.H13CORE.0.SM.B of width 16.\n",
      "Using motif -FER3L.H13CORE.0.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FEV.H13CORE.0.S.B of width 11.\n",
      "Using motif -FEV.H13CORE.0.S.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.928578\n",
      "#   Estimated pi_0=0.937042\n",
      "Using motif +FEZF1.H13CORE.0.P.B of width 12.\n",
      "Using motif -FEZF1.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FIGLA.H13CORE.0.SM.B of width 11.\n",
      "Using motif -FIGLA.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FIZ1.H13CORE.0.PSGI.A of width 17.\n",
      "Using motif -FIZ1.H13CORE.0.PSGI.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FLI1.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -FLI1.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.866228\n",
      "#   Estimated pi_0=0.869916\n",
      "Using motif +FLI1.H13CORE.1.P.B of width 18.\n",
      "Using motif -FLI1.H13CORE.1.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.775449\n",
      "#   Estimated pi_0=0.777494\n",
      "Using motif +FLYWCH1.H13CORE.0.PSGB.A of width 26.\n",
      "Using motif -FLYWCH1.H13CORE.0.PSGB.A of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOS.H13CORE.0.P.B of width 11.\n",
      "Using motif -FOS.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOS.H13CORE.1.S.C of width 10.\n",
      "Using motif -FOS.H13CORE.1.S.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.929036\n",
      "#   Estimated pi_0=0.936078\n",
      "Using motif +FOS.H13CORE.2.S.C of width 14.\n",
      "Using motif -FOS.H13CORE.2.S.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.987305\n",
      "#   Estimated pi_0=1\n",
      "Using motif +FOSB.H13CORE.0.P.B of width 9.\n",
      "Using motif -FOSB.H13CORE.0.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOSB.H13CORE.1.M.C of width 11.\n",
      "Using motif -FOSB.H13CORE.1.M.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.951705\n",
      "#   Estimated pi_0=0.960531\n",
      "Using motif +FOSB.H13CORE.2.M.C of width 16.\n",
      "Using motif -FOSB.H13CORE.2.M.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 1.00044\n",
      "#   Estimated pi_0=1\n",
      "Using motif +FOSL1.H13CORE.0.P.B of width 11.\n",
      "Using motif -FOSL1.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOSL2.H13CORE.0.P.B of width 12.\n",
      "Using motif -FOSL2.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXA1.H13CORE.0.P.B of width 11.\n",
      "Using motif -FOXA1.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXA1.H13CORE.1.S.C of width 16.\n",
      "Using motif -FOXA1.H13CORE.1.S.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXA2.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -FOXA2.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXA2.H13CORE.1.SM.B of width 16.\n",
      "Using motif -FOXA2.H13CORE.1.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXA3.H13CORE.0.PS.A of width 12.\n",
      "Using motif -FOXA3.H13CORE.0.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXA3.H13CORE.1.M.C of width 19.\n",
      "Using motif -FOXA3.H13CORE.1.M.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXB1.H13CORE.0.SM.B of width 19.\n",
      "Using motif -FOXB1.H13CORE.0.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXC1.H13CORE.0.S.B of width 19.\n",
      "Using motif -FOXC1.H13CORE.0.S.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXC1.H13CORE.1.S.C of width 14.\n",
      "Using motif -FOXC1.H13CORE.1.S.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXC2.H13CORE.0.SM.B of width 17.\n",
      "Using motif -FOXC2.H13CORE.0.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXD2.H13CORE.0.S.B of width 20.\n",
      "Using motif -FOXD2.H13CORE.0.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXD2.H13CORE.1.SM.B of width 15.\n",
      "Using motif -FOXD2.H13CORE.1.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXD3.H13CORE.0.P.C of width 14.\n",
      "Using motif -FOXD3.H13CORE.0.P.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXD3.H13CORE.1.S.B of width 20.\n",
      "Using motif -FOXD3.H13CORE.1.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXE1.H13CORE.0.SM.B of width 16.\n",
      "Using motif -FOXE1.H13CORE.0.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXE1.H13CORE.1.S.C of width 11.\n",
      "Using motif -FOXE1.H13CORE.1.S.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXF1.H13CORE.0.P.C of width 12.\n",
      "Using motif -FOXF1.H13CORE.0.P.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXG1.H13CORE.0.PSM.A of width 15.\n",
      "Using motif -FOXG1.H13CORE.0.PSM.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXH1.H13CORE.0.P.B of width 9.\n",
      "Using motif -FOXH1.H13CORE.0.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXI1.H13CORE.0.SM.B of width 18.\n",
      "Using motif -FOXI1.H13CORE.0.SM.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXJ2.H13CORE.0.PSM.A of width 14.\n",
      "Using motif -FOXJ2.H13CORE.0.PSM.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXJ2.H13CORE.1.SM.B of width 17.\n",
      "Using motif -FOXJ2.H13CORE.1.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXJ2.H13CORE.2.S.B of width 20.\n",
      "Using motif -FOXJ2.H13CORE.2.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXJ3.H13CORE.0.SM.B of width 12.\n",
      "Using motif -FOXJ3.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXK1.H13CORE.0.PS.A of width 10.\n",
      "Using motif -FOXK1.H13CORE.0.PS.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXK2.H13CORE.0.P.B of width 11.\n",
      "Using motif -FOXK2.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXL1.H13CORE.0.S.B of width 20.\n",
      "Using motif -FOXL1.H13CORE.0.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXL2.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -FOXL2.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXM1.H13CORE.0.P.B of width 12.\n",
      "Using motif -FOXM1.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXO1.H13CORE.0.PS.A of width 9.\n",
      "Using motif -FOXO1.H13CORE.0.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXO3.H13CORE.0.PS.A of width 9.\n",
      "Using motif -FOXO3.H13CORE.0.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXO4.H13CORE.0.SM.B of width 11.\n",
      "Using motif -FOXO4.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXO6.H13CORE.0.S.B of width 12.\n",
      "Using motif -FOXO6.H13CORE.0.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXP1.H13CORE.0.PS.A of width 9.\n",
      "Using motif -FOXP1.H13CORE.0.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXP2.H13CORE.0.P.B of width 8.\n",
      "Using motif -FOXP2.H13CORE.0.P.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXP3.H13CORE.0.P.B of width 10.\n",
      "Using motif -FOXP3.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +FOXR2.H13CORE.0.M.C of width 17.\n",
      "Using motif -FOXR2.H13CORE.0.M.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GABPA.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -GABPA.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.9112\n",
      "#   Estimated pi_0=0.921598\n",
      "Using motif +GATA1.H13CORE.0.P.B of width 18.\n",
      "Using motif -GATA1.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GATA1.H13CORE.1.PSM.A of width 10.\n",
      "Using motif -GATA1.H13CORE.1.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GATA2.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -GATA2.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GATA2.H13CORE.1.P.B of width 18.\n",
      "Using motif -GATA2.H13CORE.1.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GATA3.H13CORE.0.PS.A of width 9.\n",
      "Using motif -GATA3.H13CORE.0.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GATA3.H13CORE.1.SM.B of width 11.\n",
      "Using motif -GATA3.H13CORE.1.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GATA4.H13CORE.0.PSM.A of width 9.\n",
      "Using motif -GATA4.H13CORE.0.PSM.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GATA5.H13CORE.0.SM.B of width 10.\n",
      "Using motif -GATA5.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GATA6.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -GATA6.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GATAD2A.H13CORE.0.SG.A of width 21.\n",
      "Using motif -GATAD2A.H13CORE.0.SG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GBX1.H13CORE.0.S.B of width 12.\n",
      "Using motif -GBX1.H13CORE.0.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GBX1.H13CORE.1.S.B of width 20.\n",
      "Using motif -GBX1.H13CORE.1.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GBX2.H13CORE.0.SM.B of width 12.\n",
      "Using motif -GBX2.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GBX2.H13CORE.1.S.B of width 20.\n",
      "Using motif -GBX2.H13CORE.1.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GCM1.H13CORE.0.S.B of width 11.\n",
      "Using motif -GCM1.H13CORE.0.S.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GCM2.H13CORE.0.SM.B of width 10.\n",
      "Using motif -GCM2.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GCR.H13CORE.0.PS.A of width 19.\n",
      "Using motif -GCR.H13CORE.0.PS.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GCR.H13CORE.1.S.B of width 9.\n",
      "Using motif -GCR.H13CORE.1.S.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GFI1.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -GFI1.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GFI1B.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -GFI1B.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GFI1B.H13CORE.1.S.C of width 21.\n",
      "Using motif -GFI1B.H13CORE.1.S.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GLI1.H13CORE.0.P.C of width 12.\n",
      "Using motif -GLI1.H13CORE.0.P.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.551516\n",
      "#   Estimated pi_0=0.553906\n",
      "Using motif +GLI2.H13CORE.0.PSM.A of width 16.\n",
      "Using motif -GLI2.H13CORE.0.PSM.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.507812\n",
      "#   Estimated pi_0=0.514633\n",
      "Using motif +GLI3.H13CORE.0.PS.A of width 16.\n",
      "Using motif -GLI3.H13CORE.0.PS.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.479116\n",
      "#   Estimated pi_0=0.479116\n",
      "Using motif +GLI3.H13CORE.1.M.C of width 15.\n",
      "Using motif -GLI3.H13CORE.1.M.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GLI4.H13CORE.0.P.C of width 23.\n",
      "Using motif -GLI4.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GLIS1.H13CORE.0.P.B of width 14.\n",
      "Using motif -GLIS1.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.49707\n",
      "#   Estimated pi_0=0.499884\n",
      "Using motif +GLIS1.H13CORE.1.P.B of width 11.\n",
      "Using motif -GLIS1.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.557528\n",
      "#   Estimated pi_0=0.557528\n",
      "Using motif +GLIS1.H13CORE.2.SM.B of width 19.\n",
      "Using motif -GLIS1.H13CORE.2.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 7 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 768 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.628064\n",
      "#   Estimated pi_0=0.633514\n",
      "Using motif +GLIS2.H13CORE.0.SM.B of width 16.\n",
      "Using motif -GLIS2.H13CORE.0.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 7 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.5625\n",
      "#   Estimated pi_0=0.56303\n",
      "Using motif +GLIS3.H13CORE.0.P.C of width 13.\n",
      "Using motif -GLIS3.H13CORE.0.P.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.623264\n",
      "#   Estimated pi_0=0.633899\n",
      "Using motif +GLIS3.H13CORE.1.S.C of width 12.\n",
      "Using motif -GLIS3.H13CORE.1.S.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 8 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.907695\n",
      "#   Estimated pi_0=0.908929\n",
      "Using motif +GLYR1.H13CORE.0.B.B of width 7.\n",
      "Using motif -GLYR1.H13CORE.0.B.B of width 7.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GMEB1.H13CORE.0.P.C of width 9.\n",
      "Using motif -GMEB1.H13CORE.0.P.C of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GMEB1.H13CORE.1.P.C of width 14.\n",
      "Using motif -GMEB1.H13CORE.1.P.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.374023\n",
      "#   Estimated pi_0=0.374023\n",
      "Using motif +GMEB1.H13CORE.2.SM.B of width 15.\n",
      "Using motif -GMEB1.H13CORE.2.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GMEB2.H13CORE.0.P.C of width 14.\n",
      "Using motif -GMEB2.H13CORE.0.P.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.359684\n",
      "#   Estimated pi_0=0.361903\n",
      "Using motif +GMEB2.H13CORE.1.P.C of width 9.\n",
      "Using motif -GMEB2.H13CORE.1.P.C of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GMEB2.H13CORE.2.SM.B of width 15.\n",
      "Using motif -GMEB2.H13CORE.2.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GRHL1.H13CORE.0.PSM.A of width 19.\n",
      "Using motif -GRHL1.H13CORE.0.PSM.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GRHL1.H13CORE.1.PS.A of width 12.\n",
      "Using motif -GRHL1.H13CORE.1.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GRHL2.H13CORE.0.P.B of width 13.\n",
      "Using motif -GRHL2.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GRHL2.H13CORE.1.P.B of width 11.\n",
      "Using motif -GRHL2.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GRHL3.H13CORE.0.SB.A of width 18.\n",
      "Using motif -GRHL3.H13CORE.0.SB.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GSC.H13CORE.0.SM.B of width 10.\n",
      "Using motif -GSC.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GSC2.H13CORE.0.SM.B of width 9.\n",
      "Using motif -GSC2.H13CORE.0.SM.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GSX1.H13CORE.0.SM.B of width 9.\n",
      "Using motif -GSX1.H13CORE.0.SM.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GSX2.H13CORE.0.SM.B of width 12.\n",
      "Using motif -GSX2.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +GTF2IRD2.H13CORE.0.PS.A of width 10.\n",
      "Using motif -GTF2IRD2.H13CORE.0.PS.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.779474\n",
      "#   Estimated pi_0=0.785627\n",
      "Using motif +HAND1.H13CORE.0.P.B of width 17.\n",
      "Using motif -HAND1.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HAND1.H13CORE.1.P.B of width 11.\n",
      "Using motif -HAND1.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HAND2.H13CORE.0.P.B of width 18.\n",
      "Using motif -HAND2.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HAND2.H13CORE.1.P.B of width 14.\n",
      "Using motif -HAND2.H13CORE.1.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HAND2.H13CORE.2.S.C of width 12.\n",
      "Using motif -HAND2.H13CORE.2.S.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.715625\n",
      "#   Estimated pi_0=0.730378\n",
      "Using motif +HEN1.H13CORE.0.S.B of width 14.\n",
      "Using motif -HEN1.H13CORE.0.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HEN2.H13CORE.0.SM.B of width 16.\n",
      "Using motif -HEN2.H13CORE.0.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.71808\n",
      "#   Estimated pi_0=0.719975\n",
      "Using motif +HES1.H13CORE.0.S.C of width 23.\n",
      "Using motif -HES1.H13CORE.0.S.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HES1.H13CORE.1.S.C of width 13.\n",
      "Using motif -HES1.H13CORE.1.S.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.570193\n",
      "#   Estimated pi_0=0.576811\n",
      "Using motif +HES2.H13CORE.0.S.C of width 14.\n",
      "Using motif -HES2.H13CORE.0.S.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.544361\n",
      "#   Estimated pi_0=0.544361\n",
      "Using motif +HES5.H13CORE.0.SM.B of width 14.\n",
      "Using motif -HES5.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.677734\n",
      "#   Estimated pi_0=0.683594\n",
      "Using motif +HES7.H13CORE.0.S.B of width 10.\n",
      "Using motif -HES7.H13CORE.0.S.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.825496\n",
      "#   Estimated pi_0=0.825496\n",
      "Using motif +HESX1.H13CORE.0.SM.B of width 11.\n",
      "Using motif -HESX1.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HESX1.H13CORE.1.S.B of width 16.\n",
      "Using motif -HESX1.H13CORE.1.S.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HEY1.H13CORE.0.S.B of width 12.\n",
      "Using motif -HEY1.H13CORE.0.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.696937\n",
      "#   Estimated pi_0=0.700061\n",
      "Using motif +HEY2.H13CORE.0.S.B of width 13.\n",
      "Using motif -HEY2.H13CORE.0.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.550347\n",
      "#   Estimated pi_0=0.550347\n",
      "Using motif +HIC1.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -HIC1.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.784473\n",
      "#   Estimated pi_0=0.784473\n",
      "Using motif +HIC2.H13CORE.0.P.B of width 8.\n",
      "Using motif -HIC2.H13CORE.0.P.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3584 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.965162\n",
      "#   Estimated pi_0=0.965162\n",
      "Using motif +HIF1A.H13CORE.0.P.B of width 11.\n",
      "Using motif -HIF1A.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HIF1A.H13CORE.1.P.B of width 19.\n",
      "Using motif -HIF1A.H13CORE.1.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HIF3A.H13CORE.0.P.C of width 18.\n",
      "Using motif -HIF3A.H13CORE.0.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HIF3A.H13CORE.1.P.C of width 8.\n",
      "Using motif -HIF3A.H13CORE.1.P.C of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HINFP.H13CORE.0.P.B of width 20.\n",
      "Using motif -HINFP.H13CORE.0.P.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 6 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 512 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.230469\n",
      "#   Estimated pi_0=0.230469\n",
      "Using motif +HINFP.H13CORE.1.P.B of width 8.\n",
      "Using motif -HINFP.H13CORE.1.P.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3584 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.850446\n",
      "#   Estimated pi_0=0.850446\n",
      "Using motif +HINFP.H13CORE.2.S.B of width 11.\n",
      "Using motif -HINFP.H13CORE.2.S.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.696023\n",
      "#   Estimated pi_0=0.705904\n",
      "Using motif +HLF.H13CORE.0.P.B of width 11.\n",
      "Using motif -HLF.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HLF.H13CORE.1.SM.B of width 11.\n",
      "Using motif -HLF.H13CORE.1.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HMBX1.H13CORE.0.PSM.A of width 13.\n",
      "Using motif -HMBX1.H13CORE.0.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HME1.H13CORE.0.PSM.A of width 13.\n",
      "Using motif -HME1.H13CORE.0.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HME2.H13CORE.0.SM.B of width 10.\n",
      "Using motif -HME2.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HMG20A.H13CORE.0.SB.A of width 12.\n",
      "Using motif -HMG20A.H13CORE.0.SB.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HMGA1.H13CORE.0.P.B of width 11.\n",
      "Using motif -HMGA1.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HMX1.H13CORE.0.SM.B of width 11.\n",
      "Using motif -HMX1.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HMX2.H13CORE.0.SM.B of width 11.\n",
      "Using motif -HMX2.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HMX3.H13CORE.0.S.B of width 13.\n",
      "Using motif -HMX3.H13CORE.0.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HNF1A.H13CORE.0.PS.A of width 15.\n",
      "Using motif -HNF1A.H13CORE.0.PS.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HNF1B.H13CORE.0.PS.A of width 15.\n",
      "Using motif -HNF1B.H13CORE.0.PS.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HNF1B.H13CORE.1.P.D of width 11.\n",
      "Using motif -HNF1B.H13CORE.1.P.D of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HNF4A.H13CORE.0.PSM.A of width 15.\n",
      "Using motif -HNF4A.H13CORE.0.PSM.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HNF4G.H13CORE.0.P.B of width 15.\n",
      "Using motif -HNF4G.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HNF6.H13CORE.0.PM.A of width 10.\n",
      "Using motif -HNF6.H13CORE.0.PM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HNF6.H13CORE.1.S.B of width 11.\n",
      "Using motif -HNF6.H13CORE.1.S.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HNF6.H13CORE.2.S.B of width 20.\n",
      "Using motif -HNF6.H13CORE.2.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HNF6.H13CORE.3.M.B of width 15.\n",
      "Using motif -HNF6.H13CORE.3.M.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HOMEZ.H13CORE.0.S.B of width 12.\n",
      "Using motif -HOMEZ.H13CORE.0.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HSF1.H13CORE.0.PSM.A of width 18.\n",
      "Using motif -HSF1.H13CORE.0.PSM.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HSF1.H13CORE.1.P.B of width 9.\n",
      "Using motif -HSF1.H13CORE.1.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HSF1.H13CORE.2.P.B of width 16.\n",
      "Using motif -HSF1.H13CORE.2.P.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.876238\n",
      "#   Estimated pi_0=0.884846\n",
      "Using motif +HSF2.H13CORE.0.PSM.A of width 17.\n",
      "Using motif -HSF2.H13CORE.0.PSM.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HSF4.H13CORE.0.SM.B of width 19.\n",
      "Using motif -HSF4.H13CORE.0.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HSF5.H13CORE.0.M.C of width 12.\n",
      "Using motif -HSF5.H13CORE.0.M.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HSF5.H13CORE.1.S.C of width 12.\n",
      "Using motif -HSF5.H13CORE.1.S.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HSFY1.H13CORE.0.S.B of width 12.\n",
      "Using motif -HSFY1.H13CORE.0.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HSFY1.H13CORE.1.S.B of width 18.\n",
      "Using motif -HSFY1.H13CORE.1.S.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HSFY1.H13CORE.2.M.B of width 19.\n",
      "Using motif -HSFY1.H13CORE.2.M.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HTF4.H13CORE.0.PSM.A of width 9.\n",
      "Using motif -HTF4.H13CORE.0.PSM.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA1.H13CORE.0.SM.B of width 12.\n",
      "Using motif -HXA1.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA1.H13CORE.1.S.B of width 17.\n",
      "Using motif -HXA1.H13CORE.1.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA10.H13CORE.0.S.B of width 10.\n",
      "Using motif -HXA10.H13CORE.0.S.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA10.H13CORE.1.M.B of width 11.\n",
      "Using motif -HXA10.H13CORE.1.M.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA11.H13CORE.0.PS.A of width 14.\n",
      "Using motif -HXA11.H13CORE.0.PS.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA11.H13CORE.1.SM.B of width 12.\n",
      "Using motif -HXA11.H13CORE.1.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA13.H13CORE.0.P.B of width 10.\n",
      "Using motif -HXA13.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA13.H13CORE.1.SM.B of width 12.\n",
      "Using motif -HXA13.H13CORE.1.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA2.H13CORE.0.SM.B of width 13.\n",
      "Using motif -HXA2.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA4.H13CORE.0.SM.B of width 12.\n",
      "Using motif -HXA4.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA5.H13CORE.0.SM.B of width 12.\n",
      "Using motif -HXA5.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA6.H13CORE.0.SM.B of width 12.\n",
      "Using motif -HXA6.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA7.H13CORE.0.SM.B of width 13.\n",
      "Using motif -HXA7.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA9.H13CORE.0.P.B of width 12.\n",
      "Using motif -HXA9.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA9.H13CORE.1.SM.B of width 11.\n",
      "Using motif -HXA9.H13CORE.1.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXA9.H13CORE.2.P.B of width 10.\n",
      "Using motif -HXA9.H13CORE.2.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB1.H13CORE.0.SM.B of width 12.\n",
      "Using motif -HXB1.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB13.H13CORE.0.PS.A of width 12.\n",
      "Using motif -HXB13.H13CORE.0.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB13.H13CORE.1.SM.B of width 12.\n",
      "Using motif -HXB13.H13CORE.1.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB2.H13CORE.0.SM.B of width 12.\n",
      "Using motif -HXB2.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB3.H13CORE.0.S.B of width 9.\n",
      "Using motif -HXB3.H13CORE.0.S.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB4.H13CORE.0.P.B of width 9.\n",
      "Using motif -HXB4.H13CORE.0.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB4.H13CORE.1.PSM.A of width 13.\n",
      "Using motif -HXB4.H13CORE.1.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB5.H13CORE.0.SM.B of width 11.\n",
      "Using motif -HXB5.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB6.H13CORE.0.SM.B of width 13.\n",
      "Using motif -HXB6.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB7.H13CORE.0.P.C of width 14.\n",
      "Using motif -HXB7.H13CORE.0.P.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB7.H13CORE.1.SM.B of width 11.\n",
      "Using motif -HXB7.H13CORE.1.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB8.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -HXB8.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB9.H13CORE.0.S.C of width 10.\n",
      "Using motif -HXB9.H13CORE.0.S.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXB9.H13CORE.1.M.C of width 11.\n",
      "Using motif -HXB9.H13CORE.1.M.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXC10.H13CORE.0.P.B of width 11.\n",
      "Using motif -HXC10.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXC10.H13CORE.1.S.B of width 12.\n",
      "Using motif -HXC10.H13CORE.1.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXC10.H13CORE.2.SM.B of width 12.\n",
      "Using motif -HXC10.H13CORE.2.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXC11.H13CORE.0.SM.B of width 11.\n",
      "Using motif -HXC11.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXC11.H13CORE.1.S.B of width 10.\n",
      "Using motif -HXC11.H13CORE.1.S.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXC12.H13CORE.0.SM.B of width 11.\n",
      "Using motif -HXC12.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXC13.H13CORE.0.SM.B of width 12.\n",
      "Using motif -HXC13.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXC4.H13CORE.0.SM.B of width 13.\n",
      "Using motif -HXC4.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXC8.H13CORE.0.SM.B of width 13.\n",
      "Using motif -HXC8.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXC9.H13CORE.0.P.B of width 12.\n",
      "Using motif -HXC9.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXC9.H13CORE.1.S.C of width 12.\n",
      "Using motif -HXC9.H13CORE.1.S.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXC9.H13CORE.2.M.C of width 13.\n",
      "Using motif -HXC9.H13CORE.2.M.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXD1.H13CORE.0.SM.B of width 13.\n",
      "Using motif -HXD1.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXD10.H13CORE.0.SM.B of width 12.\n",
      "Using motif -HXD10.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXD11.H13CORE.0.PS.A of width 13.\n",
      "Using motif -HXD11.H13CORE.0.PS.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXD11.H13CORE.1.SM.B of width 20.\n",
      "Using motif -HXD11.H13CORE.1.SM.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXD12.H13CORE.0.SM.B of width 11.\n",
      "Using motif -HXD12.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXD13.H13CORE.0.PS.A of width 11.\n",
      "Using motif -HXD13.H13CORE.0.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXD13.H13CORE.1.SM.B of width 11.\n",
      "Using motif -HXD13.H13CORE.1.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXD3.H13CORE.0.SM.B of width 13.\n",
      "Using motif -HXD3.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXD4.H13CORE.0.SM.B of width 12.\n",
      "Using motif -HXD4.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXD8.H13CORE.0.SM.B of width 11.\n",
      "Using motif -HXD8.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +HXD9.H13CORE.0.SM.B of width 13.\n",
      "Using motif -HXD9.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF1.H13CORE.0.P.B of width 19.\n",
      "Using motif -IRF1.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF2.H13CORE.0.PSM.A of width 17.\n",
      "Using motif -IRF2.H13CORE.0.PSM.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF3.H13CORE.0.PS.A of width 19.\n",
      "Using motif -IRF3.H13CORE.0.PS.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF3.H13CORE.1.SM.B of width 23.\n",
      "Using motif -IRF3.H13CORE.1.SM.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF4.H13CORE.0.P.B of width 17.\n",
      "Using motif -IRF4.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF4.H13CORE.1.S.B of width 14.\n",
      "Using motif -IRF4.H13CORE.1.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF4.H13CORE.2.SM.B of width 17.\n",
      "Using motif -IRF4.H13CORE.2.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF5.H13CORE.0.SM.B of width 17.\n",
      "Using motif -IRF5.H13CORE.0.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF5.H13CORE.1.S.B of width 11.\n",
      "Using motif -IRF5.H13CORE.1.S.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.983665\n",
      "#   Estimated pi_0=0.983665\n",
      "Using motif +IRF6.H13CORE.0.SM.B of width 11.\n",
      "Using motif -IRF6.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.996103\n",
      "#   Estimated pi_0=0.996103\n",
      "Using motif +IRF6.H13CORE.1.S.C of width 17.\n",
      "Using motif -IRF6.H13CORE.1.S.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF7.H13CORE.0.S.B of width 14.\n",
      "Using motif -IRF7.H13CORE.0.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF7.H13CORE.1.SM.B of width 19.\n",
      "Using motif -IRF7.H13CORE.1.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF8.H13CORE.0.P.B of width 19.\n",
      "Using motif -IRF8.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF8.H13CORE.1.SM.B of width 17.\n",
      "Using motif -IRF8.H13CORE.1.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF8.H13CORE.2.SM.B of width 14.\n",
      "Using motif -IRF8.H13CORE.2.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRF9.H13CORE.0.PSM.A of width 16.\n",
      "Using motif -IRF9.H13CORE.0.PSM.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRX1.H13CORE.0.S.C of width 9.\n",
      "Using motif -IRX1.H13CORE.0.S.C of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRX2.H13CORE.0.S.B of width 10.\n",
      "Using motif -IRX2.H13CORE.0.S.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRX3.H13CORE.0.M.C of width 11.\n",
      "Using motif -IRX3.H13CORE.0.M.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRX3.H13CORE.1.S.B of width 8.\n",
      "Using motif -IRX3.H13CORE.1.S.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRX3.H13CORE.2.S.C of width 21.\n",
      "Using motif -IRX3.H13CORE.2.S.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRX5.H13CORE.0.M.C of width 11.\n",
      "Using motif -IRX5.H13CORE.0.M.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +IRX5.H13CORE.1.SM.B of width 8.\n",
      "Using motif -IRX5.H13CORE.1.SM.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ISL1.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -ISL1.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ISL2.H13CORE.0.SM.B of width 11.\n",
      "Using motif -ISL2.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ISX.H13CORE.0.SM.B of width 12.\n",
      "Using motif -ISX.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ISX.H13CORE.1.S.B of width 17.\n",
      "Using motif -ISX.H13CORE.1.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ISX.H13CORE.2.SM.B of width 17.\n",
      "Using motif -ISX.H13CORE.2.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ITF2.H13CORE.0.S.B of width 16.\n",
      "Using motif -ITF2.H13CORE.0.S.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.447917\n",
      "#   Estimated pi_0=0.447917\n",
      "Using motif +ITF2.H13CORE.1.PSM.A of width 10.\n",
      "Using motif -ITF2.H13CORE.1.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.639974\n",
      "#   Estimated pi_0=0.639974\n",
      "Using motif +JDP2.H13CORE.0.SM.B of width 14.\n",
      "Using motif -JDP2.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +JRK.H13CORE.0.PSGB.A of width 19.\n",
      "Using motif -JRK.H13CORE.0.PSGB.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +JUN.H13CORE.0.P.B of width 12.\n",
      "Using motif -JUN.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +JUN.H13CORE.1.S.C of width 16.\n",
      "Using motif -JUN.H13CORE.1.S.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +JUNB.H13CORE.0.PM.A of width 10.\n",
      "Using motif -JUNB.H13CORE.0.PM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +JUNB.H13CORE.1.S.C of width 15.\n",
      "Using motif -JUNB.H13CORE.1.S.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +JUND.H13CORE.0.PM.A of width 11.\n",
      "Using motif -JUND.H13CORE.0.PM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +JUND.H13CORE.1.SM.B of width 11.\n",
      "Using motif -JUND.H13CORE.1.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 1.0036\n",
      "#   Estimated pi_0=1\n",
      "Using motif +KAISO.H13CORE.0.P.B of width 11.\n",
      "Using motif -KAISO.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.574574\n",
      "#   Estimated pi_0=0.574574\n",
      "Using motif +KAISO.H13CORE.1.P.B of width 14.\n",
      "Using motif -KAISO.H13CORE.1.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.46875\n",
      "#   Estimated pi_0=0.46875\n",
      "Using motif +KDM2A.H13CORE.0.PSGB.A of width 9.\n",
      "Using motif -KDM2A.H13CORE.0.PSGB.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +KLF1.H13CORE.0.P.B of width 13.\n",
      "Using motif -KLF1.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.700896\n",
      "#   Estimated pi_0=0.702768\n",
      "Using motif +KLF10.H13CORE.0.P.C of width 15.\n",
      "Using motif -KLF10.H13CORE.0.P.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.485104\n",
      "#   Estimated pi_0=0.490865\n",
      "Using motif +KLF10.H13CORE.1.PSM.A of width 24.\n",
      "Using motif -KLF10.H13CORE.1.PSM.A of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +KLF11.H13CORE.0.P.B of width 15.\n",
      "Using motif -KLF11.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.520833\n",
      "#   Estimated pi_0=0.526285\n",
      "Using motif +KLF11.H13CORE.1.SM.B of width 24.\n",
      "Using motif -KLF11.H13CORE.1.SM.B of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +KLF12.H13CORE.0.P.C of width 18.\n",
      "Using motif -KLF12.H13CORE.0.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.511719\n",
      "#   Estimated pi_0=0.511719\n",
      "Using motif +KLF12.H13CORE.1.PSM.A of width 13.\n",
      "Using motif -KLF12.H13CORE.1.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.634549\n",
      "#   Estimated pi_0=0.634549\n",
      "Using motif +KLF13.H13CORE.0.PSM.A of width 15.\n",
      "Using motif -KLF13.H13CORE.0.PSM.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.621652\n",
      "#   Estimated pi_0=0.622127\n",
      "Using motif +KLF13.H13CORE.1.P.C of width 20.\n",
      "Using motif -KLF13.H13CORE.1.P.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 512 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.685051\n",
      "#   Estimated pi_0=0.687265\n",
      "Using motif +KLF14.H13CORE.0.PSM.A of width 14.\n",
      "Using motif -KLF14.H13CORE.0.PSM.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.721519\n",
      "#   Estimated pi_0=0.725863\n",
      "Using motif +KLF14.H13CORE.1.P.C of width 23.\n",
      "Using motif -KLF14.H13CORE.1.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +KLF15.H13CORE.0.S.B of width 14.\n",
      "Using motif -KLF15.H13CORE.0.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.80719\n",
      "#   Estimated pi_0=0.811876\n",
      "Using motif +KLF15.H13CORE.1.P.C of width 20.\n",
      "Using motif -KLF15.H13CORE.1.P.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 512 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.538532\n",
      "#   Estimated pi_0=0.539982\n",
      "Using motif +KLF16.H13CORE.0.SM.B of width 11.\n",
      "Using motif -KLF16.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.758623\n",
      "#   Estimated pi_0=0.770573\n",
      "Using motif +KLF16.H13CORE.1.P.B of width 23.\n",
      "Using motif -KLF16.H13CORE.1.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +KLF17.H13CORE.0.M.C of width 14.\n",
      "Using motif -KLF17.H13CORE.0.M.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.804986\n",
      "#   Estimated pi_0=0.804986\n",
      "Using motif +KLF17.H13CORE.1.P.C of width 11.\n",
      "Using motif -KLF17.H13CORE.1.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 7 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.8124\n",
      "#   Estimated pi_0=0.824052\n",
      "Using motif +KLF2.H13CORE.0.S.C of width 11.\n",
      "Using motif -KLF2.H13CORE.0.S.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.739228\n",
      "#   Estimated pi_0=0.7413\n",
      "Using motif +KLF3.H13CORE.0.PS.A of width 18.\n",
      "Using motif -KLF3.H13CORE.0.PS.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.724457\n",
      "#   Estimated pi_0=0.724457\n",
      "Using motif +KLF3.H13CORE.1.M.C of width 13.\n",
      "Using motif -KLF3.H13CORE.1.M.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.718175\n",
      "#   Estimated pi_0=0.72001\n",
      "Using motif +KLF4.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -KLF4.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.649985\n",
      "#   Estimated pi_0=0.658523\n",
      "Using motif +KLF5.H13CORE.0.PM.A of width 13.\n",
      "Using motif -KLF5.H13CORE.0.PM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.713104\n",
      "#   Estimated pi_0=0.718721\n",
      "Using motif +KLF6.H13CORE.0.P.B of width 18.\n",
      "Using motif -KLF6.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.511259\n",
      "#   Estimated pi_0=0.518329\n",
      "Using motif +KLF6.H13CORE.1.SM.B of width 14.\n",
      "Using motif -KLF6.H13CORE.1.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.673554\n",
      "#   Estimated pi_0=0.68198\n",
      "Using motif +KLF7.H13CORE.0.P.B of width 10.\n",
      "Using motif -KLF7.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.750248\n",
      "#   Estimated pi_0=0.750248\n",
      "Using motif +KLF8.H13CORE.0.P.C of width 13.\n",
      "Using motif -KLF8.H13CORE.0.P.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.711806\n",
      "#   Estimated pi_0=0.719625\n",
      "Using motif +KLF8.H13CORE.1.P.C of width 23.\n",
      "Using motif -KLF8.H13CORE.1.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +KLF9.H13CORE.0.P.B of width 14.\n",
      "Using motif -KLF9.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.760509\n",
      "#   Estimated pi_0=0.774927\n",
      "Using motif +KLF9.H13CORE.1.P.B of width 23.\n",
      "Using motif -KLF9.H13CORE.1.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +KMT2A.H13CORE.0.P.B of width 23.\n",
      "Using motif -KMT2A.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +KMT2B.H13CORE.0.P.B of width 8.\n",
      "Using motif -KMT2B.H13CORE.0.P.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LBX2.H13CORE.0.S.B of width 14.\n",
      "Using motif -LBX2.H13CORE.0.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LBX2.H13CORE.1.S.B of width 17.\n",
      "Using motif -LBX2.H13CORE.1.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LBX2.H13CORE.2.M.C of width 14.\n",
      "Using motif -LBX2.H13CORE.2.M.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LEF1.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -LEF1.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LEF1.H13CORE.1.P.B of width 15.\n",
      "Using motif -LEF1.H13CORE.1.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LEUTX.H13CORE.0.PSGB.A of width 8.\n",
      "Using motif -LEUTX.H13CORE.0.PSGB.A of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX1.H13CORE.0.S.C of width 11.\n",
      "Using motif -LHX1.H13CORE.0.S.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX1.H13CORE.1.M.C of width 10.\n",
      "Using motif -LHX1.H13CORE.1.M.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX2.H13CORE.0.PS.A of width 9.\n",
      "Using motif -LHX2.H13CORE.0.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX2.H13CORE.1.S.C of width 19.\n",
      "Using motif -LHX2.H13CORE.1.S.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX3.H13CORE.0.P.B of width 10.\n",
      "Using motif -LHX3.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX4.H13CORE.0.SM.B of width 12.\n",
      "Using motif -LHX4.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX5.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -LHX5.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX6.H13CORE.0.P.B of width 10.\n",
      "Using motif -LHX6.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX6.H13CORE.1.S.B of width 14.\n",
      "Using motif -LHX6.H13CORE.1.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX6.H13CORE.2.M.C of width 13.\n",
      "Using motif -LHX6.H13CORE.2.M.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX8.H13CORE.0.S.B of width 13.\n",
      "Using motif -LHX8.H13CORE.0.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX8.H13CORE.1.M.C of width 13.\n",
      "Using motif -LHX8.H13CORE.1.M.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX9.H13CORE.0.PS.A of width 12.\n",
      "Using motif -LHX9.H13CORE.0.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LHX9.H13CORE.1.M.B of width 12.\n",
      "Using motif -LHX9.H13CORE.1.M.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LMX1A.H13CORE.0.SM.B of width 12.\n",
      "Using motif -LMX1A.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LMX1B.H13CORE.0.P.C of width 8.\n",
      "Using motif -LMX1B.H13CORE.0.P.C of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LMX1B.H13CORE.1.SM.B of width 12.\n",
      "Using motif -LMX1B.H13CORE.1.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LMX1B.H13CORE.2.S.B of width 20.\n",
      "Using motif -LMX1B.H13CORE.2.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LRRF1.H13CORE.0.P.B of width 10.\n",
      "Using motif -LRRF1.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 13 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.43099\n",
      "#   Estimated pi_0=0.43099\n",
      "Using motif +LYL1.H13CORE.0.P.C of width 13.\n",
      "Using motif -LYL1.H13CORE.0.P.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +LYL1.H13CORE.1.P.C of width 15.\n",
      "Using motif -LYL1.H13CORE.1.P.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.731764\n",
      "#   Estimated pi_0=0.731764\n",
      "Using motif +MAD4.H13CORE.0.P.C of width 8.\n",
      "Using motif -MAD4.H13CORE.0.P.C of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 7 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3584 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.545328\n",
      "#   Estimated pi_0=0.549613\n",
      "Using motif +MAF.H13CORE.0.PM.A of width 18.\n",
      "Using motif -MAF.H13CORE.0.PM.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAF.H13CORE.1.PSM.A of width 13.\n",
      "Using motif -MAF.H13CORE.1.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAFA.H13CORE.0.PSM.A of width 18.\n",
      "Using motif -MAFA.H13CORE.0.PSM.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAFA.H13CORE.1.M.C of width 19.\n",
      "Using motif -MAFA.H13CORE.1.M.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAFB.H13CORE.0.PS.A of width 20.\n",
      "Using motif -MAFB.H13CORE.0.PS.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAFB.H13CORE.1.P.B of width 10.\n",
      "Using motif -MAFB.H13CORE.1.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAFF.H13CORE.0.PSM.A of width 22.\n",
      "Using motif -MAFF.H13CORE.0.PSM.A of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAFF.H13CORE.1.PSM.A of width 14.\n",
      "Using motif -MAFF.H13CORE.1.PSM.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAFG.H13CORE.0.PSM.A of width 18.\n",
      "Using motif -MAFG.H13CORE.0.PSM.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAFG.H13CORE.1.PSM.A of width 15.\n",
      "Using motif -MAFG.H13CORE.1.PSM.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAFK.H13CORE.0.PS.A of width 18.\n",
      "Using motif -MAFK.H13CORE.0.PS.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAFK.H13CORE.1.P.B of width 13.\n",
      "Using motif -MAFK.H13CORE.1.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAFK.H13CORE.2.S.B of width 12.\n",
      "Using motif -MAFK.H13CORE.2.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAX.H13CORE.0.PS.A of width 11.\n",
      "Using motif -MAX.H13CORE.0.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.915948\n",
      "#   Estimated pi_0=0.919766\n",
      "Using motif +MAX.H13CORE.1.M.C of width 14.\n",
      "Using motif -MAX.H13CORE.1.M.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAX.H13CORE.2.S.C of width 22.\n",
      "Using motif -MAX.H13CORE.2.S.C of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MAZ.H13CORE.0.P.B of width 13.\n",
      "Using motif -MAZ.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 15 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.481298\n",
      "#   Estimated pi_0=0.487206\n",
      "Using motif +MAZ.H13CORE.1.P.B of width 21.\n",
      "Using motif -MAZ.H13CORE.1.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.465883\n",
      "#   Estimated pi_0=0.46875\n",
      "Using motif +MBD1.H13CORE.0.PSIB.A of width 11.\n",
      "Using motif -MBD1.H13CORE.0.PSIB.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.842766\n",
      "#   Estimated pi_0=0.846499\n",
      "Using motif +MBD3.H13CORE.0.P.B of width 21.\n",
      "Using motif -MBD3.H13CORE.0.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.191483\n",
      "#   Estimated pi_0=0.191483\n",
      "Using motif +MBNL2.H13CORE.0.B.B of width 8.\n",
      "Using motif -MBNL2.H13CORE.0.B.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3584 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.863464\n",
      "#   Estimated pi_0=0.869728\n",
      "Using motif +MCR.H13CORE.0.S.B of width 17.\n",
      "Using motif -MCR.H13CORE.0.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MCR.H13CORE.1.SM.B of width 9.\n",
      "Using motif -MCR.H13CORE.1.SM.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEF2A.H13CORE.0.P.B of width 13.\n",
      "Using motif -MEF2A.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEF2B.H13CORE.0.PS.A of width 14.\n",
      "Using motif -MEF2B.H13CORE.0.PS.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEF2B.H13CORE.1.SM.B of width 18.\n",
      "Using motif -MEF2B.H13CORE.1.SM.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEF2C.H13CORE.0.P.B of width 14.\n",
      "Using motif -MEF2C.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEF2C.H13CORE.1.SM.B of width 17.\n",
      "Using motif -MEF2C.H13CORE.1.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEF2D.H13CORE.0.PS.A of width 13.\n",
      "Using motif -MEF2D.H13CORE.0.PS.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEF2D.H13CORE.1.M.C of width 16.\n",
      "Using motif -MEF2D.H13CORE.1.M.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEIS1.H13CORE.0.P.B of width 13.\n",
      "Using motif -MEIS1.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEIS1.H13CORE.1.PS.A of width 11.\n",
      "Using motif -MEIS1.H13CORE.1.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEIS2.H13CORE.0.P.C of width 13.\n",
      "Using motif -MEIS2.H13CORE.0.P.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEIS2.H13CORE.1.PSM.A of width 12.\n",
      "Using motif -MEIS2.H13CORE.1.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEIS2.H13CORE.2.SM.B of width 14.\n",
      "Using motif -MEIS2.H13CORE.2.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEIS3.H13CORE.0.SM.B of width 8.\n",
      "Using motif -MEIS3.H13CORE.0.SM.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEIS3.H13CORE.1.S.B of width 20.\n",
      "Using motif -MEIS3.H13CORE.1.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEOX1.H13CORE.0.SM.B of width 13.\n",
      "Using motif -MEOX1.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MEOX2.H13CORE.0.SM.B of width 11.\n",
      "Using motif -MEOX2.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MESP1.H13CORE.0.S.C of width 11.\n",
      "Using motif -MESP1.H13CORE.0.S.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MESP2.H13CORE.0.S.C of width 14.\n",
      "Using motif -MESP2.H13CORE.0.S.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MGA.H13CORE.0.PSG.A of width 19.\n",
      "Using motif -MGA.H13CORE.0.PSG.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MGAP.H13CORE.0.P.C of width 10.\n",
      "Using motif -MGAP.H13CORE.0.P.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 7 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.592448\n",
      "#   Estimated pi_0=0.594155\n",
      "Using motif +MGAP.H13CORE.1.S.B of width 18.\n",
      "Using motif -MGAP.H13CORE.1.S.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MITF.H13CORE.0.P.B of width 11.\n",
      "Using motif -MITF.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.506392\n",
      "#   Estimated pi_0=0.5115\n",
      "Using motif +MIXL1.H13CORE.0.SM.B of width 16.\n",
      "Using motif -MIXL1.H13CORE.0.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MIXL1.H13CORE.1.S.B of width 12.\n",
      "Using motif -MIXL1.H13CORE.1.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MKX.H13CORE.0.PSGIB.A of width 8.\n",
      "Using motif -MKX.H13CORE.0.PSGIB.A of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MLX.H13CORE.0.PM.A of width 9.\n",
      "Using motif -MLX.H13CORE.0.PM.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MLX.H13CORE.1.SM.B of width 11.\n",
      "Using motif -MLX.H13CORE.1.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.886364\n",
      "#   Estimated pi_0=0.895744\n",
      "Using motif +MLXPL.H13CORE.0.S.B of width 14.\n",
      "Using motif -MLXPL.H13CORE.0.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.913008\n",
      "#   Estimated pi_0=0.914289\n",
      "Using motif +MNT.H13CORE.0.PS.A of width 9.\n",
      "Using motif -MNT.H13CORE.0.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 6 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3328 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.674743\n",
      "#   Estimated pi_0=0.680312\n",
      "Using motif +MNX1.H13CORE.0.SM.B of width 12.\n",
      "Using motif -MNX1.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MSANTD1.H13CORE.0.PSGIB.A of width 10.\n",
      "Using motif -MSANTD1.H13CORE.0.PSGIB.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MSANTD4.H13CORE.0.SGIB.A of width 15.\n",
      "Using motif -MSANTD4.H13CORE.0.SGIB.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MSGN1.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -MSGN1.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MSX1.H13CORE.0.SM.B of width 11.\n",
      "Using motif -MSX1.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MSX1.H13CORE.1.S.C of width 22.\n",
      "Using motif -MSX1.H13CORE.1.S.C of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MSX2.H13CORE.0.SM.B of width 9.\n",
      "Using motif -MSX2.H13CORE.0.SM.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MSX2.H13CORE.1.SM.B of width 20.\n",
      "Using motif -MSX2.H13CORE.1.SM.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MTF1.H13CORE.0.SM.B of width 17.\n",
      "Using motif -MTF1.H13CORE.0.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MUSC.H13CORE.0.SM.B of width 12.\n",
      "Using motif -MUSC.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MXI1.H13CORE.0.P.B of width 11.\n",
      "Using motif -MXI1.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 7 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.629972\n",
      "#   Estimated pi_0=0.633579\n",
      "Using motif +MYB.H13CORE.0.P.B of width 11.\n",
      "Using motif -MYB.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYBA.H13CORE.0.P.B of width 15.\n",
      "Using motif -MYBA.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYBA.H13CORE.1.PS.A of width 9.\n",
      "Using motif -MYBA.H13CORE.1.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYBA.H13CORE.2.S.B of width 14.\n",
      "Using motif -MYBA.H13CORE.2.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYBB.H13CORE.0.S.B of width 15.\n",
      "Using motif -MYBB.H13CORE.0.S.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYBB.H13CORE.1.S.B of width 9.\n",
      "Using motif -MYBB.H13CORE.1.S.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYC.H13CORE.0.P.B of width 10.\n",
      "Using motif -MYC.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.652318\n",
      "#   Estimated pi_0=0.652318\n",
      "Using motif +MYCN.H13CORE.0.PS.A of width 11.\n",
      "Using motif -MYCN.H13CORE.0.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 19 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.712698\n",
      "#   Estimated pi_0=0.714847\n",
      "Using motif +MYCN.H13CORE.1.M.C of width 12.\n",
      "Using motif -MYCN.H13CORE.1.M.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.720788\n",
      "#   Estimated pi_0=0.720788\n",
      "Using motif +MYF5.H13CORE.0.P.B of width 12.\n",
      "Using motif -MYF5.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYF6.H13CORE.0.SM.B of width 14.\n",
      "Using motif -MYF6.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYNN.H13CORE.0.P.B of width 18.\n",
      "Using motif -MYNN.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYOD1.H13CORE.0.P.B of width 15.\n",
      "Using motif -MYOD1.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYOD1.H13CORE.1.PSM.A of width 10.\n",
      "Using motif -MYOD1.H13CORE.1.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYOG.H13CORE.0.PSM.A of width 16.\n",
      "Using motif -MYOG.H13CORE.0.PSM.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYPOP.H13CORE.0.PSGB.A of width 9.\n",
      "Using motif -MYPOP.H13CORE.0.PSGB.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYRFL.H13CORE.0.P.C of width 18.\n",
      "Using motif -MYRFL.H13CORE.0.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYSM1.H13CORE.0.P.B of width 17.\n",
      "Using motif -MYSM1.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYT1.H13CORE.0.SG.A of width 18.\n",
      "Using motif -MYT1.H13CORE.0.SG.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MYT1L.H13CORE.0.P.C of width 8.\n",
      "Using motif -MYT1L.H13CORE.0.P.C of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +MZF1.H13CORE.0.P.B of width 12.\n",
      "Using motif -MZF1.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NACC2.H13CORE.0.PSGIB.A of width 10.\n",
      "Using motif -NACC2.H13CORE.0.PSGIB.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NANOG.H13CORE.0.P.B of width 17.\n",
      "Using motif -NANOG.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NANOG.H13CORE.1.P.B of width 11.\n",
      "Using motif -NANOG.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NANOG.H13CORE.2.M.C of width 12.\n",
      "Using motif -NANOG.H13CORE.2.M.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NDF1.H13CORE.0.P.B of width 11.\n",
      "Using motif -NDF1.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NDF1.H13CORE.1.S.C of width 12.\n",
      "Using motif -NDF1.H13CORE.1.S.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NDF2.H13CORE.0.P.B of width 11.\n",
      "Using motif -NDF2.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NDF2.H13CORE.1.S.B of width 11.\n",
      "Using motif -NDF2.H13CORE.1.S.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NF2L1.H13CORE.0.P.B of width 16.\n",
      "Using motif -NF2L1.H13CORE.0.P.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NF2L2.H13CORE.0.P.B of width 15.\n",
      "Using motif -NF2L2.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NF2L2.H13CORE.1.P.B of width 15.\n",
      "Using motif -NF2L2.H13CORE.1.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAC1.H13CORE.0.P.B of width 16.\n",
      "Using motif -NFAC1.H13CORE.0.P.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAC1.H13CORE.1.PS.A of width 11.\n",
      "Using motif -NFAC1.H13CORE.1.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAC1.H13CORE.2.SM.B of width 20.\n",
      "Using motif -NFAC1.H13CORE.2.SM.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAC1.H13CORE.3.SM.B of width 14.\n",
      "Using motif -NFAC1.H13CORE.3.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAC2.H13CORE.0.P.B of width 20.\n",
      "Using motif -NFAC2.H13CORE.0.P.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAC2.H13CORE.1.PS.A of width 11.\n",
      "Using motif -NFAC2.H13CORE.1.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAC2.H13CORE.2.S.C of width 20.\n",
      "Using motif -NFAC2.H13CORE.2.S.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAC2.H13CORE.3.M.C of width 11.\n",
      "Using motif -NFAC2.H13CORE.3.M.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.972296\n",
      "#   Estimated pi_0=0.978992\n",
      "Using motif +NFAC2.H13CORE.4.M.B of width 19.\n",
      "Using motif -NFAC2.H13CORE.4.M.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAC3.H13CORE.0.SM.B of width 14.\n",
      "Using motif -NFAC3.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAC3.H13CORE.1.P.C of width 10.\n",
      "Using motif -NFAC3.H13CORE.1.P.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAC4.H13CORE.0.P.C of width 10.\n",
      "Using motif -NFAC4.H13CORE.0.P.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAC4.H13CORE.1.SM.B of width 12.\n",
      "Using motif -NFAC4.H13CORE.1.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAT5.H13CORE.0.SM.B of width 13.\n",
      "Using motif -NFAT5.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFAT5.H13CORE.1.PS.A of width 14.\n",
      "Using motif -NFAT5.H13CORE.1.PS.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFE2.H13CORE.0.P.B of width 12.\n",
      "Using motif -NFE2.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFE2.H13CORE.1.SM.B of width 19.\n",
      "Using motif -NFE2.H13CORE.1.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFIA.H13CORE.0.P.B of width 8.\n",
      "Using motif -NFIA.H13CORE.0.P.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3584 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.853571\n",
      "#   Estimated pi_0=0.862144\n",
      "Using motif +NFIA.H13CORE.1.PS.A of width 20.\n",
      "Using motif -NFIA.H13CORE.1.PS.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFIB.H13CORE.0.PSM.A of width 20.\n",
      "Using motif -NFIB.H13CORE.0.PSM.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFIB.H13CORE.1.PS.A of width 9.\n",
      "Using motif -NFIB.H13CORE.1.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3328 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.952293\n",
      "#   Estimated pi_0=0.957508\n",
      "Using motif +NFIC.H13CORE.0.PSM.A of width 17.\n",
      "Using motif -NFIC.H13CORE.0.PSM.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFIC.H13CORE.1.PSM.A of width 8.\n",
      "Using motif -NFIC.H13CORE.1.PSM.A of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3584 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.956501\n",
      "#   Estimated pi_0=0.96412\n",
      "Using motif +NFIL3.H13CORE.0.P.B of width 10.\n",
      "Using motif -NFIL3.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFIL3.H13CORE.1.S.B of width 18.\n",
      "Using motif -NFIL3.H13CORE.1.S.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFIX.H13CORE.0.SM.B of width 17.\n",
      "Using motif -NFIX.H13CORE.0.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFIX.H13CORE.1.S.B of width 8.\n",
      "Using motif -NFIX.H13CORE.1.S.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3584 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.736607\n",
      "#   Estimated pi_0=0.741534\n",
      "Using motif +NFKB1.H13CORE.0.PS.A of width 12.\n",
      "Using motif -NFKB1.H13CORE.0.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFKB2.H13CORE.0.PSM.A of width 16.\n",
      "Using motif -NFKB2.H13CORE.0.PSM.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFYA.H13CORE.0.P.B of width 15.\n",
      "Using motif -NFYA.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFYB.H13CORE.0.P.B of width 14.\n",
      "Using motif -NFYB.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NFYC.H13CORE.0.P.B of width 15.\n",
      "Using motif -NFYC.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NGN1.H13CORE.0.SM.B of width 14.\n",
      "Using motif -NGN1.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NGN2.H13CORE.0.P.B of width 11.\n",
      "Using motif -NGN2.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NGN2.H13CORE.1.SM.B of width 14.\n",
      "Using motif -NGN2.H13CORE.1.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX21.H13CORE.0.P.B of width 11.\n",
      "Using motif -NKX21.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX22.H13CORE.0.P.B of width 10.\n",
      "Using motif -NKX22.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX23.H13CORE.0.SM.B of width 10.\n",
      "Using motif -NKX23.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX25.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -NKX25.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX25.H13CORE.1.P.B of width 13.\n",
      "Using motif -NKX25.H13CORE.1.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX28.H13CORE.0.SM.B of width 10.\n",
      "Using motif -NKX28.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX31.H13CORE.0.PSM.A of width 14.\n",
      "Using motif -NKX31.H13CORE.0.PSM.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX32.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -NKX32.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX32.H13CORE.1.S.B of width 20.\n",
      "Using motif -NKX32.H13CORE.1.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX61.H13CORE.0.PS.A of width 20.\n",
      "Using motif -NKX61.H13CORE.0.PS.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX61.H13CORE.1.P.B of width 8.\n",
      "Using motif -NKX61.H13CORE.1.P.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX61.H13CORE.2.S.B of width 13.\n",
      "Using motif -NKX61.H13CORE.2.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX62.H13CORE.0.SM.B of width 12.\n",
      "Using motif -NKX62.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NKX63.H13CORE.0.SM.B of width 14.\n",
      "Using motif -NKX63.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NOTO.H13CORE.0.SM.B of width 13.\n",
      "Using motif -NOTO.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NPAS2.H13CORE.0.M.B of width 20.\n",
      "Using motif -NPAS2.H13CORE.0.M.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NPAS2.H13CORE.1.M.C of width 11.\n",
      "Using motif -NPAS2.H13CORE.1.M.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NPAS2.H13CORE.2.S.B of width 21.\n",
      "Using motif -NPAS2.H13CORE.2.S.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NPAS3.H13CORE.0.P.C of width 11.\n",
      "Using motif -NPAS3.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NPAS4.H13CORE.0.P.B of width 11.\n",
      "Using motif -NPAS4.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.689631\n",
      "#   Estimated pi_0=0.689631\n",
      "Using motif +NR1D1.H13CORE.0.P.B of width 15.\n",
      "Using motif -NR1D1.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1D1.H13CORE.1.SM.B of width 18.\n",
      "Using motif -NR1D1.H13CORE.1.SM.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1D2.H13CORE.0.P.B of width 18.\n",
      "Using motif -NR1D2.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1D2.H13CORE.1.SM.B of width 17.\n",
      "Using motif -NR1D2.H13CORE.1.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1H2.H13CORE.0.P.C of width 18.\n",
      "Using motif -NR1H2.H13CORE.0.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1H2.H13CORE.1.P.C of width 19.\n",
      "Using motif -NR1H2.H13CORE.1.P.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1H3.H13CORE.0.P.B of width 17.\n",
      "Using motif -NR1H3.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1H3.H13CORE.1.P.B of width 13.\n",
      "Using motif -NR1H3.H13CORE.1.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1H4.H13CORE.0.P.B of width 20.\n",
      "Using motif -NR1H4.H13CORE.0.P.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1H4.H13CORE.1.P.B of width 18.\n",
      "Using motif -NR1H4.H13CORE.1.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1H4.H13CORE.2.P.B of width 7.\n",
      "Using motif -NR1H4.H13CORE.2.P.B of width 7.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1I2.H13CORE.0.M.C of width 19.\n",
      "Using motif -NR1I2.H13CORE.0.M.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1I2.H13CORE.1.S.C of width 12.\n",
      "Using motif -NR1I2.H13CORE.1.S.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1I3.H13CORE.0.P.B of width 19.\n",
      "Using motif -NR1I3.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1I3.H13CORE.1.PSM.A of width 17.\n",
      "Using motif -NR1I3.H13CORE.1.PSM.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR1I3.H13CORE.2.M.C of width 20.\n",
      "Using motif -NR1I3.H13CORE.2.M.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR2C1.H13CORE.0.S.C of width 12.\n",
      "Using motif -NR2C1.H13CORE.0.S.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR2C1.H13CORE.1.PM.A of width 16.\n",
      "Using motif -NR2C1.H13CORE.1.PM.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR2C2.H13CORE.0.SM.B of width 12.\n",
      "Using motif -NR2C2.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR2C2.H13CORE.1.PS.A of width 15.\n",
      "Using motif -NR2C2.H13CORE.1.PS.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR2E1.H13CORE.0.SM.B of width 10.\n",
      "Using motif -NR2E1.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR2E1.H13CORE.1.S.B of width 17.\n",
      "Using motif -NR2E1.H13CORE.1.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR2F6.H13CORE.0.P.C of width 14.\n",
      "Using motif -NR2F6.H13CORE.0.P.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR2F6.H13CORE.1.PSM.A of width 12.\n",
      "Using motif -NR2F6.H13CORE.1.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR2F6.H13CORE.2.SM.B of width 18.\n",
      "Using motif -NR2F6.H13CORE.2.SM.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.714231\n",
      "#   Estimated pi_0=0.717435\n",
      "Using motif +NR2F6.H13CORE.3.SM.B of width 16.\n",
      "Using motif -NR2F6.H13CORE.3.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR4A1.H13CORE.0.PSM.A of width 8.\n",
      "Using motif -NR4A1.H13CORE.0.PSM.A of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR4A2.H13CORE.0.SM.B of width 10.\n",
      "Using motif -NR4A2.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR5A2.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -NR5A2.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR6A1.H13CORE.0.P.C of width 14.\n",
      "Using motif -NR6A1.H13CORE.0.P.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR6A1.H13CORE.1.P.C of width 12.\n",
      "Using motif -NR6A1.H13CORE.1.P.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NR6A1.H13CORE.2.SM.B of width 19.\n",
      "Using motif -NR6A1.H13CORE.2.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NRF1.H13CORE.0.PS.A of width 16.\n",
      "Using motif -NRF1.H13CORE.0.PS.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.450521\n",
      "#   Estimated pi_0=0.461342\n",
      "Using motif +NRL.H13CORE.0.SM.B of width 13.\n",
      "Using motif -NRL.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +NRL.H13CORE.1.SM.B of width 19.\n",
      "Using motif -NRL.H13CORE.1.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +OLIG1.H13CORE.0.S.B of width 16.\n",
      "Using motif -OLIG1.H13CORE.0.S.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +OLIG2.H13CORE.0.P.B of width 19.\n",
      "Using motif -OLIG2.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +OLIG2.H13CORE.1.P.B of width 9.\n",
      "Using motif -OLIG2.H13CORE.1.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +OLIG2.H13CORE.2.SM.B of width 11.\n",
      "Using motif -OLIG2.H13CORE.2.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +OLIG3.H13CORE.0.SM.B of width 13.\n",
      "Using motif -OLIG3.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ONEC2.H13CORE.0.S.B of width 12.\n",
      "Using motif -ONEC2.H13CORE.0.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ONEC2.H13CORE.1.PM.A of width 13.\n",
      "Using motif -ONEC2.H13CORE.1.PM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ONEC2.H13CORE.2.S.B of width 13.\n",
      "Using motif -ONEC2.H13CORE.2.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ONEC2.H13CORE.3.P.B of width 22.\n",
      "Using motif -ONEC2.H13CORE.3.P.B of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ONEC3.H13CORE.0.S.B of width 14.\n",
      "Using motif -ONEC3.H13CORE.0.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ONEC3.H13CORE.1.S.C of width 11.\n",
      "Using motif -ONEC3.H13CORE.1.S.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ONEC3.H13CORE.2.M.C of width 13.\n",
      "Using motif -ONEC3.H13CORE.2.M.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +OSR1.H13CORE.0.SM.B of width 11.\n",
      "Using motif -OSR1.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.930258\n",
      "#   Estimated pi_0=0.930258\n",
      "Using motif +OSR2.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -OSR2.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.832818\n",
      "#   Estimated pi_0=0.839971\n",
      "Using motif +OSR2.H13CORE.1.M.C of width 18.\n",
      "Using motif -OSR2.H13CORE.1.M.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +OTX1.H13CORE.0.SM.B of width 12.\n",
      "Using motif -OTX1.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +OTX1.H13CORE.1.S.B of width 14.\n",
      "Using motif -OTX1.H13CORE.1.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +OTX2.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -OTX2.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +OVOL1.H13CORE.0.SM.B of width 14.\n",
      "Using motif -OVOL1.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +OVOL2.H13CORE.0.PSM.A of width 13.\n",
      "Using motif -OVOL2.H13CORE.0.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +OZF.H13CORE.0.P.B of width 25.\n",
      "Using motif -OZF.H13CORE.0.P.B of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +P53.H13CORE.0.P.B of width 21.\n",
      "Using motif -P53.H13CORE.0.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +P53.H13CORE.1.S.C of width 13.\n",
      "Using motif -P53.H13CORE.1.S.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +P5F1B.H13CORE.0.S.B of width 17.\n",
      "Using motif -P5F1B.H13CORE.0.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +P63.H13CORE.0.PS.A of width 19.\n",
      "Using motif -P63.H13CORE.0.PS.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +P63.H13CORE.1.PS.A of width 12.\n",
      "Using motif -P63.H13CORE.1.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +P73.H13CORE.0.PS.A of width 19.\n",
      "Using motif -P73.H13CORE.0.PS.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +P73.H13CORE.1.P.B of width 11.\n",
      "Using motif -P73.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PATZ1.H13CORE.0.P.B of width 23.\n",
      "Using motif -PATZ1.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PATZ1.H13CORE.1.P.C of width 10.\n",
      "Using motif -PATZ1.H13CORE.1.P.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 9 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.750227\n",
      "#   Estimated pi_0=0.750227\n",
      "Using motif +PAX1.H13CORE.0.SM.B of width 18.\n",
      "Using motif -PAX1.H13CORE.0.SM.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX2.H13CORE.0.SM.B of width 19.\n",
      "Using motif -PAX2.H13CORE.0.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX3.H13CORE.0.PS.A of width 14.\n",
      "Using motif -PAX3.H13CORE.0.PS.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX3.H13CORE.1.SM.B of width 15.\n",
      "Using motif -PAX3.H13CORE.1.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX3.H13CORE.2.P.C of width 12.\n",
      "Using motif -PAX3.H13CORE.2.P.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX3.H13CORE.3.S.B of width 20.\n",
      "Using motif -PAX3.H13CORE.3.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX4.H13CORE.0.SM.B of width 9.\n",
      "Using motif -PAX4.H13CORE.0.SM.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX4.H13CORE.1.S.C of width 16.\n",
      "Using motif -PAX4.H13CORE.1.S.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX5.H13CORE.0.PS.A of width 18.\n",
      "Using motif -PAX5.H13CORE.0.PS.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX6.H13CORE.0.PSM.A of width 18.\n",
      "Using motif -PAX6.H13CORE.0.PSM.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX7.H13CORE.0.P.B of width 12.\n",
      "Using motif -PAX7.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX7.H13CORE.1.S.B of width 14.\n",
      "Using motif -PAX7.H13CORE.1.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX7.H13CORE.2.PM.A of width 14.\n",
      "Using motif -PAX7.H13CORE.2.PM.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX8.H13CORE.0.PSM.A of width 20.\n",
      "Using motif -PAX8.H13CORE.0.PSM.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PAX9.H13CORE.0.SM.B of width 17.\n",
      "Using motif -PAX9.H13CORE.0.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PBX1.H13CORE.0.P.B of width 12.\n",
      "Using motif -PBX1.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PBX1.H13CORE.1.P.B of width 10.\n",
      "Using motif -PBX1.H13CORE.1.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PBX1.H13CORE.2.P.B of width 10.\n",
      "Using motif -PBX1.H13CORE.2.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PBX2.H13CORE.0.P.B of width 11.\n",
      "Using motif -PBX2.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PBX2.H13CORE.1.P.C of width 9.\n",
      "Using motif -PBX2.H13CORE.1.P.C of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PBX2.H13CORE.2.P.C of width 9.\n",
      "Using motif -PBX2.H13CORE.2.P.C of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PBX3.H13CORE.0.P.B of width 12.\n",
      "Using motif -PBX3.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PBX3.H13CORE.1.P.B of width 10.\n",
      "Using motif -PBX3.H13CORE.1.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PDX1.H13CORE.0.P.B of width 10.\n",
      "Using motif -PDX1.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PDX1.H13CORE.1.PSM.A of width 13.\n",
      "Using motif -PDX1.H13CORE.1.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PHF21A.H13CORE.0.B.B of width 11.\n",
      "Using motif -PHF21A.H13CORE.0.B.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PHX2A.H13CORE.0.PS.A of width 20.\n",
      "Using motif -PHX2A.H13CORE.0.PS.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PHX2A.H13CORE.1.SM.B of width 12.\n",
      "Using motif -PHX2A.H13CORE.1.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PHX2B.H13CORE.0.PSM.A of width 14.\n",
      "Using motif -PHX2B.H13CORE.0.PSM.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PHX2B.H13CORE.1.SM.B of width 11.\n",
      "Using motif -PHX2B.H13CORE.1.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PIT1.H13CORE.0.SM.B of width 14.\n",
      "Using motif -PIT1.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PIT1.H13CORE.1.S.B of width 15.\n",
      "Using motif -PIT1.H13CORE.1.S.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PITX1.H13CORE.0.PSM.A of width 10.\n",
      "Using motif -PITX1.H13CORE.0.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PITX1.H13CORE.1.S.B of width 20.\n",
      "Using motif -PITX1.H13CORE.1.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PITX2.H13CORE.0.SM.B of width 10.\n",
      "Using motif -PITX2.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PITX3.H13CORE.0.SM.B of width 9.\n",
      "Using motif -PITX3.H13CORE.0.SM.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PITX3.H13CORE.1.S.B of width 20.\n",
      "Using motif -PITX3.H13CORE.1.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PKNX1.H13CORE.0.P.B of width 12.\n",
      "Using motif -PKNX1.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PKNX1.H13CORE.1.P.B of width 11.\n",
      "Using motif -PKNX1.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PKNX1.H13CORE.2.SM.B of width 8.\n",
      "Using motif -PKNX1.H13CORE.2.SM.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PKNX1.H13CORE.3.SM.B of width 14.\n",
      "Using motif -PKNX1.H13CORE.3.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PKNX2.H13CORE.0.SM.B of width 15.\n",
      "Using motif -PKNX2.H13CORE.0.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PKNX2.H13CORE.1.SM.B of width 11.\n",
      "Using motif -PKNX2.H13CORE.1.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO2F1.H13CORE.0.PS.A of width 11.\n",
      "Using motif -PO2F1.H13CORE.0.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO2F2.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -PO2F2.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO2F2.H13CORE.1.M.C of width 15.\n",
      "Using motif -PO2F2.H13CORE.1.M.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO2F2.H13CORE.2.S.B of width 13.\n",
      "Using motif -PO2F2.H13CORE.2.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO2F3.H13CORE.0.PS.A of width 13.\n",
      "Using motif -PO2F3.H13CORE.0.PS.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO2F3.H13CORE.1.S.C of width 19.\n",
      "Using motif -PO2F3.H13CORE.1.S.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO3F1.H13CORE.0.P.B of width 15.\n",
      "Using motif -PO3F1.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO3F1.H13CORE.1.PS.A of width 15.\n",
      "Using motif -PO3F1.H13CORE.1.PS.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO3F1.H13CORE.2.S.C of width 9.\n",
      "Using motif -PO3F1.H13CORE.2.S.C of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO3F1.H13CORE.3.M.C of width 17.\n",
      "Using motif -PO3F1.H13CORE.3.M.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO3F2.H13CORE.0.P.B of width 16.\n",
      "Using motif -PO3F2.H13CORE.0.P.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO3F2.H13CORE.1.S.B of width 15.\n",
      "Using motif -PO3F2.H13CORE.1.S.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO3F2.H13CORE.2.SM.B of width 16.\n",
      "Using motif -PO3F2.H13CORE.2.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO3F3.H13CORE.0.P.C of width 12.\n",
      "Using motif -PO3F3.H13CORE.0.P.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO3F3.H13CORE.1.P.C of width 18.\n",
      "Using motif -PO3F3.H13CORE.1.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO3F3.H13CORE.2.S.B of width 12.\n",
      "Using motif -PO3F3.H13CORE.2.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO3F4.H13CORE.0.SM.B of width 15.\n",
      "Using motif -PO3F4.H13CORE.0.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO4F1.H13CORE.0.S.B of width 13.\n",
      "Using motif -PO4F1.H13CORE.0.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO4F1.H13CORE.1.SM.B of width 17.\n",
      "Using motif -PO4F1.H13CORE.1.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO4F2.H13CORE.0.S.B of width 17.\n",
      "Using motif -PO4F2.H13CORE.0.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO4F3.H13CORE.0.SM.B of width 17.\n",
      "Using motif -PO4F3.H13CORE.0.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO5F1.H13CORE.0.P.B of width 17.\n",
      "Using motif -PO5F1.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO5F1.H13CORE.1.P.B of width 11.\n",
      "Using motif -PO5F1.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO5F1.H13CORE.2.SM.B of width 17.\n",
      "Using motif -PO5F1.H13CORE.2.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO6F1.H13CORE.0.SM.B of width 12.\n",
      "Using motif -PO6F1.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PO6F2.H13CORE.0.S.B of width 13.\n",
      "Using motif -PO6F2.H13CORE.0.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +POGK.H13CORE.0.SI.A of width 26.\n",
      "Using motif -POGK.H13CORE.0.SI.A of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PPARA.H13CORE.0.P.B of width 17.\n",
      "Using motif -PPARA.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PPARA.H13CORE.1.P.B of width 9.\n",
      "Using motif -PPARA.H13CORE.1.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PPARD.H13CORE.0.PSM.A of width 15.\n",
      "Using motif -PPARD.H13CORE.0.PSM.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PPARG.H13CORE.0.P.B of width 17.\n",
      "Using motif -PPARG.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PPARG.H13CORE.1.P.B of width 11.\n",
      "Using motif -PPARG.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRD13.H13CORE.0.P.B of width 11.\n",
      "Using motif -PRD13.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.627841\n",
      "#   Estimated pi_0=0.627841\n",
      "Using motif +PRD14.H13CORE.0.P.B of width 15.\n",
      "Using motif -PRD14.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRD15.H13CORE.0.P.B of width 18.\n",
      "Using motif -PRD15.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRD16.H13CORE.0.P.B of width 11.\n",
      "Using motif -PRD16.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRDM1.H13CORE.0.S.B of width 12.\n",
      "Using motif -PRDM1.H13CORE.0.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRDM1.H13CORE.1.PS.A of width 14.\n",
      "Using motif -PRDM1.H13CORE.1.PS.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRDM10.H13CORE.0.PSGI.A of width 15.\n",
      "Using motif -PRDM10.H13CORE.0.PSGI.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRDM13.H13CORE.0.PSGI.A of width 12.\n",
      "Using motif -PRDM13.H13CORE.0.PSGI.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRDM4.H13CORE.0.PSM.A of width 20.\n",
      "Using motif -PRDM4.H13CORE.0.PSM.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRDM5.H13CORE.0.PSGI.A of width 26.\n",
      "Using motif -PRDM5.H13CORE.0.PSGI.A of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRDM6.H13CORE.0.P.B of width 12.\n",
      "Using motif -PRDM6.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRDM9.H13CORE.0.P.B of width 20.\n",
      "Using motif -PRDM9.H13CORE.0.P.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRGR.H13CORE.0.P.B of width 15.\n",
      "Using motif -PRGR.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRGR.H13CORE.1.P.B of width 7.\n",
      "Using motif -PRGR.H13CORE.1.P.B of width 7.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PROP1.H13CORE.0.PSM.A of width 16.\n",
      "Using motif -PROP1.H13CORE.0.PSM.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PROP1.H13CORE.1.P.C of width 10.\n",
      "Using motif -PROP1.H13CORE.1.P.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PROX1.H13CORE.0.SM.B of width 13.\n",
      "Using motif -PROX1.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRRX1.H13CORE.0.S.B of width 13.\n",
      "Using motif -PRRX1.H13CORE.0.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRRX1.H13CORE.1.S.B of width 17.\n",
      "Using motif -PRRX1.H13CORE.1.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRRX2.H13CORE.0.SM.B of width 10.\n",
      "Using motif -PRRX2.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PRRX2.H13CORE.1.S.B of width 16.\n",
      "Using motif -PRRX2.H13CORE.1.S.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PTF1A.H13CORE.0.P.B of width 19.\n",
      "Using motif -PTF1A.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PTF1A.H13CORE.1.PS.A of width 10.\n",
      "Using motif -PTF1A.H13CORE.1.PS.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +PURB.H13CORE.0.B.B of width 10.\n",
      "Using motif -PURB.H13CORE.0.B.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RARA.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -RARA.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RARA.H13CORE.1.P.B of width 19.\n",
      "Using motif -RARA.H13CORE.1.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RARA.H13CORE.2.P.B of width 17.\n",
      "Using motif -RARA.H13CORE.2.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RARA.H13CORE.3.P.B of width 21.\n",
      "Using motif -RARA.H13CORE.3.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RARB.H13CORE.0.SM.B of width 10.\n",
      "Using motif -RARB.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RARB.H13CORE.1.P.B of width 12.\n",
      "Using motif -RARB.H13CORE.1.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RARG.H13CORE.0.P.B of width 18.\n",
      "Using motif -RARG.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RARG.H13CORE.1.PSM.A of width 11.\n",
      "Using motif -RARG.H13CORE.1.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RARG.H13CORE.2.SM.B of width 17.\n",
      "Using motif -RARG.H13CORE.2.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RARG.H13CORE.3.S.B of width 17.\n",
      "Using motif -RARG.H13CORE.3.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RAX2.H13CORE.0.S.B of width 14.\n",
      "Using motif -RAX2.H13CORE.0.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RBAK.H13CORE.0.P.C of width 16.\n",
      "Using motif -RBAK.H13CORE.0.P.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +REL.H13CORE.0.P.B of width 15.\n",
      "Using motif -REL.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RELB.H13CORE.0.P.B of width 11.\n",
      "Using motif -RELB.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +REST.H13CORE.0.P.B of width 22.\n",
      "Using motif -REST.H13CORE.0.P.B of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RFX1.H13CORE.0.PM.A of width 12.\n",
      "Using motif -RFX1.H13CORE.0.PM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RFX1.H13CORE.1.PSM.A of width 18.\n",
      "Using motif -RFX1.H13CORE.1.PSM.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RFX1.H13CORE.2.S.C of width 11.\n",
      "Using motif -RFX1.H13CORE.2.S.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RFX2.H13CORE.0.PS.A of width 14.\n",
      "Using motif -RFX2.H13CORE.0.PS.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RFX2.H13CORE.1.PSM.A of width 16.\n",
      "Using motif -RFX2.H13CORE.1.PSM.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RFX3.H13CORE.0.PSM.A of width 14.\n",
      "Using motif -RFX3.H13CORE.0.PSM.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RFX3.H13CORE.1.PS.A of width 14.\n",
      "Using motif -RFX3.H13CORE.1.PS.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RFX3.H13CORE.2.S.B of width 20.\n",
      "Using motif -RFX3.H13CORE.2.S.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RFX4.H13CORE.0.SM.B of width 16.\n",
      "Using motif -RFX4.H13CORE.0.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RFX5.H13CORE.0.PSG.A of width 19.\n",
      "Using motif -RFX5.H13CORE.0.PSG.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RFX6.H13CORE.0.P.C of width 11.\n",
      "Using motif -RFX6.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.777699\n",
      "#   Estimated pi_0=0.777699\n",
      "Using motif +RFX7.H13CORE.0.SM.B of width 12.\n",
      "Using motif -RFX7.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RHXF1.H13CORE.0.SM.B of width 13.\n",
      "Using motif -RHXF1.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RHXF2.H13CORE.0.SM.B of width 11.\n",
      "Using motif -RHXF2.H13CORE.0.SM.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RLF.H13CORE.0.PSGI.A of width 11.\n",
      "Using motif -RLF.H13CORE.0.PSGI.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RORA.H13CORE.0.P.C of width 13.\n",
      "Using motif -RORA.H13CORE.0.P.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RORB.H13CORE.0.SM.B of width 17.\n",
      "Using motif -RORB.H13CORE.0.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RORG.H13CORE.0.M.C of width 20.\n",
      "Using motif -RORG.H13CORE.0.M.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RORG.H13CORE.1.PS.A of width 12.\n",
      "Using motif -RORG.H13CORE.1.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RUNX1.H13CORE.0.P.B of width 10.\n",
      "Using motif -RUNX1.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RUNX2.H13CORE.0.P.B of width 13.\n",
      "Using motif -RUNX2.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RUNX2.H13CORE.1.S.B of width 11.\n",
      "Using motif -RUNX2.H13CORE.1.S.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RUNX2.H13CORE.2.M.C of width 18.\n",
      "Using motif -RUNX2.H13CORE.2.M.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RUNX3.H13CORE.0.P.B of width 11.\n",
      "Using motif -RUNX3.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RUNX3.H13CORE.1.S.B of width 19.\n",
      "Using motif -RUNX3.H13CORE.1.S.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RUNX3.H13CORE.2.S.B of width 11.\n",
      "Using motif -RUNX3.H13CORE.2.S.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RX.H13CORE.0.SM.B of width 16.\n",
      "Using motif -RX.H13CORE.0.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RXRA.H13CORE.0.PS.A of width 9.\n",
      "Using motif -RXRA.H13CORE.0.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RXRA.H13CORE.1.S.C of width 14.\n",
      "Using motif -RXRA.H13CORE.1.S.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RXRA.H13CORE.2.SM.B of width 15.\n",
      "Using motif -RXRA.H13CORE.2.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RXRA.H13CORE.3.P.B of width 21.\n",
      "Using motif -RXRA.H13CORE.3.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.520833\n",
      "#   Estimated pi_0=0.520833\n",
      "Using motif +RXRB.H13CORE.0.P.C of width 14.\n",
      "Using motif -RXRB.H13CORE.0.P.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RXRB.H13CORE.1.SM.B of width 17.\n",
      "Using motif -RXRB.H13CORE.1.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1280 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.725759\n",
      "#   Estimated pi_0=0.730537\n",
      "Using motif +RXRB.H13CORE.2.PS.A of width 11.\n",
      "Using motif -RXRB.H13CORE.2.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RXRG.H13CORE.0.P.B of width 10.\n",
      "Using motif -RXRG.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RXRG.H13CORE.1.SM.B of width 16.\n",
      "Using motif -RXRG.H13CORE.1.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +RXRG.H13CORE.2.P.B of width 23.\n",
      "Using motif -RXRG.H13CORE.2.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SALL1.H13CORE.0.P.C of width 16.\n",
      "Using motif -SALL1.H13CORE.0.P.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.647286\n",
      "#   Estimated pi_0=0.655493\n",
      "Using motif +SALL3.H13CORE.0.PSG.A of width 19.\n",
      "Using motif -SALL3.H13CORE.0.PSG.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SALL4.H13CORE.0.P.B of width 11.\n",
      "Using motif -SALL4.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.782454\n",
      "#   Estimated pi_0=0.783637\n",
      "Using motif +SATB1.H13CORE.0.P.B of width 19.\n",
      "Using motif -SATB1.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SATB1.H13CORE.1.P.B of width 12.\n",
      "Using motif -SATB1.H13CORE.1.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SCRT1.H13CORE.0.PSM.A of width 13.\n",
      "Using motif -SCRT1.H13CORE.0.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SCRT2.H13CORE.0.PSM.A of width 16.\n",
      "Using motif -SCRT2.H13CORE.0.PSM.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.872269\n",
      "#   Estimated pi_0=0.875466\n",
      "Using motif +SETBP1.H13CORE.0.B.B of width 14.\n",
      "Using motif -SETBP1.H13CORE.0.B.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SHOX.H13CORE.0.SM.B of width 19.\n",
      "Using motif -SHOX.H13CORE.0.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SHOX.H13CORE.1.S.B of width 13.\n",
      "Using motif -SHOX.H13CORE.1.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SHOX2.H13CORE.0.S.B of width 18.\n",
      "Using motif -SHOX2.H13CORE.0.S.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SHOX2.H13CORE.1.S.C of width 13.\n",
      "Using motif -SHOX2.H13CORE.1.S.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SIX1.H13CORE.0.P.B of width 14.\n",
      "Using motif -SIX1.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SIX1.H13CORE.1.SM.B of width 13.\n",
      "Using motif -SIX1.H13CORE.1.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SIX2.H13CORE.0.P.B of width 14.\n",
      "Using motif -SIX2.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SIX2.H13CORE.1.M.C of width 15.\n",
      "Using motif -SIX2.H13CORE.1.M.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SIX2.H13CORE.2.SM.B of width 9.\n",
      "Using motif -SIX2.H13CORE.2.SM.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SIX3.H13CORE.0.SM.B of width 9.\n",
      "Using motif -SIX3.H13CORE.0.SM.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SIX4.H13CORE.0.P.B of width 14.\n",
      "Using motif -SIX4.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SIX4.H13CORE.1.M.C of width 15.\n",
      "Using motif -SIX4.H13CORE.1.M.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SIX6.H13CORE.0.SM.B of width 9.\n",
      "Using motif -SIX6.H13CORE.0.SM.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SKOR1.H13CORE.0.M.C of width 18.\n",
      "Using motif -SKOR1.H13CORE.0.M.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SKOR1.H13CORE.1.S.C of width 16.\n",
      "Using motif -SKOR1.H13CORE.1.S.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SKOR2.H13CORE.0.SM.B of width 15.\n",
      "Using motif -SKOR2.H13CORE.0.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SLC2A4RG.H13CORE.0.PSG.A of width 10.\n",
      "Using motif -SLC2A4RG.H13CORE.0.PSG.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.126953\n",
      "#   Estimated pi_0=0.126953\n",
      "Using motif +SMAD2.H13CORE.0.P.B of width 12.\n",
      "Using motif -SMAD2.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SMAD2.H13CORE.1.P.B of width 10.\n",
      "Using motif -SMAD2.H13CORE.1.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SMAD3.H13CORE.0.S.B of width 11.\n",
      "Using motif -SMAD3.H13CORE.0.S.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SMAD3.H13CORE.1.P.B of width 9.\n",
      "Using motif -SMAD3.H13CORE.1.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SMAD3.H13CORE.2.P.C of width 14.\n",
      "Using motif -SMAD3.H13CORE.2.P.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SMAD4.H13CORE.0.P.B of width 8.\n",
      "Using motif -SMAD4.H13CORE.0.P.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SMAD4.H13CORE.1.P.B of width 13.\n",
      "Using motif -SMAD4.H13CORE.1.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SMAD5.H13CORE.0.M.C of width 10.\n",
      "Using motif -SMAD5.H13CORE.0.M.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SNAI1.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -SNAI1.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SNAI2.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -SNAI2.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SNAI3.H13CORE.0.S.C of width 11.\n",
      "Using motif -SNAI3.H13CORE.0.S.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOLH2.H13CORE.0.S.C of width 11.\n",
      "Using motif -SOLH2.H13CORE.0.S.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.632102\n",
      "#   Estimated pi_0=0.632102\n",
      "Using motif +SOX1.H13CORE.0.S.B of width 17.\n",
      "Using motif -SOX1.H13CORE.0.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX1.H13CORE.1.S.B of width 9.\n",
      "Using motif -SOX1.H13CORE.1.S.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX10.H13CORE.0.PSM.A of width 23.\n",
      "Using motif -SOX10.H13CORE.0.PSM.A of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX10.H13CORE.1.PSM.A of width 11.\n",
      "Using motif -SOX10.H13CORE.1.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX10.H13CORE.2.S.C of width 14.\n",
      "Using motif -SOX10.H13CORE.2.S.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX11.H13CORE.0.P.B of width 12.\n",
      "Using motif -SOX11.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX11.H13CORE.1.S.B of width 13.\n",
      "Using motif -SOX11.H13CORE.1.S.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX12.H13CORE.0.SM.B of width 14.\n",
      "Using motif -SOX12.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX13.H13CORE.0.P.B of width 12.\n",
      "Using motif -SOX13.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX14.H13CORE.0.SM.B of width 19.\n",
      "Using motif -SOX14.H13CORE.0.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX15.H13CORE.0.P.B of width 8.\n",
      "Using motif -SOX15.H13CORE.0.P.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX17.H13CORE.0.PS.A of width 12.\n",
      "Using motif -SOX17.H13CORE.0.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX17.H13CORE.1.P.B of width 15.\n",
      "Using motif -SOX17.H13CORE.1.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX17.H13CORE.2.S.C of width 18.\n",
      "Using motif -SOX17.H13CORE.2.S.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX18.H13CORE.0.SM.B of width 9.\n",
      "Using motif -SOX18.H13CORE.0.SM.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX2.H13CORE.0.P.B of width 12.\n",
      "Using motif -SOX2.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX2.H13CORE.1.P.B of width 17.\n",
      "Using motif -SOX2.H13CORE.1.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX2.H13CORE.2.S.B of width 10.\n",
      "Using motif -SOX2.H13CORE.2.S.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX2.H13CORE.3.S.B of width 12.\n",
      "Using motif -SOX2.H13CORE.3.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX21.H13CORE.0.S.B of width 18.\n",
      "Using motif -SOX21.H13CORE.0.S.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX3.H13CORE.0.PM.A of width 11.\n",
      "Using motif -SOX3.H13CORE.0.PM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX3.H13CORE.1.S.C of width 16.\n",
      "Using motif -SOX3.H13CORE.1.S.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX30.H13CORE.0.P.C of width 11.\n",
      "Using motif -SOX30.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX30.H13CORE.1.SM.B of width 10.\n",
      "Using motif -SOX30.H13CORE.1.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX4.H13CORE.0.PSM.A of width 13.\n",
      "Using motif -SOX4.H13CORE.0.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX5.H13CORE.0.P.B of width 9.\n",
      "Using motif -SOX5.H13CORE.0.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX6.H13CORE.0.P.B of width 10.\n",
      "Using motif -SOX6.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX7.H13CORE.0.PS.A of width 9.\n",
      "Using motif -SOX7.H13CORE.0.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX7.H13CORE.1.SM.B of width 17.\n",
      "Using motif -SOX7.H13CORE.1.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX8.H13CORE.0.PSM.A of width 17.\n",
      "Using motif -SOX8.H13CORE.0.PSM.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX8.H13CORE.1.PS.A of width 11.\n",
      "Using motif -SOX8.H13CORE.1.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX9.H13CORE.0.PS.A of width 21.\n",
      "Using motif -SOX9.H13CORE.0.PS.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX9.H13CORE.1.PS.A of width 10.\n",
      "Using motif -SOX9.H13CORE.1.PS.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SOX9.H13CORE.2.S.B of width 17.\n",
      "Using motif -SOX9.H13CORE.2.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SP1.H13CORE.0.P.B of width 14.\n",
      "Using motif -SP1.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.788225\n",
      "#   Estimated pi_0=0.795296\n",
      "Using motif +SP1.H13CORE.1.SM.B of width 14.\n",
      "Using motif -SP1.H13CORE.1.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.765306\n",
      "#   Estimated pi_0=0.770195\n",
      "Using motif +SP1.H13CORE.2.P.B of width 21.\n",
      "Using motif -SP1.H13CORE.2.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.669643\n",
      "#   Estimated pi_0=0.671601\n",
      "Using motif +SP100.H13CORE.0.PSGI.A of width 11.\n",
      "Using motif -SP100.H13CORE.0.PSGI.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SP140.H13CORE.0.PSGIB.A of width 11.\n",
      "Using motif -SP140.H13CORE.0.PSGIB.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SP140L.H13CORE.0.PSGIB.A of width 9.\n",
      "Using motif -SP140L.H13CORE.0.PSGIB.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SP2.H13CORE.0.P.D of width 12.\n",
      "Using motif -SP2.H13CORE.0.P.D of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.738804\n",
      "#   Estimated pi_0=0.738804\n",
      "Using motif +SP2.H13CORE.1.P.B of width 21.\n",
      "Using motif -SP2.H13CORE.1.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.755381\n",
      "#   Estimated pi_0=0.756527\n",
      "Using motif +SP3.H13CORE.0.P.B of width 19.\n",
      "Using motif -SP3.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 768 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.535654\n",
      "#   Estimated pi_0=0.535775\n",
      "Using motif +SP3.H13CORE.1.S.B of width 11.\n",
      "Using motif -SP3.H13CORE.1.S.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.860574\n",
      "#   Estimated pi_0=0.864533\n",
      "Using motif +SP4.H13CORE.0.P.C of width 15.\n",
      "Using motif -SP4.H13CORE.0.P.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.565848\n",
      "#   Estimated pi_0=0.570191\n",
      "Using motif +SP4.H13CORE.1.S.B of width 12.\n",
      "Using motif -SP4.H13CORE.1.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.825264\n",
      "#   Estimated pi_0=0.82905\n",
      "Using motif +SP4.H13CORE.2.P.B of width 21.\n",
      "Using motif -SP4.H13CORE.2.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.71697\n",
      "#   Estimated pi_0=0.71697\n",
      "Using motif +SP5.H13CORE.0.P.B of width 14.\n",
      "Using motif -SP5.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.654468\n",
      "#   Estimated pi_0=0.658392\n",
      "Using motif +SP5.H13CORE.1.P.B of width 25.\n",
      "Using motif -SP5.H13CORE.1.P.B of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SP7.H13CORE.0.P.B of width 14.\n",
      "Using motif -SP7.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SP8.H13CORE.0.SM.B of width 12.\n",
      "Using motif -SP8.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SP9.H13CORE.0.P.B of width 14.\n",
      "Using motif -SP9.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SP9.H13CORE.1.S.C of width 12.\n",
      "Using motif -SP9.H13CORE.1.S.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SPDEF.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -SPDEF.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SPDEF.H13CORE.1.SM.B of width 20.\n",
      "Using motif -SPDEF.H13CORE.1.SM.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SPI1.H13CORE.0.P.B of width 14.\n",
      "Using motif -SPI1.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SPI1.H13CORE.1.S.B of width 12.\n",
      "Using motif -SPI1.H13CORE.1.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SPIB.H13CORE.0.P.B of width 16.\n",
      "Using motif -SPIB.H13CORE.0.P.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SPIB.H13CORE.1.S.C of width 13.\n",
      "Using motif -SPIB.H13CORE.1.S.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SPIB.H13CORE.2.SM.B of width 16.\n",
      "Using motif -SPIB.H13CORE.2.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SRBP1.H13CORE.0.P.B of width 12.\n",
      "Using motif -SRBP1.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SRBP1.H13CORE.1.S.C of width 12.\n",
      "Using motif -SRBP1.H13CORE.1.S.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.816406\n",
      "#   Estimated pi_0=0.831586\n",
      "Using motif +SRBP2.H13CORE.0.S.B of width 12.\n",
      "Using motif -SRBP2.H13CORE.0.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SRBP2.H13CORE.1.P.B of width 13.\n",
      "Using motif -SRBP2.H13CORE.1.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SRBP2.H13CORE.2.M.C of width 12.\n",
      "Using motif -SRBP2.H13CORE.2.M.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SRF.H13CORE.0.PSM.A of width 17.\n",
      "Using motif -SRF.H13CORE.0.PSM.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SRY.H13CORE.0.S.B of width 12.\n",
      "Using motif -SRY.H13CORE.0.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SRY.H13CORE.1.S.B of width 17.\n",
      "Using motif -SRY.H13CORE.1.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +STA5A.H13CORE.0.P.B of width 11.\n",
      "Using motif -STA5A.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +STA5B.H13CORE.0.P.B of width 13.\n",
      "Using motif -STA5B.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +STAT1.H13CORE.0.P.B of width 20.\n",
      "Using motif -STAT1.H13CORE.0.P.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +STAT1.H13CORE.1.P.B of width 18.\n",
      "Using motif -STAT1.H13CORE.1.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +STAT2.H13CORE.0.P.B of width 18.\n",
      "Using motif -STAT2.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +STAT3.H13CORE.0.P.B of width 13.\n",
      "Using motif -STAT3.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +STAT4.H13CORE.0.P.B of width 11.\n",
      "Using motif -STAT4.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +STAT6.H13CORE.0.P.B of width 13.\n",
      "Using motif -STAT6.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +STF1.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -STF1.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +SUH.H13CORE.0.P.B of width 11.\n",
      "Using motif -SUH.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.689131\n",
      "#   Estimated pi_0=0.695346\n",
      "Using motif +TAF1.H13CORE.0.P.B of width 15.\n",
      "Using motif -TAF1.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.384547\n",
      "#   Estimated pi_0=0.385259\n",
      "Using motif +TAL1.H13CORE.0.P.B of width 19.\n",
      "Using motif -TAL1.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TAL1.H13CORE.1.P.B of width 14.\n",
      "Using motif -TAL1.H13CORE.1.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TAL1.H13CORE.2.P.B of width 14.\n",
      "Using motif -TAL1.H13CORE.2.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBP.H13CORE.0.P.B of width 10.\n",
      "Using motif -TBP.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBR1.H13CORE.0.PS.A of width 18.\n",
      "Using motif -TBR1.H13CORE.0.PS.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX1.H13CORE.0.S.B of width 15.\n",
      "Using motif -TBX1.H13CORE.0.S.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX15.H13CORE.0.SM.B of width 13.\n",
      "Using motif -TBX15.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX18.H13CORE.0.M.C of width 12.\n",
      "Using motif -TBX18.H13CORE.0.M.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX19.H13CORE.0.PS.A of width 22.\n",
      "Using motif -TBX19.H13CORE.0.PS.A of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX19.H13CORE.1.SM.B of width 13.\n",
      "Using motif -TBX19.H13CORE.1.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX2.H13CORE.0.SM.B of width 10.\n",
      "Using motif -TBX2.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX20.H13CORE.0.P.C of width 14.\n",
      "Using motif -TBX20.H13CORE.0.P.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX20.H13CORE.1.M.C of width 11.\n",
      "Using motif -TBX20.H13CORE.1.M.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX20.H13CORE.2.SM.B of width 18.\n",
      "Using motif -TBX20.H13CORE.2.SM.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX21.H13CORE.0.PSM.A of width 13.\n",
      "Using motif -TBX21.H13CORE.0.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX3.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -TBX3.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX4.H13CORE.0.PS.A of width 9.\n",
      "Using motif -TBX4.H13CORE.0.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX4.H13CORE.1.S.B of width 23.\n",
      "Using motif -TBX4.H13CORE.1.S.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX5.H13CORE.0.PS.A of width 9.\n",
      "Using motif -TBX5.H13CORE.0.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX5.H13CORE.1.P.B of width 13.\n",
      "Using motif -TBX5.H13CORE.1.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX5.H13CORE.2.S.B of width 26.\n",
      "Using motif -TBX5.H13CORE.2.S.B of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBX6.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -TBX6.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBXT.H13CORE.0.PS.A of width 20.\n",
      "Using motif -TBXT.H13CORE.0.PS.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TBXT.H13CORE.1.PSM.A of width 12.\n",
      "Using motif -TBXT.H13CORE.1.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TCF20.H13CORE.0.B.B of width 10.\n",
      "Using motif -TCF20.H13CORE.0.B.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TCF21.H13CORE.0.PS.A of width 12.\n",
      "Using motif -TCF21.H13CORE.0.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TCF21.H13CORE.1.SM.B of width 12.\n",
      "Using motif -TCF21.H13CORE.1.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TCF7.H13CORE.0.PSM.A of width 15.\n",
      "Using motif -TCF7.H13CORE.0.PSM.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TEAD1.H13CORE.0.PSM.A of width 13.\n",
      "Using motif -TEAD1.H13CORE.0.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TEAD1.H13CORE.1.P.C of width 13.\n",
      "Using motif -TEAD1.H13CORE.1.P.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TEAD1.H13CORE.2.S.C of width 19.\n",
      "Using motif -TEAD1.H13CORE.2.S.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TEAD2.H13CORE.0.PSM.A of width 13.\n",
      "Using motif -TEAD2.H13CORE.0.PSM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TEAD3.H13CORE.0.PS.A of width 13.\n",
      "Using motif -TEAD3.H13CORE.0.PS.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TEAD3.H13CORE.1.S.B of width 18.\n",
      "Using motif -TEAD3.H13CORE.1.S.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TEAD4.H13CORE.0.PS.A of width 12.\n",
      "Using motif -TEAD4.H13CORE.0.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TEAD4.H13CORE.1.P.B of width 16.\n",
      "Using motif -TEAD4.H13CORE.1.P.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TEF.H13CORE.0.SM.B of width 13.\n",
      "Using motif -TEF.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TERF1.H13CORE.0.PSGB.A of width 8.\n",
      "Using motif -TERF1.H13CORE.0.PSGB.A of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TET3.H13CORE.0.PSGIB.A of width 18.\n",
      "Using motif -TET3.H13CORE.0.PSGIB.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.462891\n",
      "#   Estimated pi_0=0.462891\n",
      "Using motif +TF2L1.H13CORE.0.PSM.A of width 20.\n",
      "Using motif -TF2L1.H13CORE.0.PSM.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TF2L1.H13CORE.1.P.B of width 9.\n",
      "Using motif -TF2L1.H13CORE.1.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TF2LX.H13CORE.0.SM.B of width 8.\n",
      "Using motif -TF2LX.H13CORE.0.SM.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TF2LX.H13CORE.1.SM.B of width 16.\n",
      "Using motif -TF2LX.H13CORE.1.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.90019\n",
      "#   Estimated pi_0=0.90019\n",
      "Using motif +TF2LY.H13CORE.0.SM.B of width 14.\n",
      "Using motif -TF2LY.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TF2LY.H13CORE.1.SM.B of width 14.\n",
      "Using motif -TF2LY.H13CORE.1.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TF65.H13CORE.0.P.B of width 13.\n",
      "Using motif -TF65.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TF7L1.H13CORE.0.PM.A of width 12.\n",
      "Using motif -TF7L1.H13CORE.0.PM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TF7L1.H13CORE.1.P.C of width 12.\n",
      "Using motif -TF7L1.H13CORE.1.P.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TF7L1.H13CORE.2.S.C of width 17.\n",
      "Using motif -TF7L1.H13CORE.2.S.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TF7L2.H13CORE.0.P.B of width 12.\n",
      "Using motif -TF7L2.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TFAP4.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -TFAP4.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TFCP2.H13CORE.0.SM.B of width 26.\n",
      "Using motif -TFCP2.H13CORE.0.SM.B of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TFCP2.H13CORE.1.SM.B of width 21.\n",
      "Using motif -TFCP2.H13CORE.1.SM.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TFCP2.H13CORE.2.S.C of width 13.\n",
      "Using motif -TFCP2.H13CORE.2.S.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TFDP1.H13CORE.0.P.B of width 11.\n",
      "Using motif -TFDP1.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.330502\n",
      "#   Estimated pi_0=0.330502\n",
      "Using motif +TFE2.H13CORE.0.PS.A of width 10.\n",
      "Using motif -TFE2.H13CORE.0.PS.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.610026\n",
      "#   Estimated pi_0=0.610026\n",
      "Using motif +TFE3.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -TFE3.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.768717\n",
      "#   Estimated pi_0=0.768717\n",
      "Using motif +TFEB.H13CORE.0.SM.B of width 12.\n",
      "Using motif -TFEB.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.682031\n",
      "#   Estimated pi_0=0.682031\n",
      "Using motif +TFEB.H13CORE.1.P.C of width 11.\n",
      "Using motif -TFEB.H13CORE.1.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TFEC.H13CORE.0.M.B of width 14.\n",
      "Using motif -TFEC.H13CORE.0.M.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.787109\n",
      "#   Estimated pi_0=0.787109\n",
      "Using motif +TFEC.H13CORE.1.SM.B of width 13.\n",
      "Using motif -TFEC.H13CORE.1.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.807292\n",
      "#   Estimated pi_0=0.807292\n",
      "Using motif +TGIF1.H13CORE.0.PSM.A of width 8.\n",
      "Using motif -TGIF1.H13CORE.0.PSM.A of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TGIF1.H13CORE.1.S.B of width 17.\n",
      "Using motif -TGIF1.H13CORE.1.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TGIF2.H13CORE.0.S.B of width 16.\n",
      "Using motif -TGIF2.H13CORE.0.S.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TGIF2.H13CORE.1.PSM.A of width 10.\n",
      "Using motif -TGIF2.H13CORE.1.PSM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +THA.H13CORE.0.P.B of width 17.\n",
      "Using motif -THA.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +THA.H13CORE.1.PS.A of width 8.\n",
      "Using motif -THA.H13CORE.1.PS.A of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +THA.H13CORE.2.S.B of width 16.\n",
      "Using motif -THA.H13CORE.2.S.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +THAP1.H13CORE.0.P.B of width 23.\n",
      "Using motif -THAP1.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +THB.H13CORE.0.P.B of width 17.\n",
      "Using motif -THB.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +THB.H13CORE.1.PSM.A of width 9.\n",
      "Using motif -THB.H13CORE.1.PSM.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +THB.H13CORE.2.SM.B of width 14.\n",
      "Using motif -THB.H13CORE.2.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +THB.H13CORE.3.S.C of width 22.\n",
      "Using motif -THB.H13CORE.3.S.C of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TIGD3.H13CORE.0.PSGIB.A of width 17.\n",
      "Using motif -TIGD3.H13CORE.0.PSGIB.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TIGD4.H13CORE.0.PSGI.A of width 17.\n",
      "Using motif -TIGD4.H13CORE.0.PSGI.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TIGD5.H13CORE.0.SGI.A of width 18.\n",
      "Using motif -TIGD5.H13CORE.0.SGI.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TIGD7.H13CORE.0.SGIB.A of width 19.\n",
      "Using motif -TIGD7.H13CORE.0.SGIB.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TLX2.H13CORE.0.SM.B of width 12.\n",
      "Using motif -TLX2.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TLX3.H13CORE.0.SM.B of width 12.\n",
      "Using motif -TLX3.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TPRX1.H13CORE.0.PSGIB.A of width 15.\n",
      "Using motif -TPRX1.H13CORE.0.PSGIB.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TRPS1.H13CORE.0.P.B of width 8.\n",
      "Using motif -TRPS1.H13CORE.0.P.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TSHZ2.H13CORE.0.SG.A of width 15.\n",
      "Using motif -TSHZ2.H13CORE.0.SG.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TTF1.H13CORE.0.PSG.A of width 21.\n",
      "Using motif -TTF1.H13CORE.0.PSG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TWST1.H13CORE.0.P.B of width 18.\n",
      "Using motif -TWST1.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TWST1.H13CORE.1.P.B of width 16.\n",
      "Using motif -TWST1.H13CORE.1.P.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TWST2.H13CORE.0.P.B of width 19.\n",
      "Using motif -TWST2.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TWST2.H13CORE.1.P.B of width 10.\n",
      "Using motif -TWST2.H13CORE.1.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TYY1.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -TYY1.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TYY1.H13CORE.1.S.B of width 11.\n",
      "Using motif -TYY1.H13CORE.1.S.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TYY2.H13CORE.0.PS.A of width 12.\n",
      "Using motif -TYY2.H13CORE.0.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TYY2.H13CORE.1.SM.B of width 14.\n",
      "Using motif -TYY2.H13CORE.1.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +TZAP.H13CORE.0.P.B of width 14.\n",
      "Using motif -TZAP.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +UBIP1.H13CORE.0.SM.B of width 21.\n",
      "Using motif -UBIP1.H13CORE.0.SM.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +UNC4.H13CORE.0.S.B of width 15.\n",
      "Using motif -UNC4.H13CORE.0.S.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +UNC4.H13CORE.1.S.B of width 12.\n",
      "Using motif -UNC4.H13CORE.1.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +USF1.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -USF1.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.766859\n",
      "#   Estimated pi_0=0.766984\n",
      "Using motif +USF2.H13CORE.0.PSM.A of width 18.\n",
      "Using motif -USF2.H13CORE.0.PSM.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.675781\n",
      "#   Estimated pi_0=0.675781\n",
      "Using motif +USF3.H13CORE.0.PSGIB.A of width 12.\n",
      "Using motif -USF3.H13CORE.0.PSGIB.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.699219\n",
      "#   Estimated pi_0=0.699219\n",
      "Using motif +VAX1.H13CORE.0.SM.B of width 12.\n",
      "Using motif -VAX1.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +VAX2.H13CORE.0.SM.B of width 13.\n",
      "Using motif -VAX2.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +VDR.H13CORE.0.PS.A of width 16.\n",
      "Using motif -VDR.H13CORE.0.PS.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +VDR.H13CORE.1.PS.A of width 9.\n",
      "Using motif -VDR.H13CORE.1.PS.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +VENTX.H13CORE.0.SM.B of width 10.\n",
      "Using motif -VENTX.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +VENTX.H13CORE.1.S.C of width 18.\n",
      "Using motif -VENTX.H13CORE.1.S.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +VEZF1.H13CORE.0.P.C of width 11.\n",
      "Using motif -VEZF1.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 6 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.6957\n",
      "#   Estimated pi_0=0.701004\n",
      "Using motif +VEZF1.H13CORE.1.P.B of width 23.\n",
      "Using motif -VEZF1.H13CORE.1.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +VSX1.H13CORE.0.SM.B of width 12.\n",
      "Using motif -VSX1.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +VSX1.H13CORE.1.SM.B of width 17.\n",
      "Using motif -VSX1.H13CORE.1.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +VSX2.H13CORE.0.PSM.A of width 12.\n",
      "Using motif -VSX2.H13CORE.0.PSM.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +VSX2.H13CORE.1.S.B of width 17.\n",
      "Using motif -VSX2.H13CORE.1.S.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +WT1.H13CORE.0.P.B of width 10.\n",
      "Using motif -WT1.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.920003\n",
      "#   Estimated pi_0=0.930112\n",
      "Using motif +WT1.H13CORE.1.P.B of width 21.\n",
      "Using motif -WT1.H13CORE.1.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +XBP1.H13CORE.0.PS.A of width 12.\n",
      "Using motif -XBP1.H13CORE.0.PS.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +XBP1.H13CORE.1.M.B of width 14.\n",
      "Using motif -XBP1.H13CORE.1.M.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.829121\n",
      "#   Estimated pi_0=0.838529\n",
      "Using motif +YBOX1.H13CORE.0.SM.B of width 13.\n",
      "Using motif -YBOX1.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +Z280A.H13CORE.0.P.B of width 15.\n",
      "Using motif -Z280A.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 10 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.989168\n",
      "#   Estimated pi_0=0.991004\n",
      "Using motif +Z324A.H13CORE.0.P.B of width 24.\n",
      "Using motif -Z324A.H13CORE.0.P.B of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +Z354A.H13CORE.0.P.B of width 23.\n",
      "Using motif -Z354A.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +Z585A.H13CORE.0.P.C of width 23.\n",
      "Using motif -Z585A.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +Z585B.H13CORE.0.P.C of width 24.\n",
      "Using motif -Z585B.H13CORE.0.P.C of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +Z705G.H13CORE.0.P.C of width 22.\n",
      "Using motif -Z705G.H13CORE.0.P.C of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +Z780A.H13CORE.0.P.C of width 22.\n",
      "Using motif -Z780A.H13CORE.0.P.C of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBED1.H13CORE.0.S.B of width 14.\n",
      "Using motif -ZBED1.H13CORE.0.S.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBED2.H13CORE.0.PSGIB.A of width 9.\n",
      "Using motif -ZBED2.H13CORE.0.PSGIB.A of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3328 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 1.00039\n",
      "#   Estimated pi_0=1\n",
      "Using motif +ZBED4.H13CORE.0.SGIB.A of width 20.\n",
      "Using motif -ZBED4.H13CORE.0.SGIB.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBED5.H13CORE.0.PSGIB.A of width 10.\n",
      "Using motif -ZBED5.H13CORE.0.PSGIB.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT12.H13CORE.0.P.B of width 12.\n",
      "Using motif -ZBT12.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.777102\n",
      "#   Estimated pi_0=0.777174\n",
      "Using motif +ZBT14.H13CORE.0.P.C of width 13.\n",
      "Using motif -ZBT14.H13CORE.0.P.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 27 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.404242\n",
      "#   Estimated pi_0=0.412326\n",
      "Using motif +ZBT14.H13CORE.1.S.B of width 16.\n",
      "Using motif -ZBT14.H13CORE.1.S.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT16.H13CORE.0.P.B of width 10.\n",
      "Using motif -ZBT16.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT17.H13CORE.0.P.B of width 18.\n",
      "Using motif -ZBT17.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.686466\n",
      "#   Estimated pi_0=0.696791\n",
      "Using motif +ZBT18.H13CORE.0.PSM.A of width 11.\n",
      "Using motif -ZBT18.H13CORE.0.PSM.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT20.H13CORE.0.SM.B of width 14.\n",
      "Using motif -ZBT20.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT22.H13CORE.0.SM.B of width 18.\n",
      "Using motif -ZBT22.H13CORE.0.SM.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT24.H13CORE.0.P.B of width 20.\n",
      "Using motif -ZBT24.H13CORE.0.P.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 512 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.887784\n",
      "#   Estimated pi_0=0.887784\n",
      "Using motif +ZBT26.H13CORE.0.PSM.A of width 8.\n",
      "Using motif -ZBT26.H13CORE.0.PSM.A of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT26.H13CORE.1.SM.B of width 24.\n",
      "Using motif -ZBT26.H13CORE.1.SM.B of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT37.H13CORE.0.M.C of width 11.\n",
      "Using motif -ZBT37.H13CORE.0.M.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.760857\n",
      "#   Estimated pi_0=0.760857\n",
      "Using motif +ZBT40.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZBT40.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT42.H13CORE.0.P.C of width 11.\n",
      "Using motif -ZBT42.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT43.H13CORE.0.S.C of width 10.\n",
      "Using motif -ZBT43.H13CORE.0.S.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.804688\n",
      "#   Estimated pi_0=0.808968\n",
      "Using motif +ZBT44.H13CORE.0.P.C of width 17.\n",
      "Using motif -ZBT44.H13CORE.0.P.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT45.H13CORE.0.SM.B of width 13.\n",
      "Using motif -ZBT45.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT49.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZBT49.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBT7A.H13CORE.0.P.B of width 9.\n",
      "Using motif -ZBT7A.H13CORE.0.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3328 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.795118\n",
      "#   Estimated pi_0=0.81211\n",
      "Using motif +ZBT7A.H13CORE.1.SM.B of width 16.\n",
      "Using motif -ZBT7A.H13CORE.1.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.721331\n",
      "#   Estimated pi_0=0.73002\n",
      "Using motif +ZBT7B.H13CORE.0.SM.B of width 12.\n",
      "Using motif -ZBT7B.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.533594\n",
      "#   Estimated pi_0=0.533594\n",
      "Using motif +ZBT7C.H13CORE.0.S.B of width 10.\n",
      "Using motif -ZBT7C.H13CORE.0.S.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.734195\n",
      "#   Estimated pi_0=0.734195\n",
      "Using motif +ZBTB2.H13CORE.0.SM.B of width 12.\n",
      "Using motif -ZBTB2.H13CORE.0.SM.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBTB24.H13CORE.0.SG.A of width 15.\n",
      "Using motif -ZBTB24.H13CORE.0.SG.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBTB40.H13CORE.0.PSG.A of width 19.\n",
      "Using motif -ZBTB40.H13CORE.0.PSG.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBTB41.H13CORE.0.PSG.A of width 19.\n",
      "Using motif -ZBTB41.H13CORE.0.PSG.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 6 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 768 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.412116\n",
      "#   Estimated pi_0=0.416667\n",
      "Using motif +ZBTB47.H13CORE.0.PSG.A of width 20.\n",
      "Using motif -ZBTB47.H13CORE.0.PSG.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBTB5.H13CORE.0.SGI.A of width 14.\n",
      "Using motif -ZBTB5.H13CORE.0.SGI.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBTB6.H13CORE.0.P.B of width 15.\n",
      "Using motif -ZBTB6.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.550223\n",
      "#   Estimated pi_0=0.550223\n",
      "Using motif +ZBTB8A.H13CORE.0.PSGI.A of width 20.\n",
      "Using motif -ZBTB8A.H13CORE.0.PSGI.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZBTB8B.H13CORE.0.PSGI.A of width 18.\n",
      "Using motif -ZBTB8B.H13CORE.0.PSGI.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.634282\n",
      "#   Estimated pi_0=0.646617\n",
      "Using motif +ZEB1.H13CORE.0.P.B of width 11.\n",
      "Using motif -ZEB1.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.735795\n",
      "#   Estimated pi_0=0.745069\n",
      "Using motif +ZEB2.H13CORE.0.P.B of width 8.\n",
      "Using motif -ZEB2.H13CORE.0.P.B of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZF64B.H13CORE.0.P.B of width 16.\n",
      "Using motif -ZF64B.H13CORE.0.P.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZF64B.H13CORE.1.P.B of width 14.\n",
      "Using motif -ZF64B.H13CORE.1.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.46582\n",
      "#   Estimated pi_0=0.468943\n",
      "Using motif +ZF69B.H13CORE.0.P.C of width 18.\n",
      "Using motif -ZF69B.H13CORE.0.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.914352\n",
      "#   Estimated pi_0=0.918463\n",
      "Using motif +ZFAT.H13CORE.0.PSGI.A of width 16.\n",
      "Using motif -ZFAT.H13CORE.0.PSGI.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFP1.H13CORE.0.PSM.A of width 23.\n",
      "Using motif -ZFP1.H13CORE.0.PSM.A of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFP14.H13CORE.0.P.C of width 18.\n",
      "Using motif -ZFP14.H13CORE.0.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFP28.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZFP28.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFP3.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZFP3.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFP37.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZFP37.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFP41.H13CORE.0.SM.B of width 18.\n",
      "Using motif -ZFP41.H13CORE.0.SM.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFP42.H13CORE.0.PM.A of width 13.\n",
      "Using motif -ZFP42.H13CORE.0.PM.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFP57.H13CORE.0.P.B of width 9.\n",
      "Using motif -ZFP57.H13CORE.0.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 12 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3328 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.461538\n",
      "#   Estimated pi_0=0.461538\n",
      "Using motif +ZFP69.H13CORE.0.P.C of width 19.\n",
      "Using motif -ZFP69.H13CORE.0.P.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFP82.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZFP82.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFP90.H13CORE.0.P.C of width 13.\n",
      "Using motif -ZFP90.H13CORE.0.P.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFP90.H13CORE.1.P.C of width 23.\n",
      "Using motif -ZFP90.H13CORE.1.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFP91.H13CORE.0.SGI.A of width 21.\n",
      "Using motif -ZFP91.H13CORE.0.SGI.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFPM1.H13CORE.0.I.C of width 9.\n",
      "Using motif -ZFPM1.H13CORE.0.I.C of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZFTA.H13CORE.0.PSGIB.A of width 11.\n",
      "Using motif -ZFTA.H13CORE.0.PSGIB.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.433949\n",
      "#   Estimated pi_0=0.433949\n",
      "Using motif +ZFX.H13CORE.0.P.B of width 10.\n",
      "Using motif -ZFX.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZGLP1.H13CORE.0.SGB.A of width 14.\n",
      "Using motif -ZGLP1.H13CORE.0.SGB.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZIC1.H13CORE.0.P.B of width 10.\n",
      "Using motif -ZIC1.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 15 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.610677\n",
      "#   Estimated pi_0=0.610677\n",
      "Using motif +ZIC1.H13CORE.1.M.B of width 16.\n",
      "Using motif -ZIC1.H13CORE.1.M.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 21 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.698061\n",
      "#   Estimated pi_0=0.707566\n",
      "Using motif +ZIC1.H13CORE.2.S.B of width 16.\n",
      "Using motif -ZIC1.H13CORE.2.S.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 20 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.605648\n",
      "#   Estimated pi_0=0.605648\n",
      "Using motif +ZIC2.H13CORE.0.P.B of width 15.\n",
      "Using motif -ZIC2.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 11 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.670759\n",
      "#   Estimated pi_0=0.682998\n",
      "Using motif +ZIC3.H13CORE.0.P.B of width 15.\n",
      "Using motif -ZIC3.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 12 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.473804\n",
      "#   Estimated pi_0=0.476122\n",
      "Using motif +ZIC3.H13CORE.1.S.B of width 16.\n",
      "Using motif -ZIC3.H13CORE.1.S.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 18 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.651042\n",
      "#   Estimated pi_0=0.651985\n",
      "Using motif +ZIC3.H13CORE.2.M.B of width 16.\n",
      "Using motif -ZIC3.H13CORE.2.M.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 21 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.65357\n",
      "#   Estimated pi_0=0.660342\n",
      "Using motif +ZIC4.H13CORE.0.SM.B of width 15.\n",
      "Using motif -ZIC4.H13CORE.0.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 19 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.684246\n",
      "#   Estimated pi_0=0.686583\n",
      "Using motif +ZIC5.H13CORE.0.P.B of width 15.\n",
      "Using motif -ZIC5.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 18 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.395889\n",
      "#   Estimated pi_0=0.400352\n",
      "Using motif +ZIC5.H13CORE.1.M.C of width 17.\n",
      "Using motif -ZIC5.H13CORE.1.M.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 15 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1280 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.595519\n",
      "#   Estimated pi_0=0.598958\n",
      "Using motif +ZIC5.H13CORE.2.S.C of width 17.\n",
      "Using motif -ZIC5.H13CORE.2.S.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 16 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1280 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.616554\n",
      "#   Estimated pi_0=0.62627\n",
      "Using motif +ZIK1.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZIK1.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZIM3.H13CORE.0.P.B of width 19.\n",
      "Using motif -ZIM3.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZKSC1.H13CORE.0.P.B of width 19.\n",
      "Using motif -ZKSC1.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZKSC5.H13CORE.0.P.C of width 13.\n",
      "Using motif -ZKSC5.H13CORE.0.P.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.930687\n",
      "#   Estimated pi_0=0.934289\n",
      "Using motif +ZKSC8.H13CORE.0.P.B of width 31.\n",
      "Using motif -ZKSC8.H13CORE.0.P.B of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZKSC8.H13CORE.1.P.B of width 15.\n",
      "Using motif -ZKSC8.H13CORE.1.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZKSCAN4.H13CORE.0.SG.A of width 18.\n",
      "Using motif -ZKSCAN4.H13CORE.0.SG.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN100.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN100.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN100.H13CORE.1.P.B of width 14.\n",
      "Using motif -ZN100.H13CORE.1.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.636719\n",
      "#   Estimated pi_0=0.651304\n",
      "Using motif +ZN101.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN101.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN112.H13CORE.0.P.C of width 24.\n",
      "Using motif -ZN112.H13CORE.0.P.C of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN121.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN121.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN124.H13CORE.0.P.C of width 12.\n",
      "Using motif -ZN124.H13CORE.0.P.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.73087\n",
      "#   Estimated pi_0=0.73087\n",
      "Using motif +ZN132.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN132.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN133.H13CORE.0.P.B of width 25.\n",
      "Using motif -ZN133.H13CORE.0.P.B of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN134.H13CORE.0.P.C of width 17.\n",
      "Using motif -ZN134.H13CORE.0.P.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN134.H13CORE.1.P.C of width 22.\n",
      "Using motif -ZN134.H13CORE.1.P.C of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN135.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN135.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN136.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN136.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN140.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN140.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN140.H13CORE.1.SM.B of width 15.\n",
      "Using motif -ZN140.H13CORE.1.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN141.H13CORE.0.P.C of width 17.\n",
      "Using motif -ZN141.H13CORE.0.P.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN143.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN143.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN148.H13CORE.0.P.B of width 19.\n",
      "Using motif -ZN148.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN154.H13CORE.0.P.C of width 31.\n",
      "Using motif -ZN154.H13CORE.0.P.C of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN157.H13CORE.0.P.B of width 30.\n",
      "Using motif -ZN157.H13CORE.0.P.B of width 30.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN160.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN160.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN165.H13CORE.0.P.C of width 16.\n",
      "Using motif -ZN165.H13CORE.0.P.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.43059\n",
      "#   Estimated pi_0=0.43124\n",
      "Using motif +ZN174.H13CORE.0.M.C of width 17.\n",
      "Using motif -ZN174.H13CORE.0.M.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN177.H13CORE.0.SM.B of width 24.\n",
      "Using motif -ZN177.H13CORE.0.SM.B of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN180.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN180.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN184.H13CORE.0.P.B of width 31.\n",
      "Using motif -ZN184.H13CORE.0.P.B of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN189.H13CORE.0.P.B of width 15.\n",
      "Using motif -ZN189.H13CORE.0.P.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.716412\n",
      "#   Estimated pi_0=0.718667\n",
      "Using motif +ZN189.H13CORE.1.P.B of width 24.\n",
      "Using motif -ZN189.H13CORE.1.P.B of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN197.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN197.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN205.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN205.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN211.H13CORE.0.P.B of width 13.\n",
      "Using motif -ZN211.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN211.H13CORE.1.P.C of width 23.\n",
      "Using motif -ZN211.H13CORE.1.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN213.H13CORE.0.P.C of width 17.\n",
      "Using motif -ZN213.H13CORE.0.P.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN214.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN214.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN222.H13CORE.0.P.C of width 12.\n",
      "Using motif -ZN222.H13CORE.0.P.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN223.H13CORE.0.P.C of width 16.\n",
      "Using motif -ZN223.H13CORE.0.P.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 4 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.497628\n",
      "#   Estimated pi_0=0.497628\n",
      "Using motif +ZN224.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN224.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN239.H13CORE.0.P.C of width 28.\n",
      "Using motif -ZN239.H13CORE.0.P.C of width 28.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN248.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN248.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN250.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN250.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN250.H13CORE.1.M.C of width 11.\n",
      "Using motif -ZN250.H13CORE.1.M.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN251.H13CORE.0.P.C of width 19.\n",
      "Using motif -ZN251.H13CORE.0.P.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN254.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZN254.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN257.H13CORE.0.P.B of width 12.\n",
      "Using motif -ZN257.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN260.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN260.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN263.H13CORE.0.PM.A of width 10.\n",
      "Using motif -ZN263.H13CORE.0.PM.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 17 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.586576\n",
      "#   Estimated pi_0=0.596401\n",
      "Using motif +ZN263.H13CORE.1.P.B of width 21.\n",
      "Using motif -ZN263.H13CORE.1.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.418527\n",
      "#   Estimated pi_0=0.418527\n",
      "Using motif +ZN264.H13CORE.0.P.B of width 18.\n",
      "Using motif -ZN264.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN266.H13CORE.0.P.B of width 21.\n",
      "Using motif -ZN266.H13CORE.0.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN267.H13CORE.0.P.C of width 31.\n",
      "Using motif -ZN267.H13CORE.0.P.C of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN273.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN273.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN274.H13CORE.0.SM.B of width 14.\n",
      "Using motif -ZN274.H13CORE.0.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN274.H13CORE.1.P.B of width 21.\n",
      "Using motif -ZN274.H13CORE.1.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN274.H13CORE.2.P.B of width 21.\n",
      "Using motif -ZN274.H13CORE.2.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN276.H13CORE.0.SM.B of width 26.\n",
      "Using motif -ZN276.H13CORE.0.SM.B of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN281.H13CORE.0.P.B of width 16.\n",
      "Using motif -ZN281.H13CORE.0.P.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN281.H13CORE.1.SM.B of width 26.\n",
      "Using motif -ZN281.H13CORE.1.SM.B of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN282.H13CORE.0.P.B of width 11.\n",
      "Using motif -ZN282.H13CORE.0.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 8 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.538352\n",
      "#   Estimated pi_0=0.541308\n",
      "Using motif +ZN283.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN283.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN284.H13CORE.0.P.C of width 16.\n",
      "Using motif -ZN284.H13CORE.0.P.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN285.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN285.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN296.H13CORE.0.S.C of width 11.\n",
      "Using motif -ZN296.H13CORE.0.S.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN302.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN302.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN304.H13CORE.0.P.C of width 31.\n",
      "Using motif -ZN304.H13CORE.0.P.C of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN316.H13CORE.0.P.C of width 24.\n",
      "Using motif -ZN316.H13CORE.0.P.C of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN317.H13CORE.0.P.B of width 20.\n",
      "Using motif -ZN317.H13CORE.0.P.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN317.H13CORE.1.P.B of width 23.\n",
      "Using motif -ZN317.H13CORE.1.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN320.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN320.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN320.H13CORE.1.P.B of width 11.\n",
      "Using motif -ZN320.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN322.H13CORE.0.P.C of width 20.\n",
      "Using motif -ZN322.H13CORE.0.P.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 512 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.440903\n",
      "#   Estimated pi_0=0.440903\n",
      "Using motif +ZN329.H13CORE.0.P.C of width 22.\n",
      "Using motif -ZN329.H13CORE.0.P.C of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN329.H13CORE.1.P.C of width 12.\n",
      "Using motif -ZN329.H13CORE.1.P.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN331.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN331.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN333.H13CORE.0.P.C of width 11.\n",
      "Using motif -ZN333.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.535511\n",
      "#   Estimated pi_0=0.535511\n",
      "Using motif +ZN335.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN335.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN335.H13CORE.1.P.D of width 7.\n",
      "Using motif -ZN335.H13CORE.1.P.D of width 7.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3840 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 1.00241\n",
      "#   Estimated pi_0=1\n",
      "Using motif +ZN337.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN337.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN33B.H13CORE.0.P.C of width 30.\n",
      "Using motif -ZN33B.H13CORE.0.P.C of width 30.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN341.H13CORE.0.P.B of width 10.\n",
      "Using motif -ZN341.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN341.H13CORE.1.P.B of width 21.\n",
      "Using motif -ZN341.H13CORE.1.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN343.H13CORE.0.PSM.A of width 15.\n",
      "Using motif -ZN343.H13CORE.0.PSM.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.754464\n",
      "#   Estimated pi_0=0.754464\n",
      "Using motif +ZN350.H13CORE.0.P.B of width 18.\n",
      "Using motif -ZN350.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN362.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN362.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN37A.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN37A.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN382.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN382.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN383.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN383.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN384.H13CORE.0.PSM.A of width 17.\n",
      "Using motif -ZN384.H13CORE.0.PSM.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN394.H13CORE.0.P.C of width 19.\n",
      "Using motif -ZN394.H13CORE.0.P.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN407.H13CORE.0.P.B of width 25.\n",
      "Using motif -ZN407.H13CORE.0.P.B of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN410.H13CORE.0.SM.B of width 15.\n",
      "Using motif -ZN410.H13CORE.0.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN416.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN416.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN417.H13CORE.0.P.C of width 15.\n",
      "Using motif -ZN417.H13CORE.0.P.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN418.H13CORE.0.P.B of width 18.\n",
      "Using motif -ZN418.H13CORE.0.P.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN423.H13CORE.0.P.C of width 14.\n",
      "Using motif -ZN423.H13CORE.0.P.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 15 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.352539\n",
      "#   Estimated pi_0=0.352539\n",
      "Using motif +ZN425.H13CORE.0.P.C of width 9.\n",
      "Using motif -ZN425.H13CORE.0.P.C of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3328 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.619815\n",
      "#   Estimated pi_0=0.619815\n",
      "Using motif +ZN429.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN429.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN430.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN430.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN431.H13CORE.0.P.C of width 15.\n",
      "Using motif -ZN431.H13CORE.0.P.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.923754\n",
      "#   Estimated pi_0=0.925832\n",
      "Using motif +ZN432.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN432.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN433.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN433.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN433.H13CORE.1.P.C of width 31.\n",
      "Using motif -ZN433.H13CORE.1.P.C of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN436.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN436.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN440.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN440.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN441.H13CORE.0.P.C of width 10.\n",
      "Using motif -ZN441.H13CORE.0.P.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN441.H13CORE.1.P.C of width 31.\n",
      "Using motif -ZN441.H13CORE.1.P.C of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN443.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN443.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN444.H13CORE.0.S.C of width 18.\n",
      "Using motif -ZN444.H13CORE.0.S.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.619141\n",
      "#   Estimated pi_0=0.619141\n",
      "Using motif +ZN449.H13CORE.0.P.B of width 10.\n",
      "Using motif -ZN449.H13CORE.0.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN449.H13CORE.1.S.C of width 15.\n",
      "Using motif -ZN449.H13CORE.1.S.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 20 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.75258\n",
      "#   Estimated pi_0=0.755148\n",
      "Using motif +ZN454.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN454.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN454.H13CORE.1.SM.B of width 14.\n",
      "Using motif -ZN454.H13CORE.1.SM.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN460.H13CORE.0.S.C of width 13.\n",
      "Using motif -ZN460.H13CORE.0.S.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.69014\n",
      "#   Estimated pi_0=0.697075\n",
      "Using motif +ZN467.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN467.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN468.H13CORE.0.P.C of width 13.\n",
      "Using motif -ZN468.H13CORE.0.P.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 11 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.552951\n",
      "#   Estimated pi_0=0.554352\n",
      "Using motif +ZN479.H13CORE.0.P.C of width 18.\n",
      "Using motif -ZN479.H13CORE.0.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN480.H13CORE.0.P.C of width 24.\n",
      "Using motif -ZN480.H13CORE.0.P.C of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN480.H13CORE.1.P.C of width 10.\n",
      "Using motif -ZN480.H13CORE.1.P.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN483.H13CORE.0.P.C of width 18.\n",
      "Using motif -ZN483.H13CORE.0.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN484.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN484.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN485.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN485.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN490.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN490.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN492.H13CORE.0.P.C of width 20.\n",
      "Using motif -ZN492.H13CORE.0.P.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN502.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZN502.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN506.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN506.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN510.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN510.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN510.H13CORE.1.P.C of width 11.\n",
      "Using motif -ZN510.H13CORE.1.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN519.H13CORE.0.P.C of width 20.\n",
      "Using motif -ZN519.H13CORE.0.P.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN524.H13CORE.0.S.B of width 12.\n",
      "Using motif -ZN524.H13CORE.0.S.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN527.H13CORE.0.P.C of width 18.\n",
      "Using motif -ZN527.H13CORE.0.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN528.H13CORE.0.P.B of width 20.\n",
      "Using motif -ZN528.H13CORE.0.P.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 512 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.73574\n",
      "#   Estimated pi_0=0.73574\n",
      "Using motif +ZN529.H13CORE.0.P.B of width 31.\n",
      "Using motif -ZN529.H13CORE.0.P.B of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN530.H13CORE.0.P.B of width 17.\n",
      "Using motif -ZN530.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1280 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.442188\n",
      "#   Estimated pi_0=0.442188\n",
      "Using motif +ZN534.H13CORE.0.P.C of width 18.\n",
      "Using motif -ZN534.H13CORE.0.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.602366\n",
      "#   Estimated pi_0=0.602366\n",
      "Using motif +ZN534.H13CORE.1.P.C of width 31.\n",
      "Using motif -ZN534.H13CORE.1.P.C of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN540.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN540.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN543.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZN543.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 6 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.170173\n",
      "#   Estimated pi_0=0.17663\n",
      "Using motif +ZN547.H13CORE.0.P.B of width 22.\n",
      "Using motif -ZN547.H13CORE.0.P.B of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN548.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN548.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN549.H13CORE.0.P.B of width 22.\n",
      "Using motif -ZN549.H13CORE.0.P.B of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN550.H13CORE.0.P.C of width 31.\n",
      "Using motif -ZN550.H13CORE.0.P.C of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN554.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZN554.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN554.H13CORE.1.P.C of width 10.\n",
      "Using motif -ZN554.H13CORE.1.P.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.868465\n",
      "#   Estimated pi_0=0.871156\n",
      "Using motif +ZN557.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN557.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN558.H13CORE.0.P.B of width 25.\n",
      "Using motif -ZN558.H13CORE.0.P.B of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN560.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN560.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN561.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN561.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN562.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN562.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN563.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZN563.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN563.H13CORE.1.P.C of width 15.\n",
      "Using motif -ZN563.H13CORE.1.P.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.738636\n",
      "#   Estimated pi_0=0.738636\n",
      "Using motif +ZN564.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN564.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN565.H13CORE.0.P.C of width 31.\n",
      "Using motif -ZN565.H13CORE.0.P.C of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN566.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN566.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN567.H13CORE.0.P.C of width 30.\n",
      "Using motif -ZN567.H13CORE.0.P.C of width 30.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN570.H13CORE.0.P.C of width 19.\n",
      "Using motif -ZN570.H13CORE.0.P.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN573.H13CORE.0.P.C of width 24.\n",
      "Using motif -ZN573.H13CORE.0.P.C of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN574.H13CORE.0.P.B of width 13.\n",
      "Using motif -ZN574.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.565104\n",
      "#   Estimated pi_0=0.566385\n",
      "Using motif +ZN577.H13CORE.0.P.B of width 19.\n",
      "Using motif -ZN577.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN578.H13CORE.0.P.C of width 24.\n",
      "Using motif -ZN578.H13CORE.0.P.C of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN580.H13CORE.0.SM.B of width 10.\n",
      "Using motif -ZN580.H13CORE.0.SM.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN580.H13CORE.1.SM.B of width 17.\n",
      "Using motif -ZN580.H13CORE.1.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN582.H13CORE.0.P.C of width 11.\n",
      "Using motif -ZN582.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN584.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN584.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN584.H13CORE.1.P.C of width 14.\n",
      "Using motif -ZN584.H13CORE.1.P.C of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN586.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN586.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN587.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN587.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN595.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN595.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN605.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN605.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN610.H13CORE.0.P.B of width 14.\n",
      "Using motif -ZN610.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 7 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.623011\n",
      "#   Estimated pi_0=0.628506\n",
      "Using motif +ZN610.H13CORE.1.P.C of width 23.\n",
      "Using motif -ZN610.H13CORE.1.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN611.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN611.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN611.H13CORE.1.P.C of width 25.\n",
      "Using motif -ZN611.H13CORE.1.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN613.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN613.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN614.H13CORE.0.P.C of width 16.\n",
      "Using motif -ZN614.H13CORE.0.P.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN615.H13CORE.0.P.C of width 20.\n",
      "Using motif -ZN615.H13CORE.0.P.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN616.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN616.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN621.H13CORE.0.P.C of width 19.\n",
      "Using motif -ZN621.H13CORE.0.P.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN623.H13CORE.0.P.C of width 31.\n",
      "Using motif -ZN623.H13CORE.0.P.C of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN626.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZN626.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN627.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN627.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN649.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN649.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN652.H13CORE.0.PS.A of width 11.\n",
      "Using motif -ZN652.H13CORE.0.PS.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN660.H13CORE.0.SM.B of width 17.\n",
      "Using motif -ZN660.H13CORE.0.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN662.H13CORE.0.P.C of width 22.\n",
      "Using motif -ZN662.H13CORE.0.P.C of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN664.H13CORE.0.P.C of width 19.\n",
      "Using motif -ZN664.H13CORE.0.P.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN664.H13CORE.1.P.C of width 11.\n",
      "Using motif -ZN664.H13CORE.1.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN667.H13CORE.0.P.C of width 20.\n",
      "Using motif -ZN667.H13CORE.0.P.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN667.H13CORE.1.P.C of width 10.\n",
      "Using motif -ZN667.H13CORE.1.P.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN669.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN669.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN669.H13CORE.1.P.C of width 12.\n",
      "Using motif -ZN669.H13CORE.1.P.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN671.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN671.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN674.H13CORE.0.P.C of width 17.\n",
      "Using motif -ZN674.H13CORE.0.P.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 7 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1280 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.51875\n",
      "#   Estimated pi_0=0.526961\n",
      "Using motif +ZN675.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN675.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN677.H13CORE.0.P.C of width 19.\n",
      "Using motif -ZN677.H13CORE.0.P.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN680.H13CORE.0.P.B of width 21.\n",
      "Using motif -ZN680.H13CORE.0.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN681.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZN681.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.554688\n",
      "#   Estimated pi_0=0.563063\n",
      "Using motif +ZN682.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN682.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN682.H13CORE.1.P.B of width 11.\n",
      "Using motif -ZN682.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN684.H13CORE.0.P.C of width 16.\n",
      "Using motif -ZN684.H13CORE.0.P.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.63815\n",
      "#   Estimated pi_0=0.642193\n",
      "Using motif +ZN684.H13CORE.1.M.C of width 15.\n",
      "Using motif -ZN684.H13CORE.1.M.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.927049\n",
      "#   Estimated pi_0=0.929847\n",
      "Using motif +ZN692.H13CORE.0.P.C of width 11.\n",
      "Using motif -ZN692.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.509943\n",
      "#   Estimated pi_0=0.509943\n",
      "Using motif +ZN701.H13CORE.0.P.B of width 19.\n",
      "Using motif -ZN701.H13CORE.0.P.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN701.H13CORE.1.P.B of width 11.\n",
      "Using motif -ZN701.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN704.H13CORE.0.SM.B of width 16.\n",
      "Using motif -ZN704.H13CORE.0.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.455729\n",
      "#   Estimated pi_0=0.455729\n",
      "Using motif +ZN707.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN707.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN708.H13CORE.0.P.B of width 24.\n",
      "Using motif -ZN708.H13CORE.0.P.B of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN708.H13CORE.1.P.B of width 10.\n",
      "Using motif -ZN708.H13CORE.1.P.B of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.830552\n",
      "#   Estimated pi_0=0.845853\n",
      "Using motif +ZN711.H13CORE.0.P.C of width 13.\n",
      "Using motif -ZN711.H13CORE.0.P.C of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.763889\n",
      "#   Estimated pi_0=0.770863\n",
      "Using motif +ZN713.H13CORE.0.SM.B of width 20.\n",
      "Using motif -ZN713.H13CORE.0.SM.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN714.H13CORE.0.P.C of width 18.\n",
      "Using motif -ZN714.H13CORE.0.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.738281\n",
      "#   Estimated pi_0=0.751337\n",
      "Using motif +ZN716.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN716.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN716.H13CORE.1.P.C of width 11.\n",
      "Using motif -ZN716.H13CORE.1.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN730.H13CORE.0.P.C of width 15.\n",
      "Using motif -ZN730.H13CORE.0.P.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN736.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN736.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN740.H13CORE.0.P.C of width 11.\n",
      "Using motif -ZN740.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 10 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.702268\n",
      "#   Estimated pi_0=0.704764\n",
      "Using motif +ZN740.H13CORE.1.S.C of width 20.\n",
      "Using motif -ZN740.H13CORE.1.S.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 512 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.620404\n",
      "#   Estimated pi_0=0.622235\n",
      "Using motif +ZN749.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN749.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN749.H13CORE.1.P.C of width 9.\n",
      "Using motif -ZN749.H13CORE.1.P.C of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN75A.H13CORE.0.SM.B of width 15.\n",
      "Using motif -ZN75A.H13CORE.0.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.82676\n",
      "#   Estimated pi_0=0.82676\n",
      "Using motif +ZN75D.H13CORE.0.P.B of width 17.\n",
      "Using motif -ZN75D.H13CORE.0.P.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN75D.H13CORE.1.P.B of width 25.\n",
      "Using motif -ZN75D.H13CORE.1.P.B of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN765.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN765.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN766.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN766.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN766.H13CORE.1.P.B of width 11.\n",
      "Using motif -ZN766.H13CORE.1.P.B of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN768.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZN768.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN768.H13CORE.1.P.B of width 23.\n",
      "Using motif -ZN768.H13CORE.1.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN770.H13CORE.0.P.B of width 12.\n",
      "Using motif -ZN770.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.603906\n",
      "#   Estimated pi_0=0.603906\n",
      "Using motif +ZN770.H13CORE.1.P.B of width 22.\n",
      "Using motif -ZN770.H13CORE.1.P.B of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN771.H13CORE.0.SM.B of width 13.\n",
      "Using motif -ZN771.H13CORE.0.SM.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.898693\n",
      "#   Estimated pi_0=0.905616\n",
      "Using motif +ZN776.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN776.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN778.H13CORE.0.P.C of width 15.\n",
      "Using motif -ZN778.H13CORE.0.P.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN778.H13CORE.1.P.B of width 31.\n",
      "Using motif -ZN778.H13CORE.1.P.B of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN782.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN782.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN784.H13CORE.0.SM.B of width 16.\n",
      "Using motif -ZN784.H13CORE.0.SM.B of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN787.H13CORE.0.M.C of width 16.\n",
      "Using motif -ZN787.H13CORE.0.M.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN787.H13CORE.1.S.C of width 12.\n",
      "Using motif -ZN787.H13CORE.1.S.C of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN789.H13CORE.0.P.C of width 20.\n",
      "Using motif -ZN789.H13CORE.0.P.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN790.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN790.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN790.H13CORE.1.P.C of width 10.\n",
      "Using motif -ZN790.H13CORE.1.P.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN791.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN791.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN793.H13CORE.0.P.B of width 22.\n",
      "Using motif -ZN793.H13CORE.0.P.B of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN793.H13CORE.1.P.C of width 15.\n",
      "Using motif -ZN793.H13CORE.1.P.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN799.H13CORE.0.P.C of width 31.\n",
      "Using motif -ZN799.H13CORE.0.P.C of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN805.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN805.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN816.H13CORE.0.P.B of width 24.\n",
      "Using motif -ZN816.H13CORE.0.P.B of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN816.H13CORE.1.P.C of width 16.\n",
      "Using motif -ZN816.H13CORE.1.P.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.849515\n",
      "#   Estimated pi_0=0.850923\n",
      "Using motif +ZN821.H13CORE.0.SM.B of width 17.\n",
      "Using motif -ZN821.H13CORE.0.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN823.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN823.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN860.H13CORE.0.P.C of width 17.\n",
      "Using motif -ZN860.H13CORE.0.P.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 17 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1280 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.515625\n",
      "#   Estimated pi_0=0.515625\n",
      "Using motif +ZN875.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZN875.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN879.H13CORE.0.P.C of width 31.\n",
      "Using motif -ZN879.H13CORE.0.P.C of width 31.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN880.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZN880.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZN891.H13CORE.0.P.C of width 15.\n",
      "Using motif -ZN891.H13CORE.0.P.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.811086\n",
      "#   Estimated pi_0=0.812075\n",
      "Using motif +ZNF10.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZNF10.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF107.H13CORE.0.PSGI.A of width 21.\n",
      "Using motif -ZNF107.H13CORE.0.PSGI.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF12.H13CORE.0.P.C of width 24.\n",
      "Using motif -ZNF12.H13CORE.0.P.C of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF12.H13CORE.1.M.C of width 21.\n",
      "Using motif -ZNF12.H13CORE.1.M.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF12.H13CORE.2.S.C of width 18.\n",
      "Using motif -ZNF12.H13CORE.2.S.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF131.H13CORE.0.PSGI.A of width 14.\n",
      "Using motif -ZNF131.H13CORE.0.PSGI.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2048 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.489258\n",
      "#   Estimated pi_0=0.489258\n",
      "Using motif +ZNF14.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZNF14.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF142.H13CORE.0.PSGI.A of width 16.\n",
      "Using motif -ZNF142.H13CORE.0.PSGI.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1536 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.670512\n",
      "#   Estimated pi_0=0.670512\n",
      "Using motif +ZNF16.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZNF16.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF160.H13CORE.0.SG.A of width 21.\n",
      "Using motif -ZNF160.H13CORE.0.SG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF18.H13CORE.0.P.B of width 9.\n",
      "Using motif -ZNF18.H13CORE.0.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF20.H13CORE.0.PSG.A of width 14.\n",
      "Using motif -ZNF20.H13CORE.0.PSG.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF208.H13CORE.0.SGI.A of width 15.\n",
      "Using motif -ZNF208.H13CORE.0.SGI.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF215.H13CORE.0.PSG.A of width 11.\n",
      "Using motif -ZNF215.H13CORE.0.PSG.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.749531\n",
      "#   Estimated pi_0=0.754616\n",
      "Using motif +ZNF226.H13CORE.0.PSGI.A of width 21.\n",
      "Using motif -ZNF226.H13CORE.0.PSGI.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF229.H13CORE.0.SG.A of width 16.\n",
      "Using motif -ZNF229.H13CORE.0.SG.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF23.H13CORE.0.P.C of width 24.\n",
      "Using motif -ZNF23.H13CORE.0.P.C of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF233.H13CORE.0.PG.A of width 25.\n",
      "Using motif -ZNF233.H13CORE.0.PG.A of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF234.H13CORE.0.PSG.A of width 17.\n",
      "Using motif -ZNF234.H13CORE.0.PSG.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF24.H13CORE.0.P.B of width 14.\n",
      "Using motif -ZNF24.H13CORE.0.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF251.H13CORE.0.PSGI.A of width 21.\n",
      "Using motif -ZNF251.H13CORE.0.PSGI.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF26.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZNF26.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF26.H13CORE.1.P.B of width 14.\n",
      "Using motif -ZNF26.H13CORE.1.P.B of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF275.H13CORE.0.SG.A of width 19.\n",
      "Using motif -ZNF275.H13CORE.0.SG.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 768 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.837191\n",
      "#   Estimated pi_0=0.842313\n",
      "Using motif +ZNF28.H13CORE.0.P.C of width 18.\n",
      "Using motif -ZNF28.H13CORE.0.P.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1024 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.424757\n",
      "#   Estimated pi_0=0.430648\n",
      "Using motif +ZNF286B.H13CORE.0.PSG.A of width 17.\n",
      "Using motif -ZNF286B.H13CORE.0.PSG.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF292.H13CORE.0.PSG.A of width 20.\n",
      "Using motif -ZNF292.H13CORE.0.PSG.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF3.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZNF3.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF30.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZNF30.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF32.H13CORE.0.SM.B of width 15.\n",
      "Using motif -ZNF32.H13CORE.0.SM.B of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF335.H13CORE.0.PSG.A of width 12.\n",
      "Using motif -ZNF335.H13CORE.0.PSG.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF34.H13CORE.0.P.C of width 20.\n",
      "Using motif -ZNF34.H13CORE.0.P.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF347.H13CORE.0.PSGI.A of width 24.\n",
      "Using motif -ZNF347.H13CORE.0.PSGI.A of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF35.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZNF35.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF35.H13CORE.1.P.C of width 10.\n",
      "Using motif -ZNF35.H13CORE.1.P.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF358.H13CORE.0.SG.A of width 25.\n",
      "Using motif -ZNF358.H13CORE.0.SG.A of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF362.H13CORE.0.PSGI.A of width 11.\n",
      "Using motif -ZNF362.H13CORE.0.PSGI.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF367.H13CORE.0.PSGI.A of width 12.\n",
      "Using motif -ZNF367.H13CORE.0.PSGI.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF395.H13CORE.0.PSGI.A of width 10.\n",
      "Using motif -ZNF395.H13CORE.0.PSGI.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.235677\n",
      "#   Estimated pi_0=0.235922\n",
      "Using motif +ZNF407.H13CORE.0.PSGI.A of width 22.\n",
      "Using motif -ZNF407.H13CORE.0.PSGI.A of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF41.H13CORE.0.P.C of width 25.\n",
      "Using motif -ZNF41.H13CORE.0.P.C of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF41.H13CORE.1.P.C of width 15.\n",
      "Using motif -ZNF41.H13CORE.1.P.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.971503\n",
      "#   Estimated pi_0=0.974702\n",
      "Using motif +ZNF43.H13CORE.0.PSGI.A of width 12.\n",
      "Using motif -ZNF43.H13CORE.0.PSGI.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF44.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZNF44.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.436581\n",
      "#   Estimated pi_0=0.436581\n",
      "Using motif +ZNF470.H13CORE.0.PSG.A of width 22.\n",
      "Using motif -ZNF470.H13CORE.0.PSG.A of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF471.H13CORE.0.PSG.A of width 17.\n",
      "Using motif -ZNF471.H13CORE.0.PSG.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF48.H13CORE.0.PSG.A of width 25.\n",
      "Using motif -ZNF48.H13CORE.0.PSG.A of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF493.H13CORE.0.PSGI.A of width 26.\n",
      "Using motif -ZNF493.H13CORE.0.PSGI.A of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF497.H13CORE.0.SG.A of width 19.\n",
      "Using motif -ZNF497.H13CORE.0.SG.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF500.H13CORE.0.PSGI.A of width 18.\n",
      "Using motif -ZNF500.H13CORE.0.PSGI.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF503.H13CORE.0.P.B of width 13.\n",
      "Using motif -ZNF503.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 14 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2304 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.606828\n",
      "#   Estimated pi_0=0.610946\n",
      "Using motif +ZNF507.H13CORE.0.I.B of width 9.\n",
      "Using motif -ZNF507.H13CORE.0.I.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF510.H13CORE.0.PSG.A of width 25.\n",
      "Using motif -ZNF510.H13CORE.0.PSG.A of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF516.H13CORE.0.S.C of width 10.\n",
      "Using motif -ZNF516.H13CORE.0.S.C of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF518B.H13CORE.0.PSG.A of width 13.\n",
      "Using motif -ZNF518B.H13CORE.0.PSG.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF526.H13CORE.0.S.C of width 16.\n",
      "Using motif -ZNF526.H13CORE.0.S.C of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF532.H13CORE.0.SG.A of width 13.\n",
      "Using motif -ZNF532.H13CORE.0.SG.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF536.H13CORE.0.SGI.A of width 8.\n",
      "Using motif -ZNF536.H13CORE.0.SGI.A of width 8.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF551.H13CORE.0.PSG.A of width 24.\n",
      "Using motif -ZNF551.H13CORE.0.PSG.A of width 24.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF568.H13CORE.0.SG.A of width 18.\n",
      "Using motif -ZNF568.H13CORE.0.SG.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF569.H13CORE.0.SG.A of width 26.\n",
      "Using motif -ZNF569.H13CORE.0.SG.A of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF57.H13CORE.0.PSG.A of width 21.\n",
      "Using motif -ZNF57.H13CORE.0.PSG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF575.H13CORE.0.PSG.A of width 17.\n",
      "Using motif -ZNF575.H13CORE.0.PSG.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 6 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1280 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.671875\n",
      "#   Estimated pi_0=0.671875\n",
      "Using motif +ZNF587B.H13CORE.0.SG.A of width 14.\n",
      "Using motif -ZNF587B.H13CORE.0.SG.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF606.H13CORE.0.PSG.A of width 16.\n",
      "Using motif -ZNF606.H13CORE.0.PSG.A of width 16.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF618.H13CORE.0.G.B of width 20.\n",
      "Using motif -ZNF618.H13CORE.0.G.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF623.H13CORE.0.PG.A of width 21.\n",
      "Using motif -ZNF623.H13CORE.0.PG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF646.H13CORE.0.PSG.A of width 25.\n",
      "Using motif -ZNF646.H13CORE.0.PSG.A of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF648.H13CORE.0.PSGI.A of width 26.\n",
      "Using motif -ZNF648.H13CORE.0.PSGI.A of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF66.H13CORE.0.PG.A of width 15.\n",
      "Using motif -ZNF66.H13CORE.0.PG.A of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1792 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.904372\n",
      "#   Estimated pi_0=0.912511\n",
      "Using motif +ZNF665.H13CORE.0.PSG.A of width 26.\n",
      "Using motif -ZNF665.H13CORE.0.PSG.A of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF668.H13CORE.0.SG.A of width 18.\n",
      "Using motif -ZNF668.H13CORE.0.SG.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF672.H13CORE.0.PSG.A of width 26.\n",
      "Using motif -ZNF672.H13CORE.0.PSG.A of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF676.H13CORE.0.PSG.A of width 21.\n",
      "Using motif -ZNF676.H13CORE.0.PSG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.455097\n",
      "#   Estimated pi_0=0.458716\n",
      "Using motif +ZNF678.H13CORE.0.PSG.A of width 21.\n",
      "Using motif -ZNF678.H13CORE.0.PSG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF683.H13CORE.0.PSG.A of width 11.\n",
      "Using motif -ZNF683.H13CORE.0.PSG.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF689.H13CORE.0.PSG.A of width 21.\n",
      "Using motif -ZNF689.H13CORE.0.PSG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF696.H13CORE.0.PSG.A of width 23.\n",
      "Using motif -ZNF696.H13CORE.0.PSG.A of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF699.H13CORE.0.PSG.A of width 18.\n",
      "Using motif -ZNF699.H13CORE.0.PSG.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF7.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZNF7.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF70.H13CORE.0.PSG.A of width 25.\n",
      "Using motif -ZNF70.H13CORE.0.PSG.A of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF700.H13CORE.0.PSG.A of width 12.\n",
      "Using motif -ZNF700.H13CORE.0.PSG.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF703.H13CORE.0.P.C of width 7.\n",
      "Using motif -ZNF703.H13CORE.0.P.C of width 7.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF705EP.H13CORE.0.P.B of width 21.\n",
      "Using motif -ZNF705EP.H13CORE.0.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.676082\n",
      "#   Estimated pi_0=0.676082\n",
      "Using motif +ZNF709.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZNF709.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 3 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.82261\n",
      "#   Estimated pi_0=0.82261\n",
      "Using motif +ZNF721.H13CORE.0.PSG.A of width 17.\n",
      "Using motif -ZNF721.H13CORE.0.PSG.A of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF724.H13CORE.0.PSG.A of width 19.\n",
      "Using motif -ZNF724.H13CORE.0.PSG.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF726.H13CORE.0.PSGI.A of width 21.\n",
      "Using motif -ZNF726.H13CORE.0.PSGI.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF728.H13CORE.0.PSG.A of width 21.\n",
      "Using motif -ZNF728.H13CORE.0.PSG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.78125\n",
      "#   Estimated pi_0=0.78125\n",
      "Using motif +ZNF729.H13CORE.0.SG.A of width 12.\n",
      "Using motif -ZNF729.H13CORE.0.SG.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.764062\n",
      "#   Estimated pi_0=0.764062\n",
      "Using motif +ZNF732.H13CORE.0.PSG.A of width 20.\n",
      "Using motif -ZNF732.H13CORE.0.PSG.A of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF74.H13CORE.0.P.C of width 22.\n",
      "Using motif -ZNF74.H13CORE.0.P.C of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF746.H13CORE.0.PSG.A of width 26.\n",
      "Using motif -ZNF746.H13CORE.0.PSG.A of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF76.H13CORE.0.P.B of width 23.\n",
      "Using motif -ZNF76.H13CORE.0.P.B of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF76.H13CORE.1.SM.B of width 19.\n",
      "Using motif -ZNF76.H13CORE.1.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF77.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZNF77.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF772.H13CORE.0.PG.A of width 21.\n",
      "Using motif -ZNF772.H13CORE.0.PG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF773.H13CORE.0.PG.A of width 21.\n",
      "Using motif -ZNF773.H13CORE.0.PG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.710227\n",
      "#   Estimated pi_0=0.717118\n",
      "Using motif +ZNF775.H13CORE.0.PSG.A of width 14.\n",
      "Using motif -ZNF775.H13CORE.0.PSG.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF780B.H13CORE.0.PSGI.A of width 12.\n",
      "Using motif -ZNF780B.H13CORE.0.PSGI.A of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF788P.H13CORE.0.S.C of width 11.\n",
      "Using motif -ZNF788P.H13CORE.0.S.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF79.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZNF79.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF8.H13CORE.0.P.C of width 21.\n",
      "Using motif -ZNF8.H13CORE.0.P.C of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF8.H13CORE.1.P.B of width 22.\n",
      "Using motif -ZNF8.H13CORE.1.P.B of width 22.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF800.H13CORE.0.PSG.A of width 21.\n",
      "Using motif -ZNF800.H13CORE.0.PSG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 5 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.166869\n",
      "#   Estimated pi_0=0.166869\n",
      "Using motif +ZNF81.H13CORE.0.P.C of width 19.\n",
      "Using motif -ZNF81.H13CORE.0.P.C of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF813.H13CORE.0.SG.A of width 26.\n",
      "Using motif -ZNF813.H13CORE.0.SG.A of width 26.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF814.H13CORE.0.PSG.A of width 19.\n",
      "Using motif -ZNF814.H13CORE.0.PSG.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 768 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.635417\n",
      "#   Estimated pi_0=0.635417\n",
      "Using motif +ZNF827.H13CORE.0.SG.A of width 11.\n",
      "Using motif -ZNF827.H13CORE.0.SG.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF83.H13CORE.0.G.C of width 15.\n",
      "Using motif -ZNF83.H13CORE.0.G.C of width 15.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF831.H13CORE.0.PSGI.A of width 11.\n",
      "Using motif -ZNF831.H13CORE.0.PSGI.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF836.H13CORE.0.PSG.A of width 18.\n",
      "Using motif -ZNF836.H13CORE.0.PSG.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF841.H13CORE.0.PSGI.A of width 19.\n",
      "Using motif -ZNF841.H13CORE.0.PSGI.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF845.H13CORE.0.PSG.A of width 21.\n",
      "Using motif -ZNF845.H13CORE.0.PSG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 256 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.835938\n",
      "#   Estimated pi_0=0.83948\n",
      "Using motif +ZNF85.H13CORE.0.P.B of width 13.\n",
      "Using motif -ZNF85.H13CORE.0.P.B of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF85.H13CORE.1.P.B of width 20.\n",
      "Using motif -ZNF85.H13CORE.1.P.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF850.H13CORE.0.PSG.A of width 18.\n",
      "Using motif -ZNF850.H13CORE.0.PSG.A of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF853.H13CORE.0.SG.A of width 13.\n",
      "Using motif -ZNF853.H13CORE.0.SG.A of width 13.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF865.H13CORE.0.PSG.A of width 21.\n",
      "Using motif -ZNF865.H13CORE.0.PSG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF878.H13CORE.0.PS.A of width 19.\n",
      "Using motif -ZNF878.H13CORE.0.PS.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF888.H13CORE.0.P.B of width 9.\n",
      "Using motif -ZNF888.H13CORE.0.P.B of width 9.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF90.H13CORE.0.P.C of width 23.\n",
      "Using motif -ZNF90.H13CORE.0.P.C of width 23.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZNF92.H13CORE.0.P.B of width 21.\n",
      "Using motif -ZNF92.H13CORE.0.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZSA5A.H13CORE.0.SM.B of width 19.\n",
      "Using motif -ZSA5A.H13CORE.0.SM.B of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZSA5C.H13CORE.0.P.C of width 17.\n",
      "Using motif -ZSA5C.H13CORE.0.P.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZSC16.H13CORE.0.PSM.A of width 19.\n",
      "Using motif -ZSC16.H13CORE.0.PSM.A of width 19.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZSC21.H13CORE.0.P.C of width 11.\n",
      "Using motif -ZSC21.H13CORE.0.P.C of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZSC22.H13CORE.0.P.C of width 17.\n",
      "Using motif -ZSC22.H13CORE.0.P.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1280 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.42739\n",
      "#   Estimated pi_0=0.42739\n",
      "Using motif +ZSC23.H13CORE.0.M.C of width 18.\n",
      "Using motif -ZSC23.H13CORE.0.M.C of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZSC29.H13CORE.0.P.B of width 12.\n",
      "Using motif -ZSC29.H13CORE.0.P.B of width 12.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 1 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2560 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.667969\n",
      "#   Estimated pi_0=0.667969\n",
      "Using motif +ZSC29.H13CORE.1.M.C of width 20.\n",
      "Using motif -ZSC29.H13CORE.1.M.C of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZSC31.H13CORE.0.P.C of width 17.\n",
      "Using motif -ZSC31.H13CORE.0.P.C of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 6 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 1280 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.66369\n",
      "#   Estimated pi_0=0.66369\n",
      "Using motif +ZSC31.H13CORE.1.SM.B of width 20.\n",
      "Using motif -ZSC31.H13CORE.1.SM.B of width 20.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZSCA1.H13CORE.0.SM.B of width 18.\n",
      "Using motif -ZSCA1.H13CORE.0.SM.B of width 18.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZSCA4.H13CORE.0.SM.B of width 17.\n",
      "Using motif -ZSCA4.H13CORE.0.SM.B of width 17.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZSCAN12.H13CORE.0.P.B of width 21.\n",
      "Using motif -ZSCAN12.H13CORE.0.P.B of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZSCAN2.H13CORE.0.PG.A of width 25.\n",
      "Using motif -ZSCAN2.H13CORE.0.PG.A of width 25.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZSCAN25.H13CORE.0.PSG.A of width 21.\n",
      "Using motif -ZSCAN25.H13CORE.0.PSG.A of width 21.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "Using motif +ZXDA.H13CORE.0.PSI.A of width 11.\n",
      "Using motif -ZXDA.H13CORE.0.PSI.A of width 11.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 2816 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.316438\n",
      "#   Estimated pi_0=0.316438\n",
      "Using motif +ZXDB.H13CORE.0.PI.A of width 10.\n",
      "Using motif -ZXDB.H13CORE.0.PI.A of width 10.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 2 sequences.\n",
      "Computing q-values.\n",
      "# Computing q-values.\n",
      "#   Estimating pi_0 from a uniformly sampled set of 3072 p-values.\n",
      "#   Estimating pi_0.\n",
      "# Minimal pi_zero = 0.448843\n",
      "#   Estimated pi_0=0.455137\n",
      "Using motif +ZXDC.H13CORE.0.PI.A of width 14.\n",
      "Using motif -ZXDC.H13CORE.0.PI.A of width 14.\n",
      "Finding best site passing the output threshold in each of the 128 sequences.\n",
      "Found a best site in 0 sequences.\n",
      "Computing q-values.\n",
      "The output directory 'results/finding_tfbs_in_variants_H13' already exists.\n",
      "Its contents will not be overwritten.\n",
      "FATAL: Couldn't create output directory results/finding_tfbs_in_variants_H13.\n",
      "\n"
     ]
    }
   ],
   "source": [
    "runContainer(\"fimo --o results/finding_tfbs_in_variants_H13 resources/hocomoco_v13/H13CORE_meme_format.meme resources/clinvar_variants_fimo_input.fa\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d0b9c7b6",
   "metadata": {},
   "source": [
    "Finally, we load the output file and merge it with our variant metadata we used as input (for both alternative and reference sequences) to create a unified, filterable table. üìäüîó"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "7d9de1e3",
   "metadata": {},
   "outputs": [],
   "source": [
    "import os\n",
    "\n",
    "# Check out FIMO output\n",
    "fimo_input = \"results/finding_tfbs_in_variants_H13\"\n",
    "fimo_result = os.path.join(fimo_input, 'fimo.tsv')\n",
    "fimo_result_df = pd.read_csv(fimo_result, sep=\"\\t\", comment='#')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "71951af2",
   "metadata": {},
   "source": [
    "##### üìä FIMO Output Columns\n",
    "\n",
    "| Column           | Description                                      |\n",
    "|------------------|--------------------------------------------------|\n",
    "| `motif_id`       | ID of the matched motif üè∑Ô∏è                       |\n",
    "| `sequence_name`  | Name of the sequence with the motif üìù           |\n",
    "| `start`/`stop`   | Start/stop positions of the motif match üìç       |\n",
    "| `strand`         | Strand (+/-) ‚û°Ô∏è                                  |\n",
    "| `matched_sequence`| Sequence matching the motif üß¨                  |\n",
    "| `p-value`        | Motif match p-value (default threshold: 1e-4) üìâ |\n",
    "| `q-value`        | FDR-corrected p-value (higher confidence = lower) ‚úÖ |\n",
    "\n",
    "> **üí° Tip:** Lower q-values indicate higher confidence in motif matches."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "d996e975",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "motif_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "motif_alt_id",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "sequence_name",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "start",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "stop",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "strand",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "score",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "p-value",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "q-value",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "matched_sequence",
         "rawType": "object",
         "type": "string"
        }
       ],
       "ref": "13225dad-cc04-4b1e-b50a-ce962ffaa7d4",
       "rows": [
        [
         "1",
         "MAZ.H13CORE.1.P.B",
         null,
         "REF_5-1295075-G-A",
         "1",
         "21",
         "+",
         "21.0364",
         "2.68e-08",
         "1.43e-06",
         "AGAGGGCGGGGCCGCGGAAAG"
        ],
        [
         "2",
         "SP1.H13CORE.2.P.B",
         null,
         "REF_5-1295075-G-A",
         "1",
         "21",
         "+",
         "21.0164",
         "4.57e-08",
         "3.28e-06",
         "AGAGGGCGGGGCCGCGGAAAG"
        ],
        [
         "3",
         "SP4.H13CORE.2.P.B",
         null,
         "REF_5-1295075-G-A",
         "1",
         "21",
         "+",
         "20.2844",
         "6.31e-08",
         "4.54e-06",
         "AGAGGGCGGGGCCGCGGAAAG"
        ],
        [
         "11",
         "MAZ.H13CORE.1.P.B",
         null,
         "ALT_5-1295075-G-A",
         "1",
         "21",
         "+",
         "17.8182",
         "4.42e-07",
         "1.18e-05",
         "AGAGGGCGGGACCGCGGAAAG"
        ],
        [
         "0",
         "KLF1.H13CORE.0.P.B",
         null,
         "REF_5-1295075-G-A",
         "2",
         "14",
         "+",
         "18.9355",
         "2.39e-08",
         "1.72e-05",
         "GAGGGCGGGGCCG"
        ],
        [
         "19",
         "SP1.H13CORE.2.P.B",
         null,
         "ALT_5-1295075-G-A",
         "1",
         "21",
         "+",
         "17.1967",
         "7.83e-07",
         "2.81e-05",
         "AGAGGGCGGGACCGCGGAAAG"
        ],
        [
         "8",
         "KLF13.H13CORE.1.P.C",
         null,
         "REF_5-1295075-G-A",
         "1",
         "20",
         "+",
         "18.3091",
         "2.09e-07",
         "3.05e-05",
         "AGAGGGCGGGGCCGCGGAAA"
        ],
        [
         "4",
         "SP4.H13CORE.0.P.C",
         null,
         "REF_5-1295075-G-A",
         "1",
         "15",
         "+",
         "18.112",
         "6.9e-08",
         "3.15e-05",
         "AGAGGGCGGGGCCGC"
        ],
        [
         "22",
         "SP4.H13CORE.2.P.B",
         null,
         "ALT_5-1295075-G-A",
         "1",
         "21",
         "+",
         "16.8349",
         "9.69e-07",
         "3.48e-05",
         "AGAGGGCGGGACCGCGGAAAG"
        ],
        [
         "50",
         "ZNF800.H13CORE.0.PSG.A",
         null,
         "REF_5-1294995-G-T",
         "1",
         "21",
         "+",
         "14.8364",
         "3.75e-06",
         "4.72e-05",
         "GGCATCGCGGGGGTGGCCGGG"
        ],
        [
         "69",
         "ZNF800.H13CORE.0.PSG.A",
         null,
         "ALT_5-1294995-G-T",
         "1",
         "21",
         "+",
         "14.0455",
         "6.43e-06",
         "4.72e-05",
         "GGCATCGCGGTGGTGGCCGGG"
        ],
        [
         "5",
         "SP1.H13CORE.0.P.B",
         null,
         "REF_5-1295075-G-A",
         "1",
         "14",
         "+",
         "20.1967",
         "9.14e-08",
         "6.5e-05",
         "AGAGGGCGGGGCCG"
        ],
        [
         "6",
         "SP5.H13CORE.0.P.B",
         null,
         "REF_5-1295075-G-A",
         "1",
         "14",
         "+",
         "18.6854",
         "1.13e-07",
         "6.6e-05",
         "AGAGGGCGGGGCCG"
        ],
        [
         "25",
         "HINFP.H13CORE.0.P.B",
         null,
         "REF_5-1295135-G-A",
         "1",
         "20",
         "+",
         "16.5132",
         "1.2e-06",
         "7.05e-05",
         "GGGGACCCGGGAGGGGTCGG"
        ],
        [
         "14",
         "MYC.H13CORE.0.P.B",
         null,
         "REF_5-1295234-A-G",
         "1",
         "10",
         "-",
         "14.9839",
         "5.03e-07",
         "0.000116",
         "CCCACGTGGC"
        ],
        [
         "13",
         "MYC.H13CORE.0.P.B",
         null,
         "ALT_5-1295234-A-G",
         "1",
         "10",
         "-",
         "14.9839",
         "5.03e-07",
         "0.000116",
         "CCCACGTGGC"
        ],
        [
         "30",
         "KLF13.H13CORE.1.P.C",
         null,
         "ALT_5-1295075-G-A",
         "1",
         "20",
         "+",
         "15.9636",
         "1.59e-06",
         "0.000116",
         "AGAGGGCGGGACCGCGGAAA"
        ],
        [
         "15",
         "MYC.H13CORE.0.P.B",
         null,
         "REF_5-1295225-C-T",
         "10",
         "19",
         "-",
         "14.9839",
         "5.03e-07",
         "0.000116",
         "CCCACGTGGC"
        ],
        [
         "7",
         "KLF8.H13CORE.0.P.C",
         null,
         "REF_5-1295075-G-A",
         "1",
         "13",
         "+",
         "18.9213",
         "1.68e-07",
         "0.000116",
         "AGAGGGCGGGGCC"
        ],
        [
         "18",
         "KLF12.H13CORE.0.P.C",
         null,
         "REF_5-1295075-G-A",
         "3",
         "20",
         "+",
         "17.2347",
         "7.31e-07",
         "0.000169",
         "AGGGCGGGGCCGCGGAAA"
        ],
        [
         "65",
         "HINFP.H13CORE.0.P.B",
         null,
         "ALT_5-1295135-G-A",
         "1",
         "20",
         "+",
         "13.8684",
         "5.91e-06",
         "0.000173",
         "GGGGACCCGGAAGGGGTCGG"
        ],
        [
         "99",
         "CTCFL.H13CORE.0.P.B",
         null,
         "REF_5-1295135-G-A",
         "6",
         "21",
         "+",
         "13.2903",
         "9.58e-06",
         "0.000195",
         "CCCGGGAGGGGTCGGG"
        ],
        [
         "101",
         "CTCFL.H13CORE.0.P.B",
         null,
         "ALT_5-1295135-G-A",
         "6",
         "21",
         "+",
         "13.2581",
         "9.77e-06",
         "0.000195",
         "CCCGGAAGGGGTCGGG"
        ],
        [
         "98",
         "CTCFL.H13CORE.0.P.B",
         null,
         "REF_5-1295136-A-G",
         "5",
         "20",
         "+",
         "13.2903",
         "9.58e-06",
         "0.000195",
         "CCCGGGAGGGGTCGGG"
        ],
        [
         "127",
         "CTCFL.H13CORE.0.P.B",
         null,
         "REF_5-1295112-A-T",
         "5",
         "20",
         "+",
         "13.0",
         "1.13e-05",
         "0.000195",
         "GCCCGGAGGGGGCTGG"
        ],
        [
         "126",
         "CTCFL.H13CORE.0.P.B",
         null,
         "REF_5-1295113-G-A",
         "4",
         "19",
         "+",
         "13.0",
         "1.13e-05",
         "0.000195",
         "GCCCGGAGGGGGCTGG"
        ],
        [
         "29",
         "KLF15.H13CORE.1.P.C",
         null,
         "REF_5-1295075-G-A",
         "2",
         "21",
         "+",
         "15.9796",
         "1.48e-06",
         "0.000195",
         "GAGGGCGGGGCCGCGGAAAG"
        ],
        [
         "20",
         "SP4.H13CORE.0.P.C",
         null,
         "ALT_5-1295075-G-A",
         "1",
         "15",
         "+",
         "15.896",
         "8.7e-07",
         "0.000198",
         "AGAGGGCGGGACCGC"
        ],
        [
         "17",
         "MAZ.H13CORE.0.P.B",
         null,
         "ALT_5-1295163-C-G",
         "8",
         "20",
         "+",
         "16.0101",
         "5.85e-07",
         "0.000231",
         "GCGGGGAGGAGGC"
        ],
        [
         "24",
         "MAZ.H13CORE.0.P.B",
         null,
         "REF_5-1295075-G-A",
         "1",
         "13",
         "+",
         "15.5152",
         "1.12e-06",
         "0.000231",
         "AGAGGGCGGGGCC"
        ],
        [
         "10",
         "ZN468.H13CORE.0.P.C",
         null,
         "ALT_5-1295163-C-G",
         "9",
         "21",
         "+",
         "18.5246",
         "3.96e-07",
         "0.000236",
         "CGGGGAGGAGGCG"
        ],
        [
         "9",
         "SP2.H13CORE.0.P.D",
         null,
         "REF_5-1295075-G-A",
         "1",
         "12",
         "+",
         "18.1515",
         "3.14e-07",
         "0.000262",
         "AGAGGGCGGGGC"
        ],
        [
         "157",
         "HINFP.H13CORE.0.P.B",
         null,
         "ALT_5-1295181-A-G",
         "1",
         "20",
         "+",
         "12.1974",
         "1.46e-05",
         "0.000285",
         "GCGGAGCTGGGAGGTGAAGG"
        ],
        [
         "12",
         "ZN263.H13CORE.0.PM.A",
         null,
         "ALT_5-1295163-C-G",
         "10",
         "19",
         "+",
         "15.7236",
         "5.03e-07",
         "0.000412",
         "GGGGAGGAGG"
        ],
        [
         "16",
         "KLF3.H13CORE.1.M.C",
         null,
         "REF_5-1295075-G-A",
         "2",
         "14",
         "+",
         "16.2727",
         "5.67e-07",
         "0.000415",
         "GAGGGCGGGGCCG"
        ],
        [
         "129",
         "ZNF28.H13CORE.0.P.C",
         null,
         "REF_5-1294995-G-T",
         "1",
         "18",
         "+",
         "13.2424",
         "1.15e-05",
         "0.000493",
         "GGCATCGCGGGGGTGGCC"
        ],
        [
         "132",
         "ZNF28.H13CORE.0.P.C",
         null,
         "REF_5-1294992-C-T",
         "4",
         "21",
         "+",
         "13.2424",
         "1.15e-05",
         "0.000493",
         "GGCATCGCGGGGGTGGCC"
        ],
        [
         "130",
         "ZNF28.H13CORE.0.P.C",
         null,
         "REF_5-1294994-G-C",
         "2",
         "19",
         "+",
         "13.2424",
         "1.15e-05",
         "0.000493",
         "GGCATCGCGGGGGTGGCC"
        ],
        [
         "131",
         "ZNF28.H13CORE.0.P.C",
         null,
         "REF_5-1294994-G-T",
         "2",
         "19",
         "+",
         "13.2424",
         "1.15e-05",
         "0.000493",
         "GGCATCGCGGGGGTGGCC"
        ],
        [
         "83",
         "KLF15.H13CORE.1.P.C",
         null,
         "ALT_5-1295075-G-A",
         "2",
         "21",
         "+",
         "13.0408",
         "7.77e-06",
         "0.000512",
         "GAGGGCGGGACCGCGGAAAG"
        ],
        [
         "26",
         "MYCN.H13CORE.0.PS.A",
         null,
         "REF_5-1295020-C-G",
         "7",
         "17",
         "+",
         "15.7742",
         "1.35e-06",
         "0.000525",
         "CCCACGTGCGC"
        ],
        [
         "23",
         "MYCN.H13CORE.0.PS.A",
         null,
         "ALT_5-1295234-A-G",
         "1",
         "11",
         "+",
         "16.3226",
         "1.09e-06",
         "0.000525",
         "GCCACGTGGGG"
        ],
        [
         "343",
         "MAZ.H13CORE.1.P.B",
         null,
         "REF_5-1295128-G-A",
         "1",
         "21",
         "+",
         "10.6364",
         "4.62e-05",
         "0.000614",
         "CTGGGCCGGGGACCCGGGAGG"
        ],
        [
         "285",
         "MAZ.H13CORE.1.P.B",
         null,
         "ALT_5-1295182-A-G",
         "1",
         "21",
         "+",
         "11.0818",
         "3.62e-05",
         "0.000614",
         "CGGAGCTGGAGGGTGAAGGGG"
        ],
        [
         "34",
         "SP1.H13CORE.0.P.B",
         null,
         "ALT_5-1295075-G-A",
         "1",
         "14",
         "+",
         "16.1148",
         "1.96e-06",
         "0.000697",
         "AGAGGGCGGGACCG"
        ],
        [
         "67",
         "TET3.H13CORE.0.PSGIB.A",
         null,
         "ALT_5-1295234-A-G",
         "1",
         "18",
         "+",
         "14.1148",
         "6.08e-06",
         "0.000748",
         "GCCACGTGGGGAGCGCGG"
        ],
        [
         "134",
         "TET3.H13CORE.0.PSGIB.A",
         null,
         "REF_5-1295225-C-T",
         "2",
         "19",
         "-",
         "13.3115",
         "1.19e-05",
         "0.000748",
         "CCCACGTGGCGGAGGGAC"
        ],
        [
         "84",
         "TET3.H13CORE.0.PSGIB.A",
         null,
         "ALT_5-1295225-C-T",
         "2",
         "19",
         "-",
         "13.7869",
         "8.05e-06",
         "0.000748",
         "CCCACGTGACGGAGGGAC"
        ],
        [
         "21",
         "KLF4.H13CORE.0.PSM.A",
         null,
         "REF_5-1295075-G-A",
         "3",
         "13",
         "+",
         "16.4082",
         "9.16e-07",
         "0.000761",
         "AGGGCGGGGCC"
        ],
        [
         "192",
         "ZN263.H13CORE.1.P.B",
         null,
         "ALT_5-1295181-A-G",
         "1",
         "21",
         "+",
         "11.5657",
         "2.1e-05",
         "0.000774",
         "GCGGAGCTGGGAGGTGAAGGG"
        ]
       ],
       "shape": {
        "columns": 10,
        "rows": 619
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>motif_id</th>\n",
       "      <th>motif_alt_id</th>\n",
       "      <th>sequence_name</th>\n",
       "      <th>start</th>\n",
       "      <th>stop</th>\n",
       "      <th>strand</th>\n",
       "      <th>score</th>\n",
       "      <th>p-value</th>\n",
       "      <th>q-value</th>\n",
       "      <th>matched_sequence</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>MAZ.H13CORE.1.P.B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>21.03640</td>\n",
       "      <td>2.680000e-08</td>\n",
       "      <td>0.000001</td>\n",
       "      <td>AGAGGGCGGGGCCGCGGAAAG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>SP1.H13CORE.2.P.B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>21.01640</td>\n",
       "      <td>4.570000e-08</td>\n",
       "      <td>0.000003</td>\n",
       "      <td>AGAGGGCGGGGCCGCGGAAAG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>SP4.H13CORE.2.P.B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>20.28440</td>\n",
       "      <td>6.310000e-08</td>\n",
       "      <td>0.000005</td>\n",
       "      <td>AGAGGGCGGGGCCGCGGAAAG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>MAZ.H13CORE.1.P.B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>17.81820</td>\n",
       "      <td>4.420000e-07</td>\n",
       "      <td>0.000012</td>\n",
       "      <td>AGAGGGCGGGACCGCGGAAAG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>KLF1.H13CORE.0.P.B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>2</td>\n",
       "      <td>14</td>\n",
       "      <td>+</td>\n",
       "      <td>18.93550</td>\n",
       "      <td>2.390000e-08</td>\n",
       "      <td>0.000017</td>\n",
       "      <td>GAGGGCGGGGCCG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>544</th>\n",
       "      <td>ETV3.H13CORE.0.SM.B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ALT_5-1295034-C-A</td>\n",
       "      <td>4</td>\n",
       "      <td>15</td>\n",
       "      <td>+</td>\n",
       "      <td>11.21150</td>\n",
       "      <td>8.540000e-05</td>\n",
       "      <td>0.086200</td>\n",
       "      <td>AGCAGGAAGCAG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>563</th>\n",
       "      <td>FEV.H13CORE.0.S.B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ALT_5-1295046-T-G</td>\n",
       "      <td>7</td>\n",
       "      <td>17</td>\n",
       "      <td>+</td>\n",
       "      <td>10.25000</td>\n",
       "      <td>8.810000e-05</td>\n",
       "      <td>0.090700</td>\n",
       "      <td>TGCCGGAAACT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>573</th>\n",
       "      <td>ELK3.H13CORE.0.PSM.A</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ALT_5-1295046-T-G</td>\n",
       "      <td>7</td>\n",
       "      <td>17</td>\n",
       "      <td>+</td>\n",
       "      <td>9.86538</td>\n",
       "      <td>9.080000e-05</td>\n",
       "      <td>0.095500</td>\n",
       "      <td>TGCCGGAAACT</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>578</th>\n",
       "      <td>ZBT7A.H13CORE.0.P.B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ALT_5-1295136-A-G</td>\n",
       "      <td>11</td>\n",
       "      <td>19</td>\n",
       "      <td>+</td>\n",
       "      <td>11.55740</td>\n",
       "      <td>9.160000e-05</td>\n",
       "      <td>0.097900</td>\n",
       "      <td>GGGGGTCGG</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>585</th>\n",
       "      <td>NFIA.H13CORE.0.P.B</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ALT_5-1295046-T-A</td>\n",
       "      <td>6</td>\n",
       "      <td>13</td>\n",
       "      <td>-</td>\n",
       "      <td>11.14750</td>\n",
       "      <td>9.260000e-05</td>\n",
       "      <td>0.121000</td>\n",
       "      <td>TCTGGCAG</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>619 rows √ó 10 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                 motif_id  motif_alt_id      sequence_name  start  stop  \\\n",
       "1       MAZ.H13CORE.1.P.B           NaN  REF_5-1295075-G-A      1    21   \n",
       "2       SP1.H13CORE.2.P.B           NaN  REF_5-1295075-G-A      1    21   \n",
       "3       SP4.H13CORE.2.P.B           NaN  REF_5-1295075-G-A      1    21   \n",
       "11      MAZ.H13CORE.1.P.B           NaN  ALT_5-1295075-G-A      1    21   \n",
       "0      KLF1.H13CORE.0.P.B           NaN  REF_5-1295075-G-A      2    14   \n",
       "..                    ...           ...                ...    ...   ...   \n",
       "544   ETV3.H13CORE.0.SM.B           NaN  ALT_5-1295034-C-A      4    15   \n",
       "563     FEV.H13CORE.0.S.B           NaN  ALT_5-1295046-T-G      7    17   \n",
       "573  ELK3.H13CORE.0.PSM.A           NaN  ALT_5-1295046-T-G      7    17   \n",
       "578   ZBT7A.H13CORE.0.P.B           NaN  ALT_5-1295136-A-G     11    19   \n",
       "585    NFIA.H13CORE.0.P.B           NaN  ALT_5-1295046-T-A      6    13   \n",
       "\n",
       "    strand     score       p-value   q-value       matched_sequence  \n",
       "1        +  21.03640  2.680000e-08  0.000001  AGAGGGCGGGGCCGCGGAAAG  \n",
       "2        +  21.01640  4.570000e-08  0.000003  AGAGGGCGGGGCCGCGGAAAG  \n",
       "3        +  20.28440  6.310000e-08  0.000005  AGAGGGCGGGGCCGCGGAAAG  \n",
       "11       +  17.81820  4.420000e-07  0.000012  AGAGGGCGGGACCGCGGAAAG  \n",
       "0        +  18.93550  2.390000e-08  0.000017          GAGGGCGGGGCCG  \n",
       "..     ...       ...           ...       ...                    ...  \n",
       "544      +  11.21150  8.540000e-05  0.086200           AGCAGGAAGCAG  \n",
       "563      +  10.25000  8.810000e-05  0.090700            TGCCGGAAACT  \n",
       "573      +   9.86538  9.080000e-05  0.095500            TGCCGGAAACT  \n",
       "578      +  11.55740  9.160000e-05  0.097900              GGGGGTCGG  \n",
       "585      -  11.14750  9.260000e-05  0.121000               TCTGGCAG  \n",
       "\n",
       "[619 rows x 10 columns]"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Preview the FIMO result table\n",
    "fimo_result_df.sort_values(by=\"q-value\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "fb1c7fb6",
   "metadata": {},
   "source": [
    "##### ‚ùì Question\n",
    "\n",
    "> **üí° What is the average number of motifs found in our 21bp sequence?**\n",
    ">\n",
    "> **üí° Are all of them true TF binders?**\n",
    "> \n",
    "> **üí° Do all motifs overlap our variant?**"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "f6769bb0",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Minimum motifs per sequence: 2\n",
      "Mean motifs per sequence: 11.90\n",
      "Maximum motifs per sequence: 40\n"
     ]
    }
   ],
   "source": [
    "# Get statistics for the number of motifs found per sequence\n",
    "sequence_counts = fimo_result_df['sequence_name'].value_counts()\n",
    "\n",
    "print(f\"Minimum motifs per sequence: {sequence_counts.min()}\")\n",
    "print(f\"Mean motifs per sequence: {sequence_counts.mean():.2f}\")\n",
    "print(f\"Maximum motifs per sequence: {sequence_counts.max()}\")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "423e0390",
   "metadata": {},
   "source": [
    "##### üîó Combine FIMO Output with Variant Information\n",
    "\n",
    "Let's merge the FIMO results with variant metadata for both reference and alternative alleles. üß©"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "57cd2ca9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "motif_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "start",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "stop",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "strand",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ALT_name",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "ALT_q-value",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "REF_name",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "REF_q-value",
         "rawType": "float64",
         "type": "float"
        }
       ],
       "ref": "b0ec6560-7a9d-4953-ba97-a7880a7092ea",
       "rows": [
        [
         "0",
         "5-1294991-G-A",
         "GLIS2.H13CORE.0.SM.B",
         "6",
         "21",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00254"
        ],
        [
         "1",
         "5-1294991-G-A",
         "GLIS3.H13CORE.1.S.C",
         "10",
         "21",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.0086"
        ],
        [
         "2",
         "5-1294991-G-A",
         "KAISO.H13CORE.1.P.B",
         "4",
         "17",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.0043"
        ],
        [
         "3",
         "5-1294991-G-A",
         "ZBT14.H13CORE.0.P.C",
         "3",
         "15",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00572"
        ],
        [
         "4",
         "5-1294991-G-A",
         "ZFP57.H13CORE.0.P.B",
         "1",
         "9",
         "+",
         "ALT_5-1294991-G-A",
         "0.0307",
         "REF_5-1294991-G-A",
         "0.0307"
        ],
        [
         "5",
         "5-1294991-G-A",
         "ZIC1.H13CORE.1.M.B",
         "5",
         "20",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00375"
        ],
        [
         "6",
         "5-1294991-G-A",
         "ZIC1.H13CORE.2.S.B",
         "6",
         "21",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00367"
        ],
        [
         "7",
         "5-1294991-G-A",
         "ZIC3.H13CORE.1.S.B",
         "6",
         "21",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00506"
        ],
        [
         "8",
         "5-1294991-G-A",
         "ZIC3.H13CORE.2.M.B",
         "5",
         "20",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00362"
        ],
        [
         "9",
         "5-1294991-G-A",
         "ZIC4.H13CORE.0.SM.B",
         "6",
         "20",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00176"
        ],
        [
         "10",
         "5-1294991-G-A",
         "ZIC5.H13CORE.1.M.C",
         "5",
         "21",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00204"
        ],
        [
         "11",
         "5-1294991-G-A",
         "ZIC5.H13CORE.2.S.C",
         "5",
         "21",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00337"
        ],
        [
         "12",
         "5-1294991-G-A",
         "ZN740.H13CORE.0.P.C",
         "11",
         "21",
         "+",
         "ALT_5-1294991-G-A",
         "0.00485",
         "REF_5-1294991-G-A",
         "0.00147"
        ],
        [
         "13",
         "5-1294992-C-T",
         "EGR2.H13CORE.0.PSM.A",
         "8",
         "21",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.0102"
        ],
        [
         "14",
         "5-1294992-C-T",
         "EGR3.H13CORE.0.PSM.A",
         "7",
         "19",
         "+",
         "ALT_5-1294992-C-T",
         "0.0206",
         null,
         null
        ],
        [
         "15",
         "5-1294992-C-T",
         "GLIS1.H13CORE.1.P.B",
         "8",
         "18",
         "+",
         "ALT_5-1294992-C-T",
         "0.0126",
         null,
         null
        ],
        [
         "16",
         "5-1294992-C-T",
         "GLIS1.H13CORE.2.SM.B",
         "3",
         "21",
         "+",
         "ALT_5-1294992-C-T",
         "0.00118",
         "REF_5-1294992-C-T",
         "0.00144"
        ],
        [
         "17",
         "5-1294992-C-T",
         "GLIS2.H13CORE.0.SM.B",
         "5",
         "20",
         "+",
         "ALT_5-1294992-C-T",
         "0.00254",
         "REF_5-1294992-C-T",
         "0.00254"
        ],
        [
         "18",
         "5-1294992-C-T",
         "GLIS3.H13CORE.1.S.C",
         "9",
         "20",
         "+",
         "ALT_5-1294992-C-T",
         "0.0086",
         "REF_5-1294992-C-T",
         "0.0086"
        ],
        [
         "19",
         "5-1294992-C-T",
         "KAISO.H13CORE.1.P.B",
         "3",
         "16",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.0043"
        ],
        [
         "20",
         "5-1294992-C-T",
         "KLF12.H13CORE.1.PSM.A",
         "9",
         "21",
         "+",
         "ALT_5-1294992-C-T",
         "0.00869",
         null,
         null
        ],
        [
         "21",
         "5-1294992-C-T",
         "KLF13.H13CORE.0.PSM.A",
         "7",
         "21",
         "+",
         "ALT_5-1294992-C-T",
         "0.0311",
         null,
         null
        ],
        [
         "22",
         "5-1294992-C-T",
         "KLF9.H13CORE.0.P.B",
         "8",
         "21",
         "+",
         "ALT_5-1294992-C-T",
         "0.0543",
         null,
         null
        ],
        [
         "23",
         "5-1294992-C-T",
         "ZBT14.H13CORE.0.P.C",
         "2",
         "14",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00572"
        ],
        [
         "24",
         "5-1294992-C-T",
         "ZIC1.H13CORE.1.M.B",
         "4",
         "19",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00375"
        ],
        [
         "25",
         "5-1294992-C-T",
         "ZIC1.H13CORE.2.S.B",
         "5",
         "20",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00367"
        ],
        [
         "26",
         "5-1294992-C-T",
         "ZIC3.H13CORE.1.S.B",
         "5",
         "20",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00506"
        ],
        [
         "27",
         "5-1294992-C-T",
         "ZIC3.H13CORE.2.M.B",
         "4",
         "19",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00362"
        ],
        [
         "28",
         "5-1294992-C-T",
         "ZIC4.H13CORE.0.SM.B",
         "5",
         "19",
         "+",
         "ALT_5-1294992-C-T",
         "0.00415",
         "REF_5-1294992-C-T",
         "0.00176"
        ],
        [
         "29",
         "5-1294992-C-T",
         "ZIC5.H13CORE.1.M.C",
         "4",
         "20",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00204"
        ],
        [
         "30",
         "5-1294992-C-T",
         "ZIC5.H13CORE.2.S.C",
         "4",
         "20",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00337"
        ],
        [
         "31",
         "5-1294992-C-T",
         "ZN740.H13CORE.0.P.C",
         "10",
         "20",
         "+",
         "ALT_5-1294992-C-T",
         "0.00147",
         "REF_5-1294992-C-T",
         "0.00147"
        ],
        [
         "32",
         "5-1294992-C-T",
         "ZNF28.H13CORE.0.P.C",
         "4",
         "21",
         "+",
         "ALT_5-1294992-C-T",
         "0.000839",
         "REF_5-1294992-C-T",
         "0.000493"
        ],
        [
         "33",
         "5-1294994-G-C",
         "EGR1.H13CORE.0.PS.A",
         "5",
         "20",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00131"
        ],
        [
         "34",
         "5-1294994-G-C",
         "EGR2.H13CORE.0.PSM.A",
         "6",
         "19",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.0102"
        ],
        [
         "35",
         "5-1294994-G-C",
         "GLI1.H13CORE.0.P.C",
         "9",
         "20",
         "+",
         "ALT_5-1294994-G-C",
         "0.00894",
         "REF_5-1294994-G-C",
         "0.0126"
        ],
        [
         "36",
         "5-1294994-G-C",
         "GLI2.H13CORE.0.PSM.A",
         "6",
         "21",
         "+",
         "ALT_5-1294994-G-C",
         "0.00367",
         "REF_5-1294994-G-C",
         "0.00367"
        ],
        [
         "37",
         "5-1294994-G-C",
         "GLI3.H13CORE.0.PS.A",
         "6",
         "21",
         "+",
         "ALT_5-1294994-G-C",
         "0.00447",
         "REF_5-1294994-G-C",
         "0.00447"
        ],
        [
         "38",
         "5-1294994-G-C",
         "GLIS1.H13CORE.2.SM.B",
         "1",
         "19",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00144"
        ],
        [
         "39",
         "5-1294994-G-C",
         "GLIS2.H13CORE.0.SM.B",
         "3",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00254"
        ],
        [
         "40",
         "5-1294994-G-C",
         "GLIS3.H13CORE.1.S.C",
         "7",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.0086"
        ],
        [
         "41",
         "5-1294994-G-C",
         "KAISO.H13CORE.1.P.B",
         "1",
         "14",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.0043"
        ],
        [
         "42",
         "5-1294994-G-C",
         "ZIC1.H13CORE.1.M.B",
         "2",
         "17",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00375"
        ],
        [
         "43",
         "5-1294994-G-C",
         "ZIC1.H13CORE.2.S.B",
         "3",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00367"
        ],
        [
         "44",
         "5-1294994-G-C",
         "ZIC3.H13CORE.1.S.B",
         "3",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00506"
        ],
        [
         "45",
         "5-1294994-G-C",
         "ZIC3.H13CORE.2.M.B",
         "2",
         "17",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00362"
        ],
        [
         "46",
         "5-1294994-G-C",
         "ZIC4.H13CORE.0.SM.B",
         "3",
         "17",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00176"
        ],
        [
         "47",
         "5-1294994-G-C",
         "ZIC5.H13CORE.1.M.C",
         "2",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00204"
        ],
        [
         "48",
         "5-1294994-G-C",
         "ZIC5.H13CORE.2.S.C",
         "2",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00337"
        ],
        [
         "49",
         "5-1294994-G-C",
         "ZN740.H13CORE.0.P.C",
         "8",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00147"
        ]
       ],
       "shape": {
        "columns": 9,
        "rows": 508
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>motif_id</th>\n",
       "      <th>start</th>\n",
       "      <th>stop</th>\n",
       "      <th>strand</th>\n",
       "      <th>ALT_name</th>\n",
       "      <th>ALT_q-value</th>\n",
       "      <th>REF_name</th>\n",
       "      <th>REF_q-value</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>GLIS2.H13CORE.0.SM.B</td>\n",
       "      <td>6</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>0.00254</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>GLIS3.H13CORE.1.S.C</td>\n",
       "      <td>10</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>0.00860</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>KAISO.H13CORE.1.P.B</td>\n",
       "      <td>4</td>\n",
       "      <td>17</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>0.00430</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>ZBT14.H13CORE.0.P.C</td>\n",
       "      <td>3</td>\n",
       "      <td>15</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>0.00572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>ZFP57.H13CORE.0.P.B</td>\n",
       "      <td>1</td>\n",
       "      <td>9</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1294991-G-A</td>\n",
       "      <td>0.0307</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>0.03070</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>503</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZN343.H13CORE.0.PSM.A</td>\n",
       "      <td>4</td>\n",
       "      <td>18</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295234-A-G</td>\n",
       "      <td>0.03250</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>504</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZN449.H13CORE.1.S.C</td>\n",
       "      <td>6</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295234-A-G</td>\n",
       "      <td>0.0243</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>505</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZN684.H13CORE.1.M.C</td>\n",
       "      <td>5</td>\n",
       "      <td>19</td>\n",
       "      <td>-</td>\n",
       "      <td>ALT_5-1295234-A-G</td>\n",
       "      <td>0.0116</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>506</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZN75A.H13CORE.0.SM.B</td>\n",
       "      <td>5</td>\n",
       "      <td>19</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295234-A-G</td>\n",
       "      <td>0.01550</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>507</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZNF215.H13CORE.0.PSG.A</td>\n",
       "      <td>8</td>\n",
       "      <td>18</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295234-A-G</td>\n",
       "      <td>0.06370</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>508 rows √ó 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        variant_id                motif_id  start  stop strand  \\\n",
       "0    5-1294991-G-A    GLIS2.H13CORE.0.SM.B      6    21      +   \n",
       "1    5-1294991-G-A     GLIS3.H13CORE.1.S.C     10    21      +   \n",
       "2    5-1294991-G-A     KAISO.H13CORE.1.P.B      4    17      +   \n",
       "3    5-1294991-G-A     ZBT14.H13CORE.0.P.C      3    15      +   \n",
       "4    5-1294991-G-A     ZFP57.H13CORE.0.P.B      1     9      +   \n",
       "..             ...                     ...    ...   ...    ...   \n",
       "503  5-1295234-A-G   ZN343.H13CORE.0.PSM.A      4    18      -   \n",
       "504  5-1295234-A-G     ZN449.H13CORE.1.S.C      6    20      +   \n",
       "505  5-1295234-A-G     ZN684.H13CORE.1.M.C      5    19      -   \n",
       "506  5-1295234-A-G    ZN75A.H13CORE.0.SM.B      5    19      +   \n",
       "507  5-1295234-A-G  ZNF215.H13CORE.0.PSG.A      8    18      -   \n",
       "\n",
       "              ALT_name  ALT_q-value           REF_name  REF_q-value  \n",
       "0                  NaN          NaN  REF_5-1294991-G-A      0.00254  \n",
       "1                  NaN          NaN  REF_5-1294991-G-A      0.00860  \n",
       "2                  NaN          NaN  REF_5-1294991-G-A      0.00430  \n",
       "3                  NaN          NaN  REF_5-1294991-G-A      0.00572  \n",
       "4    ALT_5-1294991-G-A       0.0307  REF_5-1294991-G-A      0.03070  \n",
       "..                 ...          ...                ...          ...  \n",
       "503                NaN          NaN  REF_5-1295234-A-G      0.03250  \n",
       "504  ALT_5-1295234-A-G       0.0243                NaN          NaN  \n",
       "505  ALT_5-1295234-A-G       0.0116                NaN          NaN  \n",
       "506                NaN          NaN  REF_5-1295234-A-G      0.01550  \n",
       "507                NaN          NaN  REF_5-1295234-A-G      0.06370  \n",
       "\n",
       "[508 rows x 9 columns]"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "important_cols_for_matching = ['variant_id', 'REF', 'ALT', 'ref_sequence', 'alt_sequence', 'variant_pos']\n",
    "\n",
    "# Merge FIMO with variants based on alternative and reference sequence names\n",
    "fimo_alt = fimo_result_df.merge(df_clinvar_snvs[important_cols_for_matching], left_on=\"sequence_name\", right_on=\"ALT\", how=\"inner\")\n",
    "fimo_ref = fimo_result_df.merge(df_clinvar_snvs[important_cols_for_matching], left_on=\"sequence_name\", right_on=\"REF\", how=\"inner\")\n",
    "\n",
    "# Rename columns to distinguish ALT and REF q-values\n",
    "fimo_alt = fimo_alt.rename(columns={\"q-value\": \"ALT_q-value\", \"sequence_name\": \"ALT_name\"})\n",
    "fimo_ref = fimo_ref.rename(columns={\"q-value\": \"REF_q-value\", \"sequence_name\": \"REF_name\"})\n",
    "\n",
    "# Merge ALT and REF results on ID, motif_id, start, stop, strand\n",
    "combined_fimo_df = pd.merge(\n",
    "    fimo_alt[['variant_id', 'motif_id', 'start', 'stop', 'strand', 'ALT_name', 'ALT_q-value']],\n",
    "    fimo_ref[['variant_id', 'motif_id', 'start', 'stop', 'strand', 'REF_name', 'REF_q-value']],\n",
    "    on=['variant_id', 'motif_id', 'start', 'stop', 'strand'],\n",
    "    how='outer'  # Ensure we include all motif occurrences\n",
    ")\n",
    "\n",
    "combined_fimo_df"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "379b7cdd",
   "metadata": {},
   "source": [
    "We want to look into the variant-overlapping TFBS, which requires the information of the variant position. üìç This info can be added from the variant table. ‚ûï"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "d602881d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "motif_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "start",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "stop",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "strand",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ALT_name",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "ALT_q-value",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "REF_name",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "REF_q-value",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "chrom",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pos",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "ref",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "alt",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ac",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "an",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "af",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "clinical_significance",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "variant_pos",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "ref_sequence",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "alt_sequence",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "REF",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ALT",
         "rawType": "object",
         "type": "string"
        }
       ],
       "ref": "56092f7c-3354-465b-84a5-b0f0ce6d22c7",
       "rows": [
        [
         "0",
         "5-1294991-G-A",
         "GLIS2.H13CORE.0.SM.B",
         "6",
         "21",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00254",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "1",
         "5-1294991-G-A",
         "GLIS3.H13CORE.1.S.C",
         "10",
         "21",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.0086",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "2",
         "5-1294991-G-A",
         "KAISO.H13CORE.1.P.B",
         "4",
         "17",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.0043",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "3",
         "5-1294991-G-A",
         "ZBT14.H13CORE.0.P.C",
         "3",
         "15",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00572",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "4",
         "5-1294991-G-A",
         "ZFP57.H13CORE.0.P.B",
         "1",
         "9",
         "+",
         "ALT_5-1294991-G-A",
         "0.0307",
         "REF_5-1294991-G-A",
         "0.0307",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "5",
         "5-1294991-G-A",
         "ZIC1.H13CORE.1.M.B",
         "5",
         "20",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00375",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "6",
         "5-1294991-G-A",
         "ZIC1.H13CORE.2.S.B",
         "6",
         "21",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00367",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "7",
         "5-1294991-G-A",
         "ZIC3.H13CORE.1.S.B",
         "6",
         "21",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00506",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "8",
         "5-1294991-G-A",
         "ZIC3.H13CORE.2.M.B",
         "5",
         "20",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00362",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "9",
         "5-1294991-G-A",
         "ZIC4.H13CORE.0.SM.B",
         "6",
         "20",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00176",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "10",
         "5-1294991-G-A",
         "ZIC5.H13CORE.1.M.C",
         "5",
         "21",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00204",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "11",
         "5-1294991-G-A",
         "ZIC5.H13CORE.2.S.C",
         "5",
         "21",
         "+",
         null,
         null,
         "REF_5-1294991-G-A",
         "0.00337",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "12",
         "5-1294991-G-A",
         "ZN740.H13CORE.0.P.C",
         "11",
         "21",
         "+",
         "ALT_5-1294991-G-A",
         "0.00485",
         "REF_5-1294991-G-A",
         "0.00147",
         "5",
         "1294991",
         "G",
         "A",
         "3",
         "1317992",
         "2.2761898403025207e-06",
         "Uncertain significance",
         "11",
         "GCGCGGCATCGCGGGGGTGGC",
         "GCGCGGCATCACGGGGGTGGC",
         "REF_5-1294991-G-A",
         "ALT_5-1294991-G-A"
        ],
        [
         "13",
         "5-1294992-C-T",
         "EGR2.H13CORE.0.PSM.A",
         "8",
         "21",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.0102",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "14",
         "5-1294992-C-T",
         "EGR3.H13CORE.0.PSM.A",
         "7",
         "19",
         "+",
         "ALT_5-1294992-C-T",
         "0.0206",
         null,
         null,
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "15",
         "5-1294992-C-T",
         "GLIS1.H13CORE.1.P.B",
         "8",
         "18",
         "+",
         "ALT_5-1294992-C-T",
         "0.0126",
         null,
         null,
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "16",
         "5-1294992-C-T",
         "GLIS1.H13CORE.2.SM.B",
         "3",
         "21",
         "+",
         "ALT_5-1294992-C-T",
         "0.00118",
         "REF_5-1294992-C-T",
         "0.00144",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "17",
         "5-1294992-C-T",
         "GLIS2.H13CORE.0.SM.B",
         "5",
         "20",
         "+",
         "ALT_5-1294992-C-T",
         "0.00254",
         "REF_5-1294992-C-T",
         "0.00254",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "18",
         "5-1294992-C-T",
         "GLIS3.H13CORE.1.S.C",
         "9",
         "20",
         "+",
         "ALT_5-1294992-C-T",
         "0.0086",
         "REF_5-1294992-C-T",
         "0.0086",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "19",
         "5-1294992-C-T",
         "KAISO.H13CORE.1.P.B",
         "3",
         "16",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.0043",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "20",
         "5-1294992-C-T",
         "KLF12.H13CORE.1.PSM.A",
         "9",
         "21",
         "+",
         "ALT_5-1294992-C-T",
         "0.00869",
         null,
         null,
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "21",
         "5-1294992-C-T",
         "KLF13.H13CORE.0.PSM.A",
         "7",
         "21",
         "+",
         "ALT_5-1294992-C-T",
         "0.0311",
         null,
         null,
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "22",
         "5-1294992-C-T",
         "KLF9.H13CORE.0.P.B",
         "8",
         "21",
         "+",
         "ALT_5-1294992-C-T",
         "0.0543",
         null,
         null,
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "23",
         "5-1294992-C-T",
         "ZBT14.H13CORE.0.P.C",
         "2",
         "14",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00572",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "24",
         "5-1294992-C-T",
         "ZIC1.H13CORE.1.M.B",
         "4",
         "19",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00375",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "25",
         "5-1294992-C-T",
         "ZIC1.H13CORE.2.S.B",
         "5",
         "20",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00367",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "26",
         "5-1294992-C-T",
         "ZIC3.H13CORE.1.S.B",
         "5",
         "20",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00506",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "27",
         "5-1294992-C-T",
         "ZIC3.H13CORE.2.M.B",
         "4",
         "19",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00362",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "28",
         "5-1294992-C-T",
         "ZIC4.H13CORE.0.SM.B",
         "5",
         "19",
         "+",
         "ALT_5-1294992-C-T",
         "0.00415",
         "REF_5-1294992-C-T",
         "0.00176",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "29",
         "5-1294992-C-T",
         "ZIC5.H13CORE.1.M.C",
         "4",
         "20",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00204",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "30",
         "5-1294992-C-T",
         "ZIC5.H13CORE.2.S.C",
         "4",
         "20",
         "+",
         null,
         null,
         "REF_5-1294992-C-T",
         "0.00337",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "31",
         "5-1294992-C-T",
         "ZN740.H13CORE.0.P.C",
         "10",
         "20",
         "+",
         "ALT_5-1294992-C-T",
         "0.00147",
         "REF_5-1294992-C-T",
         "0.00147",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "32",
         "5-1294992-C-T",
         "ZNF28.H13CORE.0.P.C",
         "4",
         "21",
         "+",
         "ALT_5-1294992-C-T",
         "0.000839",
         "REF_5-1294992-C-T",
         "0.000493",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T"
        ],
        [
         "33",
         "5-1294994-G-C",
         "EGR1.H13CORE.0.PS.A",
         "5",
         "20",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00131",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "34",
         "5-1294994-G-C",
         "EGR2.H13CORE.0.PSM.A",
         "6",
         "19",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.0102",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "35",
         "5-1294994-G-C",
         "GLI1.H13CORE.0.P.C",
         "9",
         "20",
         "+",
         "ALT_5-1294994-G-C",
         "0.00894",
         "REF_5-1294994-G-C",
         "0.0126",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "36",
         "5-1294994-G-C",
         "GLI2.H13CORE.0.PSM.A",
         "6",
         "21",
         "+",
         "ALT_5-1294994-G-C",
         "0.00367",
         "REF_5-1294994-G-C",
         "0.00367",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "37",
         "5-1294994-G-C",
         "GLI3.H13CORE.0.PS.A",
         "6",
         "21",
         "+",
         "ALT_5-1294994-G-C",
         "0.00447",
         "REF_5-1294994-G-C",
         "0.00447",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "38",
         "5-1294994-G-C",
         "GLIS1.H13CORE.2.SM.B",
         "1",
         "19",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00144",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "39",
         "5-1294994-G-C",
         "GLIS2.H13CORE.0.SM.B",
         "3",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00254",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "40",
         "5-1294994-G-C",
         "GLIS3.H13CORE.1.S.C",
         "7",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.0086",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "41",
         "5-1294994-G-C",
         "KAISO.H13CORE.1.P.B",
         "1",
         "14",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.0043",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "42",
         "5-1294994-G-C",
         "ZIC1.H13CORE.1.M.B",
         "2",
         "17",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00375",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "43",
         "5-1294994-G-C",
         "ZIC1.H13CORE.2.S.B",
         "3",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00367",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "44",
         "5-1294994-G-C",
         "ZIC3.H13CORE.1.S.B",
         "3",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00506",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "45",
         "5-1294994-G-C",
         "ZIC3.H13CORE.2.M.B",
         "2",
         "17",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00362",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "46",
         "5-1294994-G-C",
         "ZIC4.H13CORE.0.SM.B",
         "3",
         "17",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00176",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "47",
         "5-1294994-G-C",
         "ZIC5.H13CORE.1.M.C",
         "2",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00204",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "48",
         "5-1294994-G-C",
         "ZIC5.H13CORE.2.S.C",
         "2",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00337",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ],
        [
         "49",
         "5-1294994-G-C",
         "ZN740.H13CORE.0.P.C",
         "8",
         "18",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.00147",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C"
        ]
       ],
       "shape": {
        "columns": 22,
        "rows": 508
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>motif_id</th>\n",
       "      <th>start</th>\n",
       "      <th>stop</th>\n",
       "      <th>strand</th>\n",
       "      <th>ALT_name</th>\n",
       "      <th>ALT_q-value</th>\n",
       "      <th>REF_name</th>\n",
       "      <th>REF_q-value</th>\n",
       "      <th>chrom</th>\n",
       "      <th>...</th>\n",
       "      <th>alt</th>\n",
       "      <th>ac</th>\n",
       "      <th>an</th>\n",
       "      <th>af</th>\n",
       "      <th>clinical_significance</th>\n",
       "      <th>variant_pos</th>\n",
       "      <th>ref_sequence</th>\n",
       "      <th>alt_sequence</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>GLIS2.H13CORE.0.SM.B</td>\n",
       "      <td>6</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>0.00254</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>A</td>\n",
       "      <td>3</td>\n",
       "      <td>1317992</td>\n",
       "      <td>0.000002</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GCGCGGCATCGCGGGGGTGGC</td>\n",
       "      <td>GCGCGGCATCACGGGGGTGGC</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>ALT_5-1294991-G-A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>GLIS3.H13CORE.1.S.C</td>\n",
       "      <td>10</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>0.00860</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>A</td>\n",
       "      <td>3</td>\n",
       "      <td>1317992</td>\n",
       "      <td>0.000002</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GCGCGGCATCGCGGGGGTGGC</td>\n",
       "      <td>GCGCGGCATCACGGGGGTGGC</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>ALT_5-1294991-G-A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>KAISO.H13CORE.1.P.B</td>\n",
       "      <td>4</td>\n",
       "      <td>17</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>0.00430</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>A</td>\n",
       "      <td>3</td>\n",
       "      <td>1317992</td>\n",
       "      <td>0.000002</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GCGCGGCATCGCGGGGGTGGC</td>\n",
       "      <td>GCGCGGCATCACGGGGGTGGC</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>ALT_5-1294991-G-A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>ZBT14.H13CORE.0.P.C</td>\n",
       "      <td>3</td>\n",
       "      <td>15</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>0.00572</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>A</td>\n",
       "      <td>3</td>\n",
       "      <td>1317992</td>\n",
       "      <td>0.000002</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GCGCGGCATCGCGGGGGTGGC</td>\n",
       "      <td>GCGCGGCATCACGGGGGTGGC</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>ALT_5-1294991-G-A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>ZFP57.H13CORE.0.P.B</td>\n",
       "      <td>1</td>\n",
       "      <td>9</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1294991-G-A</td>\n",
       "      <td>0.0307</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>0.03070</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>A</td>\n",
       "      <td>3</td>\n",
       "      <td>1317992</td>\n",
       "      <td>0.000002</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GCGCGGCATCGCGGGGGTGGC</td>\n",
       "      <td>GCGCGGCATCACGGGGGTGGC</td>\n",
       "      <td>REF_5-1294991-G-A</td>\n",
       "      <td>ALT_5-1294991-G-A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>503</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZN343.H13CORE.0.PSM.A</td>\n",
       "      <td>4</td>\n",
       "      <td>18</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295234-A-G</td>\n",
       "      <td>0.03250</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>G</td>\n",
       "      <td>38593</td>\n",
       "      <td>151924</td>\n",
       "      <td>0.254028</td>\n",
       "      <td>Benign</td>\n",
       "      <td>11</td>\n",
       "      <td>GCCACGTGGGAAGCGCGGTCC</td>\n",
       "      <td>GCCACGTGGGGAGCGCGGTCC</td>\n",
       "      <td>REF_5-1295234-A-G</td>\n",
       "      <td>ALT_5-1295234-A-G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>504</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZN449.H13CORE.1.S.C</td>\n",
       "      <td>6</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295234-A-G</td>\n",
       "      <td>0.0243</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>G</td>\n",
       "      <td>38593</td>\n",
       "      <td>151924</td>\n",
       "      <td>0.254028</td>\n",
       "      <td>Benign</td>\n",
       "      <td>11</td>\n",
       "      <td>GCCACGTGGGAAGCGCGGTCC</td>\n",
       "      <td>GCCACGTGGGGAGCGCGGTCC</td>\n",
       "      <td>REF_5-1295234-A-G</td>\n",
       "      <td>ALT_5-1295234-A-G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>505</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZN684.H13CORE.1.M.C</td>\n",
       "      <td>5</td>\n",
       "      <td>19</td>\n",
       "      <td>-</td>\n",
       "      <td>ALT_5-1295234-A-G</td>\n",
       "      <td>0.0116</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>G</td>\n",
       "      <td>38593</td>\n",
       "      <td>151924</td>\n",
       "      <td>0.254028</td>\n",
       "      <td>Benign</td>\n",
       "      <td>11</td>\n",
       "      <td>GCCACGTGGGAAGCGCGGTCC</td>\n",
       "      <td>GCCACGTGGGGAGCGCGGTCC</td>\n",
       "      <td>REF_5-1295234-A-G</td>\n",
       "      <td>ALT_5-1295234-A-G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>506</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZN75A.H13CORE.0.SM.B</td>\n",
       "      <td>5</td>\n",
       "      <td>19</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295234-A-G</td>\n",
       "      <td>0.01550</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>G</td>\n",
       "      <td>38593</td>\n",
       "      <td>151924</td>\n",
       "      <td>0.254028</td>\n",
       "      <td>Benign</td>\n",
       "      <td>11</td>\n",
       "      <td>GCCACGTGGGAAGCGCGGTCC</td>\n",
       "      <td>GCCACGTGGGGAGCGCGGTCC</td>\n",
       "      <td>REF_5-1295234-A-G</td>\n",
       "      <td>ALT_5-1295234-A-G</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>507</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZNF215.H13CORE.0.PSG.A</td>\n",
       "      <td>8</td>\n",
       "      <td>18</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295234-A-G</td>\n",
       "      <td>0.06370</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>G</td>\n",
       "      <td>38593</td>\n",
       "      <td>151924</td>\n",
       "      <td>0.254028</td>\n",
       "      <td>Benign</td>\n",
       "      <td>11</td>\n",
       "      <td>GCCACGTGGGAAGCGCGGTCC</td>\n",
       "      <td>GCCACGTGGGGAGCGCGGTCC</td>\n",
       "      <td>REF_5-1295234-A-G</td>\n",
       "      <td>ALT_5-1295234-A-G</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>508 rows √ó 22 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        variant_id                motif_id  start  stop strand  \\\n",
       "0    5-1294991-G-A    GLIS2.H13CORE.0.SM.B      6    21      +   \n",
       "1    5-1294991-G-A     GLIS3.H13CORE.1.S.C     10    21      +   \n",
       "2    5-1294991-G-A     KAISO.H13CORE.1.P.B      4    17      +   \n",
       "3    5-1294991-G-A     ZBT14.H13CORE.0.P.C      3    15      +   \n",
       "4    5-1294991-G-A     ZFP57.H13CORE.0.P.B      1     9      +   \n",
       "..             ...                     ...    ...   ...    ...   \n",
       "503  5-1295234-A-G   ZN343.H13CORE.0.PSM.A      4    18      -   \n",
       "504  5-1295234-A-G     ZN449.H13CORE.1.S.C      6    20      +   \n",
       "505  5-1295234-A-G     ZN684.H13CORE.1.M.C      5    19      -   \n",
       "506  5-1295234-A-G    ZN75A.H13CORE.0.SM.B      5    19      +   \n",
       "507  5-1295234-A-G  ZNF215.H13CORE.0.PSG.A      8    18      -   \n",
       "\n",
       "              ALT_name  ALT_q-value           REF_name  REF_q-value chrom  \\\n",
       "0                  NaN          NaN  REF_5-1294991-G-A      0.00254     5   \n",
       "1                  NaN          NaN  REF_5-1294991-G-A      0.00860     5   \n",
       "2                  NaN          NaN  REF_5-1294991-G-A      0.00430     5   \n",
       "3                  NaN          NaN  REF_5-1294991-G-A      0.00572     5   \n",
       "4    ALT_5-1294991-G-A       0.0307  REF_5-1294991-G-A      0.03070     5   \n",
       "..                 ...          ...                ...          ...   ...   \n",
       "503                NaN          NaN  REF_5-1295234-A-G      0.03250     5   \n",
       "504  ALT_5-1295234-A-G       0.0243                NaN          NaN     5   \n",
       "505  ALT_5-1295234-A-G       0.0116                NaN          NaN     5   \n",
       "506                NaN          NaN  REF_5-1295234-A-G      0.01550     5   \n",
       "507                NaN          NaN  REF_5-1295234-A-G      0.06370     5   \n",
       "\n",
       "     ...  alt     ac       an        af   clinical_significance  variant_pos  \\\n",
       "0    ...    A      3  1317992  0.000002  Uncertain significance           11   \n",
       "1    ...    A      3  1317992  0.000002  Uncertain significance           11   \n",
       "2    ...    A      3  1317992  0.000002  Uncertain significance           11   \n",
       "3    ...    A      3  1317992  0.000002  Uncertain significance           11   \n",
       "4    ...    A      3  1317992  0.000002  Uncertain significance           11   \n",
       "..   ...  ...    ...      ...       ...                     ...          ...   \n",
       "503  ...    G  38593   151924  0.254028                  Benign           11   \n",
       "504  ...    G  38593   151924  0.254028                  Benign           11   \n",
       "505  ...    G  38593   151924  0.254028                  Benign           11   \n",
       "506  ...    G  38593   151924  0.254028                  Benign           11   \n",
       "507  ...    G  38593   151924  0.254028                  Benign           11   \n",
       "\n",
       "              ref_sequence           alt_sequence                REF  \\\n",
       "0    GCGCGGCATCGCGGGGGTGGC  GCGCGGCATCACGGGGGTGGC  REF_5-1294991-G-A   \n",
       "1    GCGCGGCATCGCGGGGGTGGC  GCGCGGCATCACGGGGGTGGC  REF_5-1294991-G-A   \n",
       "2    GCGCGGCATCGCGGGGGTGGC  GCGCGGCATCACGGGGGTGGC  REF_5-1294991-G-A   \n",
       "3    GCGCGGCATCGCGGGGGTGGC  GCGCGGCATCACGGGGGTGGC  REF_5-1294991-G-A   \n",
       "4    GCGCGGCATCGCGGGGGTGGC  GCGCGGCATCACGGGGGTGGC  REF_5-1294991-G-A   \n",
       "..                     ...                    ...                ...   \n",
       "503  GCCACGTGGGAAGCGCGGTCC  GCCACGTGGGGAGCGCGGTCC  REF_5-1295234-A-G   \n",
       "504  GCCACGTGGGAAGCGCGGTCC  GCCACGTGGGGAGCGCGGTCC  REF_5-1295234-A-G   \n",
       "505  GCCACGTGGGAAGCGCGGTCC  GCCACGTGGGGAGCGCGGTCC  REF_5-1295234-A-G   \n",
       "506  GCCACGTGGGAAGCGCGGTCC  GCCACGTGGGGAGCGCGGTCC  REF_5-1295234-A-G   \n",
       "507  GCCACGTGGGAAGCGCGGTCC  GCCACGTGGGGAGCGCGGTCC  REF_5-1295234-A-G   \n",
       "\n",
       "                   ALT  \n",
       "0    ALT_5-1294991-G-A  \n",
       "1    ALT_5-1294991-G-A  \n",
       "2    ALT_5-1294991-G-A  \n",
       "3    ALT_5-1294991-G-A  \n",
       "4    ALT_5-1294991-G-A  \n",
       "..                 ...  \n",
       "503  ALT_5-1295234-A-G  \n",
       "504  ALT_5-1295234-A-G  \n",
       "505  ALT_5-1295234-A-G  \n",
       "506  ALT_5-1295234-A-G  \n",
       "507  ALT_5-1295234-A-G  \n",
       "\n",
       "[508 rows x 22 columns]"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "clinvar_variant_infos = ['variant_id', 'variant_pos', 'ref_sequence', 'alt_sequence']\n",
    "variant_metadata_fimo_combined_df = combined_fimo_df.merge(df_clinvar_snvs, on=\"variant_id\", how=\"inner\")\n",
    "\n",
    "variant_metadata_fimo_combined_df['variant_pos'] = variant_metadata_fimo_combined_df['variant_pos'].astype(int)\n",
    "variant_metadata_fimo_combined_df"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9d9c81b8",
   "metadata": {},
   "source": [
    "Now we have to make sure that the motif overlaps our variant position. üéØ Therefore, we filter the combined table for motif start ‚â§ variant position ‚â§ motif end. ‚úÖ"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "00729339",
   "metadata": {},
   "outputs": [],
   "source": [
    "def motif_overlaps_variant(row, fimo_start_col='start', fimo_stop_col='stop', col_variant_pos='variant_pos'):\n",
    "    \"\"\"\n",
    "    Iterate over each variant position and return True if the start/stop coordinates\n",
    "    overlap the variant position.\n",
    "    \n",
    "    NOTE: It is important that we use 1-based coordinates for this comparison as FIMO uses 1-based coordinates.\n",
    "    \"\"\"\n",
    "    variant_pos = row[col_variant_pos]\n",
    "    if (int(row[fimo_start_col]) <= variant_pos and int(row[fimo_stop_col]) >= variant_pos):\n",
    "        return True\n",
    "    return False\n",
    "\n",
    "variant_metadata_fimo_combined_df[\"is_variant_overlapping\"] = variant_metadata_fimo_combined_df.apply(\n",
    "    lambda row: motif_overlaps_variant(row, fimo_start_col='start', fimo_stop_col='stop', col_variant_pos='variant_pos'),\n",
    "    axis=1\n",
    ")"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b3ffc80b",
   "metadata": {},
   "source": [
    "> **‚úÖ Result:**  \n",
    "> Now we have a table of significant variants and their overlapping TFBS predictions. Let's explore the numbers! üî¢üìä"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "78e3c3a9",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "motif_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "is_variant_overlapping",
         "rawType": "bool",
         "type": "boolean"
        }
       ],
       "ref": "d632abf6-3b71-407b-91b9-733b570fa569",
       "rows": [
        [
         "0",
         "5-1294991-G-A",
         "GLIS2.H13CORE.0.SM.B",
         "True"
        ],
        [
         "1",
         "5-1294991-G-A",
         "GLIS3.H13CORE.1.S.C",
         "True"
        ],
        [
         "2",
         "5-1294991-G-A",
         "KAISO.H13CORE.1.P.B",
         "True"
        ],
        [
         "3",
         "5-1294991-G-A",
         "ZBT14.H13CORE.0.P.C",
         "True"
        ],
        [
         "4",
         "5-1294991-G-A",
         "ZFP57.H13CORE.0.P.B",
         "False"
        ],
        [
         "5",
         "5-1294991-G-A",
         "ZIC1.H13CORE.1.M.B",
         "True"
        ],
        [
         "6",
         "5-1294991-G-A",
         "ZIC1.H13CORE.2.S.B",
         "True"
        ],
        [
         "7",
         "5-1294991-G-A",
         "ZIC3.H13CORE.1.S.B",
         "True"
        ],
        [
         "8",
         "5-1294991-G-A",
         "ZIC3.H13CORE.2.M.B",
         "True"
        ],
        [
         "9",
         "5-1294991-G-A",
         "ZIC4.H13CORE.0.SM.B",
         "True"
        ],
        [
         "10",
         "5-1294991-G-A",
         "ZIC5.H13CORE.1.M.C",
         "True"
        ],
        [
         "11",
         "5-1294991-G-A",
         "ZIC5.H13CORE.2.S.C",
         "True"
        ],
        [
         "12",
         "5-1294991-G-A",
         "ZN740.H13CORE.0.P.C",
         "True"
        ],
        [
         "13",
         "5-1294992-C-T",
         "EGR2.H13CORE.0.PSM.A",
         "True"
        ],
        [
         "14",
         "5-1294992-C-T",
         "EGR3.H13CORE.0.PSM.A",
         "True"
        ],
        [
         "15",
         "5-1294992-C-T",
         "GLIS1.H13CORE.1.P.B",
         "True"
        ],
        [
         "16",
         "5-1294992-C-T",
         "GLIS1.H13CORE.2.SM.B",
         "True"
        ],
        [
         "17",
         "5-1294992-C-T",
         "GLIS2.H13CORE.0.SM.B",
         "True"
        ],
        [
         "18",
         "5-1294992-C-T",
         "GLIS3.H13CORE.1.S.C",
         "True"
        ],
        [
         "19",
         "5-1294992-C-T",
         "KAISO.H13CORE.1.P.B",
         "True"
        ],
        [
         "20",
         "5-1294992-C-T",
         "KLF12.H13CORE.1.PSM.A",
         "True"
        ],
        [
         "21",
         "5-1294992-C-T",
         "KLF13.H13CORE.0.PSM.A",
         "True"
        ],
        [
         "22",
         "5-1294992-C-T",
         "KLF9.H13CORE.0.P.B",
         "True"
        ],
        [
         "23",
         "5-1294992-C-T",
         "ZBT14.H13CORE.0.P.C",
         "True"
        ],
        [
         "24",
         "5-1294992-C-T",
         "ZIC1.H13CORE.1.M.B",
         "True"
        ],
        [
         "25",
         "5-1294992-C-T",
         "ZIC1.H13CORE.2.S.B",
         "True"
        ],
        [
         "26",
         "5-1294992-C-T",
         "ZIC3.H13CORE.1.S.B",
         "True"
        ],
        [
         "27",
         "5-1294992-C-T",
         "ZIC3.H13CORE.2.M.B",
         "True"
        ],
        [
         "28",
         "5-1294992-C-T",
         "ZIC4.H13CORE.0.SM.B",
         "True"
        ],
        [
         "29",
         "5-1294992-C-T",
         "ZIC5.H13CORE.1.M.C",
         "True"
        ],
        [
         "30",
         "5-1294992-C-T",
         "ZIC5.H13CORE.2.S.C",
         "True"
        ],
        [
         "31",
         "5-1294992-C-T",
         "ZN740.H13CORE.0.P.C",
         "True"
        ],
        [
         "32",
         "5-1294992-C-T",
         "ZNF28.H13CORE.0.P.C",
         "True"
        ],
        [
         "33",
         "5-1294994-G-C",
         "EGR1.H13CORE.0.PS.A",
         "True"
        ],
        [
         "34",
         "5-1294994-G-C",
         "EGR2.H13CORE.0.PSM.A",
         "True"
        ],
        [
         "35",
         "5-1294994-G-C",
         "GLI1.H13CORE.0.P.C",
         "True"
        ],
        [
         "36",
         "5-1294994-G-C",
         "GLI2.H13CORE.0.PSM.A",
         "True"
        ],
        [
         "37",
         "5-1294994-G-C",
         "GLI3.H13CORE.0.PS.A",
         "True"
        ],
        [
         "38",
         "5-1294994-G-C",
         "GLIS1.H13CORE.2.SM.B",
         "True"
        ],
        [
         "39",
         "5-1294994-G-C",
         "GLIS2.H13CORE.0.SM.B",
         "True"
        ],
        [
         "40",
         "5-1294994-G-C",
         "GLIS3.H13CORE.1.S.C",
         "True"
        ],
        [
         "41",
         "5-1294994-G-C",
         "KAISO.H13CORE.1.P.B",
         "True"
        ],
        [
         "42",
         "5-1294994-G-C",
         "ZIC1.H13CORE.1.M.B",
         "True"
        ],
        [
         "43",
         "5-1294994-G-C",
         "ZIC1.H13CORE.2.S.B",
         "True"
        ],
        [
         "44",
         "5-1294994-G-C",
         "ZIC3.H13CORE.1.S.B",
         "True"
        ],
        [
         "45",
         "5-1294994-G-C",
         "ZIC3.H13CORE.2.M.B",
         "True"
        ],
        [
         "46",
         "5-1294994-G-C",
         "ZIC4.H13CORE.0.SM.B",
         "True"
        ],
        [
         "47",
         "5-1294994-G-C",
         "ZIC5.H13CORE.1.M.C",
         "True"
        ],
        [
         "48",
         "5-1294994-G-C",
         "ZIC5.H13CORE.2.S.C",
         "True"
        ],
        [
         "49",
         "5-1294994-G-C",
         "ZN740.H13CORE.0.P.C",
         "True"
        ]
       ],
       "shape": {
        "columns": 3,
        "rows": 508
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>motif_id</th>\n",
       "      <th>is_variant_overlapping</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>GLIS2.H13CORE.0.SM.B</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>GLIS3.H13CORE.1.S.C</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>KAISO.H13CORE.1.P.B</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>ZBT14.H13CORE.0.P.C</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>ZFP57.H13CORE.0.P.B</td>\n",
       "      <td>False</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>503</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZN343.H13CORE.0.PSM.A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>504</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZN449.H13CORE.1.S.C</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>505</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZN684.H13CORE.1.M.C</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>506</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZN75A.H13CORE.0.SM.B</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>507</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>ZNF215.H13CORE.0.PSG.A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>508 rows √ó 3 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        variant_id                motif_id  is_variant_overlapping\n",
       "0    5-1294991-G-A    GLIS2.H13CORE.0.SM.B                    True\n",
       "1    5-1294991-G-A     GLIS3.H13CORE.1.S.C                    True\n",
       "2    5-1294991-G-A     KAISO.H13CORE.1.P.B                    True\n",
       "3    5-1294991-G-A     ZBT14.H13CORE.0.P.C                    True\n",
       "4    5-1294991-G-A     ZFP57.H13CORE.0.P.B                   False\n",
       "..             ...                     ...                     ...\n",
       "503  5-1295234-A-G   ZN343.H13CORE.0.PSM.A                    True\n",
       "504  5-1295234-A-G     ZN449.H13CORE.1.S.C                    True\n",
       "505  5-1295234-A-G     ZN684.H13CORE.1.M.C                    True\n",
       "506  5-1295234-A-G    ZN75A.H13CORE.0.SM.B                    True\n",
       "507  5-1295234-A-G  ZNF215.H13CORE.0.PSG.A                    True\n",
       "\n",
       "[508 rows x 3 columns]"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Preview combined variant-TFBS table\n",
    "variant_metadata_fimo_combined_df[[\"variant_id\", \"motif_id\", \"is_variant_overlapping\"]]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "96746562",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Filter for variant-overlapping TFBS\n",
    "variant_overlapping_tfbs = variant_metadata_fimo_combined_df.loc[variant_metadata_fimo_combined_df['is_variant_overlapping']].copy()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "63d9db0e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of Motif predictions within the given sequences 495\n",
      "Number of different variants we found at least one TFBS 26\n"
     ]
    }
   ],
   "source": [
    "print(\"Number of Motif predictions within the given sequences\", variant_overlapping_tfbs.shape[0])\n",
    "print(\"Number of different variants we found at least one TFBS\", variant_overlapping_tfbs['variant_id'].nunique())"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "708f3fbd",
   "metadata": {},
   "source": [
    "### ‚öñÔ∏è Motif Gain or Loss? Assessing TFBS Changes\n",
    "\n",
    "We can compare if the motifs fit the reference or the alternative sequence better, so we know if the motif is more likely to bind at a specific allele. üî¨ But from just computational predictions we cannot tell if the motif is now bound or not bound anymore because of the SNV. However, we can increase the confidence of our predictions, which are prone to false positives. ‚úÖ"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "254f2dea",
   "metadata": {},
   "source": [
    "> **üîç Observation:**  \n",
    "> For 26 variants, we see 495 TFBS hits! üéØ  \n",
    "> - Short motifs ‚Üí many chance matches (false positives) ‚ö†Ô∏è\n",
    "> - Multiple TFBS can overlap the same variant üîÑ\n",
    ">\n",
    "> **ü§î How to prioritize?**\n",
    "> - Use more stringent q-value thresholds üìä\n",
    "> - Focus on TFs expressed in your cell type üß´\n",
    "> - Use additional data (e.g., ChIP-seq) üß™"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "242f9058",
   "metadata": {},
   "source": [
    "#### üéöÔ∏è Filter by q-value (Stringency)\n",
    "\n",
    "Let's filter for motif matches with q-value < 0.001 for higher confidence. ‚úÖ\n",
    "\n",
    "0.001 is very stringent. üìè You can play around with this value to see how many motif matches you get with different thresholds. üîß"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 31,
   "id": "6569c4a6",
   "metadata": {},
   "outputs": [],
   "source": [
    "q_value_threshold = 0.001\n",
    "\n",
    "variant_metadata_fimo_combined_overlap_filtered = variant_overlapping_tfbs.loc[\n",
    "    (variant_overlapping_tfbs['ALT_q-value'] < q_value_threshold) | (variant_overlapping_tfbs['ALT_q-value'].isna())\n",
    "].copy()\n",
    "\n",
    "variant_metadata_fimo_combined_overlap_filtered = variant_metadata_fimo_combined_overlap_filtered.loc[\n",
    "    (variant_metadata_fimo_combined_overlap_filtered['REF_q-value'] < q_value_threshold) | \n",
    "    (variant_metadata_fimo_combined_overlap_filtered['REF_q-value'].isna())\n",
    "].copy()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "ada5441d",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of Motif predictions within the given sequences 36\n",
      "Number of different variants we found at least one TFBS 16\n"
     ]
    }
   ],
   "source": [
    "print(\"Number of Motif predictions within the given sequences\", variant_metadata_fimo_combined_overlap_filtered.shape[0])\n",
    "print(\"Number of different variants we found at least one TFBS\", variant_metadata_fimo_combined_overlap_filtered['variant_id'].nunique())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "121362b0",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "motif_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "start",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "stop",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "strand",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ALT_name",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "ALT_q-value",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "REF_name",
         "rawType": "object",
         "type": "unknown"
        },
        {
         "name": "REF_q-value",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "chrom",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "pos",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "ref",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "alt",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ac",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "an",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "af",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "clinical_significance",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "variant_pos",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "ref_sequence",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "alt_sequence",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "REF",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "ALT",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "is_variant_overlapping",
         "rawType": "bool",
         "type": "boolean"
        }
       ],
       "ref": "1e9e09fa-d97a-4735-b71e-3aa2b652d4f4",
       "rows": [
        [
         "32",
         "5-1294992-C-T",
         "ZNF28.H13CORE.0.P.C",
         "4",
         "21",
         "+",
         "ALT_5-1294992-C-T",
         "0.000839",
         "REF_5-1294992-C-T",
         "0.000493",
         "5",
         "1294992",
         "C",
         "T",
         "2",
         "1312640",
         "1.5236470014627011e-06",
         "Likely benign",
         "11",
         "CGCGGCATCGCGGGGGTGGCC",
         "CGCGGCATCGTGGGGGTGGCC",
         "REF_5-1294992-C-T",
         "ALT_5-1294992-C-T",
         "True"
        ],
        [
         "50",
         "5-1294994-G-C",
         "ZNF28.H13CORE.0.P.C",
         "2",
         "19",
         "+",
         null,
         null,
         "REF_5-1294994-G-C",
         "0.000493",
         "5",
         "1294994",
         "G",
         "C",
         "2",
         "1315502",
         "1.5203321621707911e-06",
         "Uncertain significance",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGCGGGTGGCCGG",
         "REF_5-1294994-G-C",
         "ALT_5-1294994-G-C",
         "True"
        ],
        [
         "71",
         "5-1294994-G-T",
         "ZNF28.H13CORE.0.P.C",
         "2",
         "19",
         "+",
         null,
         null,
         "REF_5-1294994-G-T",
         "0.000493",
         "5",
         "1294994",
         "G",
         "T",
         "1",
         "1315610",
         "7.601036781416985e-07",
         "Likely benign",
         "11",
         "CGGCATCGCGGGGGTGGCCGG",
         "CGGCATCGCGTGGGTGGCCGG",
         "REF_5-1294994-G-T",
         "ALT_5-1294994-G-T",
         "True"
        ],
        [
         "91",
         "5-1294995-G-T",
         "ZNF28.H13CORE.0.P.C",
         "1",
         "18",
         "+",
         null,
         null,
         "REF_5-1294995-G-T",
         "0.000493",
         "5",
         "1294995",
         "G",
         "T",
         "13",
         "1315568",
         "9.881663281563553e-06",
         "Likely benign",
         "11",
         "GGCATCGCGGGGGTGGCCGGG",
         "GGCATCGCGGTGGTGGCCGGG",
         "REF_5-1294995-G-T",
         "ALT_5-1294995-G-T",
         "True"
        ],
        [
         "92",
         "5-1294995-G-T",
         "ZNF800.H13CORE.0.PSG.A",
         "1",
         "21",
         "+",
         "ALT_5-1294995-G-T",
         "4.72e-05",
         "REF_5-1294995-G-T",
         "4.72e-05",
         "5",
         "1294995",
         "G",
         "T",
         "13",
         "1315568",
         "9.881663281563553e-06",
         "Likely benign",
         "11",
         "GGCATCGCGGGGGTGGCCGGG",
         "GGCATCGCGGTGGTGGCCGGG",
         "REF_5-1294995-G-T",
         "ALT_5-1294995-G-T",
         "True"
        ],
        [
         "116",
         "5-1295020-C-G",
         "MYCN.H13CORE.0.PS.A",
         "7",
         "17",
         "+",
         null,
         null,
         "REF_5-1295020-C-G",
         "0.000525",
         "5",
         "1295020",
         "C",
         "G",
         "13",
         "1278802",
         "1.0165764520230653e-05",
         "Uncertain significance",
         "11",
         "GGGCTTCCCACGTGCGCAGCA",
         "GGGCTTCCCAGGTGCGCAGCA",
         "REF_5-1295020-C-G",
         "ALT_5-1295020-C-G",
         "True"
        ],
        [
         "200",
         "5-1295075-G-A",
         "KLF13.H13CORE.1.P.C",
         "1",
         "20",
         "+",
         "ALT_5-1295075-G-A",
         "0.000116",
         "REF_5-1295075-G-A",
         "3.05e-05",
         "5",
         "1295075",
         "G",
         "A",
         "12",
         "973884",
         "1.2321796024988602e-05",
         "Uncertain significance",
         "11",
         "AGAGGGCGGGGCCGCGGAAAG",
         "AGAGGGCGGGACCGCGGAAAG",
         "REF_5-1295075-G-A",
         "ALT_5-1295075-G-A",
         "True"
        ],
        [
         "201",
         "5-1295075-G-A",
         "KLF15.H13CORE.1.P.C",
         "2",
         "21",
         "+",
         "ALT_5-1295075-G-A",
         "0.000512",
         "REF_5-1295075-G-A",
         "0.000195",
         "5",
         "1295075",
         "G",
         "A",
         "12",
         "973884",
         "1.2321796024988602e-05",
         "Uncertain significance",
         "11",
         "AGAGGGCGGGGCCGCGGAAAG",
         "AGAGGGCGGGACCGCGGAAAG",
         "REF_5-1295075-G-A",
         "ALT_5-1295075-G-A",
         "True"
        ],
        [
         "210",
         "5-1295075-G-A",
         "MAZ.H13CORE.0.P.B",
         "1",
         "13",
         "+",
         "ALT_5-1295075-G-A",
         "0.000832",
         "REF_5-1295075-G-A",
         "0.000231",
         "5",
         "1295075",
         "G",
         "A",
         "12",
         "973884",
         "1.2321796024988602e-05",
         "Uncertain significance",
         "11",
         "AGAGGGCGGGGCCGCGGAAAG",
         "AGAGGGCGGGACCGCGGAAAG",
         "REF_5-1295075-G-A",
         "ALT_5-1295075-G-A",
         "True"
        ],
        [
         "211",
         "5-1295075-G-A",
         "MAZ.H13CORE.1.P.B",
         "1",
         "21",
         "+",
         "ALT_5-1295075-G-A",
         "1.18e-05",
         "REF_5-1295075-G-A",
         "1.43e-06",
         "5",
         "1295075",
         "G",
         "A",
         "12",
         "973884",
         "1.2321796024988602e-05",
         "Uncertain significance",
         "11",
         "AGAGGGCGGGGCCGCGGAAAG",
         "AGAGGGCGGGACCGCGGAAAG",
         "REF_5-1295075-G-A",
         "ALT_5-1295075-G-A",
         "True"
        ],
        [
         "215",
         "5-1295075-G-A",
         "SP1.H13CORE.0.P.B",
         "1",
         "14",
         "+",
         "ALT_5-1295075-G-A",
         "0.000697",
         "REF_5-1295075-G-A",
         "6.5e-05",
         "5",
         "1295075",
         "G",
         "A",
         "12",
         "973884",
         "1.2321796024988602e-05",
         "Uncertain significance",
         "11",
         "AGAGGGCGGGGCCGCGGAAAG",
         "AGAGGGCGGGACCGCGGAAAG",
         "REF_5-1295075-G-A",
         "ALT_5-1295075-G-A",
         "True"
        ],
        [
         "217",
         "5-1295075-G-A",
         "SP1.H13CORE.2.P.B",
         "1",
         "21",
         "+",
         "ALT_5-1295075-G-A",
         "2.81e-05",
         "REF_5-1295075-G-A",
         "3.28e-06",
         "5",
         "1295075",
         "G",
         "A",
         "12",
         "973884",
         "1.2321796024988602e-05",
         "Uncertain significance",
         "11",
         "AGAGGGCGGGGCCGCGGAAAG",
         "AGAGGGCGGGACCGCGGAAAG",
         "REF_5-1295075-G-A",
         "ALT_5-1295075-G-A",
         "True"
        ],
        [
         "218",
         "5-1295075-G-A",
         "SP2.H13CORE.0.P.D",
         "1",
         "12",
         "+",
         "ALT_5-1295075-G-A",
         "0.000777",
         "REF_5-1295075-G-A",
         "0.000262",
         "5",
         "1295075",
         "G",
         "A",
         "12",
         "973884",
         "1.2321796024988602e-05",
         "Uncertain significance",
         "11",
         "AGAGGGCGGGGCCGCGGAAAG",
         "AGAGGGCGGGACCGCGGAAAG",
         "REF_5-1295075-G-A",
         "ALT_5-1295075-G-A",
         "True"
        ],
        [
         "220",
         "5-1295075-G-A",
         "SP4.H13CORE.0.P.C",
         "1",
         "15",
         "+",
         "ALT_5-1295075-G-A",
         "0.000198",
         "REF_5-1295075-G-A",
         "3.15e-05",
         "5",
         "1295075",
         "G",
         "A",
         "12",
         "973884",
         "1.2321796024988602e-05",
         "Uncertain significance",
         "11",
         "AGAGGGCGGGGCCGCGGAAAG",
         "AGAGGGCGGGACCGCGGAAAG",
         "REF_5-1295075-G-A",
         "ALT_5-1295075-G-A",
         "True"
        ],
        [
         "222",
         "5-1295075-G-A",
         "SP4.H13CORE.2.P.B",
         "1",
         "21",
         "+",
         "ALT_5-1295075-G-A",
         "3.48e-05",
         "REF_5-1295075-G-A",
         "4.54e-06",
         "5",
         "1295075",
         "G",
         "A",
         "12",
         "973884",
         "1.2321796024988602e-05",
         "Uncertain significance",
         "11",
         "AGAGGGCGGGGCCGCGGAAAG",
         "AGAGGGCGGGACCGCGGAAAG",
         "REF_5-1295075-G-A",
         "ALT_5-1295075-G-A",
         "True"
        ],
        [
         "236",
         "5-1295112-A-T",
         "CTCFL.H13CORE.0.P.B",
         "5",
         "20",
         "+",
         null,
         null,
         "REF_5-1295112-A-T",
         "0.000195",
         "5",
         "1295112",
         "A",
         "T",
         "2",
         "98898",
         "2.0222855871706203e-05",
         "Uncertain significance",
         "11",
         "GAGGGCCCGGAGGGGGCTGGG",
         "GAGGGCCCGGTGGGGGCTGGG",
         "REF_5-1295112-A-T",
         "ALT_5-1295112-A-T",
         "True"
        ],
        [
         "242",
         "5-1295112-A-T",
         "ZBTB41.H13CORE.0.PSG.A",
         "1",
         "19",
         "+",
         "ALT_5-1295112-A-T",
         "0.000877",
         "REF_5-1295112-A-T",
         "0.000877",
         "5",
         "1295112",
         "A",
         "T",
         "2",
         "98898",
         "2.0222855871706203e-05",
         "Uncertain significance",
         "11",
         "GAGGGCCCGGAGGGGGCTGGG",
         "GAGGGCCCGGTGGGGGCTGGG",
         "REF_5-1295112-A-T",
         "ALT_5-1295112-A-T",
         "True"
        ],
        [
         "247",
         "5-1295112-A-T",
         "ZNF709.H13CORE.0.P.C",
         "1",
         "21",
         "+",
         null,
         null,
         "REF_5-1295112-A-T",
         "0.000967",
         "5",
         "1295112",
         "A",
         "T",
         "2",
         "98898",
         "2.0222855871706203e-05",
         "Uncertain significance",
         "11",
         "GAGGGCCCGGAGGGGGCTGGG",
         "GAGGGCCCGGTGGGGGCTGGG",
         "REF_5-1295112-A-T",
         "ALT_5-1295112-A-T",
         "True"
        ],
        [
         "248",
         "5-1295113-G-A",
         "CTCFL.H13CORE.0.P.B",
         "4",
         "19",
         "+",
         null,
         null,
         "REF_5-1295113-G-A",
         "0.000195",
         "5",
         "1295113",
         "G",
         "A",
         "62",
         "655850",
         "9.453381108485172e-05",
         "Likely pathogenic",
         "11",
         "AGGGCCCGGAGGGGGCTGGGC",
         "AGGGCCCGGAAGGGGCTGGGC",
         "REF_5-1295113-G-A",
         "ALT_5-1295113-G-A",
         "True"
        ],
        [
         "266",
         "5-1295113-G-A",
         "FLI1.H13CORE.1.P.B",
         "3",
         "20",
         "+",
         "ALT_5-1295113-G-A",
         "0.000939",
         null,
         null,
         "5",
         "1295113",
         "G",
         "A",
         "62",
         "655850",
         "9.453381108485172e-05",
         "Likely pathogenic",
         "11",
         "AGGGCCCGGAGGGGGCTGGGC",
         "AGGGCCCGGAAGGGGCTGGGC",
         "REF_5-1295113-G-A",
         "ALT_5-1295113-G-A",
         "True"
        ],
        [
         "268",
         "5-1295113-G-A",
         "MAZ.H13CORE.0.P.B",
         "4",
         "16",
         "+",
         null,
         null,
         "REF_5-1295113-G-A",
         "0.000832",
         "5",
         "1295113",
         "G",
         "A",
         "62",
         "655850",
         "9.453381108485172e-05",
         "Likely pathogenic",
         "11",
         "AGGGCCCGGAGGGGGCTGGGC",
         "AGGGCCCGGAAGGGGCTGGGC",
         "REF_5-1295113-G-A",
         "ALT_5-1295113-G-A",
         "True"
        ],
        [
         "282",
         "5-1295128-G-A",
         "MAZ.H13CORE.1.P.B",
         "1",
         "21",
         "+",
         null,
         null,
         "REF_5-1295128-G-A",
         "0.000614",
         "5",
         "1295128",
         "G",
         "A",
         "2",
         "386064",
         "5.180488209208836e-06",
         "Uncertain significance",
         "11",
         "CTGGGCCGGGGACCCGGGAGG",
         "CTGGGCCGGGAACCCGGGAGG",
         "REF_5-1295128-G-A",
         "ALT_5-1295128-G-A",
         "True"
        ],
        [
         "288",
         "5-1295135-G-A",
         "CTCFL.H13CORE.0.P.B",
         "6",
         "21",
         "+",
         "ALT_5-1295135-G-A",
         "0.000195",
         "REF_5-1295135-G-A",
         "0.000195",
         "5",
         "1295135",
         "G",
         "A",
         "13",
         "484372",
         "2.683887590529593e-05",
         "Pathogenic",
         "11",
         "GGGGACCCGGGAGGGGTCGGG",
         "GGGGACCCGGAAGGGGTCGGG",
         "REF_5-1295135-G-A",
         "ALT_5-1295135-G-A",
         "True"
        ],
        [
         "289",
         "5-1295135-G-A",
         "E4F1.H13CORE.0.P.B",
         "4",
         "16",
         "+",
         "ALT_5-1295135-G-A",
         "0.0009",
         null,
         null,
         "5",
         "1295135",
         "G",
         "A",
         "13",
         "484372",
         "2.683887590529593e-05",
         "Pathogenic",
         "11",
         "GGGGACCCGGGAGGGGTCGGG",
         "GGGGACCCGGAAGGGGTCGGG",
         "REF_5-1295135-G-A",
         "ALT_5-1295135-G-A",
         "True"
        ],
        [
         "311",
         "5-1295135-G-A",
         "HINFP.H13CORE.0.P.B",
         "1",
         "20",
         "+",
         "ALT_5-1295135-G-A",
         "0.000173",
         "REF_5-1295135-G-A",
         "7.05e-05",
         "5",
         "1295135",
         "G",
         "A",
         "13",
         "484372",
         "2.683887590529593e-05",
         "Pathogenic",
         "11",
         "GGGGACCCGGGAGGGGTCGGG",
         "GGGGACCCGGAAGGGGTCGGG",
         "REF_5-1295135-G-A",
         "ALT_5-1295135-G-A",
         "True"
        ],
        [
         "322",
         "5-1295135-G-A",
         "ZNF709.H13CORE.0.P.C",
         "1",
         "21",
         "+",
         "ALT_5-1295135-G-A",
         "0.000967",
         "REF_5-1295135-G-A",
         "0.000967",
         "5",
         "1295135",
         "G",
         "A",
         "13",
         "484372",
         "2.683887590529593e-05",
         "Pathogenic",
         "11",
         "GGGGACCCGGGAGGGGTCGGG",
         "GGGGACCCGGAAGGGGTCGGG",
         "REF_5-1295135-G-A",
         "ALT_5-1295135-G-A",
         "True"
        ],
        [
         "323",
         "5-1295136-A-G",
         "CTCFL.H13CORE.0.P.B",
         "5",
         "20",
         "+",
         "ALT_5-1295136-A-G",
         "0.000925",
         "REF_5-1295136-A-G",
         "0.000195",
         "5",
         "1295136",
         "A",
         "G",
         "0",
         "108874",
         "0.0",
         "Uncertain significance",
         "11",
         "GGGACCCGGGAGGGGTCGGGA",
         "GGGACCCGGGGGGGGTCGGGA",
         "REF_5-1295136-A-G",
         "ALT_5-1295136-A-G",
         "True"
        ],
        [
         "359",
         "5-1295162-G-A",
         "ZBTB41.H13CORE.0.PSG.A",
         "2",
         "20",
         "+",
         "ALT_5-1295162-G-A",
         "0.000949",
         "REF_5-1295162-G-A",
         "0.000877",
         "5",
         "1295162",
         "G",
         "A",
         "6",
         "434268",
         "1.3816353035452762e-05",
         "Uncertain significance",
         "11",
         "CGGGGTCCGCGCGGAGGAGGC",
         "CGGGGTCCGCACGGAGGAGGC",
         "REF_5-1295162-G-A",
         "ALT_5-1295162-G-A",
         "True"
        ],
        [
         "373",
         "5-1295163-C-G",
         "ZBTB41.H13CORE.0.PSG.A",
         "1",
         "19",
         "+",
         "ALT_5-1295163-C-G",
         "0.000877",
         "REF_5-1295163-C-G",
         "0.000877",
         "5",
         "1295163",
         "C",
         "G",
         "0",
         "433444",
         "0.0",
         "Likely benign",
         "11",
         "GGGGTCCGCGCGGAGGAGGCG",
         "GGGGTCCGCGGGGAGGAGGCG",
         "REF_5-1295163-C-G",
         "ALT_5-1295163-C-G",
         "True"
        ],
        [
         "382",
         "5-1295163-C-G",
         "ZN263.H13CORE.0.PM.A",
         "10",
         "19",
         "+",
         "ALT_5-1295163-C-G",
         "0.000412",
         null,
         null,
         "5",
         "1295163",
         "C",
         "G",
         "0",
         "433444",
         "0.0",
         "Likely benign",
         "11",
         "GGGGTCCGCGCGGAGGAGGCG",
         "GGGGTCCGCGGGGAGGAGGCG",
         "REF_5-1295163-C-G",
         "ALT_5-1295163-C-G",
         "True"
        ],
        [
         "390",
         "5-1295181-A-G",
         "HINFP.H13CORE.0.P.B",
         "1",
         "20",
         "+",
         "ALT_5-1295181-A-G",
         "0.000285",
         "REF_5-1295181-A-G",
         "0.000778",
         "5",
         "1295181",
         "A",
         "G",
         "0",
         "425236",
         "0.0",
         "Uncertain significance",
         "11",
         "GCGGAGCTGGAAGGTGAAGGG",
         "GCGGAGCTGGGAGGTGAAGGG",
         "REF_5-1295181-A-G",
         "ALT_5-1295181-A-G",
         "True"
        ],
        [
         "399",
         "5-1295181-A-G",
         "ZN263.H13CORE.1.P.B",
         "1",
         "21",
         "+",
         "ALT_5-1295181-A-G",
         "0.000774",
         "REF_5-1295181-A-G",
         "0.000774",
         "5",
         "1295181",
         "A",
         "G",
         "0",
         "425236",
         "0.0",
         "Uncertain significance",
         "11",
         "GCGGAGCTGGAAGGTGAAGGG",
         "GCGGAGCTGGGAGGTGAAGGG",
         "REF_5-1295181-A-G",
         "ALT_5-1295181-A-G",
         "True"
        ],
        [
         "403",
         "5-1295181-A-G",
         "ZN860.H13CORE.0.P.C",
         "3",
         "19",
         "-",
         "ALT_5-1295181-A-G",
         "0.000869",
         null,
         null,
         "5",
         "1295181",
         "A",
         "G",
         "0",
         "425236",
         "0.0",
         "Uncertain significance",
         "11",
         "GCGGAGCTGGAAGGTGAAGGG",
         "GCGGAGCTGGGAGGTGAAGGG",
         "REF_5-1295181-A-G",
         "ALT_5-1295181-A-G",
         "True"
        ],
        [
         "409",
         "5-1295182-A-G",
         "MAZ.H13CORE.1.P.B",
         "1",
         "21",
         "+",
         "ALT_5-1295182-A-G",
         "0.000614",
         null,
         null,
         "5",
         "1295182",
         "A",
         "G",
         "77",
         "424486",
         "0.0001813958528667612",
         "Uncertain significance",
         "11",
         "CGGAGCTGGAAGGTGAAGGGG",
         "CGGAGCTGGAGGGTGAAGGGG",
         "REF_5-1295182-A-G",
         "ALT_5-1295182-A-G",
         "True"
        ],
        [
         "470",
         "5-1295225-C-T",
         "MYC.H13CORE.0.P.B",
         "10",
         "19",
         "-",
         null,
         null,
         "REF_5-1295225-C-T",
         "0.000116",
         "5",
         "1295225",
         "C",
         "T",
         "58",
         "152198",
         "0.00038108253722125124",
         "Uncertain significance",
         "11",
         "AGTCCCTCCGCCACGTGGGAA",
         "AGTCCCTCCGTCACGTGGGAA",
         "REF_5-1295225-C-T",
         "ALT_5-1295225-C-T",
         "True"
        ],
        [
         "475",
         "5-1295225-C-T",
         "TET3.H13CORE.0.PSGIB.A",
         "2",
         "19",
         "-",
         "ALT_5-1295225-C-T",
         "0.000748",
         "REF_5-1295225-C-T",
         "0.000748",
         "5",
         "1295225",
         "C",
         "T",
         "58",
         "152198",
         "0.00038108253722125124",
         "Uncertain significance",
         "11",
         "AGTCCCTCCGCCACGTGGGAA",
         "AGTCCCTCCGTCACGTGGGAA",
         "REF_5-1295225-C-T",
         "ALT_5-1295225-C-T",
         "True"
        ]
       ],
       "shape": {
        "columns": 23,
        "rows": 36
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>motif_id</th>\n",
       "      <th>start</th>\n",
       "      <th>stop</th>\n",
       "      <th>strand</th>\n",
       "      <th>ALT_name</th>\n",
       "      <th>ALT_q-value</th>\n",
       "      <th>REF_name</th>\n",
       "      <th>REF_q-value</th>\n",
       "      <th>chrom</th>\n",
       "      <th>...</th>\n",
       "      <th>ac</th>\n",
       "      <th>an</th>\n",
       "      <th>af</th>\n",
       "      <th>clinical_significance</th>\n",
       "      <th>variant_pos</th>\n",
       "      <th>ref_sequence</th>\n",
       "      <th>alt_sequence</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>is_variant_overlapping</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>5-1294992-C-T</td>\n",
       "      <td>ZNF28.H13CORE.0.P.C</td>\n",
       "      <td>4</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1294992-C-T</td>\n",
       "      <td>0.000839</td>\n",
       "      <td>REF_5-1294992-C-T</td>\n",
       "      <td>0.000493</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>2</td>\n",
       "      <td>1312640</td>\n",
       "      <td>1.523647e-06</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>11</td>\n",
       "      <td>CGCGGCATCGCGGGGGTGGCC</td>\n",
       "      <td>CGCGGCATCGTGGGGGTGGCC</td>\n",
       "      <td>REF_5-1294992-C-T</td>\n",
       "      <td>ALT_5-1294992-C-T</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>5-1294994-G-C</td>\n",
       "      <td>ZNF28.H13CORE.0.P.C</td>\n",
       "      <td>2</td>\n",
       "      <td>19</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1294994-G-C</td>\n",
       "      <td>0.000493</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>2</td>\n",
       "      <td>1315502</td>\n",
       "      <td>1.520332e-06</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>CGGCATCGCGGGGGTGGCCGG</td>\n",
       "      <td>CGGCATCGCGCGGGTGGCCGG</td>\n",
       "      <td>REF_5-1294994-G-C</td>\n",
       "      <td>ALT_5-1294994-G-C</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>5-1294994-G-T</td>\n",
       "      <td>ZNF28.H13CORE.0.P.C</td>\n",
       "      <td>2</td>\n",
       "      <td>19</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1294994-G-T</td>\n",
       "      <td>0.000493</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>1</td>\n",
       "      <td>1315610</td>\n",
       "      <td>7.601037e-07</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>11</td>\n",
       "      <td>CGGCATCGCGGGGGTGGCCGG</td>\n",
       "      <td>CGGCATCGCGTGGGTGGCCGG</td>\n",
       "      <td>REF_5-1294994-G-T</td>\n",
       "      <td>ALT_5-1294994-G-T</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>5-1294995-G-T</td>\n",
       "      <td>ZNF28.H13CORE.0.P.C</td>\n",
       "      <td>1</td>\n",
       "      <td>18</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1294995-G-T</td>\n",
       "      <td>0.000493</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>13</td>\n",
       "      <td>1315568</td>\n",
       "      <td>9.881663e-06</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>11</td>\n",
       "      <td>GGCATCGCGGGGGTGGCCGGG</td>\n",
       "      <td>GGCATCGCGGTGGTGGCCGGG</td>\n",
       "      <td>REF_5-1294995-G-T</td>\n",
       "      <td>ALT_5-1294995-G-T</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>5-1294995-G-T</td>\n",
       "      <td>ZNF800.H13CORE.0.PSG.A</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1294995-G-T</td>\n",
       "      <td>0.000047</td>\n",
       "      <td>REF_5-1294995-G-T</td>\n",
       "      <td>0.000047</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>13</td>\n",
       "      <td>1315568</td>\n",
       "      <td>9.881663e-06</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>11</td>\n",
       "      <td>GGCATCGCGGGGGTGGCCGGG</td>\n",
       "      <td>GGCATCGCGGTGGTGGCCGGG</td>\n",
       "      <td>REF_5-1294995-G-T</td>\n",
       "      <td>ALT_5-1294995-G-T</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>116</th>\n",
       "      <td>5-1295020-C-G</td>\n",
       "      <td>MYCN.H13CORE.0.PS.A</td>\n",
       "      <td>7</td>\n",
       "      <td>17</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295020-C-G</td>\n",
       "      <td>0.000525</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>13</td>\n",
       "      <td>1278802</td>\n",
       "      <td>1.016576e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GGGCTTCCCACGTGCGCAGCA</td>\n",
       "      <td>GGGCTTCCCAGGTGCGCAGCA</td>\n",
       "      <td>REF_5-1295020-C-G</td>\n",
       "      <td>ALT_5-1295020-C-G</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>5-1295075-G-A</td>\n",
       "      <td>KLF13.H13CORE.1.P.C</td>\n",
       "      <td>1</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>0.000116</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>0.000030</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>12</td>\n",
       "      <td>973884</td>\n",
       "      <td>1.232180e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>AGAGGGCGGGGCCGCGGAAAG</td>\n",
       "      <td>AGAGGGCGGGACCGCGGAAAG</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>5-1295075-G-A</td>\n",
       "      <td>KLF15.H13CORE.1.P.C</td>\n",
       "      <td>2</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>0.000512</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>0.000195</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>12</td>\n",
       "      <td>973884</td>\n",
       "      <td>1.232180e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>AGAGGGCGGGGCCGCGGAAAG</td>\n",
       "      <td>AGAGGGCGGGACCGCGGAAAG</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>210</th>\n",
       "      <td>5-1295075-G-A</td>\n",
       "      <td>MAZ.H13CORE.0.P.B</td>\n",
       "      <td>1</td>\n",
       "      <td>13</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>0.000832</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>0.000231</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>12</td>\n",
       "      <td>973884</td>\n",
       "      <td>1.232180e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>AGAGGGCGGGGCCGCGGAAAG</td>\n",
       "      <td>AGAGGGCGGGACCGCGGAAAG</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>211</th>\n",
       "      <td>5-1295075-G-A</td>\n",
       "      <td>MAZ.H13CORE.1.P.B</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>0.000012</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>0.000001</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>12</td>\n",
       "      <td>973884</td>\n",
       "      <td>1.232180e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>AGAGGGCGGGGCCGCGGAAAG</td>\n",
       "      <td>AGAGGGCGGGACCGCGGAAAG</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>215</th>\n",
       "      <td>5-1295075-G-A</td>\n",
       "      <td>SP1.H13CORE.0.P.B</td>\n",
       "      <td>1</td>\n",
       "      <td>14</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>0.000697</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>0.000065</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>12</td>\n",
       "      <td>973884</td>\n",
       "      <td>1.232180e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>AGAGGGCGGGGCCGCGGAAAG</td>\n",
       "      <td>AGAGGGCGGGACCGCGGAAAG</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>217</th>\n",
       "      <td>5-1295075-G-A</td>\n",
       "      <td>SP1.H13CORE.2.P.B</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>0.000028</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>0.000003</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>12</td>\n",
       "      <td>973884</td>\n",
       "      <td>1.232180e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>AGAGGGCGGGGCCGCGGAAAG</td>\n",
       "      <td>AGAGGGCGGGACCGCGGAAAG</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>5-1295075-G-A</td>\n",
       "      <td>SP2.H13CORE.0.P.D</td>\n",
       "      <td>1</td>\n",
       "      <td>12</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>0.000777</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>0.000262</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>12</td>\n",
       "      <td>973884</td>\n",
       "      <td>1.232180e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>AGAGGGCGGGGCCGCGGAAAG</td>\n",
       "      <td>AGAGGGCGGGACCGCGGAAAG</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>220</th>\n",
       "      <td>5-1295075-G-A</td>\n",
       "      <td>SP4.H13CORE.0.P.C</td>\n",
       "      <td>1</td>\n",
       "      <td>15</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>0.000198</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>0.000031</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>12</td>\n",
       "      <td>973884</td>\n",
       "      <td>1.232180e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>AGAGGGCGGGGCCGCGGAAAG</td>\n",
       "      <td>AGAGGGCGGGACCGCGGAAAG</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>222</th>\n",
       "      <td>5-1295075-G-A</td>\n",
       "      <td>SP4.H13CORE.2.P.B</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>0.000035</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>0.000005</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>12</td>\n",
       "      <td>973884</td>\n",
       "      <td>1.232180e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>AGAGGGCGGGGCCGCGGAAAG</td>\n",
       "      <td>AGAGGGCGGGACCGCGGAAAG</td>\n",
       "      <td>REF_5-1295075-G-A</td>\n",
       "      <td>ALT_5-1295075-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>236</th>\n",
       "      <td>5-1295112-A-T</td>\n",
       "      <td>CTCFL.H13CORE.0.P.B</td>\n",
       "      <td>5</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295112-A-T</td>\n",
       "      <td>0.000195</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>2</td>\n",
       "      <td>98898</td>\n",
       "      <td>2.022286e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GAGGGCCCGGAGGGGGCTGGG</td>\n",
       "      <td>GAGGGCCCGGTGGGGGCTGGG</td>\n",
       "      <td>REF_5-1295112-A-T</td>\n",
       "      <td>ALT_5-1295112-A-T</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>242</th>\n",
       "      <td>5-1295112-A-T</td>\n",
       "      <td>ZBTB41.H13CORE.0.PSG.A</td>\n",
       "      <td>1</td>\n",
       "      <td>19</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295112-A-T</td>\n",
       "      <td>0.000877</td>\n",
       "      <td>REF_5-1295112-A-T</td>\n",
       "      <td>0.000877</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>2</td>\n",
       "      <td>98898</td>\n",
       "      <td>2.022286e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GAGGGCCCGGAGGGGGCTGGG</td>\n",
       "      <td>GAGGGCCCGGTGGGGGCTGGG</td>\n",
       "      <td>REF_5-1295112-A-T</td>\n",
       "      <td>ALT_5-1295112-A-T</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>247</th>\n",
       "      <td>5-1295112-A-T</td>\n",
       "      <td>ZNF709.H13CORE.0.P.C</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295112-A-T</td>\n",
       "      <td>0.000967</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>2</td>\n",
       "      <td>98898</td>\n",
       "      <td>2.022286e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GAGGGCCCGGAGGGGGCTGGG</td>\n",
       "      <td>GAGGGCCCGGTGGGGGCTGGG</td>\n",
       "      <td>REF_5-1295112-A-T</td>\n",
       "      <td>ALT_5-1295112-A-T</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>248</th>\n",
       "      <td>5-1295113-G-A</td>\n",
       "      <td>CTCFL.H13CORE.0.P.B</td>\n",
       "      <td>4</td>\n",
       "      <td>19</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295113-G-A</td>\n",
       "      <td>0.000195</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>62</td>\n",
       "      <td>655850</td>\n",
       "      <td>9.453381e-05</td>\n",
       "      <td>Likely pathogenic</td>\n",
       "      <td>11</td>\n",
       "      <td>AGGGCCCGGAGGGGGCTGGGC</td>\n",
       "      <td>AGGGCCCGGAAGGGGCTGGGC</td>\n",
       "      <td>REF_5-1295113-G-A</td>\n",
       "      <td>ALT_5-1295113-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>266</th>\n",
       "      <td>5-1295113-G-A</td>\n",
       "      <td>FLI1.H13CORE.1.P.B</td>\n",
       "      <td>3</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295113-G-A</td>\n",
       "      <td>0.000939</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>62</td>\n",
       "      <td>655850</td>\n",
       "      <td>9.453381e-05</td>\n",
       "      <td>Likely pathogenic</td>\n",
       "      <td>11</td>\n",
       "      <td>AGGGCCCGGAGGGGGCTGGGC</td>\n",
       "      <td>AGGGCCCGGAAGGGGCTGGGC</td>\n",
       "      <td>REF_5-1295113-G-A</td>\n",
       "      <td>ALT_5-1295113-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>268</th>\n",
       "      <td>5-1295113-G-A</td>\n",
       "      <td>MAZ.H13CORE.0.P.B</td>\n",
       "      <td>4</td>\n",
       "      <td>16</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295113-G-A</td>\n",
       "      <td>0.000832</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>62</td>\n",
       "      <td>655850</td>\n",
       "      <td>9.453381e-05</td>\n",
       "      <td>Likely pathogenic</td>\n",
       "      <td>11</td>\n",
       "      <td>AGGGCCCGGAGGGGGCTGGGC</td>\n",
       "      <td>AGGGCCCGGAAGGGGCTGGGC</td>\n",
       "      <td>REF_5-1295113-G-A</td>\n",
       "      <td>ALT_5-1295113-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>282</th>\n",
       "      <td>5-1295128-G-A</td>\n",
       "      <td>MAZ.H13CORE.1.P.B</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295128-G-A</td>\n",
       "      <td>0.000614</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>2</td>\n",
       "      <td>386064</td>\n",
       "      <td>5.180488e-06</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>CTGGGCCGGGGACCCGGGAGG</td>\n",
       "      <td>CTGGGCCGGGAACCCGGGAGG</td>\n",
       "      <td>REF_5-1295128-G-A</td>\n",
       "      <td>ALT_5-1295128-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>288</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>CTCFL.H13CORE.0.P.B</td>\n",
       "      <td>6</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295135-G-A</td>\n",
       "      <td>0.000195</td>\n",
       "      <td>REF_5-1295135-G-A</td>\n",
       "      <td>0.000195</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>13</td>\n",
       "      <td>484372</td>\n",
       "      <td>2.683888e-05</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>11</td>\n",
       "      <td>GGGGACCCGGGAGGGGTCGGG</td>\n",
       "      <td>GGGGACCCGGAAGGGGTCGGG</td>\n",
       "      <td>REF_5-1295135-G-A</td>\n",
       "      <td>ALT_5-1295135-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>289</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>E4F1.H13CORE.0.P.B</td>\n",
       "      <td>4</td>\n",
       "      <td>16</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295135-G-A</td>\n",
       "      <td>0.000900</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>13</td>\n",
       "      <td>484372</td>\n",
       "      <td>2.683888e-05</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>11</td>\n",
       "      <td>GGGGACCCGGGAGGGGTCGGG</td>\n",
       "      <td>GGGGACCCGGAAGGGGTCGGG</td>\n",
       "      <td>REF_5-1295135-G-A</td>\n",
       "      <td>ALT_5-1295135-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>311</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>HINFP.H13CORE.0.P.B</td>\n",
       "      <td>1</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295135-G-A</td>\n",
       "      <td>0.000173</td>\n",
       "      <td>REF_5-1295135-G-A</td>\n",
       "      <td>0.000071</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>13</td>\n",
       "      <td>484372</td>\n",
       "      <td>2.683888e-05</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>11</td>\n",
       "      <td>GGGGACCCGGGAGGGGTCGGG</td>\n",
       "      <td>GGGGACCCGGAAGGGGTCGGG</td>\n",
       "      <td>REF_5-1295135-G-A</td>\n",
       "      <td>ALT_5-1295135-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>322</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ZNF709.H13CORE.0.P.C</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295135-G-A</td>\n",
       "      <td>0.000967</td>\n",
       "      <td>REF_5-1295135-G-A</td>\n",
       "      <td>0.000967</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>13</td>\n",
       "      <td>484372</td>\n",
       "      <td>2.683888e-05</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>11</td>\n",
       "      <td>GGGGACCCGGGAGGGGTCGGG</td>\n",
       "      <td>GGGGACCCGGAAGGGGTCGGG</td>\n",
       "      <td>REF_5-1295135-G-A</td>\n",
       "      <td>ALT_5-1295135-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>323</th>\n",
       "      <td>5-1295136-A-G</td>\n",
       "      <td>CTCFL.H13CORE.0.P.B</td>\n",
       "      <td>5</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295136-A-G</td>\n",
       "      <td>0.000925</td>\n",
       "      <td>REF_5-1295136-A-G</td>\n",
       "      <td>0.000195</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>108874</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GGGACCCGGGAGGGGTCGGGA</td>\n",
       "      <td>GGGACCCGGGGGGGGTCGGGA</td>\n",
       "      <td>REF_5-1295136-A-G</td>\n",
       "      <td>ALT_5-1295136-A-G</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>359</th>\n",
       "      <td>5-1295162-G-A</td>\n",
       "      <td>ZBTB41.H13CORE.0.PSG.A</td>\n",
       "      <td>2</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295162-G-A</td>\n",
       "      <td>0.000949</td>\n",
       "      <td>REF_5-1295162-G-A</td>\n",
       "      <td>0.000877</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>6</td>\n",
       "      <td>434268</td>\n",
       "      <td>1.381635e-05</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>CGGGGTCCGCGCGGAGGAGGC</td>\n",
       "      <td>CGGGGTCCGCACGGAGGAGGC</td>\n",
       "      <td>REF_5-1295162-G-A</td>\n",
       "      <td>ALT_5-1295162-G-A</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>373</th>\n",
       "      <td>5-1295163-C-G</td>\n",
       "      <td>ZBTB41.H13CORE.0.PSG.A</td>\n",
       "      <td>1</td>\n",
       "      <td>19</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295163-C-G</td>\n",
       "      <td>0.000877</td>\n",
       "      <td>REF_5-1295163-C-G</td>\n",
       "      <td>0.000877</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>433444</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>11</td>\n",
       "      <td>GGGGTCCGCGCGGAGGAGGCG</td>\n",
       "      <td>GGGGTCCGCGGGGAGGAGGCG</td>\n",
       "      <td>REF_5-1295163-C-G</td>\n",
       "      <td>ALT_5-1295163-C-G</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>382</th>\n",
       "      <td>5-1295163-C-G</td>\n",
       "      <td>ZN263.H13CORE.0.PM.A</td>\n",
       "      <td>10</td>\n",
       "      <td>19</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295163-C-G</td>\n",
       "      <td>0.000412</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>433444</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>11</td>\n",
       "      <td>GGGGTCCGCGCGGAGGAGGCG</td>\n",
       "      <td>GGGGTCCGCGGGGAGGAGGCG</td>\n",
       "      <td>REF_5-1295163-C-G</td>\n",
       "      <td>ALT_5-1295163-C-G</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>390</th>\n",
       "      <td>5-1295181-A-G</td>\n",
       "      <td>HINFP.H13CORE.0.P.B</td>\n",
       "      <td>1</td>\n",
       "      <td>20</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295181-A-G</td>\n",
       "      <td>0.000285</td>\n",
       "      <td>REF_5-1295181-A-G</td>\n",
       "      <td>0.000778</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>425236</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GCGGAGCTGGAAGGTGAAGGG</td>\n",
       "      <td>GCGGAGCTGGGAGGTGAAGGG</td>\n",
       "      <td>REF_5-1295181-A-G</td>\n",
       "      <td>ALT_5-1295181-A-G</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>399</th>\n",
       "      <td>5-1295181-A-G</td>\n",
       "      <td>ZN263.H13CORE.1.P.B</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295181-A-G</td>\n",
       "      <td>0.000774</td>\n",
       "      <td>REF_5-1295181-A-G</td>\n",
       "      <td>0.000774</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>425236</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GCGGAGCTGGAAGGTGAAGGG</td>\n",
       "      <td>GCGGAGCTGGGAGGTGAAGGG</td>\n",
       "      <td>REF_5-1295181-A-G</td>\n",
       "      <td>ALT_5-1295181-A-G</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>403</th>\n",
       "      <td>5-1295181-A-G</td>\n",
       "      <td>ZN860.H13CORE.0.P.C</td>\n",
       "      <td>3</td>\n",
       "      <td>19</td>\n",
       "      <td>-</td>\n",
       "      <td>ALT_5-1295181-A-G</td>\n",
       "      <td>0.000869</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>0</td>\n",
       "      <td>425236</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>GCGGAGCTGGAAGGTGAAGGG</td>\n",
       "      <td>GCGGAGCTGGGAGGTGAAGGG</td>\n",
       "      <td>REF_5-1295181-A-G</td>\n",
       "      <td>ALT_5-1295181-A-G</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>409</th>\n",
       "      <td>5-1295182-A-G</td>\n",
       "      <td>MAZ.H13CORE.1.P.B</td>\n",
       "      <td>1</td>\n",
       "      <td>21</td>\n",
       "      <td>+</td>\n",
       "      <td>ALT_5-1295182-A-G</td>\n",
       "      <td>0.000614</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>77</td>\n",
       "      <td>424486</td>\n",
       "      <td>1.813959e-04</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>CGGAGCTGGAAGGTGAAGGGG</td>\n",
       "      <td>CGGAGCTGGAGGGTGAAGGGG</td>\n",
       "      <td>REF_5-1295182-A-G</td>\n",
       "      <td>ALT_5-1295182-A-G</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>470</th>\n",
       "      <td>5-1295225-C-T</td>\n",
       "      <td>MYC.H13CORE.0.P.B</td>\n",
       "      <td>10</td>\n",
       "      <td>19</td>\n",
       "      <td>-</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>REF_5-1295225-C-T</td>\n",
       "      <td>0.000116</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>58</td>\n",
       "      <td>152198</td>\n",
       "      <td>3.810825e-04</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>AGTCCCTCCGCCACGTGGGAA</td>\n",
       "      <td>AGTCCCTCCGTCACGTGGGAA</td>\n",
       "      <td>REF_5-1295225-C-T</td>\n",
       "      <td>ALT_5-1295225-C-T</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>475</th>\n",
       "      <td>5-1295225-C-T</td>\n",
       "      <td>TET3.H13CORE.0.PSGIB.A</td>\n",
       "      <td>2</td>\n",
       "      <td>19</td>\n",
       "      <td>-</td>\n",
       "      <td>ALT_5-1295225-C-T</td>\n",
       "      <td>0.000748</td>\n",
       "      <td>REF_5-1295225-C-T</td>\n",
       "      <td>0.000748</td>\n",
       "      <td>5</td>\n",
       "      <td>...</td>\n",
       "      <td>58</td>\n",
       "      <td>152198</td>\n",
       "      <td>3.810825e-04</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>11</td>\n",
       "      <td>AGTCCCTCCGCCACGTGGGAA</td>\n",
       "      <td>AGTCCCTCCGTCACGTGGGAA</td>\n",
       "      <td>REF_5-1295225-C-T</td>\n",
       "      <td>ALT_5-1295225-C-T</td>\n",
       "      <td>True</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>36 rows √ó 23 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "        variant_id                motif_id  start  stop strand  \\\n",
       "32   5-1294992-C-T     ZNF28.H13CORE.0.P.C      4    21      +   \n",
       "50   5-1294994-G-C     ZNF28.H13CORE.0.P.C      2    19      +   \n",
       "71   5-1294994-G-T     ZNF28.H13CORE.0.P.C      2    19      +   \n",
       "91   5-1294995-G-T     ZNF28.H13CORE.0.P.C      1    18      +   \n",
       "92   5-1294995-G-T  ZNF800.H13CORE.0.PSG.A      1    21      +   \n",
       "116  5-1295020-C-G     MYCN.H13CORE.0.PS.A      7    17      +   \n",
       "200  5-1295075-G-A     KLF13.H13CORE.1.P.C      1    20      +   \n",
       "201  5-1295075-G-A     KLF15.H13CORE.1.P.C      2    21      +   \n",
       "210  5-1295075-G-A       MAZ.H13CORE.0.P.B      1    13      +   \n",
       "211  5-1295075-G-A       MAZ.H13CORE.1.P.B      1    21      +   \n",
       "215  5-1295075-G-A       SP1.H13CORE.0.P.B      1    14      +   \n",
       "217  5-1295075-G-A       SP1.H13CORE.2.P.B      1    21      +   \n",
       "218  5-1295075-G-A       SP2.H13CORE.0.P.D      1    12      +   \n",
       "220  5-1295075-G-A       SP4.H13CORE.0.P.C      1    15      +   \n",
       "222  5-1295075-G-A       SP4.H13CORE.2.P.B      1    21      +   \n",
       "236  5-1295112-A-T     CTCFL.H13CORE.0.P.B      5    20      +   \n",
       "242  5-1295112-A-T  ZBTB41.H13CORE.0.PSG.A      1    19      +   \n",
       "247  5-1295112-A-T    ZNF709.H13CORE.0.P.C      1    21      +   \n",
       "248  5-1295113-G-A     CTCFL.H13CORE.0.P.B      4    19      +   \n",
       "266  5-1295113-G-A      FLI1.H13CORE.1.P.B      3    20      +   \n",
       "268  5-1295113-G-A       MAZ.H13CORE.0.P.B      4    16      +   \n",
       "282  5-1295128-G-A       MAZ.H13CORE.1.P.B      1    21      +   \n",
       "288  5-1295135-G-A     CTCFL.H13CORE.0.P.B      6    21      +   \n",
       "289  5-1295135-G-A      E4F1.H13CORE.0.P.B      4    16      +   \n",
       "311  5-1295135-G-A     HINFP.H13CORE.0.P.B      1    20      +   \n",
       "322  5-1295135-G-A    ZNF709.H13CORE.0.P.C      1    21      +   \n",
       "323  5-1295136-A-G     CTCFL.H13CORE.0.P.B      5    20      +   \n",
       "359  5-1295162-G-A  ZBTB41.H13CORE.0.PSG.A      2    20      +   \n",
       "373  5-1295163-C-G  ZBTB41.H13CORE.0.PSG.A      1    19      +   \n",
       "382  5-1295163-C-G    ZN263.H13CORE.0.PM.A     10    19      +   \n",
       "390  5-1295181-A-G     HINFP.H13CORE.0.P.B      1    20      +   \n",
       "399  5-1295181-A-G     ZN263.H13CORE.1.P.B      1    21      +   \n",
       "403  5-1295181-A-G     ZN860.H13CORE.0.P.C      3    19      -   \n",
       "409  5-1295182-A-G       MAZ.H13CORE.1.P.B      1    21      +   \n",
       "470  5-1295225-C-T       MYC.H13CORE.0.P.B     10    19      -   \n",
       "475  5-1295225-C-T  TET3.H13CORE.0.PSGIB.A      2    19      -   \n",
       "\n",
       "              ALT_name  ALT_q-value           REF_name  REF_q-value chrom  \\\n",
       "32   ALT_5-1294992-C-T     0.000839  REF_5-1294992-C-T     0.000493     5   \n",
       "50                 NaN          NaN  REF_5-1294994-G-C     0.000493     5   \n",
       "71                 NaN          NaN  REF_5-1294994-G-T     0.000493     5   \n",
       "91                 NaN          NaN  REF_5-1294995-G-T     0.000493     5   \n",
       "92   ALT_5-1294995-G-T     0.000047  REF_5-1294995-G-T     0.000047     5   \n",
       "116                NaN          NaN  REF_5-1295020-C-G     0.000525     5   \n",
       "200  ALT_5-1295075-G-A     0.000116  REF_5-1295075-G-A     0.000030     5   \n",
       "201  ALT_5-1295075-G-A     0.000512  REF_5-1295075-G-A     0.000195     5   \n",
       "210  ALT_5-1295075-G-A     0.000832  REF_5-1295075-G-A     0.000231     5   \n",
       "211  ALT_5-1295075-G-A     0.000012  REF_5-1295075-G-A     0.000001     5   \n",
       "215  ALT_5-1295075-G-A     0.000697  REF_5-1295075-G-A     0.000065     5   \n",
       "217  ALT_5-1295075-G-A     0.000028  REF_5-1295075-G-A     0.000003     5   \n",
       "218  ALT_5-1295075-G-A     0.000777  REF_5-1295075-G-A     0.000262     5   \n",
       "220  ALT_5-1295075-G-A     0.000198  REF_5-1295075-G-A     0.000031     5   \n",
       "222  ALT_5-1295075-G-A     0.000035  REF_5-1295075-G-A     0.000005     5   \n",
       "236                NaN          NaN  REF_5-1295112-A-T     0.000195     5   \n",
       "242  ALT_5-1295112-A-T     0.000877  REF_5-1295112-A-T     0.000877     5   \n",
       "247                NaN          NaN  REF_5-1295112-A-T     0.000967     5   \n",
       "248                NaN          NaN  REF_5-1295113-G-A     0.000195     5   \n",
       "266  ALT_5-1295113-G-A     0.000939                NaN          NaN     5   \n",
       "268                NaN          NaN  REF_5-1295113-G-A     0.000832     5   \n",
       "282                NaN          NaN  REF_5-1295128-G-A     0.000614     5   \n",
       "288  ALT_5-1295135-G-A     0.000195  REF_5-1295135-G-A     0.000195     5   \n",
       "289  ALT_5-1295135-G-A     0.000900                NaN          NaN     5   \n",
       "311  ALT_5-1295135-G-A     0.000173  REF_5-1295135-G-A     0.000071     5   \n",
       "322  ALT_5-1295135-G-A     0.000967  REF_5-1295135-G-A     0.000967     5   \n",
       "323  ALT_5-1295136-A-G     0.000925  REF_5-1295136-A-G     0.000195     5   \n",
       "359  ALT_5-1295162-G-A     0.000949  REF_5-1295162-G-A     0.000877     5   \n",
       "373  ALT_5-1295163-C-G     0.000877  REF_5-1295163-C-G     0.000877     5   \n",
       "382  ALT_5-1295163-C-G     0.000412                NaN          NaN     5   \n",
       "390  ALT_5-1295181-A-G     0.000285  REF_5-1295181-A-G     0.000778     5   \n",
       "399  ALT_5-1295181-A-G     0.000774  REF_5-1295181-A-G     0.000774     5   \n",
       "403  ALT_5-1295181-A-G     0.000869                NaN          NaN     5   \n",
       "409  ALT_5-1295182-A-G     0.000614                NaN          NaN     5   \n",
       "470                NaN          NaN  REF_5-1295225-C-T     0.000116     5   \n",
       "475  ALT_5-1295225-C-T     0.000748  REF_5-1295225-C-T     0.000748     5   \n",
       "\n",
       "     ...  ac       an            af   clinical_significance  variant_pos  \\\n",
       "32   ...   2  1312640  1.523647e-06           Likely benign           11   \n",
       "50   ...   2  1315502  1.520332e-06  Uncertain significance           11   \n",
       "71   ...   1  1315610  7.601037e-07           Likely benign           11   \n",
       "91   ...  13  1315568  9.881663e-06           Likely benign           11   \n",
       "92   ...  13  1315568  9.881663e-06           Likely benign           11   \n",
       "116  ...  13  1278802  1.016576e-05  Uncertain significance           11   \n",
       "200  ...  12   973884  1.232180e-05  Uncertain significance           11   \n",
       "201  ...  12   973884  1.232180e-05  Uncertain significance           11   \n",
       "210  ...  12   973884  1.232180e-05  Uncertain significance           11   \n",
       "211  ...  12   973884  1.232180e-05  Uncertain significance           11   \n",
       "215  ...  12   973884  1.232180e-05  Uncertain significance           11   \n",
       "217  ...  12   973884  1.232180e-05  Uncertain significance           11   \n",
       "218  ...  12   973884  1.232180e-05  Uncertain significance           11   \n",
       "220  ...  12   973884  1.232180e-05  Uncertain significance           11   \n",
       "222  ...  12   973884  1.232180e-05  Uncertain significance           11   \n",
       "236  ...   2    98898  2.022286e-05  Uncertain significance           11   \n",
       "242  ...   2    98898  2.022286e-05  Uncertain significance           11   \n",
       "247  ...   2    98898  2.022286e-05  Uncertain significance           11   \n",
       "248  ...  62   655850  9.453381e-05       Likely pathogenic           11   \n",
       "266  ...  62   655850  9.453381e-05       Likely pathogenic           11   \n",
       "268  ...  62   655850  9.453381e-05       Likely pathogenic           11   \n",
       "282  ...   2   386064  5.180488e-06  Uncertain significance           11   \n",
       "288  ...  13   484372  2.683888e-05              Pathogenic           11   \n",
       "289  ...  13   484372  2.683888e-05              Pathogenic           11   \n",
       "311  ...  13   484372  2.683888e-05              Pathogenic           11   \n",
       "322  ...  13   484372  2.683888e-05              Pathogenic           11   \n",
       "323  ...   0   108874  0.000000e+00  Uncertain significance           11   \n",
       "359  ...   6   434268  1.381635e-05  Uncertain significance           11   \n",
       "373  ...   0   433444  0.000000e+00           Likely benign           11   \n",
       "382  ...   0   433444  0.000000e+00           Likely benign           11   \n",
       "390  ...   0   425236  0.000000e+00  Uncertain significance           11   \n",
       "399  ...   0   425236  0.000000e+00  Uncertain significance           11   \n",
       "403  ...   0   425236  0.000000e+00  Uncertain significance           11   \n",
       "409  ...  77   424486  1.813959e-04  Uncertain significance           11   \n",
       "470  ...  58   152198  3.810825e-04  Uncertain significance           11   \n",
       "475  ...  58   152198  3.810825e-04  Uncertain significance           11   \n",
       "\n",
       "              ref_sequence           alt_sequence                REF  \\\n",
       "32   CGCGGCATCGCGGGGGTGGCC  CGCGGCATCGTGGGGGTGGCC  REF_5-1294992-C-T   \n",
       "50   CGGCATCGCGGGGGTGGCCGG  CGGCATCGCGCGGGTGGCCGG  REF_5-1294994-G-C   \n",
       "71   CGGCATCGCGGGGGTGGCCGG  CGGCATCGCGTGGGTGGCCGG  REF_5-1294994-G-T   \n",
       "91   GGCATCGCGGGGGTGGCCGGG  GGCATCGCGGTGGTGGCCGGG  REF_5-1294995-G-T   \n",
       "92   GGCATCGCGGGGGTGGCCGGG  GGCATCGCGGTGGTGGCCGGG  REF_5-1294995-G-T   \n",
       "116  GGGCTTCCCACGTGCGCAGCA  GGGCTTCCCAGGTGCGCAGCA  REF_5-1295020-C-G   \n",
       "200  AGAGGGCGGGGCCGCGGAAAG  AGAGGGCGGGACCGCGGAAAG  REF_5-1295075-G-A   \n",
       "201  AGAGGGCGGGGCCGCGGAAAG  AGAGGGCGGGACCGCGGAAAG  REF_5-1295075-G-A   \n",
       "210  AGAGGGCGGGGCCGCGGAAAG  AGAGGGCGGGACCGCGGAAAG  REF_5-1295075-G-A   \n",
       "211  AGAGGGCGGGGCCGCGGAAAG  AGAGGGCGGGACCGCGGAAAG  REF_5-1295075-G-A   \n",
       "215  AGAGGGCGGGGCCGCGGAAAG  AGAGGGCGGGACCGCGGAAAG  REF_5-1295075-G-A   \n",
       "217  AGAGGGCGGGGCCGCGGAAAG  AGAGGGCGGGACCGCGGAAAG  REF_5-1295075-G-A   \n",
       "218  AGAGGGCGGGGCCGCGGAAAG  AGAGGGCGGGACCGCGGAAAG  REF_5-1295075-G-A   \n",
       "220  AGAGGGCGGGGCCGCGGAAAG  AGAGGGCGGGACCGCGGAAAG  REF_5-1295075-G-A   \n",
       "222  AGAGGGCGGGGCCGCGGAAAG  AGAGGGCGGGACCGCGGAAAG  REF_5-1295075-G-A   \n",
       "236  GAGGGCCCGGAGGGGGCTGGG  GAGGGCCCGGTGGGGGCTGGG  REF_5-1295112-A-T   \n",
       "242  GAGGGCCCGGAGGGGGCTGGG  GAGGGCCCGGTGGGGGCTGGG  REF_5-1295112-A-T   \n",
       "247  GAGGGCCCGGAGGGGGCTGGG  GAGGGCCCGGTGGGGGCTGGG  REF_5-1295112-A-T   \n",
       "248  AGGGCCCGGAGGGGGCTGGGC  AGGGCCCGGAAGGGGCTGGGC  REF_5-1295113-G-A   \n",
       "266  AGGGCCCGGAGGGGGCTGGGC  AGGGCCCGGAAGGGGCTGGGC  REF_5-1295113-G-A   \n",
       "268  AGGGCCCGGAGGGGGCTGGGC  AGGGCCCGGAAGGGGCTGGGC  REF_5-1295113-G-A   \n",
       "282  CTGGGCCGGGGACCCGGGAGG  CTGGGCCGGGAACCCGGGAGG  REF_5-1295128-G-A   \n",
       "288  GGGGACCCGGGAGGGGTCGGG  GGGGACCCGGAAGGGGTCGGG  REF_5-1295135-G-A   \n",
       "289  GGGGACCCGGGAGGGGTCGGG  GGGGACCCGGAAGGGGTCGGG  REF_5-1295135-G-A   \n",
       "311  GGGGACCCGGGAGGGGTCGGG  GGGGACCCGGAAGGGGTCGGG  REF_5-1295135-G-A   \n",
       "322  GGGGACCCGGGAGGGGTCGGG  GGGGACCCGGAAGGGGTCGGG  REF_5-1295135-G-A   \n",
       "323  GGGACCCGGGAGGGGTCGGGA  GGGACCCGGGGGGGGTCGGGA  REF_5-1295136-A-G   \n",
       "359  CGGGGTCCGCGCGGAGGAGGC  CGGGGTCCGCACGGAGGAGGC  REF_5-1295162-G-A   \n",
       "373  GGGGTCCGCGCGGAGGAGGCG  GGGGTCCGCGGGGAGGAGGCG  REF_5-1295163-C-G   \n",
       "382  GGGGTCCGCGCGGAGGAGGCG  GGGGTCCGCGGGGAGGAGGCG  REF_5-1295163-C-G   \n",
       "390  GCGGAGCTGGAAGGTGAAGGG  GCGGAGCTGGGAGGTGAAGGG  REF_5-1295181-A-G   \n",
       "399  GCGGAGCTGGAAGGTGAAGGG  GCGGAGCTGGGAGGTGAAGGG  REF_5-1295181-A-G   \n",
       "403  GCGGAGCTGGAAGGTGAAGGG  GCGGAGCTGGGAGGTGAAGGG  REF_5-1295181-A-G   \n",
       "409  CGGAGCTGGAAGGTGAAGGGG  CGGAGCTGGAGGGTGAAGGGG  REF_5-1295182-A-G   \n",
       "470  AGTCCCTCCGCCACGTGGGAA  AGTCCCTCCGTCACGTGGGAA  REF_5-1295225-C-T   \n",
       "475  AGTCCCTCCGCCACGTGGGAA  AGTCCCTCCGTCACGTGGGAA  REF_5-1295225-C-T   \n",
       "\n",
       "                   ALT is_variant_overlapping  \n",
       "32   ALT_5-1294992-C-T                   True  \n",
       "50   ALT_5-1294994-G-C                   True  \n",
       "71   ALT_5-1294994-G-T                   True  \n",
       "91   ALT_5-1294995-G-T                   True  \n",
       "92   ALT_5-1294995-G-T                   True  \n",
       "116  ALT_5-1295020-C-G                   True  \n",
       "200  ALT_5-1295075-G-A                   True  \n",
       "201  ALT_5-1295075-G-A                   True  \n",
       "210  ALT_5-1295075-G-A                   True  \n",
       "211  ALT_5-1295075-G-A                   True  \n",
       "215  ALT_5-1295075-G-A                   True  \n",
       "217  ALT_5-1295075-G-A                   True  \n",
       "218  ALT_5-1295075-G-A                   True  \n",
       "220  ALT_5-1295075-G-A                   True  \n",
       "222  ALT_5-1295075-G-A                   True  \n",
       "236  ALT_5-1295112-A-T                   True  \n",
       "242  ALT_5-1295112-A-T                   True  \n",
       "247  ALT_5-1295112-A-T                   True  \n",
       "248  ALT_5-1295113-G-A                   True  \n",
       "266  ALT_5-1295113-G-A                   True  \n",
       "268  ALT_5-1295113-G-A                   True  \n",
       "282  ALT_5-1295128-G-A                   True  \n",
       "288  ALT_5-1295135-G-A                   True  \n",
       "289  ALT_5-1295135-G-A                   True  \n",
       "311  ALT_5-1295135-G-A                   True  \n",
       "322  ALT_5-1295135-G-A                   True  \n",
       "323  ALT_5-1295136-A-G                   True  \n",
       "359  ALT_5-1295162-G-A                   True  \n",
       "373  ALT_5-1295163-C-G                   True  \n",
       "382  ALT_5-1295163-C-G                   True  \n",
       "390  ALT_5-1295181-A-G                   True  \n",
       "399  ALT_5-1295181-A-G                   True  \n",
       "403  ALT_5-1295181-A-G                   True  \n",
       "409  ALT_5-1295182-A-G                   True  \n",
       "470  ALT_5-1295225-C-T                   True  \n",
       "475  ALT_5-1295225-C-T                   True  \n",
       "\n",
       "[36 rows x 23 columns]"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Preview filtered motif-variant table\n",
    "variant_metadata_fimo_combined_overlap_filtered"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bb254720",
   "metadata": {},
   "source": [
    "#### ü§î Which TFBS Fits Better?\n",
    "\n",
    "The q-value ranks motif matches, but longer motifs tend to have lower p/q-values. üìä That's why a very low threshold gives us always motifs of lengths around 20. Let's use a **bitscore** (normalized for motif length) to compare motif fits near the variant. ‚öñÔ∏è"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b62799da",
   "metadata": {},
   "source": [
    "![multiple_tfbs_fitting_variant_example_long_vs_short.png](resources/picture/multiple_tfbs_fitting_variant_example_long_vs_short.png) üñºÔ∏è"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "3ad747a8",
   "metadata": {},
   "source": [
    "##### üëÅÔ∏è Which TFBS fits better by eye?\n",
    "\n",
    "But by eye, I would say that the bottom one matches more important nucleotides. üéØ Note that it is known that the p- and q-values are not reliable when comparing motifs with different lengths, because longer motifs get a lower p-value more easily. However, we can use a **bitscore** üìä which is measured with a unified size around the variant position to make motifs of different sizes comparable. ‚öñÔ∏è"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "8e3c840f",
   "metadata": {},
   "outputs": [],
   "source": [
    "from Bio import motifs\n",
    "\n",
    "# Function used to compute the local bitscore\n",
    "def extract_motif_names(meme_file):\n",
    "    motif_names = []\n",
    "    with open(meme_file, \"r\") as f:\n",
    "        for line in f:\n",
    "            if line.startswith(\"MOTIF\"):\n",
    "                motif_name = line.split(\" \", 1)[1].strip()\n",
    "                motif_names.append(motif_name)\n",
    "    return motif_names\n",
    "\n",
    "\n",
    "# Read the used motif file and extract the motifs\n",
    "meme_file = \"resources/hocomoco_v13/H13CORE_meme_format.meme\"\n",
    "motif_names = extract_motif_names(meme_file)\n",
    "\n",
    "# Add the name while loading the file using Biopython\n",
    "hocomoco_motif_dict = {}\n",
    "with open(meme_file) as handle:\n",
    "    motif_list = motifs.parse(handle, \"pfm-four-columns\")\n",
    "    i = 0\n",
    "    for motif in motif_list:\n",
    "        motif.name = motif_names[i]\n",
    "        hocomoco_motif_dict[motif.name] = motif\n",
    "        i += 1\n",
    "\n",
    "\n",
    "def reverse_complement(seq):\n",
    "    complement = str.maketrans(\"ACGTacgt\", \"TGCAtgca\")\n",
    "    return seq.translate(complement)[::-1]\n",
    "\n",
    "\n",
    "def local_bitscore_tfbs(motif, sequence, motif_start, motif_end, variant_pos, strand):\n",
    "    \"\"\"\n",
    "    Calculate the local bitscore for a given motif and sequence, using a window of 5 bases around the variant position.\n",
    "    \"\"\"\n",
    "    zero_based_motif_start = motif_start - 1  # start should be 0-based and end 1-based\n",
    "    pwm = motif.pwm\n",
    "    zero_based_variant_pos = variant_pos - 1\n",
    "\n",
    "    # Get only subset of sequence which is overlapped with the motif\n",
    "    sequence_of_interest_start = max(zero_based_motif_start, zero_based_variant_pos - 5)\n",
    "    sequence_of_interest_end = min(motif_end, zero_based_variant_pos + 5)\n",
    "\n",
    "    motif_related_start = max(0, sequence_of_interest_start - zero_based_motif_start)\n",
    "    motif_related_end = min(motif_end - zero_based_motif_start, sequence_of_interest_end - zero_based_motif_start)\n",
    "\n",
    "    # Normalize the score for the length of the motif\n",
    "    additional_score = 10 - (motif_related_end - motif_related_start)\n",
    "    motif_sequence = sequence[zero_based_motif_start:motif_end]\n",
    "    \n",
    "    # Use the reverse complement sequence if the hit is on the \"-\" strand\n",
    "    if strand == \"-\":\n",
    "        motif_sequence = reverse_complement(motif_sequence)\n",
    "    \n",
    "    motif_part = motif_sequence[motif_related_start:motif_related_end]\n",
    "    \n",
    "    # Crop the PWM to the motif-related part\n",
    "    modified_pwm = {nuc: scores[motif_related_start:motif_related_end] for nuc, scores in pwm.items()}\n",
    "\n",
    "    return additional_score + round(sum(modified_pwm[nuc][i] for i, nuc in enumerate(motif_part)), 4)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "356a3785",
   "metadata": {},
   "source": [
    "Calculate a bitscore for each motif match in the context of the variant position. üßÆ We extract a sequence around the variant position with the same length as the motif and calculate the bitscore of that sequence with the motif PWM. üìä"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "id": "7e6f9f9a",
   "metadata": {},
   "outputs": [],
   "source": [
    "variant_overlapping_tfbs['REF_bit_score'] = variant_overlapping_tfbs.apply(\n",
    "    lambda row: local_bitscore_tfbs(\n",
    "        motif=hocomoco_motif_dict[row[\"motif_id\"]],\n",
    "        sequence=row['ref_sequence'],\n",
    "        motif_start=row['start'],\n",
    "        motif_end=row['stop'],\n",
    "        variant_pos=row['variant_pos'],\n",
    "        strand=row['strand']\n",
    "    ),\n",
    "    axis=1\n",
    ")\n",
    "variant_overlapping_tfbs['ALT_bit_score'] = variant_overlapping_tfbs.apply(\n",
    "    lambda row: local_bitscore_tfbs(\n",
    "        motif=hocomoco_motif_dict[row[\"motif_id\"]],\n",
    "        sequence=row['alt_sequence'],\n",
    "        motif_start=row['start'],\n",
    "        motif_end=row['stop'],\n",
    "        variant_pos=row['variant_pos'],\n",
    "        strand=row['strand']\n",
    "    ),\n",
    "    axis=1\n",
    ")\n",
    "\n",
    "variant_overlapping_tfbs['max_bit_score'] = variant_overlapping_tfbs[['REF_bit_score', 'ALT_bit_score']].max(axis=1).round(3)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f1f69826",
   "metadata": {},
   "source": [
    "> **‚úÖ Result:**  \n",
    "> The bitscore now correctly ranks motif fits. üéØ  \n",
    "> - **Higher bitscore = better motif fit** üìà\n",
    "> - Filter for a minimal bitscore to avoid weak matches ‚ö†Ô∏è"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "f3117d62",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "motif_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "clinical_significance",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "max_bit_score",
         "rawType": "float64",
         "type": "float"
        }
       ],
       "ref": "39bd2b17-8322-4ffb-8912-9fc6c5c6cd44",
       "rows": [
        [
         "436",
         "5-1295225-C-T",
         "BMAL1.H13CORE.0.PSM.A",
         "Uncertain significance",
         "9.742"
        ],
        [
         "31",
         "5-1294992-C-T",
         "ZN740.H13CORE.0.P.C",
         "Likely benign",
         "9.372"
        ],
        [
         "188",
         "5-1295075-G-A",
         "CGGBP1.H13CORE.0.PSGIB.A",
         "Uncertain significance",
         "9.323"
        ],
        [
         "271",
         "5-1295113-G-A",
         "VEZF1.H13CORE.0.P.C",
         "Likely pathogenic",
         "9.183"
        ],
        [
         "96",
         "5-1295020-C-G",
         "BMAL1.H13CORE.0.PSM.A",
         "Uncertain significance",
         "9.124"
        ],
        [
         "501",
         "5-1295234-A-G",
         "USF3.H13CORE.0.PSGIB.A",
         "Benign",
         "9.049"
        ],
        [
         "336",
         "5-1295136-A-G",
         "ZBT7A.H13CORE.0.P.B",
         "Uncertain significance",
         "9.0"
        ],
        [
         "12",
         "5-1294991-G-A",
         "ZN740.H13CORE.0.P.C",
         "Uncertain significance",
         "8.891"
        ],
        [
         "382",
         "5-1295163-C-G",
         "ZN263.H13CORE.0.PM.A",
         "Likely benign",
         "8.862"
        ],
        [
         "171",
         "5-1295046-T-G",
         "ZN335.H13CORE.1.P.D",
         "Conflicting classifications of pathogenicity",
         "8.756"
        ]
       ],
       "shape": {
        "columns": 4,
        "rows": 10
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>motif_id</th>\n",
       "      <th>clinical_significance</th>\n",
       "      <th>max_bit_score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>436</th>\n",
       "      <td>5-1295225-C-T</td>\n",
       "      <td>BMAL1.H13CORE.0.PSM.A</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>9.742</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>5-1294992-C-T</td>\n",
       "      <td>ZN740.H13CORE.0.P.C</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>9.372</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>188</th>\n",
       "      <td>5-1295075-G-A</td>\n",
       "      <td>CGGBP1.H13CORE.0.PSGIB.A</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>9.323</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>271</th>\n",
       "      <td>5-1295113-G-A</td>\n",
       "      <td>VEZF1.H13CORE.0.P.C</td>\n",
       "      <td>Likely pathogenic</td>\n",
       "      <td>9.183</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>5-1295020-C-G</td>\n",
       "      <td>BMAL1.H13CORE.0.PSM.A</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>9.124</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>501</th>\n",
       "      <td>5-1295234-A-G</td>\n",
       "      <td>USF3.H13CORE.0.PSGIB.A</td>\n",
       "      <td>Benign</td>\n",
       "      <td>9.049</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>336</th>\n",
       "      <td>5-1295136-A-G</td>\n",
       "      <td>ZBT7A.H13CORE.0.P.B</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>9.000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>5-1294991-G-A</td>\n",
       "      <td>ZN740.H13CORE.0.P.C</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>8.891</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>382</th>\n",
       "      <td>5-1295163-C-G</td>\n",
       "      <td>ZN263.H13CORE.0.PM.A</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>8.862</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>5-1295046-T-G</td>\n",
       "      <td>ZN335.H13CORE.1.P.D</td>\n",
       "      <td>Conflicting classifications of pathogenicity</td>\n",
       "      <td>8.756</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "        variant_id                  motif_id  \\\n",
       "436  5-1295225-C-T     BMAL1.H13CORE.0.PSM.A   \n",
       "31   5-1294992-C-T       ZN740.H13CORE.0.P.C   \n",
       "188  5-1295075-G-A  CGGBP1.H13CORE.0.PSGIB.A   \n",
       "271  5-1295113-G-A       VEZF1.H13CORE.0.P.C   \n",
       "96   5-1295020-C-G     BMAL1.H13CORE.0.PSM.A   \n",
       "501  5-1295234-A-G    USF3.H13CORE.0.PSGIB.A   \n",
       "336  5-1295136-A-G       ZBT7A.H13CORE.0.P.B   \n",
       "12   5-1294991-G-A       ZN740.H13CORE.0.P.C   \n",
       "382  5-1295163-C-G      ZN263.H13CORE.0.PM.A   \n",
       "171  5-1295046-T-G       ZN335.H13CORE.1.P.D   \n",
       "\n",
       "                            clinical_significance  max_bit_score  \n",
       "436                        Uncertain significance          9.742  \n",
       "31                                  Likely benign          9.372  \n",
       "188                        Uncertain significance          9.323  \n",
       "271                             Likely pathogenic          9.183  \n",
       "96                         Uncertain significance          9.124  \n",
       "501                                        Benign          9.049  \n",
       "336                        Uncertain significance          9.000  \n",
       "12                         Uncertain significance          8.891  \n",
       "382                                 Likely benign          8.862  \n",
       "171  Conflicting classifications of pathogenicity          8.756  "
      ]
     },
     "execution_count": 36,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "interesting_columns =[\"variant_id\", \"motif_id\", \"clinical_significance\", \"max_bit_score\"]\n",
    "\n",
    "variant_overlapping_tfbs.sort_values(by='max_bit_score', ascending=False).drop_duplicates(subset='variant_id', keep='first')[interesting_columns].head(10)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "39405359",
   "metadata": {},
   "source": [
    "Let's look at the highest difference between reference and alternative allele bitscores: üìäüîç"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "ef7d8ace",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "motif_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "clinical_significance",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "REF_bit_score",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "ALT_bit_score",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "delta_bit_score",
         "rawType": "float64",
         "type": "float"
        }
       ],
       "ref": "15b73a52-4eff-43c7-8947-e4a3a071bf02",
       "rows": [
        [
         "351",
         "5-1295136-A-G",
         "ZN740.H13CORE.0.P.C",
         "Uncertain significance",
         "7.607",
         "8.607",
         "1.0"
        ],
        [
         "300",
         "5-1295135-G-A",
         "ELK4.H13CORE.0.PSM.A",
         "Pathogenic",
         "5.7691",
         "6.7691",
         "1.0"
        ],
        [
         "87",
         "5-1294995-G-T",
         "ZIC5.H13CORE.1.M.C",
         "Likely benign",
         "6.4204",
         "5.4206",
         "1.0"
        ],
        [
         "132",
         "5-1295034-C-A",
         "ELF5.H13CORE.1.S.B",
         "Uncertain significance",
         "5.447",
         "6.447",
         "1.0"
        ],
        [
         "165",
         "5-1295046-T-G",
         "ELK4.H13CORE.0.PSM.A",
         "Conflicting classifications of pathogenicity",
         "7.0339",
         "8.0339",
         "1.0"
        ],
        [
         "276",
         "5-1295113-G-A",
         "ZXDB.H13CORE.0.PI.A",
         "Likely pathogenic",
         "6.7684",
         "7.7684",
         "1.0"
        ],
        [
         "49",
         "5-1294994-G-C",
         "ZN740.H13CORE.0.P.C",
         "Uncertain significance",
         "7.97",
         "6.97",
         "1.0"
        ],
        [
         "479",
         "5-1295225-C-T",
         "TFEC.H13CORE.1.SM.B",
         "Uncertain significance",
         "6.1831",
         "7.1831",
         "1.0"
        ],
        [
         "95",
         "5-1295020-C-G",
         "BHE41.H13CORE.0.PSM.A",
         "Uncertain significance",
         "6.8903",
         "5.8903",
         "1.0"
        ],
        [
         "70",
         "5-1294994-G-T",
         "ZN740.H13CORE.0.P.C",
         "Likely benign",
         "7.97",
         "6.97",
         "1.0"
        ]
       ],
       "shape": {
        "columns": 6,
        "rows": 10
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>motif_id</th>\n",
       "      <th>clinical_significance</th>\n",
       "      <th>REF_bit_score</th>\n",
       "      <th>ALT_bit_score</th>\n",
       "      <th>delta_bit_score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>351</th>\n",
       "      <td>5-1295136-A-G</td>\n",
       "      <td>ZN740.H13CORE.0.P.C</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>7.6070</td>\n",
       "      <td>8.6070</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ELK4.H13CORE.0.PSM.A</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.7691</td>\n",
       "      <td>6.7691</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>5-1294995-G-T</td>\n",
       "      <td>ZIC5.H13CORE.1.M.C</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>6.4204</td>\n",
       "      <td>5.4206</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>132</th>\n",
       "      <td>5-1295034-C-A</td>\n",
       "      <td>ELF5.H13CORE.1.S.B</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>5.4470</td>\n",
       "      <td>6.4470</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>165</th>\n",
       "      <td>5-1295046-T-G</td>\n",
       "      <td>ELK4.H13CORE.0.PSM.A</td>\n",
       "      <td>Conflicting classifications of pathogenicity</td>\n",
       "      <td>7.0339</td>\n",
       "      <td>8.0339</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>5-1295113-G-A</td>\n",
       "      <td>ZXDB.H13CORE.0.PI.A</td>\n",
       "      <td>Likely pathogenic</td>\n",
       "      <td>6.7684</td>\n",
       "      <td>7.7684</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>5-1294994-G-C</td>\n",
       "      <td>ZN740.H13CORE.0.P.C</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>7.9700</td>\n",
       "      <td>6.9700</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>479</th>\n",
       "      <td>5-1295225-C-T</td>\n",
       "      <td>TFEC.H13CORE.1.SM.B</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>6.1831</td>\n",
       "      <td>7.1831</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>5-1295020-C-G</td>\n",
       "      <td>BHE41.H13CORE.0.PSM.A</td>\n",
       "      <td>Uncertain significance</td>\n",
       "      <td>6.8903</td>\n",
       "      <td>5.8903</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>5-1294994-G-T</td>\n",
       "      <td>ZN740.H13CORE.0.P.C</td>\n",
       "      <td>Likely benign</td>\n",
       "      <td>7.9700</td>\n",
       "      <td>6.9700</td>\n",
       "      <td>1.0</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "        variant_id               motif_id  \\\n",
       "351  5-1295136-A-G    ZN740.H13CORE.0.P.C   \n",
       "300  5-1295135-G-A   ELK4.H13CORE.0.PSM.A   \n",
       "87   5-1294995-G-T     ZIC5.H13CORE.1.M.C   \n",
       "132  5-1295034-C-A     ELF5.H13CORE.1.S.B   \n",
       "165  5-1295046-T-G   ELK4.H13CORE.0.PSM.A   \n",
       "276  5-1295113-G-A    ZXDB.H13CORE.0.PI.A   \n",
       "49   5-1294994-G-C    ZN740.H13CORE.0.P.C   \n",
       "479  5-1295225-C-T    TFEC.H13CORE.1.SM.B   \n",
       "95   5-1295020-C-G  BHE41.H13CORE.0.PSM.A   \n",
       "70   5-1294994-G-T    ZN740.H13CORE.0.P.C   \n",
       "\n",
       "                            clinical_significance  REF_bit_score  \\\n",
       "351                        Uncertain significance         7.6070   \n",
       "300                                    Pathogenic         5.7691   \n",
       "87                                  Likely benign         6.4204   \n",
       "132                        Uncertain significance         5.4470   \n",
       "165  Conflicting classifications of pathogenicity         7.0339   \n",
       "276                             Likely pathogenic         6.7684   \n",
       "49                         Uncertain significance         7.9700   \n",
       "479                        Uncertain significance         6.1831   \n",
       "95                         Uncertain significance         6.8903   \n",
       "70                                  Likely benign         7.9700   \n",
       "\n",
       "     ALT_bit_score  delta_bit_score  \n",
       "351         8.6070              1.0  \n",
       "300         6.7691              1.0  \n",
       "87          5.4206              1.0  \n",
       "132         6.4470              1.0  \n",
       "165         8.0339              1.0  \n",
       "276         7.7684              1.0  \n",
       "49          6.9700              1.0  \n",
       "479         7.1831              1.0  \n",
       "95          5.8903              1.0  \n",
       "70          6.9700              1.0  "
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "interesting_columns = [\"variant_id\", \"motif_id\", \"clinical_significance\", \"REF_bit_score\", \"ALT_bit_score\", \"delta_bit_score\"]\n",
    "\n",
    "variant_overlapping_tfbs['delta_bit_score'] = (\n",
    "    variant_overlapping_tfbs['ALT_bit_score'] - variant_overlapping_tfbs['REF_bit_score']\n",
    ").round(3).abs()\n",
    "\n",
    "variant_overlapping_tfbs.sort_values(by='delta_bit_score', ascending=False).drop_duplicates(\n",
    "    subset='variant_id', keep='first'\n",
    ")[interesting_columns].head(10)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bf039d23",
   "metadata": {},
   "source": [
    "Does this result make sense? ü§î For example, the variant **5-1295135-G-A** creates a strong match for multiple motifs like ELK4 and GABPA in the alternative allele. See here: üëá"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "37b01da7",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.microsoft.datawrangler.viewer.v0+json": {
       "columns": [
        {
         "name": "index",
         "rawType": "int64",
         "type": "integer"
        },
        {
         "name": "variant_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "motif_id",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "clinical_significance",
         "rawType": "object",
         "type": "string"
        },
        {
         "name": "REF_bit_score",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "ALT_bit_score",
         "rawType": "float64",
         "type": "float"
        },
        {
         "name": "delta_bit_score",
         "rawType": "float64",
         "type": "float"
        }
       ],
       "ref": "2148066a-1dd2-493b-8bec-b68bcce9dad4",
       "rows": [
        [
         "298",
         "5-1295135-G-A",
         "ELF5.H13CORE.1.S.B",
         "Pathogenic",
         "6.4505",
         "7.4505",
         "1.0"
        ],
        [
         "293",
         "5-1295135-G-A",
         "ELF2.H13CORE.1.M.B",
         "Pathogenic",
         "5.7412",
         "6.7412",
         "1.0"
        ],
        [
         "294",
         "5-1295135-G-A",
         "ELF3.H13CORE.0.S.B",
         "Pathogenic",
         "6.0119",
         "7.0115",
         "1.0"
        ],
        [
         "300",
         "5-1295135-G-A",
         "ELK4.H13CORE.0.PSM.A",
         "Pathogenic",
         "5.7691",
         "6.7691",
         "1.0"
        ],
        [
         "299",
         "5-1295135-G-A",
         "ELK1.H13CORE.0.PSM.A",
         "Pathogenic",
         "6.1845",
         "7.1844",
         "1.0"
        ],
        [
         "310",
         "5-1295135-G-A",
         "GABPA.H13CORE.0.PSM.A",
         "Pathogenic",
         "6.1235",
         "7.1235",
         "1.0"
        ],
        [
         "296",
         "5-1295135-G-A",
         "ELF4.H13CORE.1.M.B",
         "Pathogenic",
         "5.6496",
         "6.6496",
         "1.0"
        ],
        [
         "297",
         "5-1295135-G-A",
         "ELF5.H13CORE.0.PSM.A",
         "Pathogenic",
         "5.3126",
         "6.3114",
         "0.999"
        ],
        [
         "295",
         "5-1295135-G-A",
         "ELF4.H13CORE.0.PS.A",
         "Pathogenic",
         "6.6359",
         "7.6339",
         "0.998"
        ],
        [
         "290",
         "5-1295135-G-A",
         "EHF.H13CORE.0.P.B",
         "Pathogenic",
         "5.3913",
         "6.3883",
         "0.997"
        ],
        [
         "301",
         "5-1295135-G-A",
         "ETS1.H13CORE.1.P.B",
         "Pathogenic",
         "5.6026",
         "6.5961",
         "0.994"
        ],
        [
         "291",
         "5-1295135-G-A",
         "ELF1.H13CORE.0.PSM.A",
         "Pathogenic",
         "6.4388",
         "7.4307",
         "0.992"
        ],
        [
         "292",
         "5-1295135-G-A",
         "ELF2.H13CORE.0.PS.A",
         "Pathogenic",
         "6.4356",
         "7.4256",
         "0.99"
        ],
        [
         "308",
         "5-1295135-G-A",
         "FLI1.H13CORE.0.PSM.A",
         "Pathogenic",
         "5.886",
         "6.8744",
         "0.988"
        ],
        [
         "307",
         "5-1295135-G-A",
         "ETV6.H13CORE.1.P.B",
         "Pathogenic",
         "5.347",
         "6.334",
         "0.987"
        ],
        [
         "305",
         "5-1295135-G-A",
         "ETV5.H13CORE.1.PM.A",
         "Pathogenic",
         "5.7821",
         "6.7688",
         "0.987"
        ],
        [
         "304",
         "5-1295135-G-A",
         "ETV4.H13CORE.0.P.B",
         "Pathogenic",
         "5.654",
         "6.641",
         "0.987"
        ],
        [
         "320",
         "5-1295135-G-A",
         "ZN468.H13CORE.0.P.C",
         "Pathogenic",
         "7.979",
         "6.995",
         "0.984"
        ],
        [
         "306",
         "5-1295135-G-A",
         "ETV6.H13CORE.0.PS.A",
         "Pathogenic",
         "6.0612",
         "7.0431",
         "0.982"
        ],
        [
         "314",
         "5-1295135-G-A",
         "PATZ1.H13CORE.1.P.C",
         "Pathogenic",
         "7.0531",
         "6.0719",
         "0.981"
        ],
        [
         "312",
         "5-1295135-G-A",
         "KLF1.H13CORE.0.P.B",
         "Pathogenic",
         "6.6295",
         "5.6506",
         "0.979"
        ],
        [
         "321",
         "5-1295135-G-A",
         "ZN860.H13CORE.0.P.C",
         "Pathogenic",
         "6.2591",
         "5.2818",
         "0.977"
        ],
        [
         "303",
         "5-1295135-G-A",
         "ETV2.H13CORE.0.S.B",
         "Pathogenic",
         "5.3103",
         "6.2873",
         "0.977"
        ],
        [
         "309",
         "5-1295135-G-A",
         "FLI1.H13CORE.1.P.B",
         "Pathogenic",
         "5.187",
         "6.16",
         "0.973"
        ],
        [
         "302",
         "5-1295135-G-A",
         "ETS2.H13CORE.1.P.B",
         "Pathogenic",
         "5.6446",
         "6.6046",
         "0.96"
        ],
        [
         "289",
         "5-1295135-G-A",
         "E4F1.H13CORE.0.P.B",
         "Pathogenic",
         "5.473",
         "6.428",
         "0.955"
        ],
        [
         "315",
         "5-1295135-G-A",
         "SP2.H13CORE.1.P.B",
         "Pathogenic",
         "5.3275",
         "4.3877",
         "0.94"
        ],
        [
         "313",
         "5-1295135-G-A",
         "MAZ.H13CORE.0.P.B",
         "Pathogenic",
         "6.0205",
         "5.1639",
         "0.857"
        ],
        [
         "319",
         "5-1295135-G-A",
         "ZN423.H13CORE.0.P.C",
         "Pathogenic",
         "6.3858",
         "5.655",
         "0.731"
        ],
        [
         "311",
         "5-1295135-G-A",
         "HINFP.H13CORE.0.P.B",
         "Pathogenic",
         "5.9187",
         "5.3821",
         "0.537"
        ],
        [
         "316",
         "5-1295135-G-A",
         "ZBT24.H13CORE.0.P.B",
         "Pathogenic",
         "5.838",
         "6.24",
         "0.402"
        ],
        [
         "317",
         "5-1295135-G-A",
         "ZBT7A.H13CORE.1.SM.B",
         "Pathogenic",
         "4.8205",
         "4.4736",
         "0.347"
        ],
        [
         "322",
         "5-1295135-G-A",
         "ZNF709.H13CORE.0.P.C",
         "Pathogenic",
         "5.5564",
         "5.9023",
         "0.346"
        ],
        [
         "318",
         "5-1295135-G-A",
         "ZN263.H13CORE.1.P.B",
         "Pathogenic",
         "3.9422",
         "3.8435",
         "0.099"
        ],
        [
         "288",
         "5-1295135-G-A",
         "CTCFL.H13CORE.0.P.B",
         "Pathogenic",
         "6.2007",
         "6.2184",
         "0.018"
        ]
       ],
       "shape": {
        "columns": 6,
        "rows": 35
       }
      },
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>variant_id</th>\n",
       "      <th>motif_id</th>\n",
       "      <th>clinical_significance</th>\n",
       "      <th>REF_bit_score</th>\n",
       "      <th>ALT_bit_score</th>\n",
       "      <th>delta_bit_score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>298</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ELF5.H13CORE.1.S.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.4505</td>\n",
       "      <td>7.4505</td>\n",
       "      <td>1.000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>293</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ELF2.H13CORE.1.M.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.7412</td>\n",
       "      <td>6.7412</td>\n",
       "      <td>1.000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>294</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ELF3.H13CORE.0.S.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.0119</td>\n",
       "      <td>7.0115</td>\n",
       "      <td>1.000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>300</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ELK4.H13CORE.0.PSM.A</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.7691</td>\n",
       "      <td>6.7691</td>\n",
       "      <td>1.000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>299</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ELK1.H13CORE.0.PSM.A</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.1845</td>\n",
       "      <td>7.1844</td>\n",
       "      <td>1.000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>310</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>GABPA.H13CORE.0.PSM.A</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.1235</td>\n",
       "      <td>7.1235</td>\n",
       "      <td>1.000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>296</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ELF4.H13CORE.1.M.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.6496</td>\n",
       "      <td>6.6496</td>\n",
       "      <td>1.000</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>297</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ELF5.H13CORE.0.PSM.A</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.3126</td>\n",
       "      <td>6.3114</td>\n",
       "      <td>0.999</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>295</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ELF4.H13CORE.0.PS.A</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.6359</td>\n",
       "      <td>7.6339</td>\n",
       "      <td>0.998</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>290</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>EHF.H13CORE.0.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.3913</td>\n",
       "      <td>6.3883</td>\n",
       "      <td>0.997</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>301</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ETS1.H13CORE.1.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.6026</td>\n",
       "      <td>6.5961</td>\n",
       "      <td>0.994</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>291</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ELF1.H13CORE.0.PSM.A</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.4388</td>\n",
       "      <td>7.4307</td>\n",
       "      <td>0.992</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>292</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ELF2.H13CORE.0.PS.A</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.4356</td>\n",
       "      <td>7.4256</td>\n",
       "      <td>0.990</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>308</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>FLI1.H13CORE.0.PSM.A</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.8860</td>\n",
       "      <td>6.8744</td>\n",
       "      <td>0.988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>307</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ETV6.H13CORE.1.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.3470</td>\n",
       "      <td>6.3340</td>\n",
       "      <td>0.987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>305</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ETV5.H13CORE.1.PM.A</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.7821</td>\n",
       "      <td>6.7688</td>\n",
       "      <td>0.987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>304</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ETV4.H13CORE.0.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.6540</td>\n",
       "      <td>6.6410</td>\n",
       "      <td>0.987</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>320</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ZN468.H13CORE.0.P.C</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>7.9790</td>\n",
       "      <td>6.9950</td>\n",
       "      <td>0.984</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>306</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ETV6.H13CORE.0.PS.A</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.0612</td>\n",
       "      <td>7.0431</td>\n",
       "      <td>0.982</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>314</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>PATZ1.H13CORE.1.P.C</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>7.0531</td>\n",
       "      <td>6.0719</td>\n",
       "      <td>0.981</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>312</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>KLF1.H13CORE.0.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.6295</td>\n",
       "      <td>5.6506</td>\n",
       "      <td>0.979</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>321</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ZN860.H13CORE.0.P.C</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.2591</td>\n",
       "      <td>5.2818</td>\n",
       "      <td>0.977</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>303</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ETV2.H13CORE.0.S.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.3103</td>\n",
       "      <td>6.2873</td>\n",
       "      <td>0.977</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>309</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>FLI1.H13CORE.1.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.1870</td>\n",
       "      <td>6.1600</td>\n",
       "      <td>0.973</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>302</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ETS2.H13CORE.1.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.6446</td>\n",
       "      <td>6.6046</td>\n",
       "      <td>0.960</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>289</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>E4F1.H13CORE.0.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.4730</td>\n",
       "      <td>6.4280</td>\n",
       "      <td>0.955</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>315</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>SP2.H13CORE.1.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.3275</td>\n",
       "      <td>4.3877</td>\n",
       "      <td>0.940</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>313</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>MAZ.H13CORE.0.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.0205</td>\n",
       "      <td>5.1639</td>\n",
       "      <td>0.857</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>319</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ZN423.H13CORE.0.P.C</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.3858</td>\n",
       "      <td>5.6550</td>\n",
       "      <td>0.731</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>311</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>HINFP.H13CORE.0.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.9187</td>\n",
       "      <td>5.3821</td>\n",
       "      <td>0.537</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>316</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ZBT24.H13CORE.0.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.8380</td>\n",
       "      <td>6.2400</td>\n",
       "      <td>0.402</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>317</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ZBT7A.H13CORE.1.SM.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>4.8205</td>\n",
       "      <td>4.4736</td>\n",
       "      <td>0.347</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>322</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ZNF709.H13CORE.0.P.C</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>5.5564</td>\n",
       "      <td>5.9023</td>\n",
       "      <td>0.346</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>318</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>ZN263.H13CORE.1.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>3.9422</td>\n",
       "      <td>3.8435</td>\n",
       "      <td>0.099</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>288</th>\n",
       "      <td>5-1295135-G-A</td>\n",
       "      <td>CTCFL.H13CORE.0.P.B</td>\n",
       "      <td>Pathogenic</td>\n",
       "      <td>6.2007</td>\n",
       "      <td>6.2184</td>\n",
       "      <td>0.018</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "        variant_id               motif_id clinical_significance  \\\n",
       "298  5-1295135-G-A     ELF5.H13CORE.1.S.B            Pathogenic   \n",
       "293  5-1295135-G-A     ELF2.H13CORE.1.M.B            Pathogenic   \n",
       "294  5-1295135-G-A     ELF3.H13CORE.0.S.B            Pathogenic   \n",
       "300  5-1295135-G-A   ELK4.H13CORE.0.PSM.A            Pathogenic   \n",
       "299  5-1295135-G-A   ELK1.H13CORE.0.PSM.A            Pathogenic   \n",
       "310  5-1295135-G-A  GABPA.H13CORE.0.PSM.A            Pathogenic   \n",
       "296  5-1295135-G-A     ELF4.H13CORE.1.M.B            Pathogenic   \n",
       "297  5-1295135-G-A   ELF5.H13CORE.0.PSM.A            Pathogenic   \n",
       "295  5-1295135-G-A    ELF4.H13CORE.0.PS.A            Pathogenic   \n",
       "290  5-1295135-G-A      EHF.H13CORE.0.P.B            Pathogenic   \n",
       "301  5-1295135-G-A     ETS1.H13CORE.1.P.B            Pathogenic   \n",
       "291  5-1295135-G-A   ELF1.H13CORE.0.PSM.A            Pathogenic   \n",
       "292  5-1295135-G-A    ELF2.H13CORE.0.PS.A            Pathogenic   \n",
       "308  5-1295135-G-A   FLI1.H13CORE.0.PSM.A            Pathogenic   \n",
       "307  5-1295135-G-A     ETV6.H13CORE.1.P.B            Pathogenic   \n",
       "305  5-1295135-G-A    ETV5.H13CORE.1.PM.A            Pathogenic   \n",
       "304  5-1295135-G-A     ETV4.H13CORE.0.P.B            Pathogenic   \n",
       "320  5-1295135-G-A    ZN468.H13CORE.0.P.C            Pathogenic   \n",
       "306  5-1295135-G-A    ETV6.H13CORE.0.PS.A            Pathogenic   \n",
       "314  5-1295135-G-A    PATZ1.H13CORE.1.P.C            Pathogenic   \n",
       "312  5-1295135-G-A     KLF1.H13CORE.0.P.B            Pathogenic   \n",
       "321  5-1295135-G-A    ZN860.H13CORE.0.P.C            Pathogenic   \n",
       "303  5-1295135-G-A     ETV2.H13CORE.0.S.B            Pathogenic   \n",
       "309  5-1295135-G-A     FLI1.H13CORE.1.P.B            Pathogenic   \n",
       "302  5-1295135-G-A     ETS2.H13CORE.1.P.B            Pathogenic   \n",
       "289  5-1295135-G-A     E4F1.H13CORE.0.P.B            Pathogenic   \n",
       "315  5-1295135-G-A      SP2.H13CORE.1.P.B            Pathogenic   \n",
       "313  5-1295135-G-A      MAZ.H13CORE.0.P.B            Pathogenic   \n",
       "319  5-1295135-G-A    ZN423.H13CORE.0.P.C            Pathogenic   \n",
       "311  5-1295135-G-A    HINFP.H13CORE.0.P.B            Pathogenic   \n",
       "316  5-1295135-G-A    ZBT24.H13CORE.0.P.B            Pathogenic   \n",
       "317  5-1295135-G-A   ZBT7A.H13CORE.1.SM.B            Pathogenic   \n",
       "322  5-1295135-G-A   ZNF709.H13CORE.0.P.C            Pathogenic   \n",
       "318  5-1295135-G-A    ZN263.H13CORE.1.P.B            Pathogenic   \n",
       "288  5-1295135-G-A    CTCFL.H13CORE.0.P.B            Pathogenic   \n",
       "\n",
       "     REF_bit_score  ALT_bit_score  delta_bit_score  \n",
       "298         6.4505         7.4505            1.000  \n",
       "293         5.7412         6.7412            1.000  \n",
       "294         6.0119         7.0115            1.000  \n",
       "300         5.7691         6.7691            1.000  \n",
       "299         6.1845         7.1844            1.000  \n",
       "310         6.1235         7.1235            1.000  \n",
       "296         5.6496         6.6496            1.000  \n",
       "297         5.3126         6.3114            0.999  \n",
       "295         6.6359         7.6339            0.998  \n",
       "290         5.3913         6.3883            0.997  \n",
       "301         5.6026         6.5961            0.994  \n",
       "291         6.4388         7.4307            0.992  \n",
       "292         6.4356         7.4256            0.990  \n",
       "308         5.8860         6.8744            0.988  \n",
       "307         5.3470         6.3340            0.987  \n",
       "305         5.7821         6.7688            0.987  \n",
       "304         5.6540         6.6410            0.987  \n",
       "320         7.9790         6.9950            0.984  \n",
       "306         6.0612         7.0431            0.982  \n",
       "314         7.0531         6.0719            0.981  \n",
       "312         6.6295         5.6506            0.979  \n",
       "321         6.2591         5.2818            0.977  \n",
       "303         5.3103         6.2873            0.977  \n",
       "309         5.1870         6.1600            0.973  \n",
       "302         5.6446         6.6046            0.960  \n",
       "289         5.4730         6.4280            0.955  \n",
       "315         5.3275         4.3877            0.940  \n",
       "313         6.0205         5.1639            0.857  \n",
       "319         6.3858         5.6550            0.731  \n",
       "311         5.9187         5.3821            0.537  \n",
       "316         5.8380         6.2400            0.402  \n",
       "317         4.8205         4.4736            0.347  \n",
       "322         5.5564         5.9023            0.346  \n",
       "318         3.9422         3.8435            0.099  \n",
       "288         6.2007         6.2184            0.018  "
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "variant_overlapping_tfbs[variant_overlapping_tfbs['variant_id'] == \"5-1295135-G-A\"].sort_values(\n",
    "    by='delta_bit_score', ascending=False\n",
    ")[interesting_columns]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "01cdbb10",
   "metadata": {},
   "source": [
    "Do we have more functional evidence? **YES!** üéâ Saturation mutagenesis experiments (so-called Massively Parallel Reporter Assays, MPRAs) have shown that this variant creates a new binding site for GABPA, which leads to increased TERT expression and is linked to cancer. üî¨üìà\n",
    "\n",
    "Please go to https://kircherlab.bihealth.org/satMutMPRA/ ‚Üí **Promoter: TERT**, **Variant plot** and locate your variants. What do you see? üëÄ\n",
    "\n",
    "**ü§î BTW, what is an MPRA?** That you will learn next in `Epigenetic Data and Regulatory Codes: Machine Learning Perspectives` üöÄ"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "jupyter",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
